<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2001 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2001.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2001</h1><p>Number of papers published in 2001: <strong>90</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Allele loss on chromosome 1p36 in epithelial ovarian cancers.</h4>
<p class="authors"><span class="author">Alvarez AA</span>, <span class="author">Lambers AR</span>, <span class="author">Lancaster JM</span>, <span class="author">Maxwell GL</span>, <span class="author">Ali S</span>, <span class="author">Gumbs C</span>, <span class="author">Berchuck A</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">Department of Obstetrics and Gynecology/Division of Gynecologic Oncology, Duke University Medical Center, Durham, North Carolina, 27710, USA.</p><p class="abstract"><strong>Objectives</strong>: Prior studies have shown that allelic loss on chromosome 1p36 occurs frequently in ovarian as well as several other types of cancer. This suggests that inactivation of gene(s) in this region may play a role in the pathogenesis of these cancers. The aim of this study was to further delineate the region of loss on chromosome 1p36 in ovarian cancers and to identify associated patient or tumor characteristics.</p><p class="abstract"><strong>Methods</strong>: Paired normal/cancer DNA samples from 75 ovarian cancers (21 early stage I/II and 54 advanced stage III/IV) were analyzed using microsatellite markers.</p><p class="abstract"><strong>Results</strong>: Forty-nine of 75 (65%) ovarian cancers had loss of at least one marker. The marker demonstrating the most frequent loss was D1S1597, which was lost in 29/57 (51%) informative cases. Allele loss on 1p36 was significantly more common in poorly differentiated ovarian cancers (73%) relative to well or moderately differentiated cases (48%) (P = 0.03). Evidence was obtained for two common regions of deletion: one flanked by D1S1646/D1S244 and another more proximally by D1S244/D1S228.</p><p class="abstract"><strong>Conclusion</strong>: These findings further delineate regions on chromosome 1p36 proposed to contain tumor suppressor gene(s) that may play a role in the development and/or progression of epithelial ovarian carcinoma. Allele loss on 1p36 is associated with poor histologic grade.</p>
<p class="publication"><span class="title">Gynecologic oncology</span>&nbsp;<span class="year">2001</span>;<span class="volume">82</span>;<span class="number">1</span>;<span class="pagerange">94-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11426968" rel="external">11426968</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgyno.2001.6175" rel="external">10.1006/gyno.2001.6175</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The InterPro database, an integrated documentation resource for protein families, domains and functional sites.</h4>
<p class="authors"><span class="author">Apweiler R</span>, <span class="author">Attwood TK</span>, <span class="author">Bairoch A</span>, <span class="author">Bateman A</span>, <span class="author">Birney E</span>, <span class="author">Biswas M</span>, <span class="author">Bucher P</span>, <span class="author">Cerutti L</span>, <span class="author">Corpet F</span>, <span class="author">Croning MD</span>, <span class="author">Durbin R</span>, <span class="author">Falquet L</span>, <span class="author">Fleischmann W</span>, <span class="author">Gouzy J</span>, <span class="author">Hermjakob H</span>, <span class="author">Hulo N</span>, <span class="author">Jonassen I</span>, <span class="author">Kahn D</span>, <span class="author">Kanapin A</span>, <span class="author">Karavidopoulou Y</span>, <span class="author">Lopez R</span>, <span class="author">Marx B</span>, <span class="author">Mulder NJ</span>, <span class="author">Oinn TM</span>, <span class="author">Pagni M</span>, <span class="author">Servant F</span>, <span class="author">Sigrist CJ</span> and <span class="author">Zdobnov EM</span></p>
<p class="affiliation">EMBL Outstation - European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%72%6F%6C%66.%61%70%77%65%69%6C%65%72%40%65%62%69.%61%63.%75%6B">&#114;&#111;&#108;&#102;&#46;&#97;&#112;&#119;&#101;&#105;&#108;&#101;&#114;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Signature databases are vital tools for identifying distant relationships in novel sequences and hence for inferring protein function. InterPro is an integrated documentation resource for protein families, domains and functional sites, which amalgamates the efforts of the PROSITE, PRINTS, Pfam and ProDom database projects. Each InterPro entry includes a functional description, annotation, literature references and links back to the relevant member database(s). Release 2.0 of InterPro (October 2000) contains over 3000 entries, representing families, domains, repeats and sites of post-translational modification encoded by a total of 6804 different regular expressions, profiles, fingerprints and Hidden Markov Models. Each InterPro entry lists all the matches against SWISS-PROT and TrEMBL (more than 1,000,000 hits from 462,500 proteins in SWISS-PROT and TrEMBL). The database is accessible for text- and sequence-based searches at http://www.ebi.ac.uk/interpro/. Questions can be emailed to <a href="mailto:%69%6E%74%65%72%68%65%6C%70%40%65%62%69.%61%63.%75%6B">&#105;&#110;&#116;&#101;&#114;&#104;&#101;&#108;&#112;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a>.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">1</span>;<span class="pagerange">37-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11125043" rel="external">11125043</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC29841" rel="external">29841</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomic structure and promoter region of the human parkin gene.</h4>
<p class="authors"><span class="author">Asakawa S</span>, <span class="author">Tsunematsu Ki</span>, <span class="author">Takayanagi A</span>, <span class="author">Sasaki T</span>, <span class="author">Shimizu A</span>, <span class="author">Shintani A</span>, <span class="author">Kawasaki K</span>, <span class="author">Mungall AJ</span>, <span class="author">Beck S</span>, <span class="author">Minoshima S</span> and <span class="author">Shimizu N</span></p>
<p class="affiliation">Department of Molecular Biology, Keio University School of Medicine, 35 Shinanomachi, Shinjuku-ku, Tokyo, 160-8582, Japan.</p><p class="abstract">Parkin has been identified as a causative gene of the autosomal recessive juvenile parkinsonism (AR-JP). In this study, we determined the genomic structure of the Parkin gene and identified a core promoter region based on the DNA sequence of 1.4 Mb. The 5&#39;-flanking region contained no apparent TATA or CAAT box elements but several putative cis-elements for various transcription factors. The GC- and CpG-rich regions were observed not only in the 5&#39;-flanking sequence but also in the 5&#39;-part of the first intron of Parkin. We identified an exact starting point of Parkin transcription. A core promoter region was determined by transfecting a series of deletion constructs with a dual luciferase reporter system into human neuroblastoma cells. Furthermore, we located a neighboring novel gene in a head-to-head direction with Parkin with only a 198-bp interval.</p>
<p class="publication"><span class="title">Biochemical and biophysical research communications</span>&nbsp;<span class="year">2001</span>;<span class="volume">286</span>;<span class="number">5</span>;<span class="pagerange">863-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11527378" rel="external">11527378</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fbbrc.2001.5490" rel="external">10.1006/bbrc.2001.5490</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fission yeast Pom1p kinase activity is cell cycle regulated and essential for cellular symmetry during growth and division.</h4>
<p class="authors"><span class="author">B&auml;hler J</span> and <span class="author">Nurse P</span></p>
<p class="affiliation">Imperial Cancer Research Fund, Cell Cycle Laboratory, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, UK. <a href="mailto:%6A%75%72%67%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#117;&#114;&#103;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Schizosaccharomyces pombe cells grow from both ends during most of interphase and divide symmetrically into two daughter cells. The pom1 gene, encoding a member of the Dyrk family of protein kinases, has been identified through a mutant showing abnormal cellular morphogenesis. Here we show that Pom1p kinase activity is cell cycle regulated in correlation with the state of cellular symmetry: the activity is high during symmetrical growth and division, but lower when cells grow at just one end. Point mutations in the catalytic domain lead to asymmetry during both cell growth and division, whilst cells overexpressing Pom1p form additional growing ends. Manipulations of kinase activity indicate a negative role for Pom1p in microtubule growth at cell ends. Pom1p is present in a large protein complex and requires its non-catalytic domain to localize to the cell periphery and its kinase activity to localize to cell ends. These data establish that Pom1p kinase activity plays an important role in generating cellular symmetry and suggest that there may be related roles of homologous protein kinases ubiquitously present in all eukaryotes.</p>
<p class="publication"><span class="title">The EMBO journal</span>&nbsp;<span class="year">2001</span>;<span class="volume">20</span>;<span class="number">5</span>;<span class="pagerange">1064-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11230130" rel="external">11230130</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC145493" rel="external">145493</a>; DOI: <a href="http://dx.doi.org/10.1093%2Femboj%2F20.5.1064" rel="external">10.1093/emboj/20.5.1064</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regulation of the stem cell leukemia (SCL) gene: a tale of two fishes.</h4>
<p class="authors"><span class="author">Barton LM</span>, <span class="author">Gottgens B</span>, <span class="author">Gering M</span>, <span class="author">Gilbert JG</span>, <span class="author">Grafham D</span>, <span class="author">Rogers J</span>, <span class="author">Bentley D</span>, <span class="author">Patient R</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Hematology, Cambridge Institute for Medical Research, University of Cambridge, Addenbrookes Hospital, Cambridge CB2 2XY, United Kingdom.</p><p class="abstract">The stem cell leukemia (SCL) gene encodes a tissue-specific basic helix-loop-helix (bHLH) protein with a pivotal role in hemopoiesis and vasculogenesis. Several enhancers have been identified within the murine SCL locus that direct reporter gene expression to subdomains of the normal SCL expression pattern, and long-range sequence comparisons of the human and murine SCL loci have identified additional candidate enhancers. To facilitate the characterization of regulatory elements, we have sequenced and analyzed 33 kb of the SCL genomic locus from the pufferfish Fugu rubripes, a species with a highly compact genome. Although the pattern of SCL expression is highly conserved from mammals to teleost fish, the genes flanking pufferfish SCL were unrelated to those known to flank both avian and mammalian SCL genes. These data suggest that SCL regulatory elements are confined to the region between the upstream and downstream flanking genes, a region of 65 kb in human and 8.5 kb in pufferfish. Consistent with this hypothesis, the entire 33-kb pufferfish SCL locus directed appropriate expression to hemopoietic and neural tissue in transgenic zebrafish embryos, as did a 10.4-kb fragment containing the SCL gene and extending to the 5&#39; and 3&#39; flanking genes. These results demonstrate the power of combining the compact genome of the pufferfish with the advantages that zebrafish provide for studies of gene regulation during development. Furthermore, the pufferfish SCL locus provides a powerful tool for the manipulation of hemopoiesis and vasculogenesis in vivo.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2001</span>;<span class="volume">98</span>;<span class="number">12</span>;<span class="pagerange">6747-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11381108" rel="external">11381108</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC34424" rel="external">34424</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.101532998" rel="external">10.1073/pnas.101532998</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome acrobatics: understanding complex genomes.</h4>
<p class="authors"><span class="author">Beck S</span></p>
<p class="affiliation">Head of Human Sequencing, The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, tel: +44 1223 494952; fax: +44 1223 494919, Cambridge, UK</p>
<p class="publication"><span class="title">Drug discovery today</span>&nbsp;<span class="year">2001</span>;<span class="volume">6</span>;<span class="number">23</span>;<span class="pagerange">1181-1182</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11722860" rel="external">11722860</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From first base: the sequence of the tip of the X chromosome of Drosophila melanogaster, a comparison of two sequencing strategies.</h4>
<p class="authors"><span class="author">Benos PV</span>, <span class="author">Gatt MK</span>, <span class="author">Murphy L</span>, <span class="author">Harris D</span>, <span class="author">Barrell B</span>, <span class="author">Ferraz C</span>, <span class="author">Vidal S</span>, <span class="author">Brun C</span>, <span class="author">Demaille J</span>, <span class="author">Cadieu E</span>, <span class="author">Dreano S</span>, <span class="author">Gloux S</span>, <span class="author">Lelaure V</span>, <span class="author">Mottier S</span>, <span class="author">Galibert F</span>, <span class="author">Borkova D</span>, <span class="author">Mi&ntilde;ana B</span>, <span class="author">Kafatos FC</span>, <span class="author">Bolshakov S</span>, <span class="author">Sid&eacute;n-Kiamos I</span>, <span class="author">Papagiannakis G</span>, <span class="author">Spanos L</span>, <span class="author">Louis C</span>, <span class="author">Madue&ntilde;o E</span>, <span class="author">de Pablos B</span>, <span class="author">Modolell J</span>, <span class="author">Peter A</span>, <span class="author">Sch&ouml;ttler P</span>, <span class="author">Werner M</span>, <span class="author">Mourkioti F</span>, <span class="author">Beinert N</span>, <span class="author">Dowe G</span>, <span class="author">Sch&auml;fer U</span>, <span class="author">J&auml;ckle H</span>, <span class="author">Bucheton A</span>, <span class="author">Callister D</span>, <span class="author">Campbell L</span>, <span class="author">Henderson NS</span>, <span class="author">McMillan PJ</span>, <span class="author">Salles C</span>, <span class="author">Tait E</span>, <span class="author">Valenti P</span>, <span class="author">Saunders RD</span>, <span class="author">Billaud A</span>, <span class="author">Pachter L</span>, <span class="author">Glover DM</span> and <span class="author">Ashburner M</span></p>
<p class="affiliation">EMBL Outstation, The European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK.</p><p class="abstract">We present the sequence of a contiguous 2.63 Mb of DNA extending from the tip of the X chromosome of Drosophila melanogaster. Within this sequence, we predict 277 protein coding genes, of which 94 had been sequenced already in the course of studying the biology of their gene products, and examples of 12 different transposable elements. We show that an interval between bands 3A2 and 3C2, believed in the 1970s to show a correlation between the number of bands on the polytene chromosomes and the 20 genes identified by conventional genetics, is predicted to contain 45 genes from its DNA sequence. We have determined the insertion sites of P-elements from 111 mutant lines, about half of which are in a position likely to affect the expression of novel predicted genes, thus representing a resource for subsequent functional genomic analysis. We compare the European Drosophila Genome Project sequence with the corresponding part of the independently assembled and annotated Joint Sequence determined through &quot;shotgun&quot; sequencing. Discounting differences in the distribution of known transposable elements between the strains sequenced in the two projects, we detected three major sequence differences, two of which are probably explained by errors in assembly; the origin of the third major difference is unclear. In addition there are eight sequence gaps within the Joint Sequence. At least six of these eight gaps are likely to be sites of transposable elements; the other two are complex. Of the 275 genes in common to both projects, 60% are identical within 1% of their predicted amino-acid sequence and 31% show minor differences such as in choice of translation initiation or termination codons; the remaining 9% show major differences in interpretation.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">5</span>;<span class="pagerange">710-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11337470" rel="external">11337470</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311117" rel="external">311117</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.173801" rel="external">10.1101/gr.173801</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The physical maps for sequencing human chromosomes 1, 6, 9, 10, 13, 20 and X.</h4>
<p class="authors"><span class="author">Bentley DR</span>, <span class="author">Deloukas P</span>, <span class="author">Dunham A</span>, <span class="author">French L</span>, <span class="author">Gregory SG</span>, <span class="author">Humphray SJ</span>, <span class="author">Mungall AJ</span>, <span class="author">Ross MT</span>, <span class="author">Carter NP</span>, <span class="author">Dunham I</span>, <span class="author">Scott CE</span>, <span class="author">Ashcroft KJ</span>, <span class="author">Atkinson AL</span>, <span class="author">Aubin K</span>, <span class="author">Beare DM</span>, <span class="author">Bethel G</span>, <span class="author">Brady N</span>, <span class="author">Brook JC</span>, <span class="author">Burford DC</span>, <span class="author">Burrill WD</span>, <span class="author">Burrows C</span>, <span class="author">Butler AP</span>, <span class="author">Carder C</span>, <span class="author">Catanese JJ</span>, <span class="author">Clee CM</span>, <span class="author">Clegg SM</span>, <span class="author">Cobley V</span>, <span class="author">Coffey AJ</span>, <span class="author">Cole CG</span>, <span class="author">Collins JE</span>, <span class="author">Conquer JS</span>, <span class="author">Cooper RA</span>, <span class="author">Culley KM</span>, <span class="author">Dawson E</span>, <span class="author">Dearden FL</span>, <span class="author">Durbin RM</span>, <span class="author">de Jong PJ</span>, <span class="author">Dhami PD</span>, <span class="author">Earthrowl ME</span>, <span class="author">Edwards CA</span>, <span class="author">Evans RS</span>, <span class="author">Gillson CJ</span>, <span class="author">Ghori J</span>, <span class="author">Green L</span>, <span class="author">Gwilliam R</span>, <span class="author">Halls KS</span>, <span class="author">Hammond S</span>, <span class="author">Harper GL</span>, <span class="author">Heathcott RW</span>, <span class="author">Holden JL</span>, <span class="author">Holloway E</span>, <span class="author">Hopkins BL</span>, <span class="author">Howard PJ</span>, <span class="author">Howell GR</span>, <span class="author">Huckle EJ</span>, <span class="author">Hughes J</span>, <span class="author">Hunt PJ</span>, <span class="author">Hunt SE</span>, <span class="author">Izmajlowicz M</span>, <span class="author">Jones CA</span>, <span class="author">Joseph SS</span>, <span class="author">Laird G</span>, <span class="author">Langford CF</span>, <span class="author">Lehvaslaiho MH</span>, <span class="author">Leversha MA</span>, <span class="author">McCann OT</span>, <span class="author">McDonald LM</span>, <span class="author">McDowall J</span>, <span class="author">Maslen GL</span>, <span class="author">Mistry D</span>, <span class="author">Moschonas NK</span>, <span class="author">Neocleous V</span>, <span class="author">Pearson DM</span>, <span class="author">Phillips KJ</span>, <span class="author">Porter KM</span>, <span class="author">Prathalingam SR</span>, <span class="author">Ramsey YH</span>, <span class="author">Ranby SA</span>, <span class="author">Rice CM</span>, <span class="author">Rogers J</span>, <span class="author">Rogers LJ</span>, <span class="author">Sarafidou T</span>, <span class="author">Scott DJ</span>, <span class="author">Sharp GJ</span>, <span class="author">Shaw-Smith CJ</span>, <span class="author">Smink LJ</span>, <span class="author">Soderlund C</span>, <span class="author">Sotheran EC</span>, <span class="author">Steingruber HE</span>, <span class="author">Sulston JE</span>, <span class="author">Taylor A</span>, <span class="author">Taylor RG</span>, <span class="author">Thorpe AA</span>, <span class="author">Tinsley E</span>, <span class="author">Warry GL</span>, <span class="author">Whittaker A</span>, <span class="author">Whittaker P</span>, <span class="author">Williams SH</span>, <span class="author">Wilmer TE</span>, <span class="author">Wooster R</span> and <span class="author">Wright CL</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridge, UK. <a href="mailto:%64%72%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#114;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We constructed maps for eight chromosomes (1, 6, 9, 10, 13, 20, X and (previously) 22), representing one-third of the genome, by building landmark maps, isolating bacterial clones and assembling contigs. By this approach, we could establish the long-range organization of the maps early in the project, and all contig extension, gap closure and problem-solving was simplified by containment within local regions. The maps currently represent more than 94% of the euchromatic (gene-containing) regions of these chromosomes in 176 contigs, and contain 96% of the chromosome-specific markers in the human gene map. By measuring the remaining gaps, we can assess chromosome length and coverage in sequenced clones.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">942-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237015">11237015</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057165">10.1038/35057165</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parasites are GO.</h4>
<p class="authors"><span class="author">Berriman M</span>, <span class="author">Aslett M</span>, <span class="author">Hall N</span> and <span class="author">Ivens A</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust, Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%62%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#98;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2001</span>;<span class="volume">17</span>;<span class="number">10</span>;<span class="pagerange">463-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11642257" rel="external">11642257</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insulin signalling.</h4>
<p class="authors"><span class="author">Bevan P</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK. <a href="mailto:%61%62%36%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#98;&#54;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2001</span>;<span class="volume">114</span>;<span class="number">Pt 8</span>;<span class="pagerange">1429-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11282018">11282018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mining the draft human genome.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Bateman A</span>, <span class="author">Clamp ME</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">The European Bioinformatics Institute, Hinxton, Cambridge, UK. <a href="mailto:%62%69%72%6E%65%79%40%65%62%69.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Now that the draft human genome sequence is available, everyone wants to be able to use it. However, we have perhaps become complacent about our ability to turn new genomes into lists of genes. The higher volume of data associated with a larger genome is accompanied by a much greater increase in complexity. We need to appreciate both the scale of the challenge of vertebrate genome analysis and the limitations of current gene prediction methods and understanding.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">827-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11236999">11236999</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2658632">2658632</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057004">10.1038/35057004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A novel poly(A)-binding protein gene (PABPC5) maps to an X-specific subinterval in the Xq21.3/Yp11.2 homology block of the human sex chromosomes.</h4>
<p class="authors"><span class="author">Blanco P</span>, <span class="author">Sargent CA</span>, <span class="author">Boucher CA</span>, <span class="author">Howell G</span>, <span class="author">Ross M</span> and <span class="author">Affara NA</span></p>
<p class="affiliation">Human Molecular Genetics Group, Division of Cellular and Molecular Pathology, Department of Pathology, University of Cambridge, Tennis Court Road, Cambridge, CB2 1QP, United Kingdom.</p><p class="abstract">The gene-poor human-specific Xq21.3/Yp11.2 block of homology exhibits 99% nucleotide identity, with the exception of an internal X-specific region containing the marker DXS214. This paper describes the characterization of a novel gene (PABPC5) from this X-specific subinterval that belongs to the poly(A)-binding protein gene family. The genomic structure of PABPC5 covers 4061 bp of an uninterrupted open reading frame (ORF) and a 5&#39;UTR spanning across two exons and associated with a CpG island; the potential 382-amino-acid protein contains four RNA recognition motif domains. PABPC5 has 73% nucleotide identity with PABPC4 over 1801 bp of the ORF. At the protein level, 60% identity and 75% similarity are obtained in the comparison with human PABPC4, as well as human, mouse, and Xenopus PABPC1. RT-PCR indicates that PABPC5 is expressed in fetal brain and in a range of adult tissues. Conservation of the PABPC5 ORF and genomic structure is shown in primates and rodents. The close proximity of this gene to translocation breakpoints associated with premature ovarian failure makes it a potential candidate for this condition.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2001</span>;<span class="volume">74</span>;<span class="number">1</span>;<span class="pagerange">1-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11374897" rel="external">11374897</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2001.6530" rel="external">10.1006/geno.2001.6530</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and characterization of KLHL4, a novel human homologue of the Drosophila Kelch gene that maps within the X-linked cleft palate and Ankyloglossia (CPX) critical region.</h4>
<p class="authors"><span class="author">Braybrook C</span>, <span class="author">Warry G</span>, <span class="author">Howell G</span>, <span class="author">Arnason A</span>, <span class="author">Bjornsson A</span>, <span class="author">Moore GE</span>, <span class="author">Ross MT</span> and <span class="author">Stanier P</span></p>
<p class="affiliation">Institute of Reproductive and Developmental Biology, Imperial College, Hammersmith Campus, Du Cane Road, London, W12 ONN, United Kingdom.</p><p class="abstract">X-linked cleft palate (CPX) is a rare nonsyndromic form of orofacial clefting that is, unlike more common forms, inherited as a highly penetrant Mendelian trait. Linkage studies using a large Icelandic kindred localized the gene to Xq21.3, and a physical map defining a 2.0-Mb candidate region was subsequently constructed. Genomic sequence is now available for much of the critical region and has been surveyed for potential transcriptional units. Through this analysis, we have identified a novel human homologue of Kelch, KLHL4. The transcript represents a mRNA of approximately 3.6 kb and encodes a protein of 718 amino acids. Protein domain analysis reveals six tandem repeats (Kelch repeats) at the C-terminus and a POZ/BTB protein-binding domain toward the N-terminus, characteristic of Drosophila Kelch and other family members. KLHL4 consists of 11 exons spanning a genomic interval of approximately 150 kb. From EST sequences and RT-PCR analysis, there is evidence for the use of alternative 3&#39; UTRs. The mRNA is expressed in a range of fetal tissues including tongue, palate, and mandible. Mutational analysis in affected CPX patients revealed one sequence alteration that was most likely to be a silent polymorphism.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2001</span>;<span class="volume">72</span>;<span class="number">2</span>;<span class="pagerange">128-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11401425" rel="external">11401425</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2000.6478" rel="external">10.1006/geno.2000.6478</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical and transcriptional mapping of the X-linked cleft palate and ankyloglossia (CPX) critical region.</h4>
<p class="authors"><span class="author">Braybrook C</span>, <span class="author">Warry G</span>, <span class="author">Howell G</span>, <span class="author">Mandryko V</span>, <span class="author">Arnason A</span>, <span class="author">Bjornsson A</span>, <span class="author">Ross MT</span>, <span class="author">Moore GE</span> and <span class="author">Stanier P</span></p>
<p class="affiliation">Institute of Reproductive and Developmental Biology, Imperial College School of Medicine, London, UK.</p><p class="abstract">Cleft palate most commonly occurs as a sporadic multifactorial disorder with a clear but difficult to define genetic component. As a semi-dominant disorder, X-linked cleft palate (CPX) provides a useful model to investigate a congenital defect that is little influenced by non-genetic factors. By using an Icelandic kindred, CPX has been localised between DXS1196 and DXS1217 and mapped, in a 3-Mb yeast artificial chromosome contig, at Xq21.3. Markers generated from this physical map have now been used to construct a contig of P1 and bacterial artificial chromosome clones for genomic DNA sequencing. Genomic DNA sequence analysis has revealed two novel expressed genes and two pseudogenes in the order Cen-KLHL4-LAMRL5-CAPZA1P-CPXCR1-Tel. KLHL4 and CPXCR1 are widely expressed in fetal tissues, including the tongue, mandible and palate. DNA mutation screening of CPXCR1 has revealed several sequence variants present on all affected CPX chromosomes. However, these variants have also been detected at a lower frequency on unaffected chromosomes, indicating that they are polymorphisms that are unlikely to cause the CPX phenotype.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">108</span>;<span class="number">6</span>;<span class="pagerange">537-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11499681" rel="external">11499681</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SpliceDB: database of canonical and non-canonical mammalian splice sites.</h4>
<p class="authors"><span class="author">Burset M</span>, <span class="author">Seledtsov IA</span> and <span class="author">Solovyev VV</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridge CB10 1SA, UK and Softberry Inc., 108 Corporate Park Drive, Suite 120, White Plains, NY 10604, USA.</p><p class="abstract">A database (SpliceDB) of known mammalian splice site sequences has been developed. We extracted 43 337 splice pairs from mammalian divisions of the gene-centered Infogene database, including sites from incomplete or alternatively spliced genes. Known EST sequences supported 22 815 of them. After discarding sequences with putative errors and ambiguous location of splice junctions the verified dataset includes 22 489 entries. Of these, 98.71% contain canonical GT-AG junctions (22 199 entries) and 0.56% have non-canonical GC-AG splice site pairs. The remainder (0.73%) occurs in a lot of small groups (with a maximum size of 0.05%). We especially studied non-canonical splice sites, which comprise 3.73% of GenBank annotated splice pairs. EST alignments allowed us to verify only the exonic part of splice sites. To check the conservative dinucleotides we compared sequences of human non-canonical splice sites with sequences from the high throughput genome sequencing project (HTG). Out of 171 human non-canonical and EST-supported splice pairs, 156 (91.23%) had a clear match in the human HTG. They can be classified after sequence analysis as: 79 GC-AG pairs (of which one was an error that corrected to GC-AG), 61 errors corrected to GT-AG canonical pairs, six AT-AC pairs (of which two were errors corrected to AT-AC), one case was produced from a non-existent intron, seven cases were found in HTG that were deposited to GenBank and finally there were only two other cases left of supported non-canonical splice pairs. The information about verified splice site sequences for canonical and non-canonical sites is presented in SpliceDB with the supporting evidence. We also built weight matrices for the major splice groups, which can be incorporated into gene prediction programs. SpliceDB is available at the computational genomic Web server of the Sanger Centre: http://genomic.sanger.ac. uk/spldb/SpliceDB.html and at http://www.softberry. com/spldb/SpliceDB.html.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">1</span>;<span class="pagerange">255-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11125105" rel="external">11125105</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC29840" rel="external">29840</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A clone-array pooled shotgun strategy for sequencing large genomes.</h4>
<p class="authors"><span class="author">Cai WW</span>, <span class="author">Chen R</span>, <span class="author">Gibbs RA</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">A simplified strategy for sequencing large genomes is proposed. Clone-Array Pooled Shotgun Sequencing (CAPSS) is based on pooling rows and columns of arrayed genomic clones, for shotgun library construction. Random sequences are accumulated, and the data are processed by sequential comparison of rows and columns to assemble the sequence of clones at points of intersection. Compared with either a clone-by-clone approach or whole-genome shotgun sequencing, CAPSS requires relatively few library constructions and only minimal computational power for a complete genome assembly. The strategy is suitable for sequencing large genomes for which there are no sequence-ready maps, but for which relatively high resolution STS maps and highly redundant BAC libraries are available. It is immediately applicable to the sequencing of mouse, rat, zebrafish, and other important genomes, and can be managed in a cooperative fashion to take advantage of a distributed international DNA sequencing capacity.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">1619-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11591638" rel="external">11591638</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.198101" rel="external">10.1101/gr.198101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An SSLP marker-anchored BAC framework map of the mouse genome.</h4>
<p class="authors"><span class="author">Cai WW</span>, <span class="author">Chow CW</span>, <span class="author">Damani S</span>, <span class="author">Gregory SG</span>, <span class="author">Marra M</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, Texas 77030, USA.</p><p class="abstract">We have constructed a BAC framework map of the mouse genome consisting of 2,808 PCR-confirmed BAC clusters, using a previously described method. Fingerprints of BACs from selected clusters confirm the accuracy of the map. Combined with BAC fingerprint data, the framework map covers 37% of the mouse genome.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">2</span>;<span class="pagerange">133-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11586294">11586294</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1001-133">10.1038/ng1001-133</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lost and found.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Thomson N</span>, <span class="author">Holden M</span>, <span class="author">Sebaihia M</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2001</span>;<span class="volume">9</span>;<span class="number">11</span>;<span class="pagerange">526-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11825692" rel="external">11825692</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integration of cytogenetic landmarks into the draft sequence of the human genome.</h4>
<p class="authors"><span class="author">Cheung VG</span>, <span class="author">Nowak N</span>, <span class="author">Jang W</span>, <span class="author">Kirsch IR</span>, <span class="author">Zhao S</span>, <span class="author">Chen XN</span>, <span class="author">Furey TS</span>, <span class="author">Kim UJ</span>, <span class="author">Kuo WL</span>, <span class="author">Olivier M</span>, <span class="author">Conroy J</span>, <span class="author">Kasprzyk A</span>, <span class="author">Massa H</span>, <span class="author">Yonescu R</span>, <span class="author">Sait S</span>, <span class="author">Thoreen C</span>, <span class="author">Snijders A</span>, <span class="author">Lemyre E</span>, <span class="author">Bailey JA</span>, <span class="author">Bruzel A</span>, <span class="author">Burrill WD</span>, <span class="author">Clegg SM</span>, <span class="author">Collins S</span>, <span class="author">Dhami P</span>, <span class="author">Friedman C</span>, <span class="author">Han CS</span>, <span class="author">Herrick S</span>, <span class="author">Lee J</span>, <span class="author">Ligon AH</span>, <span class="author">Lowry S</span>, <span class="author">Morley M</span>, <span class="author">Narasimhan S</span>, <span class="author">Osoegawa K</span>, <span class="author">Peng Z</span>, <span class="author">Plajzer-Frick I</span>, <span class="author">Quade BJ</span>, <span class="author">Scott D</span>, <span class="author">Sirotkin K</span>, <span class="author">Thorpe AA</span>, <span class="author">Gray JW</span>, <span class="author">Hudson J</span>, <span class="author">Pinkel D</span>, <span class="author">Ried T</span>, <span class="author">Rowen L</span>, <span class="author">Shen-Ong GL</span>, <span class="author">Strausberg RL</span>, <span class="author">Birney E</span>, <span class="author">Callen DF</span>, <span class="author">Cheng JF</span>, <span class="author">Cox DR</span>, <span class="author">Doggett NA</span>, <span class="author">Carter NP</span>, <span class="author">Eichler EE</span>, <span class="author">Haussler D</span>, <span class="author">Korenberg JR</span>, <span class="author">Morton CC</span>, <span class="author">Albertson D</span>, <span class="author">Schuler G</span>, <span class="author">de Jong PJ</span>, <span class="author">Trask BJ</span> and <span class="author">BAC Resource Consortium</span></p>
<p class="affiliation">Department of Pediatrics, University of Pennsylvania, The Children&#39;s Hospital of Philadelphia, 19104, USA.</p><p class="abstract">We have placed 7,600 cytogenetically defined landmarks on the draft sequence of the human genome to help with the characterization of genes altered by gross chromosomal aberrations that cause human disease. The landmarks are large-insert clones mapped to chromosome bands by fluorescence in situ hybridization. Each clone contains a sequence tag that is positioned on the genomic sequence. This genome-wide set of sequence-anchored clones allows structural and functional analyses of the genome. This resource represents the first comprehensive integration of cytogenetic, radiation hybrid, linkage and sequence maps of the human genome; provides an independent validation of the sequence map and framework for contig order and orientation; surveys the genome for large-scale duplications, which are likely to require special attention during sequence assembly; and allows a stringent assessment of sequence differences between the dark and light bands of chromosomes. It also provides insight into large-scale chromatin structure and the evolution of chromosomes and gene families and will accelerate our understanding of the molecular bases of human disease and cancer.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">953-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237021">11237021</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057192">10.1038/35057192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The morning after.</h4>
<p class="authors"><span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust, Sanger Institute, Hinxton, CB10 1SA, Cambridge, UK. <a href="mailto:%6D%69%63%68%65%6C%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#104;&#101;&#108;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2001</span>;<span class="volume">17</span>;<span class="number">12</span>;<span class="pagerange">688-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11718908" rel="external">11718908</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression microarray analysis in cancer biology, pharmacology, and drug development: progress and potential.</h4>
<p class="authors"><span class="author">Clarke PA</span>, <span class="author">te Poele R</span>, <span class="author">Wooster R</span> and <span class="author">Workman P</span></p>
<p class="affiliation">Cancer Research Campaign Centre for Cancer Therapeutics, E Block, Institute of Cancer Research, 15 Cotswold Road, SM2 5NG, Sutton, Surrey, UK.</p><p class="abstract">With the imminent completion of the Human Genome Project, biomedical research is being revolutionised by the ability to carry out investigations on a genome wide scale. This is particularly important in cancer, a disease that is caused by accumulating abnormalities in the sequence and expression of a number of critical genes. Gene expression microarray technology is gaining increasingly widespread use as a means to determine the expression of potentially all human genes at the level of messenger RNA. In this commentary, we review developments in gene expression microarray technology and illustrate the progress and potential of the methodology in cancer biology, pharmacology, and drug development. Important applications include: (a) development of a more global understanding of the gene expression abnormalities that contribute to malignant progression; (b) discovery of new diagnostic and prognostic indicators and biomarkers of therapeutic response; (c) identification and validation of new molecular targets for drug development; (d) provision of an improved understanding of the molecular mode of action during lead identification and optimisation, including structure-activity relationships for on-target versus off-target effects; (e) prediction of potential side-effects during preclinical development and toxicology studies; (f) confirmation of a molecular mode of action during hypothesis-testing clinical trials; (g) identification of genes involved in conferring drug sensitivity and resistance; and (h) prediction of patients most likely to benefit from the drug and use in general pharmacogenomic studies. As a result of further technological improvements and decreasing costs, the use of microarrays will become an essential and potentially routine tool for cancer and biomedical research.</p>
<p class="publication"><span class="title">Biochemical pharmacology</span>&nbsp;<span class="year">2001</span>;<span class="volume">62</span>;<span class="number">10</span>;<span class="pagerange">1311-36</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11709192" rel="external">11709192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Disruption of an imprinted gene cluster by a targeted chromosomal translocation in mice.</h4>
<p class="authors"><span class="author">Cleary MA</span>, <span class="author">van Raamsdonk CD</span>, <span class="author">Levorse J</span>, <span class="author">Zheng B</span>, <span class="author">Bradley A</span> and <span class="author">Tilghman SM</span></p>
<p class="affiliation">Howard Hughes Medical Institute and Department of Molecular Biology, Princeton University, Princeton, New Jersey, USA.</p><p class="abstract">Genomic imprinting is an epigenetic process in which the activity of a gene is determined by its parent of origin. Mechanisms governing genomic imprinting are just beginning to be understood. However, the tendency of imprinted genes to exist in chromosomal clusters suggests a sharing of regulatory elements. To better understand imprinted gene clustering, we disrupted a cluster of imprinted genes on mouse distal chromosome 7 using the Cre/loxP recombination system. In mice carrying a site-specific translocation separating Cdkn1c and Kcnq1, imprinting of the genes retained on chromosome 7, including Kcnq1, Kcnq1ot1, Ascl2, H19 and Igf2, is unaffected, demonstrating that these genes are not regulated by elements near or telomeric to Cdkn1c. In contrast, expression and imprinting of the translocated Cdkn1c, Slc22a1l and Tssc3 on chromosome 11 are affected, consistent with the hypothesis that elements regulating both expression and imprinting of these genes lie within or proximal to Kcnq1. These data support the proposal that chromosomal abnormalities, including translocations, within KCNQ1 that are associated with the human disease Beckwith-Wiedemann syndrome (BWS) may disrupt CDKN1C expression. These results underscore the importance of gene clustering for the proper regulation of imprinted genes.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">1</span>;<span class="pagerange">78-82</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11528397">11528397</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng715">10.1038/ng715</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Massive gene decay in the leprosy bacillus.</h4>
<p class="authors"><span class="author">Cole ST</span>, <span class="author">Eiglmeier K</span>, <span class="author">Parkhill J</span>, <span class="author">James KD</span>, <span class="author">Thomson NR</span>, <span class="author">Wheeler PR</span>, <span class="author">Honor&eacute; N</span>, <span class="author">Garnier T</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Mungall K</span>, <span class="author">Basham D</span>, <span class="author">Brown D</span>, <span class="author">Chillingworth T</span>, <span class="author">Connor R</span>, <span class="author">Davies RM</span>, <span class="author">Devlin K</span>, <span class="author">Duthoy S</span>, <span class="author">Feltwell T</span>, <span class="author">Fraser A</span>, <span class="author">Hamlin N</span>, <span class="author">Holroyd S</span>, <span class="author">Hornsby T</span>, <span class="author">Jagels K</span>, <span class="author">Lacroix C</span>, <span class="author">Maclean J</span>, <span class="author">Moule S</span>, <span class="author">Murphy L</span>, <span class="author">Oliver K</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Rutherford KM</span>, <span class="author">Rutter S</span>, <span class="author">Seeger K</span>, <span class="author">Simon S</span>, <span class="author">Simmonds M</span>, <span class="author">Skelton J</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Stevens K</span>, <span class="author">Taylor K</span>, <span class="author">Whitehead S</span>, <span class="author">Woodward JR</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, Paris, France. <a href="mailto:%73%74%63%6F%6C%65%40%70%61%73%74%65%75%72.%66%72">&#115;&#116;&#99;&#111;&#108;&#101;&#64;&#112;&#97;&#115;&#116;&#101;&#117;&#114;&#46;&#102;&#114;</a></p><p class="abstract">Leprosy, a chronic human neurological disease, results from infection with the obligate intracellular pathogen Mycobacterium leprae, a close relative of the tubercle bacillus. Mycobacterium leprae has the longest doubling time of all known bacteria and has thwarted every effort at culture in the laboratory. Comparing the 3.27-megabase (Mb) genome sequence of an armadillo-derived Indian isolate of the leprosy bacillus with that of Mycobacterium tuberculosis (4.41 Mb) provides clear explanations for these properties and reveals an extreme case of reductive evolution. Less than half of the genome contains functional genes but pseudogenes, with intact counterparts in M. tuberculosis, abound. Genome downsizing and the current mosaic arrangement appear to have resulted from extensive recombination events between dispersed repetitive sequences. Gene deletion and decay have eliminated many important metabolic activities including siderophore production, part of the oxidative and most of the microaerophilic and anaerobic respiratory chains, and numerous catabolic systems and their regulatory circuits.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6823</span>;<span class="pagerange">1007-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11234002">11234002</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35059006">10.1038/35059006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence, structure and pathology of the fully annotated terminal 2 Mb of the short arm of human chromosome 16.</h4>
<p class="authors"><span class="author">Daniels RJ</span>, <span class="author">Peden JF</span>, <span class="author">Lloyd C</span>, <span class="author">Horsley SW</span>, <span class="author">Clark K</span>, <span class="author">Tufarelli C</span>, <span class="author">Kearney L</span>, <span class="author">Buckle VJ</span>, <span class="author">Doggett NA</span>, <span class="author">Flint J</span> and <span class="author">Higgs DR</span></p>
<p class="affiliation">MRC Molecular Haematology Unit, Weatherall Institute for Molecular Medicine, John Radcliffe Hospital, Oxford OX3 9DS, UK.</p><p class="abstract">We have sequenced 1949 kb from the terminal Giemsa light band of human chromosome 16p, enabling us to fully annotate the region extending from the telomeric repeats to the previously published tuberous sclerosis disease 2 (TSC2) and polycystic kidney disease 1 (PKD1) genes. This region can be subdivided into two GC-rich, Alu-rich domains and one GC-rich, Alu-poor domain. The entire region is extremely gene rich, containing 100 confirmed genes and 20 predicted genes. Many of the genes encode widely expressed proteins orchestrating basic cellular processes (e.g. DNA recombination, repair, transcription, RNA processing, signal transduction, intracellular signalling and mRNA translation). Others, such as the alpha globin genes (HBA1 and HBA2), PDIP and BAIAP3, are specialized tissue-restricted genes. Some of the genes have been previously implicated in the pathophysiology of important human genetic diseases (e.g. asthma, cataracts and the ATR-16 syndrome). Others are known disease genes for alpha thalassaemia, adult polycystic kidney disease and tuberous sclerosis. There is also linkage evidence for bipolar affective disorder, epilepsy and autism in this region. Sixty-three chromosomal deletions reported here and elsewhere allow us to interpret the results of removing progressively larger numbers of genes from this well defined human telomeric region.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">10</span>;<span class="number">4</span>;<span class="pagerange">339-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11157797" rel="external">11157797</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A SNP resource for human chromosome 22: extracting dense clusters of SNPs from the genomic sequence.</h4>
<p class="authors"><span class="author">Dawson E</span>, <span class="author">Chen Y</span>, <span class="author">Hunt S</span>, <span class="author">Smink LJ</span>, <span class="author">Hunt A</span>, <span class="author">Rice K</span>, <span class="author">Livingston S</span>, <span class="author">Bumpstead S</span>, <span class="author">Bruskiewich R</span>, <span class="author">Sham P</span>, <span class="author">Ganske R</span>, <span class="author">Adams M</span>, <span class="author">Kawasaki K</span>, <span class="author">Shimizu N</span>, <span class="author">Minoshima S</span>, <span class="author">Roe B</span>, <span class="author">Bentley D</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The recent publication of the complete sequence of human chromosome 22 provides a platform from which to investigate genomic sequence variation. We report the identification and characterization of 12,267 potential variants (SNPs and other small insertions/deletions) of human chromosome 22, discovered in the overlaps of 460 clones used for the chromosome sequencing. We found, on average, 1 potential variant every 1.07 kb and approximately 18% of the potential variants involve insertions/deletions. The SNPs have been positioned both relative to each other, and to genes, predicted genes, repeat sequences, other genetic markers, and the 2730 SNPs previously identified on the chromosome. A subset of the SNPs were verified experimentally using either PCR-RFLP or genomic Invader assays. These experiments confirmed 92% of the potential variants in a panel of 92 individuals. [Details of the SNPs and RFLP assays can be found at http://www.sanger.ac.uk and in dbSNP.]</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">1</span>;<span class="pagerange">170-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11156626">11156626</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311026">311026</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">HOMSTRAD: adding sequence information to structure-based alignments of homologous protein families.</h4>
<p class="authors"><span class="author">de Bakker PI</span>, <span class="author">Bateman A</span>, <span class="author">Burke DF</span>, <span class="author">Miguel RN</span>, <span class="author">Mizuguchi K</span>, <span class="author">Shi J</span>, <span class="author">Shirai H</span> and <span class="author">Blundell TL</span></p>
<p class="affiliation">Department of Biochemistry, University of Cambridge, 80 Tennis Court Road, Cambridge CB2 1GA, UK.</p><p class="abstract">summary: We describe an extension to the Homologous Structure Alignment Database (HOMSTRAD; Mizuguchi et al., Protein Sci., 7, 2469-2471, 1998a) to include homologous sequences derived from the protein families database Pfam (Bateman et al., Nucleic Acids Res., 28, 263-266, 2000). HOMSTRAD is integrated with the server FUGUE (Shi et al., submitted, 2001) for recognition and alignment of homologues, benefitting from the combination of abundant sequence information and accurate structure-based alignments. AVAILABILITY The HOMSTRAD database is available at: http://www-cryst.bioc.cam.ac.uk/homstrad/. Query sequences can be submitted to the homology recognition/alignment server FUGUE at: http://www-cryst.bioc.cam.ac.uk/fugue/.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2001</span>;<span class="volume">17</span>;<span class="number">8</span>;<span class="pagerange">748-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11524380" rel="external">11524380</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parasite genome initiatives.</h4>
<p class="authors"><span class="author">Degrave WM</span>, <span class="author">Melville S</span>, <span class="author">Ivens A</span> and <span class="author">Aslett M</span></p>
<p class="affiliation">DBBM - IOC/Fiocruz, Av. Brasil 4365, Manguinhos, 21045-900, Rio de Janeiro, Brazil. <a href="mailto:%77%64%65%67%72%61%76%65%40%64%62%62%6D.%66%69%6F%63%72%75%7A.%62%72">&#119;&#100;&#101;&#103;&#114;&#97;&#118;&#101;&#64;&#100;&#98;&#98;&#109;&#46;&#102;&#105;&#111;&#99;&#114;&#117;&#122;&#46;&#98;&#114;</a></p><p class="abstract">During 1993-1994, scientists from developing and developed countries planned and initiated a number of parasite genome projects and several consortiums for the mapping and sequencing of these medium-sized genomes were established, often based on already ongoing scientific collaborations. Financial and other support came from WHO/TDR, Wellcome Trust and other funding agencies. Thus, the genomes of Plasmodium falciparum, Schistosoma mansoni, Trypanosoma cruzi, Leishmania major, Trypanosoma brucei, Brugia malayi and other pathogenic nematodes are now under study. From an initial phase of network formation, mapping efforts and resource building (EST, GSS, phage, cosmid, BAC and YAC library constructions), sequencing was initiated in gene discovery projects but soon also on a small chromosome, and now on a fully fledged genome scale. Proteomics, functional analysis, genetic manipulation and microarray analysis are ongoing to different degrees in the respective genome initiatives, and as the funding for the whole genome sequencing becomes secured, most of the participating laboratories, apart from larger sequencing centres, become oriented to post-genomics. Bioinformatics networks are being expanded, including in developing countries, for data mining, annotation and in-depth analysis.</p>
<p class="publication"><span class="title">International journal for parasitology</span>&nbsp;<span class="year">2001</span>;<span class="volume">31</span>;<span class="number">5-6</span>;<span class="pagerange">532-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11334938" rel="external">11334938</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A superfamily of variant genes encoded in the subtelomeric region of Plasmodium vivax.</h4>
<p class="authors"><span class="author">del Portillo HA</span>, <span class="author">Fernandez-Becerra C</span>, <span class="author">Bowman S</span>, <span class="author">Oliver K</span>, <span class="author">Preuss M</span>, <span class="author">Sanchez CP</span>, <span class="author">Schneider NK</span>, <span class="author">Villalobos JM</span>, <span class="author">Rajandream MA</span>, <span class="author">Harris D</span>, <span class="author">Pereira da Silva LH</span>, <span class="author">Barrell B</span> and <span class="author">Lanzer M</span></p>
<p class="affiliation">Departamento de Parasitologia, Instituto de Ci&ecirc;ncias Biom&eacute;dicas, Universidade de S&atilde;o Paulo, Av. Lineu Prestes 1374, S&atilde;o Paulo, SP 05508-900, Brazil. <a href="mailto:%68%65%72%6E%61%6E%64%6F%40%69%63%62.%75%73%70.%62%72">&#104;&#101;&#114;&#110;&#97;&#110;&#100;&#111;&#64;&#105;&#99;&#98;&#46;&#117;&#115;&#112;&#46;&#98;&#114;</a></p><p class="abstract">The malarial parasite Plasmodium vivax causes disease in humans, including chronic infections and recurrent relapses, but the course of infection is rarely fatal, unlike that caused by Plasmodium falciparum. To investigate differences in pathogenicity between P. vivax and P. falciparum, we have compared the subtelomeric domains in the DNA of these parasites. In P. falciparum, subtelomeric domains are conserved and contain ordered arrays of members of multigene families, such as var, rif and stevor, encoding virulence determinants of cytoadhesion and antigenic variation. Here we identify, through the analysis of a continuous 155,711-base-pair sequence of a P. vivax chromosome end, a multigene family called vir, which is specific to P. vivax. The vir genes are present at about 600-1,000 copies per haploid genome and encode proteins that are immunovariant in natural infections, indicating that they may have a functional role in establishing chronic infection through antigenic variation.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">410</span>;<span class="number">6830</span>;<span class="pagerange">839-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11298455">11298455</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35071118">10.1038/35071118</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Construction of transcript maps by somatic cell/radiation hybrid mapping. The human gene map.</h4>
<p class="authors"><span class="author">Deloukas P</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2001</span>;<span class="volume">175</span>;<span class="pagerange">155-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11462833" rel="external">11462833</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-235-X%3A155" rel="external">10.1385/1-59259-235-X:155</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Map integration. From a genetic map to a physical gene map and ultimately to the sequence map.</h4>
<p class="authors"><span class="author">Deloukas P</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2001</span>;<span class="volume">175</span>;<span class="pagerange">129-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11462831" rel="external">11462831</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-235-X%3A129" rel="external">10.1385/1-59259-235-X:129</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence and comparative analysis of human chromosome 20.</h4>
<p class="authors"><span class="author">Deloukas P</span>, <span class="author">Matthews LH</span>, <span class="author">Ashurst J</span>, <span class="author">Burton J</span>, <span class="author">Gilbert JG</span>, <span class="author">Jones M</span>, <span class="author">Stavrides G</span>, <span class="author">Almeida JP</span>, <span class="author">Babbage AK</span>, <span class="author">Bagguley CL</span>, <span class="author">Bailey J</span>, <span class="author">Barlow KF</span>, <span class="author">Bates KN</span>, <span class="author">Beard LM</span>, <span class="author">Beare DM</span>, <span class="author">Beasley OP</span>, <span class="author">Bird CP</span>, <span class="author">Blakey SE</span>, <span class="author">Bridgeman AM</span>, <span class="author">Brown AJ</span>, <span class="author">Buck D</span>, <span class="author">Burrill W</span>, <span class="author">Butler AP</span>, <span class="author">Carder C</span>, <span class="author">Carter NP</span>, <span class="author">Chapman JC</span>, <span class="author">Clamp M</span>, <span class="author">Clark G</span>, <span class="author">Clark LN</span>, <span class="author">Clark SY</span>, <span class="author">Clee CM</span>, <span class="author">Clegg S</span>, <span class="author">Cobley VE</span>, <span class="author">Collier RE</span>, <span class="author">Connor R</span>, <span class="author">Corby NR</span>, <span class="author">Coulson A</span>, <span class="author">Coville GJ</span>, <span class="author">Deadman R</span>, <span class="author">Dhami P</span>, <span class="author">Dunn M</span>, <span class="author">Ellington AG</span>, <span class="author">Frankland JA</span>, <span class="author">Fraser A</span>, <span class="author">French L</span>, <span class="author">Garner P</span>, <span class="author">Grafham DV</span>, <span class="author">Griffiths C</span>, <span class="author">Griffiths MN</span>, <span class="author">Gwilliam R</span>, <span class="author">Hall RE</span>, <span class="author">Hammond S</span>, <span class="author">Harley JL</span>, <span class="author">Heath PD</span>, <span class="author">Ho S</span>, <span class="author">Holden JL</span>, <span class="author">Howden PJ</span>, <span class="author">Huckle E</span>, <span class="author">Hunt AR</span>, <span class="author">Hunt SE</span>, <span class="author">Jekosch K</span>, <span class="author">Johnson CM</span>, <span class="author">Johnson D</span>, <span class="author">Kay MP</span>, <span class="author">Kimberley AM</span>, <span class="author">King A</span>, <span class="author">Knights A</span>, <span class="author">Laird GK</span>, <span class="author">Lawlor S</span>, <span class="author">Lehvaslaiho MH</span>, <span class="author">Leversha M</span>, <span class="author">Lloyd C</span>, <span class="author">Lloyd DM</span>, <span class="author">Lovell JD</span>, <span class="author">Marsh VL</span>, <span class="author">Martin SL</span>, <span class="author">McConnachie LJ</span>, <span class="author">McLay K</span>, <span class="author">McMurray AA</span>, <span class="author">Milne S</span>, <span class="author">Mistry D</span>, <span class="author">Moore MJ</span>, <span class="author">Mullikin JC</span>, <span class="author">Nickerson T</span>, <span class="author">Oliver K</span>, <span class="author">Parker A</span>, <span class="author">Patel R</span>, <span class="author">Pearce TA</span>, <span class="author">Peck AI</span>, <span class="author">Phillimore BJ</span>, <span class="author">Prathalingam SR</span>, <span class="author">Plumb RW</span>, <span class="author">Ramsay H</span>, <span class="author">Rice CM</span>, <span class="author">Ross MT</span>, <span class="author">Scott CE</span>, <span class="author">Sehra HK</span>, <span class="author">Shownkeen R</span>, <span class="author">Sims S</span>, <span class="author">Skuce CD</span>, <span class="author">Smith ML</span>, <span class="author">Soderlund C</span>, <span class="author">Steward CA</span>, <span class="author">Sulston JE</span>, <span class="author">Swann M</span>, <span class="author">Sycamore N</span>, <span class="author">Taylor R</span>, <span class="author">Tee L</span>, <span class="author">Thomas DW</span>, <span class="author">Thorpe A</span>, <span class="author">Tracey A</span>, <span class="author">Tromans AC</span>, <span class="author">Vaudin M</span>, <span class="author">Wall M</span>, <span class="author">Wallis JM</span>, <span class="author">Whitehead SL</span>, <span class="author">Whittaker P</span>, <span class="author">Willey DL</span>, <span class="author">Williams L</span>, <span class="author">Williams SA</span>, <span class="author">Wilming L</span>, <span class="author">Wray PW</span>, <span class="author">Hubbard T</span>, <span class="author">Durbin RM</span>, <span class="author">Bentley DR</span>, <span class="author">Beck S</span> and <span class="author">Rogers J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%70%61%6E%6F%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#110;&#111;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The finished sequence of human chromosome 20 comprises 59,187,298 base pairs (bp) and represents 99.4% of the euchromatic DNA. A single contig of 26 megabases (Mb) spans the entire short arm, and five contigs separated by gaps totalling 320 kb span the long arm of this metacentric chromosome. An additional 234,339 bp of sequence has been determined within the pericentromeric region of the long arm. We annotated 727 genes and 168 pseudogenes in the sequence. About 64% of these genes have a 5&#39; and a 3&#39; untranslated region and a complete open reading frame. Comparative analysis of the sequence of chromosome 20 to whole-genome shotgun-sequence data of two other vertebrates, the mouse Mus musculus and the puffer fish Tetraodon nigroviridis, provides an independent measure of the efficiency of gene annotation, and indicates that this analysis may account for more than 95% of all coding exons and almost all genes.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">414</span>;<span class="number">6866</span>;<span class="pagerange">865-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11780052">11780052</a>; DOI: <a href="http://dx.doi.org/10.1038%2F414865a">10.1038/414865a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome comparison of Campylobacter jejuni human isolates using a low-cost microarray reveals extensive genetic diversity.</h4>
<p class="authors"><span class="author">Dorrell N</span>, <span class="author">Mangan JA</span>, <span class="author">Laing KG</span>, <span class="author">Hinds J</span>, <span class="author">Linton D</span>, <span class="author">Al-Ghusein H</span>, <span class="author">Barrell BG</span>, <span class="author">Parkhill J</span>, <span class="author">Stoker NG</span>, <span class="author">Karlyshev AV</span>, <span class="author">Butcher PD</span> and <span class="author">Wren BW</span></p>
<p class="affiliation">Department of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK.</p><p class="abstract">Campylobacter jejuni is the leading cause of bacterial food-borne diarrhoeal disease throughout the world, and yet is still a poorly understood pathogen. Whole genome microarray comparisons of 11 C. jejuni strains of diverse origin identified genes in up to 30 NCTC 11168 loci ranging from 0.7 to 18.7 kb that are either absent or highly divergent in these isolates. Many of these regions are associated with the biosynthesis of surface structures including flagella, lipo-oligosaccharide, and the newly identified capsule. Other strain-variable genes of known function include those responsible for iron acquisition, DNA restriction/modification, and sialylation. In fact, at least 21% of genes in the sequenced strain appear dispensable as they are absent or highly divergent in one or more of the isolates tested, thus defining 1300 C. jejuni core genes. Such core genes contribute mainly to metabolic, biosynthetic, cellular, and regulatory processes, but many virulence determinants are also conserved. Comparison of the capsule biosynthesis locus revealed conservation of all the genes in this region in strains with the same Penner serotype as strain NCTC 11168. By contrast, between 5 and 17 NCTC 11168 genes in this region are either absent or highly divergent in strains of a different serotype from the sequenced strain, providing further evidence that the capsule accounts for Penner serotype specificity. These studies reveal extensive genetic diversity among C. jejuni strains and pave the way toward identifying correlates of pathogenicity and developing improved epidemiological tools for this problematic pathogen.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062511</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">1706-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11591647" rel="external">11591647</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311159" rel="external">311159</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.185801" rel="external">10.1101/gr.185801</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The decaying genome of Mycobacterium leprae.</h4>
<p class="authors"><span class="author">Eiglmeier K</span>, <span class="author">Parkhill J</span>, <span class="author">Honor&eacute; N</span>, <span class="author">Garnier T</span>, <span class="author">Tekaia F</span>, <span class="author">Telenti A</span>, <span class="author">Klatser P</span>, <span class="author">James KD</span>, <span class="author">Thomson NR</span>, <span class="author">Wheeler PR</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Mungall K</span>, <span class="author">Barrell BG</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, 28 Rue du Docteur Roux, 75724 Paris, France.</p><p class="abstract">Everything that we need to know about Mycobacterium leprae, a close relative of the tubercle bacillus, is encrypted in its genome. Inspection of the 3.27 Mb genome sequence of an armadillo-derived Indian isolate of the leprosy bacillus identified 1,605 genes encoding proteins and 50 genes for stable RNA species. Comparison with the genome sequence of Mycobacterium tuberculosis revealed an extreme case of reductive evolution, since less than half of the genome contains functional genes while inactivated or pseudogenes are highly abundant. The level of gene duplication was approximately 34% and, on classification of the proteins into families, the largest functional groups were found to be involved in the metabolism and modification of fatty acids and polyketides, transport of metabolites, cell envelope synthesis and gene regulation. Reductive evolution, gene decay and genome downsizing have eliminated entire metabolic pathways, together with their regulatory circuits and accessory functions, particularly those involved in catabolism. This may explain the unusually long generation time and account for our inability to culture the leprosy bacillus.</p>
<p class="publication"><span class="title">Leprosy review</span>&nbsp;<span class="year">2001</span>;<span class="volume">72</span>;<span class="number">4</span>;<span class="pagerange">387-98</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11826475" rel="external">11826475</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A 6.9-Mb high-resolution BAC/PAC contig of human 4p15.3-p16.1, a candidate region for bipolar affective disorder.</h4>
<p class="authors"><span class="author">Evans KL</span>, <span class="author">Le Hellard S</span>, <span class="author">Morris SW</span>, <span class="author">Lawson D</span>, <span class="author">Whitton C</span>, <span class="author">Semple CA</span>, <span class="author">Fantes JA</span>, <span class="author">Torrance HS</span>, <span class="author">Malloy MP</span>, <span class="author">Maule JC</span>, <span class="author">Humphray SJ</span>, <span class="author">Ross MT</span>, <span class="author">Bentley DR</span>, <span class="author">Muir WJ</span>, <span class="author">Blackwood DH</span> and <span class="author">Porteous DJ</span></p>
<p class="affiliation">Medical Genetics Section, MRC Human Genetics Unit, University of Edinburgh, Molecular Medicine Centre, Crewe Road, Edinburgh, EH4 2XU, United Kingdom. <a href="mailto:%4B%61%74%68%79.%45%76%61%6E%73%40%65%64.%61%63.%75%6B">&#75;&#97;&#116;&#104;&#121;&#46;&#69;&#118;&#97;&#110;&#115;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Bipolar affective disorder (BPAD) is a complex disease with a significant genetic component and a population lifetime risk of 1%. Our previous work identified a region of human chromosome 4p that showed significant linkage to BPAD in a large pedigree. Here, we report the construction of an accurate, high-resolution physical map of 6.9 Mb of human chromosome 4p15.3-p16.1, which includes an 11-cM (5.8 Mb) critical region for BPAD. The map consists of 460 PAC and BAC clones ordered by a combination of STS content analysis and restriction fragment fingerprinting, with a single approximately 300-kb gap remaining. A total of 289 new and existing markers from a wide range of sources have been localized on the contig, giving an average marker resolution of 1 marker/23 kb. The STSs include 57 ESTs, 9 of which represent known genes. This contig is an essential preliminary to the identification of candidate genes that predispose to bipolar affective disorder, to the completion of the sequence of the region, and to the development of a high-density SNP map.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2001</span>;<span class="volume">71</span>;<span class="number">3</span>;<span class="pagerange">315-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11170748" rel="external">11170748</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2000.6432" rel="external">10.1006/geno.2000.6432</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer and genomics.</h4>
<p class="authors"><span class="author">Futreal PA</span>, <span class="author">Kasprzyk A</span>, <span class="author">Birney E</span>, <span class="author">Mullikin JC</span>, <span class="author">Wooster R</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Sanger Centre, Cambridge, UK.</p><p class="abstract">Identification of the genes that cause oncogenesis is a central aim of cancer research. We searched the proteins predicted from the draft human genome sequence for paralogues of known tumour suppressor genes, but no novel genes were identified. We then assessed whether it was possible to search directly for oncogenic sequence changes in cancer cells by comparing cancer genome sequences against the draft genome. Apparently chimaeric transcripts (from oncogenic fusion genes generated by chromosomal translocations, the ends of which mapped to different genomic locations) were detected to the same degree in both normal and neoplastic tissues, indicating a significant level of false positives. Our experiment underscores the limited amount and variable quality of DNA sequence from cancer cells that is currently available.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">850-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237008">11237008</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057046">10.1038/35057046</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Long-range comparison of human and mouse SCL loci: localized regions of sensitivity to restriction endonucleases correspond precisely with peaks of conserved noncoding sequences.</h4>
<p class="authors"><span class="author">G&ouml;ttgens B</span>, <span class="author">Gilbert JG</span>, <span class="author">Barton LM</span>, <span class="author">Grafham D</span>, <span class="author">Rogers J</span>, <span class="author">Bentley DR</span> and <span class="author">Green AR</span></p>
<p class="affiliation">The Wellcome Trust Centre for Molecular Mechanisms in Disease, Cambridge Institute for Medical Research, Addenbrooke&#39;s Hospital Site, Cambridge CB2 2XY, UK. <a href="mailto:%62%67%32%30%30%40%63%61%6D.%61%63.%75%6B">&#98;&#103;&#50;&#48;&#48;&#64;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Long-range comparative sequence analysis provides a powerful strategy for identifying conserved regulatory elements. The stem cell leukemia (SCL) gene encodes a bHLH transcription factor with a pivotal role in hemopoiesis and vasculogenesis, and it displays a highly conserved expression pattern. We present here a detailed sequence comparison of 193 kb of the human SCL locus to 234 kb of the mouse SCL locus. Four new genes have been identified together with an ancient mitochondrial insertion in the human locus. The SCL gene is flanked upstream by the SIL gene and downstream by the MAP17 gene in both species, but the gene order is not collinear downstream from MAP17. To facilitate rapid identification of candidate regulatory elements, we have developed a new sequence analysis tool (SynPlot) that automates the graphical display of large-scale sequence alignments. Unlike existing programs, SynPlot can display the locus features of more than one sequence, thereby indicating the position of homology peaks relative to the structure of all sequences in the alignment. In addition, high-resolution analysis of the chromatin structure of the mouse SCL gene permitted the accurate positioning of localized zones accessible to restriction endonucleases. Zones known to be associated with functional regulatory regions were found to correspond precisely with peaks of human/mouse homology, thus demonstrating that long-range human/mouse sequence comparisons allow accurate prediction of the extent of accessible DNA associated with active regulatory regions.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">1</span>;<span class="pagerange">87-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11156618" rel="external">11156618</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311011" rel="external">311011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complex repeats of Dictyostelium discoideum.</h4>
<p class="authors"><span class="author">Gl&ouml;ckner G</span>, <span class="author">Szafranski K</span>, <span class="author">Winckler T</span>, <span class="author">Dingermann T</span>, <span class="author">Quail MA</span>, <span class="author">Cox E</span>, <span class="author">Eichinger L</span>, <span class="author">Noegel AA</span> and <span class="author">Rosenthal A</span></p>
<p class="affiliation">IMB Jena, Department of Genome Analysis, D-07745 Jena, Germany. <a href="mailto:%67%65%72%6E%6F%74%40%69%6D%62-%6A%65%6E%61.%64%65">&#103;&#101;&#114;&#110;&#111;&#116;&#64;&#105;&#109;&#98;&#45;&#106;&#101;&#110;&#97;&#46;&#100;&#101;</a></p><p class="abstract">In the course of determining the sequence of the Dictyostelium discoideum genome we have characterized in detail the quantity and nature of interspersed repetitive elements present in this species. Several of the most abundant small complex repeats and transposons (DIRS-1; TRE3-A,B; TRE5-A; skipper; Tdd-4; H3R) have been described previously. In our analysis we have identified additional elements. Thus, we can now present a complete list of complex repetitive elements in D. discoideum. All elements add up to 10% of the genome. Some of the newly described elements belong to established classes (TRE3-C, D; TRE5-B,C; DGLT-A,P; Tdd-5). However, we have also defined two new classes of DNA transposable elements (DDT and thug) that have not been described thus far. Based on the nucleotide amount, we calculated the least copy number in each family. These vary between &lt;10 up to &gt;200 copies. Unique sequences adjacent to the element ends and truncation points in elements gave a measure for the fragmentation of the elements. Furthermore, we describe the diversity of single elements with regard to polymorphisms and conserved structures. All elements show insertion preference into loci in which other elements of the same family reside. The analysis of the complex repeats is a valuable data resource for the ongoing assembly of whole D. discoideum chromosomes.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">585-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11282973" rel="external">11282973</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311061" rel="external">311061</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.162201" rel="external">10.1101/gr.162201</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomics of Mycobacterium bovis.</h4>
<p class="authors"><span class="author">Gordon SV</span>, <span class="author">Eiglmeier K</span>, <span class="author">Garnier T</span>, <span class="author">Brosch R</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell B</span>, <span class="author">Cole ST</span> and <span class="author">Hewinson RG</span></p>
<p class="affiliation">Veterinary Laboratories Agency, Woodham Lane, New Haw, Addlestone, Surrey, UK. <a href="mailto:%61%70%37%35%40%67%74%6E%65%74.%67%6F%76.%75%6B">&#97;&#112;&#55;&#53;&#64;&#103;&#116;&#110;&#101;&#116;&#46;&#103;&#111;&#118;&#46;&#117;&#107;</a></p><p class="abstract">The imminent completion of the genome sequence of Mycobacterium bovis will reveal the genetic blueprint for this most successful pathogen. Comparative analysis with the genome sequences of M. tuberculosis and M. bovis BCG promises to expose the genetic basis for the phenotypic differences between the tubercle bacilli, offering unparalleled insight into the virulence factors of the M. tuberculosis complex. Initial analysis of the sequence data has already revealed a novel deletion from M. bovis, as well as identifying variation in members of the PPE family of proteins. As the study of bacterial pathogenicity enters the postgenomic phase, the genome sequence of M. bovis promises to serve as a cornerstone of mycobacterial genetics.</p>
<p class="publication"><span class="title">Tuberculosis (Edinburgh, Scotland)</span>&nbsp;<span class="year">2001</span>;<span class="volume">81</span>;<span class="number">1-2</span>;<span class="pagerange">157-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11463237" rel="external">11463237</a>; DOI: <a href="http://dx.doi.org/10.1054%2Ftube.2000.0269" rel="external">10.1054/tube.2000.0269</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing bacterial artificial chromosomes.</h4>
<p class="authors"><span class="author">Harris DE</span> and <span class="author">Murphy L</span></p>
<p class="affiliation">The Sanger Centre, Cambridge, UK.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2001</span>;<span class="volume">175</span>;<span class="pagerange">217-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11462837" rel="external">11462837</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-235-X%3A217" rel="external">10.1385/1-59259-235-X:217</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation and haplotype analysis of the CFTR gene in atypically mild cystic fibrosis patients from Northern Ireland.</h4>
<p class="authors"><span class="author">Hughes D</span>, <span class="author">D&ouml;rk T</span>, <span class="author">Stuhrmann M</span> and <span class="author">Graham C</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">38</span>;<span class="number">2</span>;<span class="pagerange">136-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11288718" rel="external">11288718</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1734815" rel="external">1734815</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Subtelomeric sequence from the right arm of Schizosaccharomyces pombe chromosome I contains seven permease genes.</h4>
<p class="authors"><span class="author">Hunt C</span>, <span class="author">Moore K</span>, <span class="author">Xiang Z</span>, <span class="author">Hurst SM</span>, <span class="author">McDougall RC</span>, <span class="author">Rajandream MA</span>, <span class="author">Barrell BG</span>, <span class="author">Gwilliam R</span>, <span class="author">Wood V</span>, <span class="author">Lyne MH</span> and <span class="author">Aves SJ</span></p>
<p class="affiliation">School of Biological Sciences, University of Exeter, Washington Singer Laboratories, Perry Road, Exeter EX4 4QG, UK.</p><p class="abstract">The sequence has been determined of 80 888 bp of contiguous subtelomeric DNA, including the isp5 gene, from the right arm of chromosome I of Schizosaccharomyces pombe; 27 open reading frames (ORFs) longer than 100 codons are present, giving a density of one gene per 3.0 kb. Seven of the predicted proteins are members of the major facilitator superfamily (MFS) of transport proteins, including four amino acid permease homologues, bringing this family of amino acid permease sequences to 17 in Sz. pombe, and a phylogenetic analysis is presented. Also encoded is an allantoate permease homologue, a sulphate permease homologue and a probable urea active transporter. Predicted non-membrane proteins include a 1-aminocyclopropane-1-carboxylate deaminase (ACC deaminase), a class III aminotransferase, serine acetyltransferase, protein-L-isoaspartate O-methyltransferase, alpha-glucosidase, alpha-galactosidase, esterase/lipase, oxidoreductase of the short-chain dehydrogenase/reductase (SDR) family, aldehyde dehydrogenase, formamidase, amidase, flavohaemoprotein, a putative translation initiation inhibitor and a protein with similarity to a filamentous fungal conidiation-specific protein. The remaining six ORFs are likely to encode proteins, either because they have sequence similarity with hypothetical proteins or because they are known to be transcribed. Introns are scarce in the sequenced region: only three ORFs contain introns, with only one having multiple introns. The sequenced region also contains a single Tf1 transposon long terminal repeat (LTR). The sequence is derived from cosmid clones c869, c922 and c1039 and has been submitted to the EMBL database under entries SPAC869 (Accession No. AL132779), SPAC922 (AL133522) and SPAC1039 (AL133521).</p>
<p class="publication"><span class="title">Yeast (Chichester, England)</span>&nbsp;<span class="year">2001</span>;<span class="volume">18</span>;<span class="number">4</span>;<span class="pagerange">355-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11223945" rel="external">11223945</a>; DOI: <a href="http://dx.doi.org/10.1002%2F1097-0061(20010315)18%3A4%3C355%3A%3AAID-YEA676%3E3.0.CO%3B2-P" rel="external">10.1002/1097-0061(20010315)18:4&lt;355::AID-YEA676&gt;3.0.CO;2-P</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene discovery in Plasmodium chabaudi by genome survey sequencing.</h4>
<p class="authors"><span class="author">Janssen CS</span>, <span class="author">Barrett MP</span>, <span class="author">Lawson D</span>, <span class="author">Quail MA</span>, <span class="author">Harris D</span>, <span class="author">Bowman S</span>, <span class="author">Phillips RS</span> and <span class="author">Turner CM</span></p>
<p class="affiliation">Division of Infection &amp; Immunity, IBLS, University of Glasgow, G12 8QQ, Glasgow, UK. <a href="mailto:%63.%6A%61%6E%73%73%65%6E%40%62%69%6F.%67%6C%61.%61%63.%75%6B">&#99;&#46;&#106;&#97;&#110;&#115;&#115;&#101;&#110;&#64;&#98;&#105;&#111;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The first genome survey sequencing of the rodent malaria parasite Plasmodium chabaudi is presented here. In 766 sequences, 131 putative gene sequences have been identified by sequence similarity database searches. Further, 7 potential gene families, four of which have not previously been described, were discovered. These genes may be important in understanding the biology of malaria, as well as offering potential new drug targets. We have also identified a number of candidate minisatellite sequences that could be helpful in genetic studies. Genome survey sequencing in P. chabaudi is a productive strategy in further developing this in vivo model of malaria, in the context of the malaria genome projects.</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2001</span>;<span class="volume">113</span>;<span class="number">2</span>;<span class="pagerange">251-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11295179" rel="external">11295179</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Karyotyping mouse chromosomes by multiplex-FISH (M-FISH).</h4>
<p class="authors"><span class="author">Jentsch I</span>, <span class="author">Adler ID</span>, <span class="author">Carter NP</span> and <span class="author">Speicher MR</span></p>
<p class="affiliation">Institut f&uuml;r Anthropologie und Humangenetik, Ludwig-Maximilians-Universit&auml;t M&uuml;nchen, Germany.</p><p class="abstract">Karyotyping of mouse chromosomes is a skillful art, which is laborious work even for experienced cytogeneticists. With the growing number of mouse models for human diseases, there is an increasing demand for automated mouse karyotyping systems. Here, such a karyotyping system for mouse chromosomes based on the multiplex-fluorescence in-situ hybridization (M-FISH) technology is shown. The system was tested on a number of individual mice with numerical and structural aberrations and its reproducibility and robustness verified. Mouse M-FISH should be a valuable tool for the analysis of chromosomal rearrangements in mice.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2001</span>;<span class="volume">9</span>;<span class="number">3</span>;<span class="pagerange">211-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11330395" rel="external">11330395</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Functional annotation of a full-length mouse cDNA collection.</h4>
<p class="authors"><span class="author">Kawai J</span>, <span class="author">Shinagawa A</span>, <span class="author">Shibata K</span>, <span class="author">Yoshino M</span>, <span class="author">Itoh M</span>, <span class="author">Ishii Y</span>, <span class="author">Arakawa T</span>, <span class="author">Hara A</span>, <span class="author">Fukunishi Y</span>, <span class="author">Konno H</span>, <span class="author">Adachi J</span>, <span class="author">Fukuda S</span>, <span class="author">Aizawa K</span>, <span class="author">Izawa M</span>, <span class="author">Nishi K</span>, <span class="author">Kiyosawa H</span>, <span class="author">Kondo S</span>, <span class="author">Yamanaka I</span>, <span class="author">Saito T</span>, <span class="author">Okazaki Y</span>, <span class="author">Gojobori T</span>, <span class="author">Bono H</span>, <span class="author">Kasukawa T</span>, <span class="author">Saito R</span>, <span class="author">Kadota K</span>, <span class="author">Matsuda H</span>, <span class="author">Ashburner M</span>, <span class="author">Batalov S</span>, <span class="author">Casavant T</span>, <span class="author">Fleischmann W</span>, <span class="author">Gaasterland T</span>, <span class="author">Gissi C</span>, <span class="author">King B</span>, <span class="author">Kochiwa H</span>, <span class="author">Kuehl P</span>, <span class="author">Lewis S</span>, <span class="author">Matsuo Y</span>, <span class="author">Nikaido I</span>, <span class="author">Pesole G</span>, <span class="author">Quackenbush J</span>, <span class="author">Schriml LM</span>, <span class="author">Staubli F</span>, <span class="author">Suzuki R</span>, <span class="author">Tomita M</span>, <span class="author">Wagner L</span>, <span class="author">Washio T</span>, <span class="author">Sakai K</span>, <span class="author">Okido T</span>, <span class="author">Furuno M</span>, <span class="author">Aono H</span>, <span class="author">Baldarelli R</span>, <span class="author">Barsh G</span>, <span class="author">Blake J</span>, <span class="author">Boffelli D</span>, <span class="author">Bojunga N</span>, <span class="author">Carninci P</span>, <span class="author">de Bonaldo MF</span>, <span class="author">Brownstein MJ</span>, <span class="author">Bult C</span>, <span class="author">Fletcher C</span>, <span class="author">Fujita M</span>, <span class="author">Gariboldi M</span>, <span class="author">Gustincich S</span>, <span class="author">Hill D</span>, <span class="author">Hofmann M</span>, <span class="author">Hume DA</span>, <span class="author">Kamiya M</span>, <span class="author">Lee NH</span>, <span class="author">Lyons P</span>, <span class="author">Marchionni L</span>, <span class="author">Mashima J</span>, <span class="author">Mazzarelli J</span>, <span class="author">Mombaerts P</span>, <span class="author">Nordone P</span>, <span class="author">Ring B</span>, <span class="author">Ringwald M</span>, <span class="author">Rodriguez I</span>, <span class="author">Sakamoto N</span>, <span class="author">Sasaki H</span>, <span class="author">Sato K</span>, <span class="author">Sch&ouml;nbach C</span>, <span class="author">Seya T</span>, <span class="author">Shibata Y</span>, <span class="author">Storch KF</span>, <span class="author">Suzuki H</span>, <span class="author">Toyo-oka K</span>, <span class="author">Wang KH</span>, <span class="author">Weitz C</span>, <span class="author">Whittaker C</span>, <span class="author">Wilming L</span>, <span class="author">Wynshaw-Boris A</span>, <span class="author">Yoshida K</span>, <span class="author">Hasegawa Y</span>, <span class="author">Kawaji H</span>, <span class="author">Kohtsuki S</span>, <span class="author">Hayashizaki Y</span> and <span class="author">RIKEN Genome Exploration Research Group Phase II Team and the FANTOM Consortium</span></p>
<p class="affiliation">Laboratory for Genome Exploration Research Group, RIKEN Genomic Sciences Center, Yokohama Institute, Kanagawa, Japan.</p><p class="abstract">The RIKEN Mouse Gene Encyclopaedia Project, a systematic approach to determining the full coding potential of the mouse genome, involves collection and sequencing of full-length complementary DNAs and physical mapping of the corresponding genes to the mouse genome. We organized an international functional annotation meeting (FANTOM) to annotate the first 21,076 cDNAs to be analysed in this project. Here we describe the first RIKEN clone collection, which is one of the largest described for any organism. Analysis of these cDNAs extends known gene families and identifies new ones.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6821</span>;<span class="pagerange">685-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11217851">11217851</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35055500">10.1038/35055500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed structure of Pneumocystis carinii chromosome ends.</h4>
<p class="authors"><span class="author">Keely SP</span>, <span class="author">Wakefield AE</span>, <span class="author">Cushion MT</span>, <span class="author">Smulian AG</span>, <span class="author">Hall N</span>, <span class="author">Barrell BG</span> and <span class="author">Stringer JR</span></p>
<p class="affiliation">Department of Molecular Genetics, Biochemistry &amp; Microbiology, University of Cincinnati, OH, USA.</p>
<p class="grants">Funded by: FIC NIH HHS: <span class="grant">TW01200-02</span>; NIAID NIH HHS: <span class="grant">R01AI36701</span>, <span class="grant">R01AI44651</span></p>
<p class="publication"><span class="title">The Journal of eukaryotic microbiology</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl</span>;<span class="pagerange">118S-120S</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11906018" rel="external">11906018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of functional genomics in C. elegans for studying human development and disease.</h4>
<p class="authors"><span class="author">Kuwabara PE</span> and <span class="author">O&#39;Neil N</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, UK. <a href="mailto:%70%65%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#101;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The 100 Mb Caenorhabditis elegans genome sequence is the first animal genome to be sequenced in its entirety. Many reverse-genetics tools have been developed to mine the genome sequence and to facilitate the jump between the identification of a gene sequence and the understanding of its function. Here we discuss how C. elegans can contribute to understanding of the function of genes involved in human development and disease.</p>
<p class="publication"><span class="title">Journal of inherited metabolic disease</span>&nbsp;<span class="year">2001</span>;<span class="volume">24</span>;<span class="number">2</span>;<span class="pagerange">127-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11405335" rel="external">11405335</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">It ain&#39;t over till it&#39;s ova: germline sex determination in C. elegans.</h4>
<p class="authors"><span class="author">Kuwabara PE</span> and <span class="author">Perry MD</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%70%65%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#101;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Sex determination in most organisms involves a simple binary fate choice between male or female development; the outcome of this decision has profound effects on organismal biology, biochemistry and behaviour. In the nematode C. elegans, there is also a binary choice, either male or hermaphrodite. In C. elegans, distinct genetic pathways control somatic and germline sexual cell fate. Both pathways share a common set of globally acting regulatory genes; however, germline-specific regulatory genes also participate in the decision to make male or female gametes. The determination of sexual fate in the germline of the facultative hermaphrodite poses a special problem, because first sperm then oocytes are produced. It has emerged that additional layers of post-transcriptional regulation have been imposed to modulate the activities of the global sex-determining genes, tra-2 and fem-3; the balance between these activities is crucial in controlling sexual cell fate in the hermaphrodite germline.</p>
<p class="publication"><span class="title">BioEssays : news and reviews in molecular, cellular and developmental biology</span>&nbsp;<span class="year">2001</span>;<span class="volume">23</span>;<span class="number">7</span>;<span class="pagerange">596-604</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11462213" rel="external">11462213</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fbies.1085" rel="external">10.1002/bies.1085</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Initial sequencing and analysis of the human genome.</h4>
<p class="authors"><span class="author">Lander ES</span>, <span class="author">Linton LM</span>, <span class="author">Birren B</span>, <span class="author">Nusbaum C</span>, <span class="author">Zody MC</span>, <span class="author">Baldwin J</span>, <span class="author">Devon K</span>, <span class="author">Dewar K</span>, <span class="author">Doyle M</span>, <span class="author">FitzHugh W</span>, <span class="author">Funke R</span>, <span class="author">Gage D</span>, <span class="author">Harris K</span>, <span class="author">Heaford A</span>, <span class="author">Howland J</span>, <span class="author">Kann L</span>, <span class="author">Lehoczky J</span>, <span class="author">LeVine R</span>, <span class="author">McEwan P</span>, <span class="author">McKernan K</span>, <span class="author">Meldrim J</span>, <span class="author">Mesirov JP</span>, <span class="author">Miranda C</span>, <span class="author">Morris W</span>, <span class="author">Naylor J</span>, <span class="author">Raymond C</span>, <span class="author">Rosetti M</span>, <span class="author">Santos R</span>, <span class="author">Sheridan A</span>, <span class="author">Sougnez C</span>, <span class="author">Stange-Thomann N</span>, <span class="author">Stojanovic N</span>, <span class="author">Subramanian A</span>, <span class="author">Wyman D</span>, <span class="author">Rogers J</span>, <span class="author">Sulston J</span>, <span class="author">Ainscough R</span>, <span class="author">Beck S</span>, <span class="author">Bentley D</span>, <span class="author">Burton J</span>, <span class="author">Clee C</span>, <span class="author">Carter N</span>, <span class="author">Coulson A</span>, <span class="author">Deadman R</span>, <span class="author">Deloukas P</span>, <span class="author">Dunham A</span>, <span class="author">Dunham I</span>, <span class="author">Durbin R</span>, <span class="author">French L</span>, <span class="author">Grafham D</span>, <span class="author">Gregory S</span>, <span class="author">Hubbard T</span>, <span class="author">Humphray S</span>, <span class="author">Hunt A</span>, <span class="author">Jones M</span>, <span class="author">Lloyd C</span>, <span class="author">McMurray A</span>, <span class="author">Matthews L</span>, <span class="author">Mercer S</span>, <span class="author">Milne S</span>, <span class="author">Mullikin JC</span>, <span class="author">Mungall A</span>, <span class="author">Plumb R</span>, <span class="author">Ross M</span>, <span class="author">Shownkeen R</span>, <span class="author">Sims S</span>, <span class="author">Waterston RH</span>, <span class="author">Wilson RK</span>, <span class="author">Hillier LW</span>, <span class="author">McPherson JD</span>, <span class="author">Marra MA</span>, <span class="author">Mardis ER</span>, <span class="author">Fulton LA</span>, <span class="author">Chinwalla AT</span>, <span class="author">Pepin KH</span>, <span class="author">Gish WR</span>, <span class="author">Chissoe SL</span>, <span class="author">Wendl MC</span>, <span class="author">Delehaunty KD</span>, <span class="author">Miner TL</span>, <span class="author">Delehaunty A</span>, <span class="author">Kramer JB</span>, <span class="author">Cook LL</span>, <span class="author">Fulton RS</span>, <span class="author">Johnson DL</span>, <span class="author">Minx PJ</span>, <span class="author">Clifton SW</span>, <span class="author">Hawkins T</span>, <span class="author">Branscomb E</span>, <span class="author">Predki P</span>, <span class="author">Richardson P</span>, <span class="author">Wenning S</span>, <span class="author">Slezak T</span>, <span class="author">Doggett N</span>, <span class="author">Cheng JF</span>, <span class="author">Olsen A</span>, <span class="author">Lucas S</span>, <span class="author">Elkin C</span>, <span class="author">Uberbacher E</span>, <span class="author">Frazier M</span>, <span class="author">Gibbs RA</span>, <span class="author">Muzny DM</span>, <span class="author">Scherer SE</span>, <span class="author">Bouck JB</span>, <span class="author">Sodergren EJ</span>, <span class="author">Worley KC</span>, <span class="author">Rives CM</span>, <span class="author">Gorrell JH</span>, <span class="author">Metzker ML</span>, <span class="author">Naylor SL</span>, <span class="author">Kucherlapati RS</span>, <span class="author">Nelson DL</span>, <span class="author">Weinstock GM</span>, <span class="author">Sakaki Y</span>, <span class="author">Fujiyama A</span>, <span class="author">Hattori M</span>, <span class="author">Yada T</span>, <span class="author">Toyoda A</span>, <span class="author">Itoh T</span>, <span class="author">Kawagoe C</span>, <span class="author">Watanabe H</span>, <span class="author">Totoki Y</span>, <span class="author">Taylor T</span>, <span class="author">Weissenbach J</span>, <span class="author">Heilig R</span>, <span class="author">Saurin W</span>, <span class="author">Artiguenave F</span>, <span class="author">Brottier P</span>, <span class="author">Bruls T</span>, <span class="author">Pelletier E</span>, <span class="author">Robert C</span>, <span class="author">Wincker P</span>, <span class="author">Smith DR</span>, <span class="author">Doucette-Stamm L</span>, <span class="author">Rubenfield M</span>, <span class="author">Weinstock K</span>, <span class="author">Lee HM</span>, <span class="author">Dubois J</span>, <span class="author">Rosenthal A</span>, <span class="author">Platzer M</span>, <span class="author">Nyakatura G</span>, <span class="author">Taudien S</span>, <span class="author">Rump A</span>, <span class="author">Yang H</span>, <span class="author">Yu J</span>, <span class="author">Wang J</span>, <span class="author">Huang G</span>, <span class="author">Gu J</span>, <span class="author">Hood L</span>, <span class="author">Rowen L</span>, <span class="author">Madan A</span>, <span class="author">Qin S</span>, <span class="author">Davis RW</span>, <span class="author">Federspiel NA</span>, <span class="author">Abola AP</span>, <span class="author">Proctor MJ</span>, <span class="author">Myers RM</span>, <span class="author">Schmutz J</span>, <span class="author">Dickson M</span>, <span class="author">Grimwood J</span>, <span class="author">Cox DR</span>, <span class="author">Olson MV</span>, <span class="author">Kaul R</span>, <span class="author">Raymond C</span>, <span class="author">Shimizu N</span>, <span class="author">Kawasaki K</span>, <span class="author">Minoshima S</span>, <span class="author">Evans GA</span>, <span class="author">Athanasiou M</span>, <span class="author">Schultz R</span>, <span class="author">Roe BA</span>, <span class="author">Chen F</span>, <span class="author">Pan H</span>, <span class="author">Ramser J</span>, <span class="author">Lehrach H</span>, <span class="author">Reinhardt R</span>, <span class="author">McCombie WR</span>, <span class="author">de la Bastide M</span>, <span class="author">Dedhia N</span>, <span class="author">Bl&ouml;cker H</span>, <span class="author">Hornischer K</span>, <span class="author">Nordsiek G</span>, <span class="author">Agarwala R</span>, <span class="author">Aravind L</span>, <span class="author">Bailey JA</span>, <span class="author">Bateman A</span>, <span class="author">Batzoglou S</span>, <span class="author">Birney E</span>, <span class="author">Bork P</span>, <span class="author">Brown DG</span>, <span class="author">Burge CB</span>, <span class="author">Cerutti L</span>, <span class="author">Chen HC</span>, <span class="author">Church D</span>, <span class="author">Clamp M</span>, <span class="author">Copley RR</span>, <span class="author">Doerks T</span>, <span class="author">Eddy SR</span>, <span class="author">Eichler EE</span>, <span class="author">Furey TS</span>, <span class="author">Galagan J</span>, <span class="author">Gilbert JG</span>, <span class="author">Harmon C</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Haussler D</span>, <span class="author">Hermjakob H</span>, <span class="author">Hokamp K</span>, <span class="author">Jang W</span>, <span class="author">Johnson LS</span>, <span class="author">Jones TA</span>, <span class="author">Kasif S</span>, <span class="author">Kaspryzk A</span>, <span class="author">Kennedy S</span>, <span class="author">Kent WJ</span>, <span class="author">Kitts P</span>, <span class="author">Koonin EV</span>, <span class="author">Korf I</span>, <span class="author">Kulp D</span>, <span class="author">Lancet D</span>, <span class="author">Lowe TM</span>, <span class="author">McLysaght A</span>, <span class="author">Mikkelsen T</span>, <span class="author">Moran JV</span>, <span class="author">Mulder N</span>, <span class="author">Pollara VJ</span>, <span class="author">Ponting CP</span>, <span class="author">Schuler G</span>, <span class="author">Schultz J</span>, <span class="author">Slater G</span>, <span class="author">Smit AF</span>, <span class="author">Stupka E</span>, <span class="author">Szustakowski J</span>, <span class="author">Thierry-Mieg D</span>, <span class="author">Thierry-Mieg J</span>, <span class="author">Wagner L</span>, <span class="author">Wallis J</span>, <span class="author">Wheeler R</span>, <span class="author">Williams A</span>, <span class="author">Wolf YI</span>, <span class="author">Wolfe KH</span>, <span class="author">Yang SP</span>, <span class="author">Yeh RF</span>, <span class="author">Collins F</span>, <span class="author">Guyer MS</span>, <span class="author">Peterson J</span>, <span class="author">Felsenfeld A</span>, <span class="author">Wetterstrand KA</span>, <span class="author">Patrinos A</span>, <span class="author">Morgan MJ</span>, <span class="author">de Jong P</span>, <span class="author">Catanese JJ</span>, <span class="author">Osoegawa K</span>, <span class="author">Shizuya H</span>, <span class="author">Choi S</span>, <span class="author">Chen YJ</span>, <span class="author">Szustakowki J</span> and <span class="author">International Human Genome Sequencing Consortium</span></p>
<p class="affiliation">Whitehead Institute for Biomedical Research, Center for Genome Research, Cambridge, Massachusetts 02142, USA. <a href="mailto:%6C%61%6E%64%65%72%40%67%65%6E%6F%6D%65.%77%69.%6D%69%74.%65%64%75">&#108;&#97;&#110;&#100;&#101;&#114;&#64;&#103;&#101;&#110;&#111;&#109;&#101;&#46;&#119;&#105;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The human genome holds an extraordinary trove of information about human development, physiology, medicine and evolution. Here we report the results of an international collaboration to produce and make freely available a draft sequence of the human genome. We also present an initial analysis of the data, describing some of the insights that can be gleaned from the sequence.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">860-921</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237011">11237011</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057062">10.1038/35057062</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">David Bentley discusses life after the Human Genome Project. Interviewed by Rebecca Lawrence.</h4>
<p class="authors"><span class="author">Lawrence R</span></p>
<p class="affiliation">David Bentley, The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, CB10 1SA tel.: +44-1223-834244; fax: +44-1223-494919; <a href="mailto:%64%72%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#114;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a>, Cambridge, UK</p>
<p class="publication"><span class="title">Drug discovery today</span>&nbsp;<span class="year">2001</span>;<span class="volume">6</span>;<span class="number">1</span>;<span class="pagerange">13-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11165165" rel="external">11165165</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessment of novel fold targets in CASP4: predictions of three-dimensional structures, secondary structures, and interresidue contacts.</h4>
<p class="authors"><span class="author">Lesk AM</span>, <span class="author">Lo Conte L</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge Clinical School, Cambridge Institute for Medical Research, Cambridge, United Kingdom. <a href="mailto:%61%6D%6C%32%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#97;&#109;&#108;&#50;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In the Novel Fold category, three types of predictions were assessed: three-dimensional structures, secondary structures, and residue-residue contacts. For predictions of three-dimensional models, CASP4 targets included 5 domains or structures with novel folds, and 13 on the borderline between Novel Fold and Fold Recognition categories. These elicited 1863 predictions of these and other targets by methods more general than comparative modeling or fold recognition techniques. The group of Bonneau, Tsai, Ruczinski, and Baker stood out as performing well with the greatest consistency. In many cases, several groups were able to predict fragments of the target correctly-often at a level somewhat larger than standard supersecondary structures-but were not able to assemble fragments into a correct global topology. The methods of Bonneau, Tsai, Ruczinski, and Baker have been successful in addressing the fragment assembly problem for many but not all the target structures.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl 5</span>;<span class="pagerange">98-118</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11835487">11835487</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A computational scan for U12-dependent introns in the human genome sequence.</h4>
<p class="authors"><span class="author">Levine A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">U12-dependent introns are found in small numbers in most eukaryotic genomes, but their scarcity makes accurate characterisation of their properties challenging. A computational search for U12-dependent introns was performed using the draft version of the human genome sequence. Human expressed sequences confirmed 404 U12-dependent introns within the human genome, a 6-fold increase over the total number of non-redundant U12-dependent introns previously identified in all genomes. Although most of these introns had AT-AC or GT-AG terminal dinucleotides, small numbers of introns with a surprising diversity of termini were found, suggesting that many of the non-canonical introns found in the human genome may be variants of U12-dependent introns and, thus, spliced by the minor spliceosome. Comparisons with U2-dependent introns revealed that the U12-dependent intron set lacks the &#39;short intron&#39; peak characteristic of U2-dependent introns. Analysis of this U12-dependent intron set confirmed reports of a biased distribution of U12-dependent introns in the genome and allowed the identification of several alternative splicing events as well as a surprising number of apparent splicing errors. This new larger reference set of U12-dependent introns will serve as a resource for future studies of both the properties and evolution of the U12 spliceosome.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">19</span>;<span class="pagerange">4006-13</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11574683" rel="external">11574683</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC60238" rel="external">60238</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Progress in sequencing the mouse genome.</h4>
<p class="authors"><span class="author">Lindblad-Toh K</span>, <span class="author">Lander ES</span>, <span class="author">McPherson JD</span>, <span class="author">Waterston RH</span>, <span class="author">Rodgers J</span> and <span class="author">Birney E</span></p>
<p class="publication"><span class="title">Genesis (New York, N.Y. : 2000)</span>&nbsp;<span class="year">2001</span>;<span class="volume">31</span>;<span class="number">4</span>;<span class="pagerange">137-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11783003" rel="external">11783003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tbx1 haploinsufficieny in the DiGeorge syndrome region causes aortic arch defects in mice.</h4>
<p class="authors"><span class="author">Lindsay EA</span>, <span class="author">Vitelli F</span>, <span class="author">Su H</span>, <span class="author">Morishima M</span>, <span class="author">Huynh T</span>, <span class="author">Pramparo T</span>, <span class="author">Jurecic V</span>, <span class="author">Ogunrinu G</span>, <span class="author">Sutherland HF</span>, <span class="author">Scambler PJ</span>, <span class="author">Bradley A</span> and <span class="author">Baldini A</span></p>
<p class="affiliation">Department of Pediatrics, Baylor College of Medicine, Houston, Texas 77030, USA.</p><p class="abstract">DiGeorge syndrome is characterized by cardiovascular, thymus and parathyroid defects and craniofacial anomalies, and is usually caused by a heterozygous deletion of chromosomal region 22q11.2 (del22q11) (ref. 1). A targeted, heterozygous deletion, named Df(16)1, encompassing around 1 megabase of the homologous region in mouse causes cardiovascular abnormalities characteristic of the human disease. Here we have used a combination of chromosome engineering and P1 artificial chromosome transgenesis to localize the haploinsufficient gene in the region, Tbx1. We show that Tbx1, a member of the T-box transcription factor family, is required for normal development of the pharyngeal arch arteries in a gene dosage-dependent manner. Deletion of one copy of Tbx1 affects the development of the fourth pharyngeal arch arteries, whereas homozygous mutation severely disrupts the pharyngeal arch artery system. Our data show that haploinsufficiency of Tbx1 is sufficient to generate at least one important component of the DiGeorge syndrome phenotype in mice, and demonstrate the suitability of the mouse for the genetic dissection of microdeletion syndromes.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">410</span>;<span class="number">6824</span>;<span class="pagerange">97-101</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11242049">11242049</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35065105">10.1038/35065105</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Independent regulation of initiation and maintenance phases of Hoxa3 expression in the vertebrate hindbrain involve auto- and cross-regulatory mechanisms.</h4>
<p class="authors"><span class="author">Manzanares M</span>, <span class="author">Bel-Vialar S</span>, <span class="author">Ariza-McNaughton L</span>, <span class="author">Ferretti E</span>, <span class="author">Marshall H</span>, <span class="author">Maconochie MM</span>, <span class="author">Blasi F</span> and <span class="author">Krumlauf R</span></p>
<p class="affiliation">Division of Developmental Neurobiology, MRC National Institute for Medical Research, The Ridgeway, Mill Hill, London NW7 1AA, UK.</p><p class="abstract">During development of the vertebrate hindbrain, Hox genes play multiple roles in the segmental processes that regulate anteroposterior (AP) patterning. Paralogous Hox genes, such as Hoxa3, Hoxb3 and Hoxd3, generally have very similar patterns of expression, and gene targeting experiments have shown that members of paralogy group 3 can functionally compensate for each other. Hence, distinct functions for individual members of this family may primarily depend upon differences in their expression domains. The earliest domains of expression of the Hoxa3 and Hoxb3 genes in hindbrain rhombomeric (r) segments are transiently regulated by kreisler, a conserved Maf b-Zip protein, but the mechanisms that maintain expression in later stages are unknown. In this study, we have compared the segmental expression and regulation of Hoxa3 and Hoxb3 in mouse and chick embryos to investigate how they are controlled after initial activation. We found that the patterns of Hoxa3 and Hoxb3 expression in r5 and r6 in later stages during mouse and chick hindbrain development were differentially regulated. Hoxa3 expression was maintained in r5 and r6, while Hoxb3 was downregulated. Regulatory comparisons of cis-elements from the chick and mouse Hoxa3 locus in both transgenic mouse and chick embryos have identified a conserved enhancer that mediates the late phase of Hoxa3 expression through a conserved auto/cross-regulatory loop. This block of similarity is also present in the human and horn shark loci, and contains two bipartite Hox/Pbx-binding sites that are necessary for its in vivo activity in the hindbrain. These HOX/PBC sites are positioned near a conserved kreisler-binding site (KrA) that is involved in activating early expression in r5 and r6, but their activity is independent of kreisler. This work demonstrates that separate elements are involved in initiating and maintaining Hoxa3 expression during hindbrain segmentation, and that it is regulated in a manner different from Hoxb3 in later stages. Together, these findings add further strength to the emerging importance of positive auto- and cross-regulatory interactions between Hox genes as a general mechanism for maintaining their correct spatial patterns in the vertebrate nervous system.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2001</span>;<span class="volume">128</span>;<span class="number">18</span>;<span class="pagerange">3595-607</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11566863" rel="external">11566863</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Vanin genes are clustered (human 6q22-24 and mouse 10A2B1) and encode isoforms of pantetheinase ectoenzymes.</h4>
<p class="authors"><span class="author">Martin F</span>, <span class="author">Malergue F</span>, <span class="author">Pitari G</span>, <span class="author">Philippe JM</span>, <span class="author">Philips S</span>, <span class="author">Chabret C</span>, <span class="author">Granjeaud S</span>, <span class="author">Mattei MG</span>, <span class="author">Mungall AJ</span>, <span class="author">Naquet P</span> and <span class="author">Galland F</span></p>
<p class="affiliation">Centre d&#39;Immunologie de Marseille-Luminy, INSERM-CNRS-Universit&eacute; de la M&eacute;diterran&eacute;e, France.</p><p class="abstract">The mouse Vanin-1 molecule plays a role in thymic reconstitution following damage by irradiation. We recently demonstrated that it is a membrane pantetheinase (EC 3.56.1.-). This molecule is the prototypic member of a larger Vanin family encoded by at least two mouse (Vanin-1 and Vanin-3) and three human (VNN1, VNN2, VNN3) orthologous genes. We now report (1) the structural characterization of the human and mouse Vanin genes and their organization in clusters on the 6q22-24 and 10A2B1 chromosomes, respectively; (2) identification of the human VNN3 gene and the demonstration that the mouse Vanin-3 molecule is secreted by cells, and (3) that the Vanin genes encode different isoforms of the mammalian pantetheinase activity. Thus, the Vanin family represents a novel class of secreted or membrane-associated ectoenzymes. We discuss here their possible role in processes pertaining to tissue repair in the context of oxidative stress.</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">53</span>;<span class="number">4</span>;<span class="pagerange">296-306</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11491533" rel="external">11491533</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localization of the gene for distal hereditary motor neuronopathy VII (dHMN-VII) to chromosome 2q14.</h4>
<p class="authors"><span class="author">McEntagart M</span>, <span class="author">Norton N</span>, <span class="author">Williams H</span>, <span class="author">Teare MD</span>, <span class="author">Dunstan M</span>, <span class="author">Baker P</span>, <span class="author">Houlden H</span>, <span class="author">Reilly M</span>, <span class="author">Wood N</span>, <span class="author">Harper PS</span>, <span class="author">Futreal PA</span>, <span class="author">Williams N</span> and <span class="author">Rahman N</span></p>
<p class="affiliation">Institute of Medical Genetics, University Hospital of Wales, Cardiff CF4 4XW, United Kingdom.</p><p class="abstract">Distal hereditary motor neuronopathy type VII (dHMN-VII) is an autosomal dominant disorder characterized by distal muscular atrophy and vocal cord paralysis. We performed a genomewide linkage search in a large Welsh pedigree with dHMN-VII and established linkage to chromosome 2q14. Analyses of a second family with dHMN-VII confirmed the location of the gene and provided evidence for a founder mutation segregating in both pedigrees. The maximum three-point LOD score in the combined pedigree was 7.49 at D2S274. Expansion of a polyalanine tract in Engrailed-1, a transcription factor strongly expressed in the spinal cord, was excluded as the cause of dHMN-VII.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">68</span>;<span class="number">5</span>;<span class="pagerange">1270-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11294660" rel="external">11294660</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1226107" rel="external">1226107</a>; DOI: <a href="http://dx.doi.org/10.1086%2F320122" rel="external">10.1086/320122</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">From mouse to man: generating megabase chromosome rearrangements.</h4>
<p class="authors"><span class="author">Mills AA</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, 1 BungtownRoad, Cold Spring Harbor, NY 11724, USA. <a href="mailto:%6D%69%6C%6C%73%40%63%70%68%6C.%6F%72%67">&#109;&#105;&#108;&#108;&#115;&#64;&#99;&#112;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">Experimental approaches for deciphering the function of human genes rely heavily on our ability to generate mutations in model organisms such as the mouse. However, because recessive mutations are masked by the wild-type allele in the diploid context, conventional mutagenesis and screening is often laborious and costly. Chromosome engineering combines the power of gene targeting in embryonic stem (ES) cells with Cre--loxP technology to create mice that are functionally haploid in discrete portions of the genome. Chromosome deletions, duplications and inversions can be tagged with visible markers, facilitating strain maintenance. These approaches allow for more refined mutagenesis screens that will greatly accelerate functional mouse genomics and generate mammalian models for developmental processes and cancer.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2001</span>;<span class="volume">17</span>;<span class="number">6</span>;<span class="pagerange">331-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11377795" rel="external">11377795</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Live observation of fission yeast meiosis in recombination-deficient mutants: a study on achiasmate chromosome segregation.</h4>
<p class="authors"><span class="author">Molnar M</span>, <span class="author">B&auml;hler J</span>, <span class="author">Kohli J</span> and <span class="author">Hiraoka Y</span></p>
<p class="affiliation">Institute of Cell Biology, University of Bern, Switzerland.</p><p class="abstract">Regular segregation of homologous chromosomes during meiotic divisions is essential for the generation of viable progeny. In recombination-proficient organisms, chromosome disjunction at meiosis I generally occurs by chiasma formation between the homologs (chiasmate meiosis). We have studied meiotic stages in living rec8 and rec7 mutant cells of fission yeast, with special attention to prophase and the first meiotic division. Both rec8 and rec7 are early recombination mutants, and in rec7 mutants, chromosome segregation at meiosis I occurs without any recombination (achiasmate meiosis). Both mutants showed distinct irregularities in nuclear prophase movements. Additionally, rec7 showed an extended first division of variable length and with single chromosomes changing back and forth between the cell poles. Two other early recombination deficient mutants (rec14 and rec15) showed very similar phenotypes to rec7 during the first meiotic division, and the fidelity of achiasmate chromosome segregation slightly exceeded the expected random level. We discuss possible regulatory mechanisms of fission yeast to deal with achiasmate chromosome segregation.</p>
<p class="publication"><span class="title">Journal of cell science</span>&nbsp;<span class="year">2001</span>;<span class="volume">114</span>;<span class="number">Pt 15</span>;<span class="pagerange">2843-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11683417" rel="external">11683417</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Critical assessment of methods of protein structure prediction (CASP): round IV.</h4>
<p class="authors"><span class="author">Moult J</span>, <span class="author">Fidelis K</span>, <span class="author">Zemla A</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">Center for Advanced Research in Biotechnology, University of Maryland Biotechnology Institute, Rockville, Maryland 20850, USA. <a href="mailto:%6A%6D%6F%75%6C%74%40%74%75%6E%63.%6F%72%67">&#106;&#109;&#111;&#117;&#108;&#116;&#64;&#116;&#117;&#110;&#99;&#46;&#111;&#114;&#103;</a></p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM/DK61967</span>; NLM NIH HHS: <span class="grant">LM07085</span></p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl 5</span>;<span class="pagerange">2-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11835476">11835476</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-density transcript map of the human dominant optic atrophy OPA1 gene locus and re-evaluation of evidence for a founder haplotype.</h4>
<p class="authors"><span class="author">Murton NJ</span>, <span class="author">French L</span>, <span class="author">Toomes C</span>, <span class="author">Joseph SS</span>, <span class="author">Rehman I</span>, <span class="author">Hopkins BL</span>, <span class="author">Inglehearn CF</span> and <span class="author">Churchill AJ</span></p>
<p class="affiliation">Molecular Medicine Unit, St. James&#39;s University Hospital, Leeds, UK.</p><p class="abstract">Dominant optic atrophy (DOA, gene OPA1) is the commonest form of inherited optic atrophy. Linkage studies have shown that a locus for this disease lies in a 1.4-cM region at chromosome 3q28--&gt;q29 and have suggested a founder haplotype for as many as 95% of the linked families. To aid the identification of candidate genes for this disease, we have constructed a Bacterial Artificial Chromosome (BAC) contig covering approximately 3.3 Mb and encompassing the OPA1 critical region (flanking markers D3S3669 and D3S3562). This physical map corrects errors in the marker order reported in the literature, allowing the OPA1 critical region to be precisely defined. A reassessment of the founder effect in the light of the revised marker order suggests that it may not be as significant as had previously been suggested. A high-density transcript map was created by precisely mapping genes and expressed sequence tags (ESTs) from GeneMap&#39;99, that have been loosely assigned to the region by radiation hybrid mapping. One known gene (KIAA0567 protein) and 15 ESTs were found to lie within the minimal disease region. Analysis of the sequence data already available from within the OPA1 critical region allowed the identification and mapping of a further 31 ESTs. The work presented in this study provides the basis for the characterisation of candidate genes and the ultimate identification of the gene mutated in DOA.</p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">92</span>;<span class="number">1-2</span>;<span class="pagerange">97-102</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11306804" rel="external">11306804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Prediction targets of CASP4.</h4>
<p class="authors"><span class="author">Murzin A</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Centre for Protein Engineering, MRC Centre, Cambridge, UK.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl 5</span>;<span class="pagerange">8-12</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11835477">11835477</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">CASP2 knowledge-based approach to distant homology recognition and fold prediction in CASP4.</h4>
<p class="authors"><span class="author">Murzin AG</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Centre for Protein Engineering, MRC Centre, Cambridge, United Kingdom. <a href="mailto:%61%67%6D%40%6D%72%63-%6C%6D%62.%63%61%6D.%61%63.%75%6B">&#97;&#103;&#109;&#64;&#109;&#114;&#99;&#45;&#108;&#109;&#98;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In 1996, in CASP2, we presented a semimanual approach to the prediction of protein structure that was aimed at the recognition of probable distant homology, where it existed, between a given target protein and a protein of known structure (Murzin and Bateman, Proteins 1997; Suppl 1:105-112). Central to our method was the knowledge of all known structural and probable evolutionary relationships among proteins of known structure classified in the SCOP database (Murzin et al., J Mol Biol 1995;247:536-540). It was demonstrated that a knowledge-based approach could compete successfully with the best computational methods of the time in the correct recognition of the target protein fold. Four years later, in CASP4, we have applied essentially the same knowledge-based approach to distant homology recognition, concentrating our effort on the improvement of the completeness and alignment accuracy of our models. The manifold increase of available sequence and structure data was to our advantage, as well as was the experience and expertise obtained through the classification of these data. In particular, we were able to model most of our predictions from several distantly related structures rather than from a single parent structure, and we could use more superfamily characteristic features for the refinement of our alignments. Our predictions for each of the attempted distant homology recognition targets ranked among the few top predictions for each of these targets, with the predictions for the hypothetical protein HI0065 (T0104) and the C-terminal domain of the ABC transporter MalK (T0121C) being particularly successful. We also have attempted the prediction of protein folds of some of the targets tentatively assigned to new superfamilies. The average quality of our fold predictions was far less than the quality of our distant homology recognition models, but for the two targets, chorismate lyase (T0086) and Appr&gt;p cyclic phosphodiesterase (T0094), our predictions achieved the top ranking.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl 5</span>;<span class="pagerange">76-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11835484" rel="external">11835484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Breast cancer genetics: what we know and what we need.</h4>
<p class="authors"><span class="author">Nathanson KL</span>, <span class="author">Wooster R</span>, <span class="author">Weber BL</span> and <span class="author">Nathanson KN</span></p>
<p class="affiliation">Abramson Family Cancer Research Institute, University of Pennsylvania, Philadelphia, Pennsylvania, USA.</p><p class="abstract">Breast cancer results from genetic and environmental factors leading to the accumulation of mutations in essential genes. Genetic predisposition may have a strong, almost singular effect, as with BRCA1 and BRCA2, or may represent the cumulative effects of multiple low-penetrance susceptibility alleles. Here we review high- and low-penetrance breast-cancer-susceptibility alleles and discuss ongoing efforts to identify additional susceptibility genes. Ultimately these discoveries will lead to individualized breast cancer risk assessment and a reduction in breast cancer incidence.</p>
<p class="publication"><span class="title">Nature medicine</span>&nbsp;<span class="year">2001</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">552-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11329055">11329055</a>; DOI: <a href="http://dx.doi.org/10.1038%2F87876">10.1038/87876</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localisation of a novel region of recurrent amplification in follicular lymphoma to an approximately 6.8 Mb region of 13q32-33.</h4>
<p class="authors"><span class="author">Neat MJ</span>, <span class="author">Foot N</span>, <span class="author">Jenner M</span>, <span class="author">Goff L</span>, <span class="author">Ashcroft K</span>, <span class="author">Burford D</span>, <span class="author">Dunham A</span>, <span class="author">Norton A</span>, <span class="author">Lister TA</span> and <span class="author">Fitzgibbon J</span></p>
<p class="affiliation">ICRF Medical Oncology Unit, St. Bartholomew&#39;s Hospital, Charterhouse Square, London, UK. <a href="mailto:%6E%65%61%74%40%69%63%72%66.%69%63%6E%65%74.%75%6B">&#110;&#101;&#97;&#116;&#64;&#105;&#99;&#114;&#102;&#46;&#105;&#99;&#110;&#101;&#116;&#46;&#117;&#107;</a></p><p class="abstract">Follicular lymphoma (FL) is characterised by the presence of the t(14;18)(q32;q21) and represents approximately 25% of new cases of non-Hodgkin&#39;s lymphoma. While the t(14;18) is a well-documented rearrangement, the role of secondary cytogenetic abnormalities in the development and progression of these tumours remains unclear. Comparative genomic hybridisation was used to characterise changes in DNA copy number in tumour DNA from patients with this malignancy. The mean numbers of deletion and amplification events found in each of the 45 samples studied were 1.8 and 2.3, respectively. Regions of recurrent (&gt;10% tumour samples) gain involved chromosomes 2p13-16 (16%), 7 (20%), 12 (16%), 13q21-33 (18%), 18 (27%), and X (36%) and frequent losses localised to 6q (29%) and 17p (20%). Amplification of chromosome 13 represents a novel finding in FL. The minimal amplified region was refined to a 6.8-Mb interval of 13q32-33 between the BAC clones 88K16 and 44H20 by fluorescence in situ hybridisation studies using metaphase chromosomes derived from tumour material. There are a number of reports in the literature suggesting that amplification of chromosome 13 also occurs in other human cancers. The location of the putative oncogene on 13q described here in follicular and transformed lymphoma may also be important in the evolution of many other malignancies.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2001</span>;<span class="volume">32</span>;<span class="number">3</span>;<span class="pagerange">236-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11579463" rel="external">11579463</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SSAHA: a fast search method for large DNA databases.</h4>
<p class="authors"><span class="author">Ning Z</span>, <span class="author">Cox AJ</span> and <span class="author">Mullikin JC</span></p>
<p class="affiliation">Informatics Division, The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">We describe an algorithm, SSAHA (Sequence Search and Alignment by Hashing Algorithm), for performing fast searches on databases containing multiple gigabases of DNA. Sequences in the database are preprocessed by breaking them into consecutive k-tuples of k contiguous bases and then using a hash table to store the position of each occurrence of each k-tuple. Searching for a query sequence in the database is done by obtaining from the hash table the &quot;hits&quot; for each k-tuple in the query sequence and then performing a sort on the results. We discuss the effect of the tuple length k on the search speed, memory usage, and sensitivity of the algorithm and present the results of computational experiments which show that SSAHA can be three to four orders of magnitude faster than BLAST or FASTA, while requiring less memory than suffix tree methods. The SSAHA algorithm is used for high-throughput single nucleotide polymorphism (SNP) detection and very large scale sequence assembly. Also, it provides Web-based sequence search facilities for Ensembl projects.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">10</span>;<span class="pagerange">1725-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11591649">11591649</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311141">311141</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.194201">10.1101/gr.194201</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An annotator&#39;s view.</h4>
<p class="authors"><span class="author">Parkhill J</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Cambridge, CB10 1SA, UK. <a href="mailto:%70%61%72%6B%68%69%6C%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#114;&#107;&#104;&#105;&#108;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2001</span>;<span class="volume">147</span>;<span class="number">Pt 1</span>;<span class="pagerange">2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11160794" rel="external">11160794</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genome sequence of a multiple drug resistant Salmonella enterica serovar Typhi CT18.</h4>
<p class="authors"><span class="author">Parkhill J</span>, <span class="author">Dougan G</span>, <span class="author">James KD</span>, <span class="author">Thomson NR</span>, <span class="author">Pickard D</span>, <span class="author">Wain J</span>, <span class="author">Churcher C</span>, <span class="author">Mungall KL</span>, <span class="author">Bentley SD</span>, <span class="author">Holden MT</span>, <span class="author">Sebaihia M</span>, <span class="author">Baker S</span>, <span class="author">Basham D</span>, <span class="author">Brooks K</span>, <span class="author">Chillingworth T</span>, <span class="author">Connerton P</span>, <span class="author">Cronin A</span>, <span class="author">Davis P</span>, <span class="author">Davies RM</span>, <span class="author">Dowd L</span>, <span class="author">White N</span>, <span class="author">Farrar J</span>, <span class="author">Feltwell T</span>, <span class="author">Hamlin N</span>, <span class="author">Haque A</span>, <span class="author">Hien TT</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Krogh A</span>, <span class="author">Larsen TS</span>, <span class="author">Leather S</span>, <span class="author">Moule S</span>, <span class="author">O&#39;Gaora P</span>, <span class="author">Parry C</span>, <span class="author">Quail M</span>, <span class="author">Rutherford K</span>, <span class="author">Simmonds M</span>, <span class="author">Skelton J</span>, <span class="author">Stevens K</span>, <span class="author">Whitehead S</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%70%61%72%6B%68%69%6C%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#114;&#107;&#104;&#105;&#108;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Salmonella enterica serovar Typhi (S. typhi) is the aetiological agent of typhoid fever, a serious invasive bacterial disease of humans with an annual global burden of approximately 16 million cases, leading to 600,000 fatalities. Many S. enterica serovars actively invade the mucosal surface of the intestine but are normally contained in healthy individuals by the local immune defence mechanisms. However, S. typhi has evolved the ability to spread to the deeper tissues of humans, including liver, spleen and bone marrow. Here we have sequenced the 4,809,037-base pair (bp) genome of a S. typhi (CT18) that is resistant to multiple drugs, revealing the presence of hundreds of insertions and deletions compared with the Escherichia coli genome, ranging in size from single genes to large islands. Notably, the genome sequence identifies over two hundred pseudogenes, several corresponding to genes that are known to contribute to virulence in Salmonella typhimurium. This genetic degradation may contribute to the human-restricted host range for S. typhi. CT18 harbours a 218,150-bp multiple-drug-resistance incH1 plasmid (pHCM1), and a 106,516-bp cryptic plasmid (pHCM2), which shows recent common ancestry with a virulence plasmid of Yersinia pestis.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">413</span>;<span class="number">6858</span>;<span class="pagerange">848-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11677608">11677608</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35101607">10.1038/35101607</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of Yersinia pestis, the causative agent of plague.</h4>
<p class="authors"><span class="author">Parkhill J</span>, <span class="author">Wren BW</span>, <span class="author">Thomson NR</span>, <span class="author">Titball RW</span>, <span class="author">Holden MT</span>, <span class="author">Prentice MB</span>, <span class="author">Sebaihia M</span>, <span class="author">James KD</span>, <span class="author">Churcher C</span>, <span class="author">Mungall KL</span>, <span class="author">Baker S</span>, <span class="author">Basham D</span>, <span class="author">Bentley SD</span>, <span class="author">Brooks K</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Chillingworth T</span>, <span class="author">Cronin A</span>, <span class="author">Davies RM</span>, <span class="author">Davis P</span>, <span class="author">Dougan G</span>, <span class="author">Feltwell T</span>, <span class="author">Hamlin N</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Karlyshev AV</span>, <span class="author">Leather S</span>, <span class="author">Moule S</span>, <span class="author">Oyston PC</span>, <span class="author">Quail M</span>, <span class="author">Rutherford K</span>, <span class="author">Simmonds M</span>, <span class="author">Skelton J</span>, <span class="author">Stevens K</span>, <span class="author">Whitehead S</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%70%61%72%6B%68%69%6C%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#114;&#107;&#104;&#105;&#108;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Gram-negative bacterium Yersinia pestis is the causative agent of the systemic invasive infectious disease classically referred to as plague, and has been responsible for three human pandemics: the Justinian plague (sixth to eighth centuries), the Black Death (fourteenth to nineteenth centuries) and modern plague (nineteenth century to the present day). The recent identification of strains resistant to multiple drugs and the potential use of Y. pestis as an agent of biological warfare mean that plague still poses a threat to human health. Here we report the complete genome sequence of Y. pestis strain CO92, consisting of a 4.65-megabase (Mb) chromosome and three plasmids of 96.2 kilobases (kb), 70.3 kb and 9.6 kb. The genome is unusually rich in insertion sequences and displays anomalies in GC base-composition bias, indicating frequent intragenomic recombination. Many genes seem to have been acquired from other bacteria and viruses (including adhesins, secretion systems and insecticidal toxins). The genome contains around 150 pseudogenes, many of which are remnants of a redundant enteropathogenic lifestyle. The evidence of ongoing genome fluidity, expansion and decay suggests Y. pestis is a pathogen that has undergone large-scale genetic flux and provides a unique insight into the ways in which new and highly virulent pathogens evolve.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">413</span>;<span class="number">6855</span>;<span class="pagerange">523-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11586360">11586360</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35097083">10.1038/35097083</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insertional events as well as translocations may arise during aberrant immunoglobulin switch recombination in a patient with multiple myeloma.</h4>
<p class="authors"><span class="author">Pratt G</span>, <span class="author">Fenton JA</span>, <span class="author">Davies FE</span>, <span class="author">Rawstron AC</span>, <span class="author">Richards SJ</span>, <span class="author">Collins JE</span>, <span class="author">Owen RG</span>, <span class="author">Jack AS</span>, <span class="author">Smith GM</span> and <span class="author">Morgan GJ</span></p>
<p class="affiliation">Academic Unit of Haematology and Oncology, Department of Haematology, The General Infirmary at Leeds, UK.</p><p class="abstract">The majority of patients with multiple myeloma have translocations involving the immunoglobulin heavy chain switch regions on chromosome 14q32 and a promiscuous range of partner chromosomes. We describe a patient with an insertion of 132 bp of chromosome 22q12 sequence into the 5&#39; region flanking S(mu) on chromosome 14q32. The 132 bp region from chromosome 22q12 contains the whole of exon 3 from a novel gene of unknown function in man. The significance of such insertional events remains unclear. The description of insertional events occurring as a result of abnormal switch recombination suggests that, in myeloma, dysregulation of oncogenes may occur by a mechanism other than chromosomal translocation.</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2001</span>;<span class="volume">112</span>;<span class="number">2</span>;<span class="pagerange">388-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11167836" rel="external">11167836</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The cluster of BTN genes in the extended major histocompatibility complex.</h4>
<p class="authors"><span class="author">Rhodes DA</span>, <span class="author">Stammers M</span>, <span class="author">Malcherek G</span>, <span class="author">Beck S</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Department of Immunology, University of Cambridge, Tennis Court Road, Cambridge, CB2 1QP, United Kingdom. <a href="mailto:%64%61%72%33%32%40%6D%6F%6C%65.%62%69%6F.%63%61%6D.%61%63.%75%6B">&#100;&#97;&#114;&#51;&#50;&#64;&#109;&#111;&#108;&#101;&#46;&#98;&#105;&#111;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We sequenced the 170-kb cluster of BTN genes in the extended major histocompatibility complex region, 4 Mb telomeric of human leukocyte antigen class I genes, at 6p22.1. The cluster consists of seven genes belonging to the expanding B7/butyrophilin-like group, a subset of the immunoglobulin gene superfamily. The main complex is composed of six genes, from two subfamilies, BTN2 and BTN3, arranged in pairs. This alternating pattern must have evolved by duplications of an original block of two genes, one from each subfamily. The sequences from the two subfamilies share approximately 50% amino acid identity. By analysis of repeat elements within each block, these duplications may be dated to approximately 100 million years ago, at about the time of the branching of the Rodentia and Primate lineages. The single BTN1A1 (butyrophilin) gene was positioned approximately 25 kb centromeric to the cluster. Each gene covers approximately 12 kb and consists of seven (BTN2 subfamily) or nine (BTN3 subfamily) coding exons. The predicted leader sequence, immunoglobulin-like IgV (variable)/IgC (constant) ectodomains, and the predicted transmembrane domain are encoded on separate exons and are separated from a B30.2 domain by a variable number of very short exons, 21 and 27 nucleotides in length. BTN transcripts were detected in all tissues examined. Alternative splicing, involving particularly the carboxyl-terminal B30.2 domain, was a notable feature. Most transcripts of BTN2 subfamily genes contained this domain, whereas BTN3 genes did not. Using immunofluorescence, we showed surface expression of BTN-green fluorescent protein fusions in mammalian cell transfectants.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2001</span>;<span class="volume">71</span>;<span class="number">3</span>;<span class="pagerange">351-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11170752" rel="external">11170752</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2000.6406" rel="external">10.1006/geno.2000.6406</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The mouse genome sequence: status and prospects.</h4>
<p class="authors"><span class="author">Rogers J</span> and <span class="author">Bradley A</span></p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2001</span>;<span class="volume">77</span>;<span class="number">3</span>;<span class="pagerange">117-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11597135" rel="external">11597135</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.2001.6637" rel="external">10.1006/geno.2001.6637</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of 41 kb of the DNA sequence from the right arm of chromosome II of Schizosaccharomyces pombe.</h4>
<p class="authors"><span class="author">S&aacute;nchez M</span>, <span class="author">Revuelta JL</span>, <span class="author">del Rey F</span>, <span class="author">Gwilliam R</span>, <span class="author">Skelton J</span>, <span class="author">Churcher C</span>, <span class="author">Rajandream MA</span>, <span class="author">Wood V</span>, <span class="author">Barrell B</span>, <span class="author">Lyne R</span>, <span class="author">Reinhardt R</span>, <span class="author">Borzym K</span>, <span class="author">Beck A</span>, <span class="author">Moreno S</span> and <span class="author">Dom&iacute;nguez A</span></p>
<p class="affiliation">Departamento de Microbiolog&iacute;a y Gen&eacute;tica, Instituto de Microbiolog&iacute;a Bioqu&iacute;mica/CSIC. Universidad de Salamanca, 37071 Salamanca, Spain.</p><p class="abstract">We report the complete sequence of cosmid c18A7 (41 046 bp insert), located on the right arm of chromosome II of the Schizosaccharomyces pombe genome. The sequence, which partially overlaps with cosmids SPBC4F6 and SPBC336, contains 16 open reading frames (ORFs) capable of coding for proteins of at least 100 amino acid residues in length (one partial) and one small nucleolar RNA (snoRNA). Four known genes were found: swi10 (encoding a mating-type switching protein also involved in nucleotide excision repair); dim1 (encoding a dimethyladenosine transferase); arf1 (encoding ADP-ribosylation factor 1); and pol3 (cdc6) the partial fragment, encoding the 125 kDa catalytic subunit of the DNA polymerase type B. Six ORFs similar to known proteins were found. They include a transporter of the major facilitator superfamily class, a vacuolar sorting protein, an asparagine synthase, a nuclear protein, a reticulum oxidoreductin and a heat shock protein. Each protein product of the other six ORFs has conserved domains and can be assigned a molecular, but not a biological, function. The sequence has been submitted to the EMBL database under Accession No. AL080287.</p>
<p class="publication"><span class="title">Yeast (Chichester, England)</span>&nbsp;<span class="year">2001</span>;<span class="volume">18</span>;<span class="number">12</span>;<span class="pagerange">1111-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11536333" rel="external">11536333</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fyea.760" rel="external">10.1002/yea.760</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms.</h4>
<p class="authors"><span class="author">Sachidanandam R</span>, <span class="author">Weissman D</span>, <span class="author">Schmidt SC</span>, <span class="author">Kakol JM</span>, <span class="author">Stein LD</span>, <span class="author">Marth G</span>, <span class="author">Sherry S</span>, <span class="author">Mullikin JC</span>, <span class="author">Mortimore BJ</span>, <span class="author">Willey DL</span>, <span class="author">Hunt SE</span>, <span class="author">Cole CG</span>, <span class="author">Coggill PC</span>, <span class="author">Rice CM</span>, <span class="author">Ning Z</span>, <span class="author">Rogers J</span>, <span class="author">Bentley DR</span>, <span class="author">Kwok PY</span>, <span class="author">Mardis ER</span>, <span class="author">Yeh RT</span>, <span class="author">Schultz B</span>, <span class="author">Cook L</span>, <span class="author">Davenport R</span>, <span class="author">Dante M</span>, <span class="author">Fulton L</span>, <span class="author">Hillier L</span>, <span class="author">Waterston RH</span>, <span class="author">McPherson JD</span>, <span class="author">Gilman B</span>, <span class="author">Schaffner S</span>, <span class="author">Van Etten WJ</span>, <span class="author">Reich D</span>, <span class="author">Higgins J</span>, <span class="author">Daly MJ</span>, <span class="author">Blumenstiel B</span>, <span class="author">Baldwin J</span>, <span class="author">Stange-Thomann N</span>, <span class="author">Zody MC</span>, <span class="author">Linton L</span>, <span class="author">Lander ES</span>, <span class="author">Altshuler D</span> and <span class="author">International SNP Map Working Group</span></p>
<p class="affiliation">Cold Spring Harbor, New York 11724, USA.</p><p class="abstract">We describe a map of 1.42 million single nucleotide polymorphisms (SNPs) distributed throughout the human genome, providing an average density on available sequence of one SNP every 1.9 kilobases. These SNPs were primarily discovered by two projects: The SNP Consortium and the analysis of clone overlaps by the International Human Genome Sequencing Consortium. The map integrates all publicly available SNPs with described genes and other genomic features. We estimate that 60,000 SNPs fall within exon (coding and untranslated regions), and 85% of exons are within 5 kb of the nearest SNP. Nucleotide diversity varies greatly across the genome, in a manner broadly consistent with a standard population genetic model of human history. This high-density SNP map provides a public resource for defining haplotype variation across the genome, and should help to identify biomedically important genes for diagnosis and therapy.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">928-33</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237013">11237013</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057149">10.1038/35057149</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification and cloning of Lmairk, a member of the Aurora/Ipl1p protein kinase family, from the human protozoan parasite Leishmania.</h4>
<p class="authors"><span class="author">Siman-Tov MM</span>, <span class="author">Ivens AC</span> and <span class="author">Jaffe CL</span></p>
<p class="affiliation">Department of Parasitology, Hebrew University, Hadassah Medical School, Jerusalem, Israel.</p><p class="abstract">Lmairk, a gene encoding a member of the Aurora/Ipl1p family of protein kinases (AIRK), was cloned from the protozoan parasite Leishmania major. Aurora kinases are key enzymes involved in the regulation of normal chromosome segregation during mitosis and cytokenesis of eukaryotic cells. This single-copy gene located on L. major chromosome 28 encodes a 301 amino acid polypeptide. All 11 conserved eukaryotic protein kinase catalytic subdomains are present and the proposed AIRK signature sequence was identified in the activation loop between subdomains VII and VIII. Lmairk is expressed, as an approximately 2.4 kb message, in at least three different species of Leishmania. This report represents the first identification of an AIRK from the trypanosomatid family of early divergent eukaryotes.</p>
<p class="publication"><span class="title">Biochimica et biophysica acta</span>&nbsp;<span class="year">2001</span>;<span class="volume">1519</span>;<span class="number">3</span>;<span class="pagerange">241-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11418192" rel="external">11418192</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cyclin D3 is a target gene of t(6;14)(p21.1;q32.3) of mature B-cell malignancies.</h4>
<p class="authors"><span class="author">Sonoki T</span>, <span class="author">Harder L</span>, <span class="author">Horsman DE</span>, <span class="author">Karran L</span>, <span class="author">Taniguchi I</span>, <span class="author">Willis TG</span>, <span class="author">Gesk S</span>, <span class="author">Steinemann D</span>, <span class="author">Zucca E</span>, <span class="author">Schlegelberger B</span>, <span class="author">Sol&eacute; F</span>, <span class="author">Mungall AJ</span>, <span class="author">Gascoyne RD</span>, <span class="author">Siebert R</span> and <span class="author">Dyer MJ</span></p>
<p class="affiliation">Academic Department of Haematology and Cytogenetics, Institute of Cancer Research, Sutton, United Kingdom.</p><p class="abstract">Chromosomal translocation t(6;14)(p21.1;q32.3) has been reported as a rare but recurrent event not only in myeloma and plasma cell leukemia but also in diffuse large B-cell non-Hodgkin lymphoma (B-NHL) (diffuse large B-cell lymphoma [DLBCL]) and splenic lymphoma with villous lymphocytes (SLVL); however, the nature of the target gene(s) has not been determined. This study identified t(6;14)(p21.1;q32.3) in 3 cases of transformed extranodal marginal zone B-NHL, in 1 case of SLVL, and in 1 case of a low-grade B-cell lymphoproliferative disorder. In a sixth case, a CD5(+) DLBCL, the translocation was identified by molecular cloning in the absence of cytogenetically detectable change. Two chromosomal translocation breakpoints were cloned by using long-distance inverse polymerase chain reaction methods. Comparison with the genomic sequence for chromosome 6p21.1 showed breakpoints approximately 59 and 73.5 kilobases 5&#39; of the cyclin D3 (CCND3) gene with no other identifiable transcribed sequences in the intervening region. Although Southern blotting with derived genomic 6p21.1 probes failed to detect other rearrangements, fluorescent in situ hybridization assays, using BAC (bacterial artificial chromosome) clones spanning and flanking the CCND3 locus, along with probes for IGH confirmed localization of 6p21.1 breakpoints within the same region, as well as fusion of the CCND3 and IGH loci. Furthermore, in all cases, high-level expression of CCND3 was demonstrated at RNA and/or protein levels by Northern and Western blotting and by immunohistochemistry. These data implicate CCND3 as a dominant oncogene in the pathogenesis and transformation in several histologic subtypes of mature B-cell malignancies with t(6;14)(p21.1;q32.3) and suggest that CCND3 overexpression seen in about 10% of DLBCL cases may have a genetic basis.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">5 U01 CA84967-02</span></p>
<p class="publication"><span class="title">Blood</span>&nbsp;<span class="year">2001</span>;<span class="volume">98</span>;<span class="number">9</span>;<span class="pagerange">2837-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11675358" rel="external">11675358</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: network access to the genome and biology of Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Stein L</span>, <span class="author">Sternberg P</span>, <span class="author">Durbin R</span>, <span class="author">Thierry-Mieg J</span> and <span class="author">Spieth J</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724, USA. <a href="mailto:%6C%73%74%65%69%6E%40%63%73%68%6C.%6F%72%67">&#108;&#115;&#116;&#101;&#105;&#110;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">WormBase (http://www.wormbase.org) is a web-based resource for the Caenorhabditis elegans genome and its biology. It builds upon the existing ACeDB database of the C.elegans genome by providing data curation services, a significantly expanded range of subject areas and a user-friendly front end.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">1</span>;<span class="pagerange">82-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11125056" rel="external">11125056</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC29781" rel="external">29781</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The human gene for mannan-binding lectin-associated serine protease-2 (MASP-2), the effector component of the lectin route of complement activation, is part of a tightly linked gene cluster on chromosome 1p36.2-3.</h4>
<p class="authors"><span class="author">Stover C</span>, <span class="author">Endo Y</span>, <span class="author">Takahashi M</span>, <span class="author">Lynch NJ</span>, <span class="author">Constantinescu C</span>, <span class="author">Vorup-Jensen T</span>, <span class="author">Thiel S</span>, <span class="author">Friedl H</span>, <span class="author">Hankeln T</span>, <span class="author">Hall R</span>, <span class="author">Gregory S</span>, <span class="author">Fujita T</span> and <span class="author">Schwaeble W</span></p>
<p class="affiliation">Department of Microbiology and Immunology, University of Leicester, Leicester, UK.</p><p class="abstract">The proteases of the lectin pathway of complement activation, MASP-1 and MASP-2, are encoded by two separate genes. The MASP1 gene is located on chromosome 3q27, the MASP2 gene on chromosome 1p36.23-31. The genes for the classical complement activation pathway proteases, C1r and C1s, are linked on chromosome 12p13. We have shown that the MASP2 gene encodes two gene products, the 76 kDa MASP-2 serine protease and a plasma protein of 19 kDa, termed MAp19 or sMAP. Both gene products are components of the lectin pathway activation complex. We present the complete primary structure of the human MASP2 gene and the tight cluster that this locus forms with non-complement genes. A comparison of the MASP2 gene with the previously characterised C1s gene revealed identical positions of introns separating orthologous coding sequences, underlining the hypothesis that the C1s and MASP2 genes arose by exon shuffling from one ancestral gene.</p>
<p class="publication"><span class="title">Genes and immunity</span>&nbsp;<span class="year">2001</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">119-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11426320" rel="external">11426320</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.gene.6363745" rel="external">10.1038/sj.gene.6363745</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Society and the human genome. Sir Frederick Gowland Hopkins Memorial Lecture.</h4>
<p class="authors"><span class="author">Sulston J</span> and <span class="author">International Human Genome Sequencing Consortium</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1RQ, U.K. <a href="mailto:%6A%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In June 2000, the draft sequence of the human genome was announced. It is, and will be for some years, incomplete, but the vast majority is now available. Currently about a third is finished (including two complete chromosomes); the rest has good coverage, but not long-range continuity. First-pass analysis indicates, among other things, fewer genes than expected: about 40000 now looks a likely number. This uncertainty illustrates the difficulty of interpretation: the sequence is not an end in itself, but a resource to be continually reanalysed as our biological understanding increases. That is the scientific reason for releasing it promptly, fully and freely. The social reasons for doing so are even more compelling.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2001</span>;<span class="volume">29</span>;<span class="number">Pt 2</span>;<span class="pagerange">27-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11356121" rel="external">11356121</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A sequence-based integrated map of chromosome 22.</h4>
<p class="authors"><span class="author">Tapper WJ</span>, <span class="author">Morton NE</span>, <span class="author">Dunham I</span>, <span class="author">Ke X</span> and <span class="author">Collins A</span></p>
<p class="affiliation">Human Genetics Research Division, Southampton General Hospital, Southampton SO16 6YD, UK. <a href="mailto:%77%6A%74%40%73%6F%74%6F%6E.%61%63.%75%6B">&#119;&#106;&#116;&#64;&#115;&#111;&#116;&#111;&#110;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The near-completion of the sequence for chromosome 22q revolutionizes map integration. We describe a sequence-based integrated map containing 968 loci including 516 known or predicted gene sequences, 317 STSs not included in these sequences, and 135 nonexpressed multinucleotide polymorphisms. The published sequence spans 34.6 Mb, inclusive of gaps estimated to total 1.1 Mb, compared with a top-down estimate of 43 Mb. This discrepancy is discussed, but will not be resolved until more of the genome is analyzed. The radiation hybrid map has 5% error in order and 34% error in location exceeding 1 Mb. The utility of a composite location based on evidence other than sequence is limited to regions not yet sequenced. A genetic map conditional on sequence order was constructed from pairwise lods. Its length of 74.8 cM in males and 80.2 cM in females is slightly less than the previous estimate not constrained by sequence order. Five recombination hot spots are detected, with differences in location between the sexes. Male recombination correlates with repetitive DNA, whereas female recombination does not. It remains to be seen whether this is true for other human chromosomes. An algorithm to improve the fit of cytogenetic bands sequence location reduces the discrepancies in cytogenetic assignment from 61 to 38. This sequence-based integrated map is represented in the genetic location database (LDB2000), which is available at http://cedar.genetics.soton.ac.uk/public_html/LDB2000.html.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">7</span>;<span class="pagerange">1290-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11435412" rel="external">11435412</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311079" rel="external">311079</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.161301" rel="external">10.1101/gr.161301</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An integrated cytogenetic, radiation-hybrid, and comparative map of dog chromosome 5.</h4>
<p class="authors"><span class="author">Thomas R</span>, <span class="author">Breen M</span>, <span class="author">Deloukas P</span>, <span class="author">Holmes NG</span> and <span class="author">Binns MM</span></p>
<p class="affiliation">Genetics Section, Animal Health Trust, Lanwades Park, Kentford, Newmarket, Suffolk, CB8 7UU, UK. <a href="mailto:%72%74%68%6F%6D%61%73%40%68%67%6D%70.%6D%72%63.%61%63.%75%6B">&#114;&#116;&#104;&#111;&#109;&#97;&#115;&#64;&#104;&#103;&#109;&#112;&#46;&#109;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The development of a detailed genome map for the domestic dog (Canis familiaris, CFA) is a prerequisite for the continued use of this species as a model system for the study of inherited traits. We present an integrated cytogenetic, radiation-hybrid, and comparative map of dog Chromosome (Chr) 5 (CFA 5). The map comprises 14 gene markers, selected from loci previously mapped within the corresponding evolutionarily conserved chromosome segments (ECCS) of the human genome. Large-insert clones representing each marker were first isolated and mapped by fluorescence in situ hybridization (FISH) analysis to determine their subchromosomal localization on CFA 5. Thirteen gene markers were subsequently mapped by using a commercially available whole genome radiation hybrid (WG-RH) panel for the dog. Nine anonymous markers were also assigned to CFA 5 by both FISH and WG-RH analysis. The 22 markers formed six RH-linkage groups, spanning each of the four ECCS comprising this 99 megabase chromosome. All cytogenetic, WG-RH, and comparative mapping data were in agreement and were combined to determine both the most likely locus order within each linkage group, and also the gross relative orientation of the corresponding ECCS. This study provides a resource for the transfer of information from the human transcript map to that of the dog, and extends existing data regarding the structural relationships between CFA 5 and its evolutionary counterparts within the human genome.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2001</span>;<span class="volume">12</span>;<span class="number">5</span>;<span class="pagerange">371-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11331945" rel="external">11331945</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs003350010287" rel="external">10.1007/s003350010287</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Secondary DNA structure analysis of the coding strand switch regions of five Leishmania major Friedlin chromosomes.</h4>
<p class="authors"><span class="author">Tosato V</span>, <span class="author">Ciarloni L</span>, <span class="author">Ivens AC</span>, <span class="author">Rajandream MA</span>, <span class="author">Barrell BG</span> and <span class="author">Bruschi CV</span></p>
<p class="affiliation">Department of Biology, University of Trieste, Italy.</p><p class="abstract">As part of the EULEISH international genome project, a region of 74,674 nucleotides from chromosome 21 of Leishmania major Friedlin was subcloned and sequenced; and 31 new coding sequences were predicted. Of particular interest was a unique coding strand switching region covering 1.6 kb of DNA; and this was subjected to further investigation. Bioinformatic analysis of this region revealed an unusually high AT composition, a lack of putative hairpins and a strong curvature of the DNA in agreement with the structural characteristics of similar regions of other Leishmania chromosomes. These observations and a comparison with the secondary DNA structure of four other Leishmania chromosomes and chromosomes of different organisms could suggest a functional role of this region in transcription and mitotic division.</p>
<p class="publication"><span class="title">Current genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">40</span>;<span class="number">3</span>;<span class="pagerange">186-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11727994" rel="external">11727994</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Salmonella enterica serovar Typhi possesses a unique repertoire of fimbrial gene sequences.</h4>
<p class="authors"><span class="author">Townsend SM</span>, <span class="author">Kramer NE</span>, <span class="author">Edwards R</span>, <span class="author">Baker S</span>, <span class="author">Hamlin N</span>, <span class="author">Simmonds M</span>, <span class="author">Stevens K</span>, <span class="author">Maloy S</span>, <span class="author">Parkhill J</span>, <span class="author">Dougan G</span> and <span class="author">B&auml;umler AJ</span></p>
<p class="affiliation">Department of Medical Microbiology and Immunology, College of Medicine, Texas A&amp;M University, College Station, Texas 77843, USA.</p><p class="abstract">Salmonella enterica serotype Typhi differs from nontyphoidal Salmonella serotypes by its strict host adaptation to humans and higher primates. Since fimbriae have been implicated in host adaptation, we investigated whether the serotype Typhi genome contains fimbrial operons which are unique to this pathogen or restricted to typhoidal Salmonella serotypes. This study established for the first time the total number of fimbrial operons present in an individual Salmonella serotype. The serotype Typhi CT18 genome, which has been sequenced by the Typhi Sequencing Group at the Sanger Centre, contained a type IV fimbrial operon, an orthologue of the agf operon, and 12 putative fimbrial operons of the chaperone-usher assembly class. In addition to sef, fim, saf, and tcf, which had been described previously in serotype Typhi, we identified eight new putative chaperone-usher-dependent fimbrial operons, which were termed bcf, sta, stb, ste, std, stc, stg, and sth. Hybridization analysis performed with 16 strains of Salmonella reference collection C and 22 strains of Salmonella reference collection B showed that all eight putative fimbrial operons of serotype Typhi were also present in a number of nontyphoidal Salmonella serotypes. Thus, a simple correlation between host range and the presence of a single fimbrial operon seems at present unlikely. However, the serotype Typhi genome differed from that of all other Salmonella serotypes investigated in that it contained a unique combination of putative fimbrial operons.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI40124</span>, <span class="grant">AI44170</span></p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2001</span>;<span class="volume">69</span>;<span class="number">5</span>;<span class="pagerange">2894-901</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11292704" rel="external">11292704</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC98240" rel="external">98240</a>; DOI: <a href="http://dx.doi.org/10.1128%2FIAI.69.5.2894-2901.2001" rel="external">10.1128/IAI.69.5.2894-2901.2001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis and assessment of comparative modeling predictions in CASP4.</h4>
<p class="authors"><span class="author">Tramontano A</span>, <span class="author">Leplae R</span> and <span class="author">Morea V</span></p>
<p class="affiliation">Department of Biochemical Sciences &quot;A. Rossi Fanelli,&quot; University of Rome &quot;La Sapienza,&quot; Rome, Italy. <a href="mailto:%41%6E%6E%61.%54%72%61%6D%6F%6E%74%61%6E%6F%40%75%6E%69%72%6F%6D%61%31.%69%74">&#65;&#110;&#110;&#97;&#46;&#84;&#114;&#97;&#109;&#111;&#110;&#116;&#97;&#110;&#111;&#64;&#117;&#110;&#105;&#114;&#111;&#109;&#97;&#49;&#46;&#105;&#116;</a></p><p class="abstract">We describe here the results of our analysis of the comparative modeling predictions submitted to the fourth round of Critical Assessment of Structure Prediction (CASP4). On the basis of a numerical evaluation of the models, we assessed their ability to predict the overall fold correctly, the relative orientation of domains in multidomain proteins, the conformation of the side chains, the loop regions, and the biologically important residues of the targets. We also discuss the performance of automatic prediction servers and compare the results of CASP4 with those obtained in CASP3.</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">2001</span>;<span class="volume">Suppl 5</span>;<span class="pagerange">22-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11835479" rel="external">11835479</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genomic context of natural killer receptor extended gene families.</h4>
<p class="authors"><span class="author">Trowsdale J</span>, <span class="author">Barten R</span>, <span class="author">Haude A</span>, <span class="author">Stewart CA</span>, <span class="author">Beck S</span> and <span class="author">Wilson MJ</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, UK. <a href="mailto:%6A%74%32%33%33%40%6D%6F%6C%65.%62%69%6F.%63%61%6D.%61%63.%75%6B">&#106;&#116;&#50;&#51;&#51;&#64;&#109;&#111;&#108;&#101;&#46;&#98;&#105;&#111;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The two sets of inhibitory and activating natural killer (NK) receptor genes belong either to the Ig or to the C-type lectin superfamilies. Both are extensive and diverse, comprising genes of varying degrees of relatedness, indicative of a process of iterative duplication. We have constructed gene maps to help understand how and when NK receptor genes developed and the nature of their polymorphism. A cluster of over 15 C-type lectin genes, the natural killer complex is located on human chromosome 12p13.1, syntenic with a region in mouse that borders multiple Ly49 loci. The equivalent locus in man is occupied by a single pseudogene, LY49L. The immunoglobulin superfamily of loci, the leukocyte receptor complex (LRC), on chromosome 19q13.4, contains many polymorphic killer cell immunoglobulin-like receptor (KIR) genes as well as multiple related sequences. These include immunoglobulin-like transcript (ILT) (or leukocyte immunoglobulin-like receptor genes), leukocyte-associated inhibitory receptor genes (LAIR), NKp46, Fc alphaR and the platelet glycoprotein receptor VI locus, which encodes a collagen-binding molecule. KIRs are expressed mostly on NK cells and some T cells. The other LRC loci are more widely expressed. Further centromeric of the LRC are sets of additional loci with weak sequence similarity to the KIRs, including the extensive CD66(CEA) and Siglec families. The LRC-syntenic region in mice contains no orthologues of KIRs. Some of the KIR genes are highly polymorphic in terms of sequence as well as for presence/absence of genes on different haplotypes. Some anchor loci, such as KIR2DL4, are present on most haplotypes. A few ILT loci, such as ILT5 and ILT8, are polymorphic, but only ILT6 exhibits presence/absence variation. This knowledge of the genomic organisation of the extensive NK superfamilies underpins efforts to understand the functions of the encoded NK receptor molecules. It leads to the conclusion that the functional homology of human KIR and mouse Ly49 genes arose by convergent evolution. NK receptor immunogenetics has interesting parallels with the major histocompatibility complex (MHC) in which some of the polymorphic genes are ligands for NK molecules. There are hints of an ancient genetic relationship between NK receptor genes and MHC-paralogous regions on chromosomes 1, 9 and 19. The picture that emerges from both complexes is of eternal evolutionary restlessness, presumably in response to resistance to disease.</p>
<p class="publication"><span class="title">Immunological reviews</span>&nbsp;<span class="year">2001</span>;<span class="volume">181</span>;<span class="pagerange">20-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11513141" rel="external">11513141</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Matroshka and ectopic polymorphisms: Two new classes of DNA sequence variation identified at the Van der Woude syndrome locus on 1q32-q41.</h4>
<p class="authors"><span class="author">Watanabe Y</span>, <span class="author">Murray JC</span>, <span class="author">Bjork BC</span>, <span class="author">Bird CP</span>, <span class="author">Chiang PW</span>, <span class="author">Gregory SG</span>, <span class="author">Kurnit DM</span> and <span class="author">Schutte BC</span></p>
<p class="affiliation">Department of Pediatrics, University of Iowa, Iowa City, Iowa, USA.</p><p class="abstract">Van der Woude syndrome (VWS) is an orofacial clefting disorder with an autosomal dominant pattern of inheritance. In our efforts to clone the VWS gene, 900 kb of genomic sequence from the VWS candidate region at chromosome 1q32-q41 was analyzed for new DNA sequence variants. We observed that in clone CTA-321i20 a 7922 bp sequence is absent relative to the sequence present in PAC clone RP4-782d21 at positions 1669-9590, suggesting the presence of a deletion/insertion (del/ins) polymorphism. Embedded in this 7922 bp region was a TTCC short tandem repeat (STR). Genotype analysis showed that both the internal STR and the (del/ins) mutation were true polymorphisms. This is a novel example of intraallelic variation, a polymorphism within a polymorphism, and we suggest that it be termed a &quot;Matroshka&quot; polymorphism. Further genetic and DNA sequence analysis indicated that the ancestral state of the 1669-9590 del/ins polymorphism was the insertion allele and that the original deletion mutation probably occurred only once. A second class of novel DNA sequence variation was discovered on chromosome 5 that shared a 328 bp identical sequence with this region on chromosome 1. A single nucleotide polymorphism (SNP) was detected by SSCP using a pair of primers derived from the chromosome 1 sequence. Surprisingly, these primers also amplified the identical locus on chromosome 5, and the SNP was only located on chromosome 5. Since the probe unexpectedly detected alleles from another locus, we suggest that this type of sequence variant be termed an &quot;ectopic&quot; polymorphism. These two novel classes of DNA sequence polymorphisms have the potential to confound genetic and DNA sequence analysis and may also contribute to variation in disease phenotypes.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD27748</span>; NIDCR NIH HHS: <span class="grant">DE08559</span>, <span class="grant">DE13076</span></p>
<p class="publication"><span class="title">Human mutation</span>&nbsp;<span class="year">2001</span>;<span class="volume">18</span>;<span class="number">5</span>;<span class="pagerange">422-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11668635" rel="external">11668635</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fhumu.1213" rel="external">10.1002/humu.1213</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detailed molecular analysis of 1p36 in neuroblastoma.</h4>
<p class="authors"><span class="author">White PS</span>, <span class="author">Thompson PM</span>, <span class="author">Seifried BA</span>, <span class="author">Sulman EP</span>, <span class="author">Jensen SJ</span>, <span class="author">Guo C</span>, <span class="author">Maris JM</span>, <span class="author">Hogarty MD</span>, <span class="author">Allen C</span>, <span class="author">Biegel JA</span>, <span class="author">Matise TC</span>, <span class="author">Gregory SG</span>, <span class="author">Reynolds CP</span> and <span class="author">Brodeur GM</span></p>
<p class="affiliation">Division of Oncology, Children&#39;s Hospital of Philadelphia, Pennsylvania 19104-4318, USA. <a href="mailto:%77%68%69%74%65%40%67%65%6E%6F%6D%65.%63%68%6F%70.%65%64%75">&#119;&#104;&#105;&#116;&#101;&#64;&#103;&#101;&#110;&#111;&#109;&#101;&#46;&#99;&#104;&#111;&#112;&#46;&#101;&#100;&#117;</a></p><p class="abstract"><strong>Background</strong>: Several lines of evidence es tablish that chromosome band 1p36 is frequently deleted in neuroblastoma primary tumors and cell lines, suggesting that a tumor suppressor gene within this region is involved in the development of this tumor.</p><p class="abstract"><strong>Procedure</strong>: We analyzed the status of 1p36 in primary neuroblastomas and cell lines to define the region of consistent rearrangement.</p><p class="abstract"><strong>Results</strong>: Loss of heterozygosity (LOH) studies of primary neuro blastomas identified allelic loss in 135 of 503 tumors (27%), with the smallest region of overlap (SRO) defined distal to D15214 (1p36.3). No homozygous deletions were detected at 120 loci mapping to 1p36.1-p36.3 in a panel of 46 neuroblastoma cell lines. A recently identified patient with neuroblastoma was found to have a constitutional deletion within 1p36.2-p36.3, and this deletion, when combined with the LOH results, defined a smaller SRO of one megabase within 1p36.3. We constructed a comprehensive integrated map of chromosome 1 containing 11,000 markers and large-insert clones, a high-resolution radiation hybrid (RH) map of 1p36, and a P1-artificial chromosome (PAC) contig spanning the SRO, to further characterize the region of interest. Over 768 kb (75%) of the SRO has been sequenced to completion. Further analysis of distal 1p identified 113 transcripts localizing to 1p36, 21 of which were mapped within the SRO.</p><p class="abstract"><strong>Conclusion</strong>: This analysis will identify suitable positional candidate transcripts for mutational screening and subsequent identification of the 1p36.3 neuroblastoma suppressor gene.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA-39771</span></p>
<p class="publication"><span class="title">Medical and pediatric oncology</span>&nbsp;<span class="year">2001</span>;<span class="volume">36</span>;<span class="number">1</span>;<span class="pagerange">37-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11464901" rel="external">11464901</a>; DOI: <a href="http://dx.doi.org/10.1002%2F1096-911X(20010101)36%3A1%3C37%3A%3AAID-MPO1010%3E3.0.CO%3B2-L" rel="external">10.1002/1096-911X(20010101)36:1&lt;37::AID-MPO1010&gt;3.0.CO;2-L</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of clustered MHC-linked olfactory receptor genes in human and mouse.</h4>
<p class="authors"><span class="author">Younger RM</span>, <span class="author">Amadou C</span>, <span class="author">Bethel G</span>, <span class="author">Ehlers A</span>, <span class="author">Lindahl KF</span>, <span class="author">Forbes S</span>, <span class="author">Horton R</span>, <span class="author">Milne S</span>, <span class="author">Mungall AJ</span>, <span class="author">Trowsdale J</span>, <span class="author">Volz A</span>, <span class="author">Ziegler A</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract">Olfactory receptor (OR) loci frequently cluster and are present on most human chromosomes. They are members of the seven transmembrane receptor (7-TM) superfamily and, as such, are part of one of the largest mammalian multigene families, with an estimated copy number of up to 1000 ORs per haploid genome. As their name implies, ORs are known to be involved in the perception of odors and possibly also in other, nonolfaction-related, functions. Here, we report the characterization of ORs that are part of the MHC-linked OR clusters in human and mouse (partial sequence only). These clusters are of particular interest because of their possible involvement in olfaction-driven mate selection. In total, we describe 50 novel OR loci (36 human, 14 murine), making the human MHC-linked cluster the largest sequenced OR cluster in any organism so far. Comparative and phylogenetic analyses confirm the cluster to be MHC-linked but divergent in both species and allow the identification of at least one ortholog that will be useful for future regulatory and functional studies. Quantitative feature analysis shows clear evidence of duplications of blocks of OR genes and reveals the entire cluster to have a genomic environment that is very different from its neighboring regions. Based on in silico transcript analysis, we also present evidence of extensive long-distance splicing in the 5&#39;-untranslated regions and, for the first time, of alternative splicing within the single coding exon of ORs. Taken together with our previous finding that ORs are also polymorphic, the presented data indicate that the expression, function, and evolution of these interesting genes might be more complex than previously thought.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2001</span>;<span class="volume">11</span>;<span class="number">4</span>;<span class="pagerange">519-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11282967" rel="external">11282967</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC311051" rel="external">311051</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.160301" rel="external">10.1101/gr.160301</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of human genetic and sequence-based physical maps.</h4>
<p class="authors"><span class="author">Yu A</span>, <span class="author">Zhao C</span>, <span class="author">Fan Y</span>, <span class="author">Jang W</span>, <span class="author">Mungall AJ</span>, <span class="author">Deloukas P</span>, <span class="author">Olsen A</span>, <span class="author">Doggett NA</span>, <span class="author">Ghebranious N</span>, <span class="author">Broman KW</span> and <span class="author">Weber JL</span></p>
<p class="affiliation">Center for Medical Genetics, Marshfield Medical Research Foundation, Wisconsin 54449, USA.</p><p class="abstract">Recombination is the exchange of information between two homologous chromosomes during meiosis. The rate of recombination per nucleotide, which profoundly affects the evolution of chromosomal segments, is calculated by comparing genetic and physical maps. Human physical maps have been constructed using cytogenetics, overlapping DNA clones and radiation hybrids; but the ultimate and by far the most accurate physical map is the actual nucleotide sequence. The completion of the draft human genomic sequence provides us with the best opportunity yet to compare the genetic and physical maps. Here we describe our estimates of female, male and sex-average recombination rates for about 60% of the genome. Recombination rates varied greatly along each chromosome, from 0 to at least 9 centiMorgans per megabase (cM Mb(-1)). Among several sequence and marker parameters tested, only relative marker position along the metacentric chromosomes in males correlated strongly with recombination rate. We identified several chromosomal regions up to 6 Mb in length with particularly low (deserts) or high (jungles) recombination rates. Linkage disequilibrium was much more common and extended for greater distances in the deserts than in the jungles.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2001</span>;<span class="volume">409</span>;<span class="number">6822</span>;<span class="pagerange">951-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11237020">11237020</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35057185">10.1038/35057185</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Engineering chromosomal rearrangements in mice.</h4>
<p class="authors"><span class="author">Yu Y</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Program in Developmental Biology, Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, Texas 77030, <a href="mailto:%55%53%41.%79%79%75%40%62%63%6D.%74%6D%63.%65%64%75">&#85;&#83;&#65;&#46;&#121;&#121;&#117;&#64;&#98;&#99;&#109;&#46;&#116;&#109;&#99;&#46;&#101;&#100;&#117;</a></p><p class="abstract">The combination of gene-targeting techniques in mouse embryonic stem cells and the Cre/loxP site-specific recombination system has resulted in the emergence of chromosomal-engineering technology in mice. This advance has opened up new opportunities for modelling human diseases that are associated with chromosomal rearrangements. It has also led to the generation of visibly marked deletions and balancer chromosomes in mice, which provide essential reagents for maximizing the efficiency of large-scale mutagenesis efforts and which will accelerate the functional annotation of mammalian genomes, including the human genome.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2001</span>;<span class="volume">2</span>;<span class="number">10</span>;<span class="pagerange">780-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11584294">11584294</a>; DOI: <a href="http://dx.doi.org/10.1038%2F35093564">10.1038/35093564</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nonredundant roles of the mPer1 and mPer2 genes in the mammalian circadian clock.</h4>
<p class="authors"><span class="author">Zheng B</span>, <span class="author">Albrecht U</span>, <span class="author">Kaasik K</span>, <span class="author">Sage M</span>, <span class="author">Lu W</span>, <span class="author">Vaishnav S</span>, <span class="author">Li Q</span>, <span class="author">Sun ZS</span>, <span class="author">Eichele G</span>, <span class="author">Bradley A</span> and <span class="author">Lee CC</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA.</p><p class="abstract">Mice carrying a null mutation in the Period 1 (mPer1) gene were generated using embryonic stem cell technology. Homozygous mPer1 mutants display a shorter circadian period with reduced precision and stability. Mice deficient in both mPer1 and mPer2 do not express circadian rhythms. While mPER2 regulates clock gene expression at the transcriptional level, mPER1 is dispensable for the rhythmic RNA expression of mPer1 and mPer2 and may instead regulate mPER2 at a posttranscriptional level. Studies of clock-controlled genes (CCGs) reveal a complex pattern of regulation by mPER1 and mPER2, suggesting independent controls by the two proteins over some output pathways. Genes encoding key enzymes in heme biosynthesis are under circadian control and are regulated by mPER1 and mPER2. Together, our studies show that mPER1 and mPER2 have distinct and complementary roles in the mouse clock mechanism.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2001</span>;<span class="volume">105</span>;<span class="number">5</span>;<span class="pagerange">683-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/11389837" rel="external">11389837</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/5f98scb8"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/5f98scb8" src="/qr/5f98scb8.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:55:34 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
