<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1998 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1998.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1998</h1><p>Number of papers published in 1998: <strong>58</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structure and distribution of pentapeptide repeats in bacteria.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Murzin AG</span> and <span class="author">Teichmann SA</span></p>
<p class="affiliation">Medical Research Council Laboratory of Molecular Biology, Cambridge, United Kingdom. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We report the discovery of a novel family of proteins, each member contains tandem pentapeptide (five residue) repeats, described by the motif A(D/N)LXX. Members of this family are both membrane bound and cytoplasmic. The function of these repeats is uncertain, but they may have a targeting or structural function rather than enzymatic activity. This family is most common in cyanobacteria, suggesting a function related to cyanobacterial-specific metabolism. Although no experimental information is available for the structure of this family, it is predicted that the tandem pentapeptide repeats will form a right-handed beta-helical structure. A structural model of the pentapeptide repeats is presented.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Protein science : a publication of the Protein Society</span>&nbsp;<span class="year">1998</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">1477-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9655353" rel="external">9655353</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2144021" rel="external">2144021</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fpro.5560070625" rel="external">10.1002/pro.5560070625</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-scale DNA sequencing: where are we?</h4>
<p class="authors"><span class="author">Beck S</span> and <span class="author">Sterk P</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK. <a href="mailto:%62%65%63%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#101;&#99;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Current opinion in biotechnology</span>&nbsp;<span class="year">1998</span>;<span class="volume">9</span>;<span class="number">1</span>;<span class="pagerange">116-21</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9503597" rel="external">9503597</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A detailed physical and transcriptional map of the region of chromosome 20 that is deleted in myeloproliferative disorders and refinement of the common deleted region.</h4>
<p class="authors"><span class="author">Bench AJ</span>, <span class="author">Aldred MA</span>, <span class="author">Humphray SJ</span>, <span class="author">Champion KM</span>, <span class="author">Gilbert JG</span>, <span class="author">Asimakopoulos FA</span>, <span class="author">Deloukas P</span>, <span class="author">Gwilliam R</span>, <span class="author">Bentley DR</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, University of Cambridge, United Kingdom.</p><p class="abstract">Acquired deletions of the long arm of chromosome 20 are the most common chromosomal abnormality seen in polycythemia vera and are also associated with other myeloid malignancies. Such deletions are believed to mark the site of one or more tumor suppressor genes, loss of which perturbs normal hematopoiesis. A common deleted region (CDR) has previously been identified on 20q. We have now constructed the most detailed physical map of this region to date--a YAC contig that encompasses the entire CDR and spans 23 cM (11 Mb). This contig contains 140 DNA markers and 65 unique expressed sequences. Our data represent a first step toward a complete transcriptional map of the CDR. The high marker density within the physical map permitted two complementary approaches to reducing the size of the CDR. Microsatellite PCR refined the centromeric boundary of the CDR to D20S465 and was used to search for homozygous deletions in 28 patients using 32 markers. No such deletions were detected. Genetic changes on the remaining chromosome 20 may therefore be too small to be detected or may occur in a subpopulation of cells.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1998</span>;<span class="volume">49</span>;<span class="number">3</span>;<span class="pagerange">351-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9615219" rel="external">9615219</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1998.5231" rel="external">10.1006/geno.1998.5231</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Coordination of human genome sequencing via a consensus framework map.</h4>
<p class="authors"><span class="author">Bentley DR</span>, <span class="author">Pruitt KD</span>, <span class="author">Deloukas P</span>, <span class="author">Schuler GD</span> and <span class="author">Ostell J</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK. <a href="mailto:%64%72%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#114;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">1998</span>;<span class="volume">14</span>;<span class="number">10</span>;<span class="pagerange">381-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9820023" rel="external">9820023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new candidate region for the positional cloning of the XLP gene.</h4>
<p class="authors"><span class="author">Bolino A</span>, <span class="author">Yin L</span>, <span class="author">Seri M</span>, <span class="author">Cusano R</span>, <span class="author">Cinti R</span>, <span class="author">Coffey A</span>, <span class="author">Brooksbank R</span>, <span class="author">Howell G</span>, <span class="author">Bentley D</span>, <span class="author">Davis JR</span>, <span class="author">Lanyi A</span>, <span class="author">Huang D</span>, <span class="author">Stark M</span>, <span class="author">Creaven M</span>, <span class="author">Bj&oslash;rkhaug L</span>, <span class="author">Heitzmann F</span>, <span class="author">Lamartine J</span>, <span class="author">Gaudi S</span>, <span class="author">Sylla BS</span>, <span class="author">Lenoir GM</span>, <span class="author">Castagnola E</span>, <span class="author">Giacchino R</span>, <span class="author">Porta G</span>, <span class="author">Franco B</span>, <span class="author">Zollo M</span>, <span class="author">Sumegi J</span> and <span class="author">Romeo G</span></p>
<p class="affiliation">International XLP Consortium, Laboratorio di Genetica Molecolare, Istituto Gaslini, Genova, Quarto, Italy.</p><p class="abstract">X-linked lymphoproliferative disease (XLP) is an inherited immunodeficiency characterised by selective susceptibility to Epstein-Barr virus and frequent association with malignant lymphomas chiefly located in the ileocecal region, liver, kidney and CNS. Taking advantage of a large bacterial clone contig, we obtained a genomic sequence of 197620 bp encompassing a deletion (XLP-D) of 116 kb in an XLP family, whose breakpoints were identified. The study of potential exons from this region in 40 unrelated XLP patients did not reveal any mutation. To define the critical region for XLP and investigate the role of the XLP-D deletion, detailed haplotypes in a region of approximately 20 cM were reconstructed in a total of 87 individuals from 7 families with recurrence of XLP. Two recombination events in a North American family and a new microdeletion (XLP-G) in an Italian family indicate that the XLP gene maps in the interval between DXS1001 and DXS8057, approximately 800 kb centromeric to the previously reported familial microdeletion XLP-D.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">1RO1 AI33532OIA3</span>; Telethon: <span class="grant">E.0633</span>, <span class="grant">TGM06S01</span></p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">1998</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">509-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9801876" rel="external">9801876</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5200249" rel="external">10.1038/sj.ejhg.5200249</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">UHX1 and PCTK1: precise characterisation and localisation within a gene-rich region in Xp11.23 and evaluation as candidate genes for retinal diseases mapped to Xp21.1-p11.2.</h4>
<p class="authors"><span class="author">Brandau O</span>, <span class="author">Nyakatura G</span>, <span class="author">Jedele KB</span>, <span class="author">Platzer M</span>, <span class="author">Achatz H</span>, <span class="author">Ross M</span>, <span class="author">Murken J</span>, <span class="author">Rosenthal A</span> and <span class="author">Meindl A</span></p>
<p class="affiliation">Abteilung f&uuml;r Medizinische Genetik, Kinderpoliklinik der Universit&auml;t, M&uuml;nchen, Germany.</p><p class="abstract">The gene for ubiquitin hydrolase on the X chromosome (UHX1), cloned and mapped to Xp21.2-p11.2, is a candidate gene for retinal diseases. We used fine mapping techniques to localise UHX1 between markers DXS1266 and DXS337, where congenital stationary night blindness (XICSNB) and retinitis pigmentosa type 2 (RP2) are also located. Reevaluation of the UHX1 gene structure demonstrated five new exons, for a total of 21 exons and a predicted protein product of 963 amino acids. Evaluation of patients revealed no UHX1 mutations using SSCP (10 CSNB1 and 20 XLRP) or deletion screening with cDNA hybridisation (13 CSNB1 and 43 XLRP). Likewise, no aberrations were found in the nearby PCTAIRE1 (PCTK1) gene in 13 CSNB1 and 43 XLRP patients by deletion screening. Thus mutations of UHX1, and probably PCTK1, do not appear to cause common X-linked eye diseases. UHX1&#39;s role in patients with mental retardation may be appropriate for further investigations into UHX1 function.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">1998</span>;<span class="volume">6</span>;<span class="number">5</span>;<span class="pagerange">459-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9801870" rel="external">9801870</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5200207" rel="external">10.1038/sj.ejhg.5200207</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships.</h4>
<p class="authors"><span class="author">Brenner SE</span>, <span class="author">Chothia C</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge CB2 2QH, United Kingdom. <a href="mailto:%62%72%65%6E%6E%65%72%40%68%79%70%65%72.%73%74%61%6E%66%6F%72%64.%65%64%75">&#98;&#114;&#101;&#110;&#110;&#101;&#114;&#64;&#104;&#121;&#112;&#101;&#114;&#46;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Pairwise sequence comparison methods have been assessed using proteins whose relationships are known reliably from their structures and functions, as described in the SCOP database [Murzin, A. G., Brenner, S. E., Hubbard, T. &amp; Chothia C. (1995) J. Mol. Biol. 247, 536-540]. The evaluation tested the programs BLAST [Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. (1990). J. Mol. Biol. 215, 403-410], WU-BLAST2 [Altschul, S. F. &amp; Gish, W. (1996) Methods Enzymol. 266, 460-480], FASTA [Pearson, W. R. &amp; Lipman, D. J. (1988) Proc. Natl. Acad. Sci. USA 85, 2444-2448], and SSEARCH [Smith, T. F. &amp; Waterman, M. S. (1981) J. Mol. Biol. 147, 195-197] and their scoring schemes. The error rate of all algorithms is greatly reduced by using statistical scores to evaluate matches rather than percentage identity or raw scores. The E-value statistical scores of SSEARCH and FASTA are reliable: the number of false positives found in our tests agrees well with the scores reported. However, the P-values reported by BLAST and WU-BLAST2 exaggerate significance by orders of magnitude. SSEARCH, FASTA ktup = 1, and WU-BLAST2 perform best, and they are capable of detecting almost all relationships between proteins whose sequence identities are &gt;30%. For more distantly related proteins, they do much less well; only one-half of the relationships between proteins with 20-30% identity are found. Because many homologs have low sequence similarity, most distant relationships cannot be detected by any pairwise comparison method; however, those which are identified may be used with confidence.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1998</span>;<span class="volume">95</span>;<span class="number">11</span>;<span class="pagerange">6073-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9600919">9600919</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC27587">27587</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Use of a Mycobacterium tuberculosis H37Rv bacterial artificial chromosome library for genome mapping, sequencing, and comparative genomics.</h4>
<p class="authors"><span class="author">Brosch R</span>, <span class="author">Gordon SV</span>, <span class="author">Billault A</span>, <span class="author">Garnier T</span>, <span class="author">Eiglmeier K</span>, <span class="author">Soravito C</span>, <span class="author">Barrell BG</span> and <span class="author">Cole ST</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bacteri&eacute;nne, Institut Pasteur, Paris, France.</p><p class="abstract">The bacterial artificial chromosome (BAC) cloning system is capable of stably propagating large, complex DNA inserts in Escherichia coli. As part of the Mycobacterium tuberculosis H37Rv genome sequencing project, a BAC library was constructed in the pBeloBAC11 vector and used for genome mapping, confirmation of sequence assembly, and sequencing. The library contains about 5,000 BAC clones, with inserts ranging in size from 25 to 104 kb, representing theoretically a 70-fold coverage of the M. tuberculosis genome (4.4 Mb). A total of 840 sequences from the T7 and SP6 termini of 420 BACs were determined and compared to those of a partial genomic database. These sequences showed excellent correlation between the estimated sizes and positions of the BAC clones and the sizes and positions of previously sequenced cosmids and the resulting contigs. Many BAC clones represent linking clones between sequenced cosmids, allowing full coverage of the H37Rv chromosome, and they are now being shotgun sequenced in the framework of the H37Rv sequencing project. Also, no chimeric, deleted, or rearranged BAC clones were detected, which was of major importance for the correct mapping and assembly of the H37Rv sequence. The minimal overlapping set contains 68 unique BAC clones and spans the whole H37Rv chromosome with the exception of a single gap of approximately 150 kb. As a postgenomic application, the canonical BAC set was used in a comparative study to reveal chromosomal polymorphisms between M. tuberculosis, M. bovis, and M. bovis BCG Pasteur, and a novel 12.7-kb segment present in M. tuberculosis but absent from M. bovis and M. bovis BCG was characterized. This region contains a set of genes whose products show low similarity to proteins involved in polysaccharide biosynthesis. The H37Rv BAC library therefore provides us with a powerful tool both for the generation and confirmation of sequence data as well as for comparative genomics and other postgenomic applications. It represents a major resource for present and future M. tuberculosis research projects.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">1998</span>;<span class="volume">66</span>;<span class="number">5</span>;<span class="pagerange">2221-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9573111" rel="external">9573111</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC108185" rel="external">108185</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic analysis of meiotic recombination in humans by use of sperm typing: reduced recombination within a heterozygous paracentric inversion of chromosome 9q32-q34.3.</h4>
<p class="authors"><span class="author">Brown GM</span>, <span class="author">Leversha M</span>, <span class="author">Hulten M</span>, <span class="author">Ferguson-Smith MA</span>, <span class="author">Affara NA</span> and <span class="author">Furlong RA</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, Cambridge, United Kingdom.</p><p class="abstract">To investigate patterns of genetic recombination within a heterozygous paracentric inversion of chromosome 9 (46XY inv[9] [q32q34.3]), we performed sperm typing using a series of polymorphic microsatellite markers spanning the inversion region. For comparison, two donors with cytogenetically normal chromosomes 9, one of whom was heterozygous for a pericentric chromosome 2 inversion (46XY inv[2] [p11q13]), were also tested. Linkage analysis was performed by use of the multilocus linkage-analysis program SPERM, and also CRI-MAP, which was adapted for sperm-typing data. Analysis of the controls generated a marker order in agreement with previously published data and revealed no significant interchromosomal effects of the inv(2) on recombination on chromosome 9. FISH employing cosmids containing appropriate chromosome 9 markers was used to localize the inversion breakpoint of inv(9). Analysis of inv(9) sperm was performed by use of a set of microsatellite markers that mapped centromeric to, telomeric to, and within the inversion breakpoints. Three distinct patterns of recombination across the region were observed. Proximal to the centromeric breakpoint, recombination was similar to normal levels. Distal to the telomeric breakpoint, there was an increase in recombination found in the inversion patient. Finally, within the inversion, recombination was dramatically reduced, but several apparent double recombinants were found. A putative model explaining these data is proposed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">62</span>;<span class="number">6</span>;<span class="pagerange">1484-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9585591" rel="external">9585591</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1377142" rel="external">1377142</a>; DOI: <a href="http://dx.doi.org/10.1086%2F301863" rel="external">10.1086/301863</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fulminant jejuno-ileitis following ablation of enteric glia in adult transgenic mice.</h4>
<p class="authors"><span class="author">Bush TG</span>, <span class="author">Savidge TC</span>, <span class="author">Freeman TC</span>, <span class="author">Cox HJ</span>, <span class="author">Campbell EA</span>, <span class="author">Mucke L</span>, <span class="author">Johnson MH</span> and <span class="author">Sofroniew MV</span></p>
<p class="affiliation">Medical Research Council Cambridge Centre for Brain Repair, Department of Anatomy, University of Cambridge, United Kingdom.</p><p class="abstract">To investigate the roles of astroglial cells, we targeted their ablation genetically. Transgenic mice were generated expressing herpes simplex virus thymidine kinase from the mouse glial fibrillary acidic protein (GFAP) promoter. In adult transgenic mice, 2 weeks of subcutaneous treatment with the antiviral agent ganciclovir preferentially ablated transgene-expressing, GFAP-positive glia from the jejunum and ileum, causing a fulminating and fatal jejuno-ileitis. This pathology was independent of bacterial overgrowth and was characterized by increased myeloperoxidase activity, moderate degeneration of myenteric neurons, and intraluminal hemorrhage. These findings demonstrate that enteric glia play an essential role in maintaining the integrity of the bowel and suggest that their loss or dysfunction may contribute to the cellular mechanisms of inflammatory bowel disease.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">1998</span>;<span class="volume">93</span>;<span class="number">2</span>;<span class="pagerange">189-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9568712" rel="external">9568712</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of the nematode C. elegans: a platform for investigating biology.</h4>
<p class="authors"><span class="author">C. elegans Sequencing Consortium</span></p><p class="abstract">The 97-megabase genomic sequence of the nematode Caenorhabditis elegans reveals over 19,000 genes. More than 40 percent of the predicted protein products find significant matches in other organisms. There is a variety of repeated sequences, both local and dispersed. The distinctive distribution of some repeats and highly conserved genes provides evidence for a regional organization of the chromosomes.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1998</span>;<span class="volume">282</span>;<span class="number">5396</span>;<span class="pagerange">2012-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9851916">9851916</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic nomenclature for Trypanosoma and Leishmania.</h4>
<p class="authors"><span class="author">Clayton C</span>, <span class="author">Adams M</span>, <span class="author">Almeida R</span>, <span class="author">Baltz T</span>, <span class="author">Barrett M</span>, <span class="author">Bastien P</span>, <span class="author">Belli S</span>, <span class="author">Beverley S</span>, <span class="author">Biteau N</span>, <span class="author">Blackwell J</span>, <span class="author">Blaineau C</span>, <span class="author">Boshart M</span>, <span class="author">Bringaud F</span>, <span class="author">Cross G</span>, <span class="author">Cruz A</span>, <span class="author">Degrave W</span>, <span class="author">Donelson J</span>, <span class="author">El-Sayed N</span>, <span class="author">Fu G</span>, <span class="author">Ersfeld K</span>, <span class="author">Gibson W</span>, <span class="author">Gull K</span>, <span class="author">Ivens A</span>, <span class="author">Kelly J</span>, <span class="author">Vanhamme L</span> <em>et al.</em></p>
<p class="affiliation">Zentrum f&uuml;r Molekulare Biologie, Heidelberg, Germany. <a href="mailto:%63%63%6C%61%79%74%6F%6E%40%73%75%6E%30.%75%72%7A.%75%6E%69-%68%65%69%64%65%6C%62%65%72%67.%64%65">&#99;&#99;&#108;&#97;&#121;&#116;&#111;&#110;&#64;&#115;&#117;&#110;&#48;&#46;&#117;&#114;&#122;&#46;&#117;&#110;&#105;&#45;&#104;&#101;&#105;&#100;&#101;&#108;&#98;&#101;&#114;&#103;&#46;&#100;&#101;</a></p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">1998</span>;<span class="volume">97</span>;<span class="number">1-2</span>;<span class="pagerange">221-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9879900" rel="external">9879900</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Host response to EBV infection in X-linked lymphoproliferative disease results from mutations in an SH2-domain encoding gene.</h4>
<p class="authors"><span class="author">Coffey AJ</span>, <span class="author">Brooksbank RA</span>, <span class="author">Brandau O</span>, <span class="author">Oohashi T</span>, <span class="author">Howell GR</span>, <span class="author">Bye JM</span>, <span class="author">Cahn AP</span>, <span class="author">Durham J</span>, <span class="author">Heath P</span>, <span class="author">Wray P</span>, <span class="author">Pavitt R</span>, <span class="author">Wilkinson J</span>, <span class="author">Leversha M</span>, <span class="author">Huckle E</span>, <span class="author">Shaw-Smith CJ</span>, <span class="author">Dunham A</span>, <span class="author">Rhodes S</span>, <span class="author">Schuster V</span>, <span class="author">Porta G</span>, <span class="author">Yin L</span>, <span class="author">Serafini P</span>, <span class="author">Sylla B</span>, <span class="author">Zollo M</span>, <span class="author">Franco B</span>, <span class="author">Bolino A</span>, <span class="author">Seri M</span>, <span class="author">Lanyi A</span>, <span class="author">Davis JR</span>, <span class="author">Webster D</span>, <span class="author">Harris A</span>, <span class="author">Lenoir G</span>, <span class="author">de St Basile G</span>, <span class="author">Jones A</span>, <span class="author">Behloradsky BH</span>, <span class="author">Achatz H</span>, <span class="author">Murken J</span>, <span class="author">Fassler R</span>, <span class="author">Sumegi J</span>, <span class="author">Romeo G</span>, <span class="author">Vaudin M</span>, <span class="author">Ross MT</span>, <span class="author">Meindl A</span> and <span class="author">Bentley DR</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK. <a href="mailto:%61%6A%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#106;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">X-linked lymphoproliferative syndrome (XLP or Duncan disease) is characterized by extreme sensitivity to Epstein-Barr virus (EBV), resulting in a complex phenotype manifested by severe or fatal infectious mononucleosis, acquired hypogammaglobulinemia and malignant lymphoma. We have identified a gene, SH2D1A, that is mutated in XLP patients and encodes a novel protein composed of a single SH2 domain. SH2D1A is expressed in many tissues involved in the immune system. The identification of SH2D1A will allow the determination of its mechanism of action as a possible regulator of the EBV-induced immune response.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">1 R01 AI33532-OIA3</span>; Telethon: <span class="grant">E.0440</span>, <span class="grant">TGM06S01</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">129-35</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9771704">9771704</a>; DOI: <a href="http://dx.doi.org/10.1038%2F2424">10.1038/2424</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of the genome of Mycobacterium tuberculosis H37Rv.</h4>
<p class="authors"><span class="author">Cole ST</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">Unit&eacute; de G&eacute;n&eacute;tique Mol&eacute;culaire Bact&eacute;rienne, Institut Pasteur, Paris, France.</p><p class="abstract">The powerful combination of genomics and bioinformatics is providing a wealth of information about Mycobacterium tuberculosis, the aetiological agent of human tuberculosis, that will facilitate the conception and development of new therapies. The starting point for genome sequencing was the integrated map of the 4.4 Mb circular chromosome of the widely used, virulent reference strain, M. tuberculosis H37Rv. Cosmids and bacterial artificial chromosomes were selected from ordered libraries and subjected to systematic shotgun sequence analysis. This approach simplified sequence assembly as the genome is rich in repetitive DNA. In common with most bacteria, &gt; 90% of the potential coding capacity is used, and probable or tentative functions could be attributed to &gt; 70% of the genes. The potential biological roles of two of the principal driving forces in genome dynamics, insertion sequence elements and polymorphic multigene families are discussed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Novartis Foundation symposium</span>&nbsp;<span class="year">1998</span>;<span class="volume">217</span>;<span class="pagerange">160-72; discussion 172-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9949807" rel="external">9949807</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deciphering the biology of Mycobacterium tuberculosis from the complete genome sequence.</h4>
<p class="authors"><span class="author">Cole ST</span>, <span class="author">Brosch R</span>, <span class="author">Parkhill J</span>, <span class="author">Garnier T</span>, <span class="author">Churcher C</span>, <span class="author">Harris D</span>, <span class="author">Gordon SV</span>, <span class="author">Eiglmeier K</span>, <span class="author">Gas S</span>, <span class="author">Barry CE</span>, <span class="author">Tekaia F</span>, <span class="author">Badcock K</span>, <span class="author">Basham D</span>, <span class="author">Brown D</span>, <span class="author">Chillingworth T</span>, <span class="author">Connor R</span>, <span class="author">Davies R</span>, <span class="author">Devlin K</span>, <span class="author">Feltwell T</span>, <span class="author">Gentles S</span>, <span class="author">Hamlin N</span>, <span class="author">Holroyd S</span>, <span class="author">Hornsby T</span>, <span class="author">Jagels K</span>, <span class="author">Krogh A</span>, <span class="author">McLean J</span>, <span class="author">Moule S</span>, <span class="author">Murphy L</span>, <span class="author">Oliver K</span>, <span class="author">Osborne J</span>, <span class="author">Quail MA</span>, <span class="author">Rajandream MA</span>, <span class="author">Rogers J</span>, <span class="author">Rutter S</span>, <span class="author">Seeger K</span>, <span class="author">Skelton J</span>, <span class="author">Squares R</span>, <span class="author">Squares S</span>, <span class="author">Sulston JE</span>, <span class="author">Taylor K</span>, <span class="author">Whitehead S</span> and <span class="author">Barrell BG</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, UK. <a href="mailto:%73%74%63%6F%6C%65%40%70%61%73%74%65%75%72.%66%72">&#115;&#116;&#99;&#111;&#108;&#101;&#64;&#112;&#97;&#115;&#116;&#101;&#117;&#114;&#46;&#102;&#114;</a></p><p class="abstract">Countless millions of people have died from tuberculosis, a chronic infectious disease caused by the tubercle bacillus. The complete genome sequence of the best-characterized strain of Mycobacterium tuberculosis, H37Rv, has been determined and analysed in order to improve our understanding of the biology of this slow-growing pathogen and to help the conception of new prophylactic and therapeutic interventions. The genome comprises 4,411,529 base pairs, contains around 4,000 genes, and has a very high guanine + cytosine content that is reflected in the biased amino-acid content of the proteins. M. tuberculosis differs radically from other bacteria in that a very large portion of its coding capacity is devoted to the production of enzymes involved in lipogenesis and lipolysis, and to two new families of glycine-rich proteins with a repetitive structure that may represent a source of antigenic variation.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">Z01 AI000783-11</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1998</span>;<span class="volume">393</span>;<span class="number">6685</span>;<span class="pagerange">537-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9634230">9634230</a>; DOI: <a href="http://dx.doi.org/10.1038%2F31159">10.1038/31159</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence assembly with CAFTOOLS.</h4>
<p class="authors"><span class="author">Dear S</span>, <span class="author">Durbin R</span>, <span class="author">Hillier L</span>, <span class="author">Marth G</span>, <span class="author">Thierry-Mieg J</span> and <span class="author">Mott R</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Large-scale genomic sequencing requires a software infrastructure to support and integrate applications that are not directly compatible. We describe a suite of software tools built around the Common Assembly Format (CAF), a comprehensive representation of a sequence assembly as a text file. These tools form the backbone of sequencing informatics at the Sanger Centre and the Genome Sequencing Center. The CAF format is intentionally flexible, and our Perl and C libraries, which parse and manipulate it, provide powerful tools for creating new applications as well as wrappers to incorporate other software. The tools are available free by anonymous FTP from ftp://ftp.sanger.ac.uk/pub/badger/.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">260-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9521929" rel="external">9521929</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310697" rel="external">310697</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A physical map of 30,000 human genes.</h4>
<p class="authors"><span class="author">Deloukas P</span>, <span class="author">Schuler GD</span>, <span class="author">Gyapay G</span>, <span class="author">Beasley EM</span>, <span class="author">Soderlund C</span>, <span class="author">Rodriguez-Tom&eacute; P</span>, <span class="author">Hui L</span>, <span class="author">Matise TC</span>, <span class="author">McKusick KB</span>, <span class="author">Beckmann JS</span>, <span class="author">Bentolila S</span>, <span class="author">Bihoreau M</span>, <span class="author">Birren BB</span>, <span class="author">Browne J</span>, <span class="author">Butler A</span>, <span class="author">Castle AB</span>, <span class="author">Chiannilkulchai N</span>, <span class="author">Clee C</span>, <span class="author">Day PJ</span>, <span class="author">Dehejia A</span>, <span class="author">Dibling T</span>, <span class="author">Drouot N</span>, <span class="author">Duprat S</span>, <span class="author">Fizames C</span>, <span class="author">Fox S</span>, <span class="author">Gelling S</span>, <span class="author">Green L</span>, <span class="author">Harrison P</span>, <span class="author">Hocking R</span>, <span class="author">Holloway E</span>, <span class="author">Hunt S</span>, <span class="author">Keil S</span>, <span class="author">Lijnzaad P</span>, <span class="author">Louis-Dit-Sully C</span>, <span class="author">Ma J</span>, <span class="author">Mendis A</span>, <span class="author">Miller J</span>, <span class="author">Morissette J</span>, <span class="author">Muselet D</span>, <span class="author">Nusbaum HC</span>, <span class="author">Peck A</span>, <span class="author">Rozen S</span>, <span class="author">Simon D</span>, <span class="author">Slonim DK</span>, <span class="author">Staples R</span>, <span class="author">Stein LD</span>, <span class="author">Stewart EA</span>, <span class="author">Suchard MA</span>, <span class="author">Thangarajah T</span>, <span class="author">Vega-Czarny N</span>, <span class="author">Webber C</span>, <span class="author">Wu X</span>, <span class="author">Hudson J</span>, <span class="author">Auffray C</span>, <span class="author">Nomura N</span>, <span class="author">Sikela JM</span>, <span class="author">Polymeropoulos MH</span>, <span class="author">James MR</span>, <span class="author">Lander ES</span>, <span class="author">Hudson TJ</span>, <span class="author">Myers RM</span>, <span class="author">Cox DR</span>, <span class="author">Weissenbach J</span>, <span class="author">Boguski MS</span> and <span class="author">Bentley DR</span></p>
<p class="affiliation">Sanger Centre, Hinxton Hall, Hinxton, Cambridge CB10 1SA UK.</p><p class="abstract">A map of 30,181 human gene-based markers was assembled and integrated with the current genetic map by radiation hybrid mapping. The new gene map contains nearly twice as many genes as the previous release, includes most genes that encode proteins of known function, and is twofold to threefold more accurate than the previous version. A redesigned, more informative and functional World Wide Web site (www.ncbi.nlm.nih.gov/genemap) provides the mapping information and associated data and annotations. This resource constitutes an important infrastructure and tool for the study of complex genetic traits, the positional cloning of disease genes, the cross-referencing of mammalian genomes, and validated human transcribed sequences for large-scale studies of gene expression.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1998</span>;<span class="volume">282</span>;<span class="number">5389</span>;<span class="pagerange">744-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9784132">9784132</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression profiling of single cells using 3 prime end amplification (TPEA) PCR.</h4>
<p class="authors"><span class="author">Dixon AK</span>, <span class="author">Richardson PJ</span>, <span class="author">Lee K</span>, <span class="author">Carter NP</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">The Sanger Centre, The Wellcome Genome Campus, Hinxton, Cambridge CB10 1SA, UK,Department of Pharmacology, University of Cambridge, Tennis Court Road, Cambridge CB2 1QJ, UK.</p><p class="abstract">The ability to relate the physiological status of individual cells to the complement of genes they express is limited by current methodological approaches for performing these analyses. We report here the development of a robust and reproducible method for amplifying 3&#39; sequences of mRNA derived from single cells and demonstrate that the amplified cDNA, derived from individual human lymphoblastoma cells, can be used for the expression profiling of up to 40 different genes per cell. In addition, we show that 3 prime end amplification (TPEA) PCR can be used to enable the detection of both high and low abundance mRNA species in samples harvested from live neurons in rat brain slices. This procedure will facilitate the study of complex tissue function at the cellular level.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1998</span>;<span class="volume">26</span>;<span class="number">19</span>;<span class="pagerange">4426-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9742245" rel="external">9742245</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC147873" rel="external">147873</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data disclosure in the human genome project.</h4>
<p class="authors"><span class="author">Dunham I</span></p>
<p class="publication"><span class="title">Molecular medicine today</span>&nbsp;<span class="year">1998</span>;<span class="volume">4</span>;<span class="number">8</span>;<span class="pagerange">335-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9755451" rel="external">9755451</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Base qualities help sequencing software.</h4>
<p class="authors"><span class="author">Durbin R</span> and <span class="author">Dear S</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%72%64%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#100;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">3</span>;<span class="pagerange">161-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9521917" rel="external">9521917</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An oligo-screening strategy to fill gaps found during shotgun sequencing projects.</h4>
<p class="authors"><span class="author">Flint J</span>, <span class="author">Sims M</span>, <span class="author">Clark K</span>, <span class="author">Staden R</span> and <span class="author">Thomas K</span></p>
<p class="affiliation">Institute of Molecular Medicine, John Radcliffe Hospital, Headington, Oxford, England.</p><p class="abstract">During the course of projects to sequence human and nematode cosmids we encountered great difficulties generating contiguous sequence in regions with repetitive DNA (Alu repeats in humans and tandem or inverted repeats in the nematode). We have developed a simple and efficient strategy to fill gaps. By screening M13, plasmid or phagemid libraries with oligonucleotides flanking the gap, clones are identified that contiguate the cosmid sequence. Our method has been integrated into the GAP4 sequence assembly program. The strategy reduces both time and costs in large scale sequencing projects.</p>
<p class="publication"><span class="title">DNA sequence : the journal of DNA sequencing and mapping</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">4</span>;<span class="pagerange">241-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10520453" rel="external">10520453</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Saccharomyces cerevisiae early secretion mutant tip20 is synthetic lethal with mutants in yeast coatomer and the SNARE proteins Sec22p and Ufe1p.</h4>
<p class="authors"><span class="author">Frigerio G</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK.</p><p class="abstract">Tip20p is an 80 kDa cytoplasmic protein bound to the cytoplasmic surface of the endoplasmic reticulum (ER) by interaction with the type II integral membrane protein Sec20p. Both proteins are required for vesicular transport between the ER and Golgi complex. Recently, sec20-1 was found to be defective in retrograde transport. A collection of temperature-sensitive tip20 mutants are shown to be lethal in combination with ufe1-1, a target SNARE of the ER and ret2-1, yeast delta-COP. A subset of tip20 mutants was found to be lethal in combination with sec20-1, sec21-1, sec22-3 and sec27-1. Since all pairwise combinations of a tip20 mutant, sec20-1, and ufe1-1 are lethal, Tip20p and Sec20p might be part of the docking complex for Golgi-derived retrograde transport vesicles. Since carboxy-terminal tip20 truncations are lethal in combination with mutants in three coatomer subunits, Tip20p might be involved in binding or uncoating of COPI coated retrograde transport vesicles.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Yeast (Chichester, England)</span>&nbsp;<span class="year">1998</span>;<span class="volume">14</span>;<span class="number">7</span>;<span class="pagerange">633-46</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9639310" rel="external">9639310</a>; DOI: <a href="http://dx.doi.org/10.1002%2F(SICI)1097-0061(199805)14%3A7%3C633%3A%3AAID-YEA267%3E3.0.CO%3B2-3" rel="external">10.1002/(SICI)1097-0061(199805)14:7&lt;633::AID-YEA267&gt;3.0.CO;2-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The pufferfish SLP-1 gene, a new member of the SCL/TAL-1 family of transcription factors.</h4>
<p class="authors"><span class="author">G&ouml;ttgens B</span>, <span class="author">Gilbert JG</span>, <span class="author">Barton LM</span>, <span class="author">Aparicio S</span>, <span class="author">Hawker K</span>, <span class="author">Mistry S</span>, <span class="author">Vaudin M</span>, <span class="author">King A</span>, <span class="author">Bentley D</span>, <span class="author">Elgar G</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, MRC Centre, University of Cambridge, United Kingdom.</p><p class="abstract">The SCL/TAL-1 gene encodes a basic helix-loop-helix (bHLH) transcription factor essential for the development of all hemopoietic lineages and also acts as a T-cell oncogene. Four related genes have been described in mammals (LYL-1, TAL-2, NSCL1, and NSCL2), all of which exhibit a high degree of sequence similarity to SCL/TAL-1 in the bHLH domain and two of which (LYL-1 and TAL-2) have also been implicated in the pathogenesis of T-cell acute lymphoblastic leukemia. In this study we describe the identification and characterization of a pufferfish gene termed SLP-1, which represents a new member of this gene family. The genomic structure and sequence of SLP-1 suggests that it forms a subfamily with SCL/TAL-1 and LYL-1 and is most closely related to SCL/TAL-1. However, unlike SCL/TAL-1, SLP-1 is widely expressed. Sequence analysis of a whole cosmid containing SLP-1 shows that SLP-1 is flanked upstream by a zinc finger gene and a fork-head-domain gene and downstream by a heme-oxygenase and a RING finger gene.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1998</span>;<span class="volume">48</span>;<span class="number">1</span>;<span class="pagerange">52-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9503016" rel="external">9503016</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1997.5162" rel="external">10.1006/geno.1997.5162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Statement on the rapid release of genomic DNA sequence.</h4>
<p class="authors"><span class="author">Guyer M</span></p>
<p class="affiliation">National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA. <a href="mailto:%6D%67%32%35%6D%40%6E%69%68.%67%6F%76">&#109;&#103;&#50;&#53;&#109;&#64;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">413</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9582183" rel="external">9582183</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TAPASIN, DAXX, RGL2, HKE2 and four new genes (BING 1, 3 to 5) form a dense cluster at the centromeric end of the MHC.</h4>
<p class="authors"><span class="author">Herberg JA</span>, <span class="author">Beck S</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Human Immunogenetics Laboratory, Imperial Cancer Research Fund, 44 Lincoln&#39;s Inn Fields, London, U.K.</p><p class="abstract">TAPASIN, a gene recently shown to be required for antigen presentation through MHC class I molecules, is located 180 kbp centromeric of HLA-DP in a region linked to several diseases, and associated with altered developmental phenotypes in the mouse. We present the genomic analysis of a 70 kbp gene-dense segment flanking the TAPASIN locus, including sequence, structure and preliminary characterisation of seven additional genes. BING1 is a Zn finger gene containing a POZ motif. BING3 is similar to myosin regulatory light chain. BING4 shows homologies only to hypothetical yeast and Caenorhabditis elegans proteins. BING5 is found within an intron of BING4 on the complementary strand, and encodes a molecule with no homologies to database proteins. Another three genes were identified whose full sequence was not previously known; namely, RGL2, DAXX (BING2) and HKE2. RGL2 encodes an effector of Ras, homologous to the mouse RalGDS protein, Rlf. DAXX encodes an effector of Fas that stimulates apoptosis through the Jun kinase (JNK) pathway. The location of DAXX is of interest given the linkage of autoimmune disease to the MHC and to apoptosis.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1998</span>;<span class="volume">277</span>;<span class="number">4</span>;<span class="pagerange">839-57</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9545376" rel="external">9545376</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1998.1637" rel="external">10.1006/jmbi.1998.1637</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic structure and domain organisation of the human Bak gene.</h4>
<p class="authors"><span class="author">Herberg JA</span>, <span class="author">Phillips S</span>, <span class="author">Beck S</span>, <span class="author">Jones T</span>, <span class="author">Sheer D</span>, <span class="author">Wu JJ</span>, <span class="author">Prochazka V</span>, <span class="author">Barr PJ</span>, <span class="author">Kiefer MC</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Imperial Cancer Research Fund Laboratories, 44 Lincoln&#39;s Inn Fields, London, UK.</p><p class="abstract">The Bcl-2 homologue, Bak, is a potent inducer of apoptosis. FISH data presented here located the gene to 6p21.3. Mapping was consistent with its location centromeric of the HSET locus and approximately 400kb from the MHC. The construction of a contig of genomic clones across the locus facilitated the sequencing of a PAC containing the gene. Comparison of the gene structure to functional and physical domains revealed a good agreement between the physical structure and the intron-exon organisation. The position of a single intron was conserved in comparison to other members of the Bcl-2 family, namely Bax, CED-9, Bcl-X and Bcl-2, but all other introns were displaced, consistent with a divergent phylogeny.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3585</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">1998</span>;<span class="volume">211</span>;<span class="number">1</span>;<span class="pagerange">87-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9573342" rel="external">9573342</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic analysis of the Tapasin gene, located close to the TAP loci in the MHC.</h4>
<p class="authors"><span class="author">Herberg JA</span>, <span class="author">Sgouros J</span>, <span class="author">Jones T</span>, <span class="author">Copeman J</span>, <span class="author">Humphray SJ</span>, <span class="author">Sheer D</span>, <span class="author">Cresswell P</span>, <span class="author">Beck S</span> and <span class="author">Trowsdale J</span></p>
<p class="affiliation">Imperial Cancer Research Fund Laboratories, London, GB.</p><p class="abstract">The Tapasin molecule is a member of the immunoglobulin (Ig) superfamily required for the association of TAP transporters and MHC class I heterodimers in the endoplasmic reticulum. In this study, the Tapasin gene was precisely mapped in relation to the MHC. The gene was centromeric of the HLA-DP locus between the HSET and HKE1.5 genes and within 500 kbp of the TAP1 and TAP2 genes. A homologous mouse EST was mapped to a syntenic position on chromosome 17, centromeric of the H-2 K locus. Similarly, the rat Tapasin gene was shown to be in an equivalent location with respect to the RT1.A locus. The localization of Tapasin, TAP, LMP and class I genes within such a short distance of each other on the chromosome implies some regulatory or functional significance. We determined the Tapasin gene sequence for comparison of its structure to that of other Ig superfamily members, such as MHC class I genes. The IgC domain was encoded by a separate exon. However, the positions of the other introns were not characteristic of other Ig superfamily genes, indicating that Tapasin has a distinct phylogeny.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A3585</span>; Wellcome Trust</p>
<p class="publication"><span class="title">European journal of immunology</span>&nbsp;<span class="year">1998</span>;<span class="volume">28</span>;<span class="number">2</span>;<span class="pagerange">459-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9521053" rel="external">9521053</a>; DOI: <a href="http://dx.doi.org/10.1002%2F(SICI)1521-4141(199802)28%3A02%26%2360%3B459%3A%3AAID-IMMU459%26%2362%3B3.0.CO%3B2-Z" rel="external">10.1002/(SICI)1521-4141(199802)28:02&amp;#60;459::AID-IMMU459&amp;#62;3.0.CO;2-Z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dynamic programming alignment accuracy.</h4>
<p class="authors"><span class="author">Holmes I</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge, England. <a href="mailto:%69%68%68,%72%64%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#104;&#104;&#44;&#114;&#100;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Algorithms for generating alignments of biological sequences have inherent statistical limitations when it comes to the accuracy of the alignments they produce. Using simulations, we measure the accuracy of the standard global dynamic programming method and show that it can be reasonably well modelled by an &quot;edge wander&quot; approximation to the distribution of the optimal scoring path around the correct path in the vicinity of a gap. We also give a table from which accuracy values can be predicted for commonly used scoring schemes and sequence divergences (the PAM and BLOSUM series). Finally we describe how to calculate the expected accuracy of a given alignment, and show how this can be used to construct an optimal accuracy alignment algorithm which generates significantly more accurate alignments than standard dynamic programming methods in simulated experiments.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">1998</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">493-504</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9773345" rel="external">9773345</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Large-scale sequence comparisons reveal unusually high levels of variation in the HLA-DQB1 locus in the class II region of the human MHC.</h4>
<p class="authors"><span class="author">Horton R</span>, <span class="author">Niblett D</span>, <span class="author">Milne S</span>, <span class="author">Palmer S</span>, <span class="author">Tubby B</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, CB10 1SA, UK.</p><p class="abstract">Comparison of genomic sequences flanking the HLA-DQB1 locus in the human MHC class II region reveals local sequence variation of up to 10%, which is the highest level of sequence variation found in the human genome so far. The variation is haplotype-specific and extends far beyond the transcriptional unit of the DQB1 gene, suggesting hitch-hiking along with functionally selected alleles as the most likely mechanism. All major insertions/deletions (indels) were found to be of retroviral origin and in the immediate upstream region of DQB1. Possible cis-acting effects of these indels on the transcriptional regulation of DQB1 are discussed.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1998</span>;<span class="volume">282</span>;<span class="number">1</span>;<span class="pagerange">71-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9733642" rel="external">9733642</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1998.2018" rel="external">10.1006/jmbi.1998.2018</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCOP, Structural Classification of Proteins database: applications to evaluation of the effectiveness of sequence alignment methods and statistics of protein structural data.</h4>
<p class="authors"><span class="author">Hubbard TJ</span>, <span class="author">Ailey B</span>, <span class="author">Brenner SE</span>, <span class="author">Murzin AG</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, England. <a href="mailto:%74%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Structural Classification of Proteins (SCOP) database provides a detailed and comprehensive description of the relationships of all known protein structures. The classification is on hierarchical levels: the first two levels, family and superfamily, describe near and far evolutionary relationships; the third, fold, describes geometrical relationships. The distinction between evolutionary relationships and those that arise from the physics and chemistry of proteins is a feature that is unique to this database, so far. The database can be used as a source of data to calibrate sequence search algorithms and for the generation of population statistics on protein structures. The database and its associated files are freely accessible from a number of WWW sites mirrored from URL http://scop. mrc-lmb.cam.ac.uk/scop/.</p>
<p class="publication"><span class="title">Acta crystallographica. Section D, Biological crystallography</span>&nbsp;<span class="year">1998</span>;<span class="volume">54</span>;<span class="number">Pt 6 Pt 1</span>;<span class="pagerange">1147-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/10089491">10089491</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the human synaptogyrin gene family.</h4>
<p class="authors"><span class="author">Kedra D</span>, <span class="author">Pan HQ</span>, <span class="author">Seroussi E</span>, <span class="author">Fransson I</span>, <span class="author">Guilbaud C</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Blennow E</span>, <span class="author">Roe BA</span>, <span class="author">Piehl F</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Hospital, Stockholm, Sweden.</p><p class="abstract">Genomic sequencing was combined with searches of databases for identification of active genes on human chromosome 22. A cosmid from 22q13, located in the telomeric vicinity of the PDGFB (platelet-derived growth factor B-chain) gene, was fully sequenced. Using an expressed sequence tag-based approach we characterized human (SYNGR1) and mouse (Syngr1) orthologs of the previously cloned rat synaptogyrin gene (RATSYNGR1). The human SYNGR1 gene reveals three (SYNGR1a, SYNGR1b, SYNGR1c) alternative transcript forms of 4.5, 1.3 and 0.9 kb, respectively. The transcription of SYNGR1 starts from two different promoters, and leads to predicted proteins with different N- and C-terminal ends. The most abundant SYNGR1 a transcript, the 4.5-kb form, which corresponds to RATSYNGR1, is highly expressed in neurons of the central nervous system and at much lower levels in other tissues, as determined by in situ hybridization histochemistry. The levels of SYNGR1b and SYNGR1c transcripts are low and limited to heart, skeletal muscle, ovary and fetal liver. We also characterized two additional members of this novel synaptogyrin gene family in human (SYNGR2 and SYNGR3), and one in mouse (Syngr2). The human SYNGR2 gene transcript of 1.6 kb is expressed at high levels in all tissues, except brain. The 2.2-kb SYNGR3 transcript was detected in brain and placenta only. The human SYNGR2 and SYNGR3 genes were mapped by fluorescence in situ hybridization to 17qtel and 16ptel, respectively. The human SYNGR2 gene has a processed pseudogene localized in 15q11. All predicted synaptogyrin proteins contain four strongly conserved transmembrane domains, which is consistent with the M-shaped topology. The C-terminal polypeptide ends are variable in length, display a low degree of sequence similarity between family members, and are therefore likely to convey the functional specificity of each protein.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">103</span>;<span class="number">2</span>;<span class="pagerange">131-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9760194" rel="external">9760194</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Detection of gains and losses in 18 meningiomas by comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Khan J</span>, <span class="author">Parsa NZ</span>, <span class="author">Harada T</span>, <span class="author">Meltzer PS</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">Laboratory of Cancer Genetics, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892-4470, USA.</p><p class="abstract">Comparative genomic hybridization (CGH) was used to examine gains and losses in 18 meningioma tumors that had been previously analyzed for loss of heterozygosity (LOH) at 22q12. Partial or complete losses were seen by CGH in only 9 of 18 cases on chromosome 22. This compares with 11 of 18 losses of single or more loci by LOH. The discrepancy in these results in probably explained by the increased sensitivity of LOH by using microsatellite markers that are able to detect small deletions, whereas losses on the order of 10-15 megabases are required for confident identification by CGH. There was no consistent pattern of gains or losses by CGH, including those tumors that lacked LOH at 22q12. In one tumor of interest in which CGH and LOH studies failed to demonstrate loss on chromosome 22, CGH identified an area of amplification at 17q22-23.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Cancer genetics and cytogenetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">103</span>;<span class="number">2</span>;<span class="pagerange">95-100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9614906" rel="external">9614906</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of an ATP-sensitive potassium channel current in rat striatal cholinergic interneurones.</h4>
<p class="authors"><span class="author">Lee K</span>, <span class="author">Dixon AK</span>, <span class="author">Freeman TC</span> and <span class="author">Richardson PJ</span></p>
<p class="affiliation">Parke Davis Neuroscience Research Centre, Cambridge University Forvie Site, UK. <a href="mailto:%6B%65%76%69%6E.%6C%65%65%40%63%61%6D%62.%77%6C.%63%6F%6D">&#107;&#101;&#118;&#105;&#110;&#46;&#108;&#101;&#101;&#64;&#99;&#97;&#109;&#98;&#46;&#119;&#108;&#46;&#99;&#111;&#109;</a></p><p class="abstract">1. Whole-cell patch-clamp recordings were made from rat striatal cholinergic interneurones in slices of brain tissue in vitro. In the absence of ATP in the electrode solution, these neurones were found to gradually hyperpolarize through the induction of an outward current at -60 mV. This outward current and the resultant hyperpolarization were blocked by the sulphonylureas tolbutamide and glibenclamide and by the photorelease of caged ATP within neurones. 2. This ATP-sensitive outward current was not observed when 2 mM ATP was present in the electrode solution. Under these conditions, 500 microM diazoxide was found to induce an outward current that was blocked by tolbutamide. 3. Using permeabilized patch recordings, neurones were shown to hyperpolarize in response to glucose deprivation or metabolic poisoning with sodium azide (NaN3). The resultant hyperpolarization was blocked by tolbutamide. 4. In cell-attached recordings, metabolic inhibition with 1 mM NaN3 revealed the presence of a tolbutamide-sensitive channel exhibiting a unitary conductance of 44.1 pS. 5. Reverse transcription followed by the polymerase chain reaction using cytoplasm from single cholinergic interneurones demonstrated the expression of the ATP-sensitive potassium (KATP) channel subunits Kir6.1 and SUR1 but not Kir6.2 or SUR2. 6. It is concluded that cholinergic interneurones within the rat striatum exhibit a KATP channel current and that this channel is formed from Kir6.1 and SUR1 subunits.</p>
<p class="publication"><span class="title">The Journal of physiology</span>&nbsp;<span class="year">1998</span>;<span class="volume">510 ( Pt 2)</span>;<span class="pagerange">441-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9705995" rel="external">9705995</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2231058" rel="external">2231058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GLASS: a tool to visualize protein structure prediction data in three dimensions and evaluate their consistency.</h4>
<p class="authors"><span class="author">Leplae R</span>, <span class="author">Hubbard T</span> and <span class="author">Tramontano A</span></p>
<p class="affiliation">Istituto di Ricerche di Biologia Molecolare, P. Angeletti, Pomezia, Italy.</p><p class="abstract">When a protein sequence does not share any significant sequence similarity with a protein of known structure, homology modeling cannot be applied. However, many novel and interesting methods, such as secondary structure prediction, fold recognition, and prediction of long-range interactions, are being developed and have been shown to be reasonably successful in predicting protein structures from sequence data and evolutionary information. The a priori evaluation of the correctness of a prediction obtained by one of these methods is however often problematic. Consequently, it is important to use all available information provided by as many different methods as possible and all the available experimental data about the protein of interest, since the consistency of the results is indicative of the reliability of the prediction. Hence the need has arisen for suitable tools able to compare results provided by different methods and evaluate their consistency. We have therefore constructed GLASS, a general platform to read, visualize, compare, and evaluate prediction results from many different sources and to project these prediction results into three dimensions. In addition, GLASS allows the comparison of selected parameters calculated for a model with the distribution observed in real protein structures, thus providing an easy way to test new methods for evaluating the likelihood of different structural models. GLASS can be considered as a &quot;workbench&quot; for structural predictions useful to both experimentalists and theoreticians.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proteins</span>&nbsp;<span class="year">1998</span>;<span class="volume">30</span>;<span class="number">4</span>;<span class="pagerange">339-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9533618">9533618</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transmission disequilibrium analysis of a triplet repeat within the hKCa3 gene using family trios with schizophrenia.</h4>
<p class="authors"><span class="author">Li T</span>, <span class="author">Hu X</span>, <span class="author">Chandy KG</span>, <span class="author">Fantino E</span>, <span class="author">Kalman K</span>, <span class="author">Gutman G</span>, <span class="author">Gargus JJ</span>, <span class="author">Freeman B</span>, <span class="author">Murray RM</span>, <span class="author">Dawson E</span>, <span class="author">Liu X</span>, <span class="author">Bruinvels AT</span>, <span class="author">Sham PC</span> and <span class="author">Collier DA</span></p>
<p class="affiliation">Department of Psychological Medicine and Centre for Social, Genetic and Developmental Psychiatry, The Institute of Psychiatry, De Crespigny Park, Denmark Hill, SE5 8AF, United Kingdom.</p><p class="abstract">hKCa3 is a neuronal small conductance calcium-activated potassium channel which contains a polyglutamine tract, encoded by a polymorphic CAG repeat in the gene. Since an association between longer alleles of the CAG repeat and schizophrenia has been reported, we performed haplotype-based haplotype relative risk (HHRR) and transmission disequilibrium (TDT) in 97 family trios with schizophrenia from SW China. We found no evidence for an excess of longer CAG repeats in the patients, and the ETDT test was not significant for either allele-wise (P = 0.31) or genotype-wise analysis (P = 0.18). However, there was a deficit of transmission of the (CAG)20 repeat allele to affected offspring when this allele was considered individually by TDT (P = 0.012; not corrected for multiple testing). These data do not support a role for larger alleles at the hKCa3 locus in psychosis in Chinese subjects.</p>
<p class="publication"><span class="title">Biochemical and biophysical research communications</span>&nbsp;<span class="year">1998</span>;<span class="volume">251</span>;<span class="number">2</span>;<span class="pagerange">662-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9792831" rel="external">9792831</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fbbrc.1998.9484" rel="external">10.1006/bbrc.1998.9484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Short-insert libraries as a method of problem solving in genome sequencing.</h4>
<p class="authors"><span class="author">McMurray AA</span>, <span class="author">Sulston JE</span> and <span class="author">Quail MA</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">As the Human Genome Project moves into its sequencing phase, a serious problem has arisen. The same problem has been increasingly vexing in the closing phase of the Caenorhabditis elegans project. The difficulty lies in sequencing efficiently through certain regions in which the templates (DNA substrates for the sequencing process) form complex folded secondary structures that are inaccessible to the enzymes. The solution, however, is simply to break them up. Specifically, the offending fragments are sonicated heavily and recloned, as much smaller fragments, into pUC vector. The sequences obtained from the resulting library can subsequently be assembled, free from the effects of secondary structure, to produce high-quality, complete sequence. Because of the success and simplicity of this procedure, we have begun to use it for the sequencing of all regions in which standard primer walking has been at all difficult.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">562-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9582200" rel="external">9582200</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310723" rel="external">310723</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in a gene encoding a novel protein tyrosine phosphatase cause progressive myoclonus epilepsy.</h4>
<p class="authors"><span class="author">Minassian BA</span>, <span class="author">Lee JR</span>, <span class="author">Herbrick JA</span>, <span class="author">Huizenga J</span>, <span class="author">Soder S</span>, <span class="author">Mungall AJ</span>, <span class="author">Dunham I</span>, <span class="author">Gardner R</span>, <span class="author">Fong CY</span>, <span class="author">Carpenter S</span>, <span class="author">Jardim L</span>, <span class="author">Satishchandra P</span>, <span class="author">Andermann E</span>, <span class="author">Snead OC</span>, <span class="author">Lopes-Cendes I</span>, <span class="author">Tsui LC</span>, <span class="author">Delgado-Escueta AV</span>, <span class="author">Rouleau GA</span> and <span class="author">Scherer SW</span></p>
<p class="affiliation">Department of Genetics, The Hospital for Sick Children, University of Toronto, Ontario, Canada.</p><p class="abstract">Lafora&#39;s disease (LD; OMIM 254780) is an autosomal recessive form of progressive myoclonus epilepsy characterized by seizures and cumulative neurological deterioration. Onset occurs during late childhood and usually results in death within ten years of the first symptoms. With few exceptions, patients follow a homogeneous clinical course despite the existence of genetic heterogeneity. Biopsy of various tissues, including brain, revealed characteristic polyglucosan inclusions called Lafora bodies, which suggested LD might be a generalized storage disease. Using a positional cloning approach, we have identified at chromosome 6q24 a novel gene, EPM2A, that encodes a protein with consensus amino acid sequence indicative of a protein tyrosine phosphatase (PTP). mRNA transcripts representing alternatively spliced forms of EPM2A were found in every tissue examined, including brain. Six distinct DNA sequence variations in EPM2A in nine families, and one homozygous microdeletion in another family, have been found to cosegregate with LD. These mutations are predicted to cause deleterious effects in the putative protein product, named laforin, resulting in LD.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">5P01-NS21908</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">171-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9771710">9771710</a>; DOI: <a href="http://dx.doi.org/10.1038%2F2470">10.1038/2470</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Alternative splicing, genomic structure, and fine chromosome localization of REV3L.</h4>
<p class="authors"><span class="author">Morelli C</span>, <span class="author">Mungall AJ</span>, <span class="author">Negrini M</span>, <span class="author">Barbanti-Brodano G</span> and <span class="author">Croce CM</span></p>
<p class="affiliation">Department of Diagnostic and Experimental Medicine, Section of Microbiology, University of Ferrara, Ferrara (Italy).</p><p class="abstract">We have localized a human homolog, REV3L, of the Saccharomyces cerevisiae REV3 gene on chromosome region 6q21. The full-length cDNA consists of 10,919 nucleotides, with a putative open reading frame of 9,159 bp for a predicted protein of 3,053 amino acids. The gene contains 33 exons in about 200 kb of genomic DNA. In contrast to the previously reported sequence, an additional exon and an alternative splicing site are demonstrated.</p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">83</span>;<span class="number">1-2</span>;<span class="pagerange">18-20</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9925914" rel="external">9925914</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inversin, a novel gene in the vertebrate left-right axis pathway, is partially deleted in the inv mouse.</h4>
<p class="authors"><span class="author">Morgan D</span>, <span class="author">Turnpenny L</span>, <span class="author">Goodship J</span>, <span class="author">Dai W</span>, <span class="author">Majumder K</span>, <span class="author">Matthews L</span>, <span class="author">Gardner A</span>, <span class="author">Schuster G</span>, <span class="author">Vien L</span>, <span class="author">Harrison W</span>, <span class="author">Elder FF</span>, <span class="author">Penman-Splitt M</span>, <span class="author">Overbeek P</span> and <span class="author">Strachan T</span></p>
<p class="affiliation">Department of Human Genetics, University of Newcastle upon Tyne, UK.</p><p class="abstract">Visceral left-right asymmetry occurs in all vertebrates, but the inversion of embryo turning (inv) mouse, which resulted following a random transgene insertion, is the only model in which these asymmetries are consistently reversed. We report positional cloning of the gene underlying this recessive phenotype. Although transgene insertion was accompanied by neighbouring deletion and duplication events, our YAC phenotype rescue studies indicate that the mutant phenotype results from the deletion. After extensively characterizing the 47-kb deleted region and flanking sequences from the wild-type mouse genome, we found evidence for only one gene sequence in the deleted region. We determined the full-length 5.5-kb cDNA sequence and identified 16 exons, of which exons 3-11 were eliminated by the deletion, causing a frameshift. The novel gene specifies a 1062-aa product with tandem ankyrin-like repeat sequences. Characterization of complementing and non-complementing YAC transgenic families revealed that correction of the inv mutant phenotype was concordant with integration and intact expression of this novel gene, which we have named inversin (Invs).</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">20</span>;<span class="number">2</span>;<span class="pagerange">149-56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9771707">9771707</a>; DOI: <a href="http://dx.doi.org/10.1038%2F2450">10.1038/2450</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Trace alignment and some of its applications.</h4>
<p class="authors"><span class="author">Mott R</span></p>
<p class="affiliation">Informatics Group, Sanger Centre, Cambridge, UK.</p><p class="abstract">MOTIVATION: Extra useful information can be extracted from a DNA chromatogram trace, over that contained in the base-called DNA sequence. Many sequencing applications can benefit from examination of these traces. RESULTS: An algorithm, based on dynamic programming, for aligning a DNA chromatogram to a DNA sequence is described and implemented. Its applications to vector clipping, EST alignment and mutation detection are discussed.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">1998</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">92-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9520506" rel="external">9520506</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fine-mapping, genomic organization, and transcript analysis of the human ubiquitin-conjugating enzyme gene UBE2L3.</h4>
<p class="authors"><span class="author">Moynihan TP</span>, <span class="author">Cole CG</span>, <span class="author">Dunham I</span>, <span class="author">O&#39;Neil L</span>, <span class="author">Markham AF</span> and <span class="author">Robinson PA</span></p>
<p class="affiliation">Clinical Sciences Building, St. James&#39;s University Hospital, Leeds, LS9 7TF, United Kingdom.</p><p class="abstract">The human UBE2L3 gene encodes the ubiquitin-conjugating enzyme UbcH7, demonstrated to participate in the ubiquitination of p53, c-Fos, and NF-kappaB in vitro. We report the fine-mapping of this four-exon gene to chromosome 22q11.2. We have constructed a comprehensive genomic clone contig across this gene, demonstrating that the gene lies adjacent to the microsatellite marker D22S446 and spans approximately 57 kb. Four mRNA species are transcribed from this gene, differing in the length of their 3&#39; UTR. Sequence comparison of the UBE2L3 cDNA with its murine homologue reveals a remarkably high degree of sequence conservation within the 3&#39;UTR.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1998</span>;<span class="volume">51</span>;<span class="number">1</span>;<span class="pagerange">124-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9693040" rel="external">9693040</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1998.5257" rel="external">10.1006/geno.1998.5257</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Estimation of distances and map construction using radiation hybrids.</h4>
<p class="authors"><span class="author">Newell W</span>, <span class="author">Beck S</span>, <span class="author">Lehrach H</span> and <span class="author">Lyall A</span></p>
<p class="affiliation">Oxford Molecular Group, The Medawar Centre, Oxford OX4 4GA, UK.</p><p class="abstract">A method of estimating distances between pairs of genetic markers is described that directly uses their observed joint frequency distribution in a panel of radiation hybrids (RHs). The distance measure is based on the strength of association between marker pairs, which is high for close markers and decays with distance. These distances are then submitted to a previous method that generates linear coordinates for the markers directly from the intermarker distance matrix. This method of map building from RH data is simpler than others, because it uses only the observed joint frequency distributions of markers in the panel, and does not attempt to model unobserved quantities such as the retention of different sized fragments that contain the markers. It also incorporates directly the observed variation in retention of different markers, without needing a model for differential fragment retention dependent on chromosomal location, which is generally not known. Only small, precise distances are used in map construction, thereby reducing any effects of different fragment retention frequencies and local variations in X-ray sensitivity. The method is tested by simulation, and known marker distances and locations are successfully recovered from RH raw data. The method is also applied to publicly available data sets related to the recent transcript map of the human genome.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">5</span>;<span class="pagerange">493-508</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9582193" rel="external">9582193</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence comparisons using multiple sequences detect three times as many remote homologues as pairwise methods.</h4>
<p class="authors"><span class="author">Park J</span>, <span class="author">Karplus K</span>, <span class="author">Barrett C</span>, <span class="author">Hughey R</span>, <span class="author">Haussler D</span>, <span class="author">Hubbard T</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Hills Road, Cambridge, CB2 2QH, UK.</p><p class="abstract">The sequences of related proteins can diverge beyond the point where their relationship can be recognised by pairwise sequence comparisons. In attempts to overcome this limitation, methods have been developed that use as a query, not a single sequence, but sets of related sequences or a representation of the characteristics shared by related sequences. Here we describe an assessment of three of these methods: the SAM-T98 implementation of a hidden Markov model procedure; PSI-BLAST; and the intermediate sequence search (ISS) procedure. We determined the extent to which these procedures can detect evolutionary relationships between the members of the sequence database PDBD40-J. This database, derived from the structural classification of proteins (SCOP), contains the sequences of proteins of known structure whose sequence identities with each other are 40% or less. The evolutionary relationships that exist between those that have low sequence identities were found by the examination of their structural details and, in many cases, their functional features. For nine false positive predictions out of a possible 432,680, i.e. at a false positive rate of about 1/50,000, SAM-T98 found 35% of the true homologous relationships in PDBD40-J, whilst PSI-BLAST found 30% and ISS found 25%. Overall, this is about twice the number of PDBD40-J relations that can be detected by the pairwise comparison procedures FASTA (17%) and GAP-BLAST (15%). For distantly related sequences in PDBD40-J, those pairs whose sequence identity is less than 30%, SAM-T98 and PSI-BLAST detect three times the number of relationships found by the pairwise methods.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1998</span>;<span class="volume">284</span>;<span class="number">4</span>;<span class="pagerange">1201-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9837738">9837738</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fjmbi.1998.2221">10.1006/jmbi.1998.2221</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SPEM: a parser for EMBL style flat file database entries.</h4>
<p class="authors"><span class="author">Pocock MR</span>, <span class="author">Hubbard T</span> and <span class="author">Birney E</span></p>
<p class="affiliation">Informatics, The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%72%70%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#112;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Summary</strong>: We present a set of Perl modules for the flexible and robust parsing and editing of EMBL/SWISS-PROT databases.</p><p class="abstract"><strong>Availability</strong>: The Web page at http://www.sanger.ac. uk/Software/PerlModule/ provides information about downloading the SPEM and PrEMBL modules, and provides links to documentation and example code.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">1998</span>;<span class="volume">14</span>;<span class="number">9</span>;<span class="pagerange">823-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9918956">9918956</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Using neural networks for prediction of the subcellular location of proteins.</h4>
<p class="authors"><span class="author">Reinhardt A</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%61%72%65%69%6E%68%61%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#114;&#101;&#105;&#110;&#104;&#97;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Neural networks have been trained to predict the subcellular location of proteins in prokaryotic or eukaryotic cells from their amino acid composition. For three possible subcellular locations in prokaryotic organisms a prediction accuracy of 81% can be achieved. Assigning a reliability index, 33% of the predictions can be made with an accuracy of 91%. For eukaryotic proteins (excluding plant sequences) an overall prediction accuracy of 66% for four locations was achieved, with 33% of the sequences being predicted with an accuracy of 82% or better. With the subcellular location restricting a protein&#39;s possible function, this method should be a useful tool for the systematic analysis of genome data and is available via a server on the world wide web.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1998</span>;<span class="volume">26</span>;<span class="number">9</span>;<span class="pagerange">2230-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9547285">9547285</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC147531">147531</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Toward a complete human genome sequence.</h4>
<p class="authors"><span class="author">Sanger Center</span> and <span class="author">Genome Sequencing Center</span></p>
<p class="affiliation">Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK;</p><p class="abstract">We have begun a joint program as part of a coordinated international effort to determine a complete human genome sequence. Our strategy is to map large-insert bacterial clones and to sequence each clone by a random shotgun approach followed by directed finishing. As of September 1998, we have identified the map positions of bacterial clones covering approximately 860 Mb for sequencing and completed &gt;98 Mb ( approximately 3.3%) of the human genome sequence. Our progress and sequencing data can be accessed via the World Wide Web (http://webace.sanger.ac.uk/HGP/ or http://genome.wustl.edu/gsc/).</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">11</span>;<span class="pagerange">1097-108</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9847074">9847074</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Deletion mapping on chromosome 10p and definition of a critical region for the second DiGeorge syndrome locus (DGS2).</h4>
<p class="authors"><span class="author">Schuffenhauer S</span>, <span class="author">Lichtner P</span>, <span class="author">Peykar-Derakhshandeh P</span>, <span class="author">Murken J</span>, <span class="author">Haas OA</span>, <span class="author">Back E</span>, <span class="author">Wolff G</span>, <span class="author">Zabel B</span>, <span class="author">Barisic I</span>, <span class="author">Rauch A</span>, <span class="author">Borochowitz Z</span>, <span class="author">Dallapiccola B</span>, <span class="author">Ross M</span> and <span class="author">Meitinger T</span></p>
<p class="affiliation">Abteilung Medizinische Genetik, Ludwig-Maximilians-Universit&auml;t M&uuml;nchen, Germany. <a href="mailto:%73%69%6D%6F%6E%65%40%70%65%64%67%65%6E.%6D%65%64.%75%6E%69-%6D%75%65%6E%63%68%65%6E.%64%65">&#115;&#105;&#109;&#111;&#110;&#101;&#64;&#112;&#101;&#100;&#103;&#101;&#110;&#46;&#109;&#101;&#100;&#46;&#117;&#110;&#105;&#45;&#109;&#117;&#101;&#110;&#99;&#104;&#101;&#110;&#46;&#100;&#101;</a></p><p class="abstract">DiGeorge syndrome (DGS) is a developmental field defect, characterised by absent/hypoplastic thymus and parathyroid, and conotruncal heart defects, with haploinsufficiency loci at 22q (DGS1) and 10p (DGS2). We performed fluorescence in situ hybridisations (FISH) and polymerase chain reaction (PCR) analyses in 12 patients with 10p deletions, nine of them with features of DGS, and in a familial translocation 10p;14q associated with midline defects. The critical DGS2 region is defined by two DGS patients, and maps within a 1 cM interval including D10S547 and D10S585. The other seven DGS patients are hemizygous for both loci. The breakpoint of the reciprocal translocation 10p;14q maps at a distance of at least 12 cM distal to the critical DGS2 region. Interstitial and terminal deletions described are in the range of 10-50 cM and enable the tentative mapping of loci for ptosis and hearing loss, features which are not part of the DGS clinical spectrum.</p>
<p class="publication"><span class="title">European journal of human genetics : EJHG</span>&nbsp;<span class="year">1998</span>;<span class="volume">6</span>;<span class="number">3</span>;<span class="pagerange">213-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9781025" rel="external">9781025</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.ejhg.5200183" rel="external">10.1038/sj.ejhg.5200183</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene number in an invertebrate chordate, Ciona intestinalis.</h4>
<p class="authors"><span class="author">Simmen MW</span>, <span class="author">Leitgeb S</span>, <span class="author">Clark VH</span>, <span class="author">Jones SJ</span> and <span class="author">Bird A</span></p>
<p class="affiliation">Institute of Cell and Molecular Biology, University of Edinburgh, King&#39;s Buildings, Edinburgh EH9 3JR, United Kingdom. <a href="mailto:%4D.%53%69%6D%6D%65%6E%40%65%64.%61%63.%75%6B">&#77;&#46;&#83;&#105;&#109;&#109;&#101;&#110;&#64;&#101;&#100;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Gene number can be considered a pragmatic measure of biological complexity, but reliable data is scarce. Estimates for vertebrates are 50-100,000 genes per haploid genome, whereas invertebrate estimates fall below 25,000. We wished to test the hypothesis that the origin of vertebrates coincided with extensive gene creation. A prediction is that gene number will differ sharply between invertebrate and vertebrate members of the chordate phylum. A gene number estimation method requiring limited sequence sampling of genomic DNA was developed and validated by using data for Caenorhabditis elegans. Using the method, we estimated that the invertebrate chordate Ciona intestinalis has 15,500 protein-coding genes (+/-3,700). This number is significantly lower than gene numbers of vertebrate chordates, but similar to those of invertebrates in distantly related phyla. The data indicate that evolution of vertebrates was accompanied by a dramatic increase in protein-coding capacity of the genome.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1998</span>;<span class="volume">95</span>;<span class="number">8</span>;<span class="pagerange">4437-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9539755" rel="external">9539755</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC22507" rel="external">22507</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Z extensions to the RHMAPPER package.</h4>
<p class="authors"><span class="author">Soderlund C</span>, <span class="author">Lau T</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%63%61%72%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">MOTIVATION: Extensions have been made to the RHMAPPER-1.1 package. One set of extensions computes the totally linked markers and uses the results as input to the salient RHMAPPER functions. The second set of extensions uses TKperl to provide an interactive interface for ease of querying the database and displaying maps. AVAILABILITY: The extensions can be obtained via ftp.sanger.ac.uk/pub/zmapper. Supplementary information: The User&#39;s Manual can be viewed from http:/www.sanger.ac.uk/Users/cari/Z.shtml. CONTACT: <a href="mailto:%63%61%72%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">1998</span>;<span class="volume">14</span>;<span class="number">6</span>;<span class="pagerange">538-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9694993" rel="external">9694993</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pfam: multiple sequence alignments and HMM-profiles of protein domains.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span>, <span class="author">Eddy SR</span>, <span class="author">Birney E</span>, <span class="author">Bateman A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Computational Biology Branch, National Center for Biotechnology Information, National Library of Medicine, Building 38A, Room 8N805, National Institutes of Health, Bethesda, MD 20894, USA. <a href="mailto:%73%6F%6E%6E%68%61%6D%6D%65%72%40%6E%63%62%69.%6E%6C%6D.%6E%69%68.%67%6F%76">&#115;&#111;&#110;&#110;&#104;&#97;&#109;&#109;&#101;&#114;&#64;&#110;&#99;&#98;&#105;&#46;&#110;&#108;&#109;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Pfam contains multiple alignments and hidden Markov model based profiles (HMM-profiles) of complete protein domains. The definition of domain boundaries, family members and alignment is done semi-automatically based on expert knowledge, sequence similarity, other protein family databases and the ability of HMM-profiles to correctly identify and align the members. Release 2.0 of Pfam contains 527 manually verified families which are available for browsing and on-line searching via the World Wide Web in the UK at http://www.sanger.ac.uk/Pfam/ and in the US at http://genome.wustl. edu/Pfam/ Pfam 2.0 matches one or more domains in 50% of Swissprot-34 sequences, and 25% of a large sample of predicted proteins from the Caenorhabditis elegans genome.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01-HG01363</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1998</span>;<span class="volume">26</span>;<span class="number">1</span>;<span class="pagerange">320-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9399864" rel="external">9399864</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC147209" rel="external">147209</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequence and structure of the mouse RING3 gene: identification of variant RING3 transcripts.</h4>
<p class="authors"><span class="author">Thorpe KL</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Sanger Centre, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, England.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Immunogenetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">48</span>;<span class="number">1</span>;<span class="pagerange">82-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9601950" rel="external">9601950</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical map of human 6p21.2-6p21.3: region flanking the centromeric end of the major histocompatibility complex.</h4>
<p class="authors"><span class="author">Tripodis N</span>, <span class="author">Mason R</span>, <span class="author">Humphray SJ</span>, <span class="author">Davies AF</span>, <span class="author">Herberg JA</span>, <span class="author">Trowsdale J</span>, <span class="author">Nizetic D</span>, <span class="author">Senger G</span> and <span class="author">Ragoussis J</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, United Medical and Dental School of Guy&#39;s and St. Thomas&#39;, Guy&#39;s Hospital, London SE1 9RT, UK.</p><p class="abstract">We have physically mapped and cloned a 2.5-Mb chromosomal segment flanking the centromeric end of the major histocompatibility complex (MHC). We characterized in detail 27 YACs, 144 cosmids, 51 PACs, and 5 BACs, which will facilitate the complete genomic sequencing of this region of chromosome 6. The contig contains the genes encoding CSBP, p21, HSU09564 serine kinase, ZNF76, TCP-11, RPS10, HMGI(Y), BAK, and the human homolog of Tctex-7 (HSET). The GLO1 gene was mapped further centromeric in the 6p21.2-6p21.1 region toward TCTE-1. The gene order of the GLO1-HMGI(Y) segment in respect to the centromere is similar to the gene order in the mouse t-chromosome distal inversion, indicating that there is conservation in gene content but not gene order between humans and mice in this region. The close linkage of the BAK and CSBP genes to the MHC is of interest because of their possible involvement in autoimmune disease.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">6</span>;<span class="pagerange">631-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9647638" rel="external">9647638</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310739" rel="external">310739</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of SCML1, a new gene in Xp22, with homology to developmental polycomb genes.</h4>
<p class="authors"><span class="author">van de Vosse E</span>, <span class="author">Walpole SM</span>, <span class="author">Nicolaou A</span>, <span class="author">van der Bent P</span>, <span class="author">Cahn A</span>, <span class="author">Vaudin M</span>, <span class="author">Ross MT</span>, <span class="author">Durham J</span>, <span class="author">Pavitt R</span>, <span class="author">Wilkinson J</span>, <span class="author">Grafham D</span>, <span class="author">Bergen AA</span>, <span class="author">van Ommen GJ</span>, <span class="author">Yates JR</span>, <span class="author">den Dunnen JT</span> and <span class="author">Trump D</span></p>
<p class="affiliation">MGC-Department of Human Genetics, Leiden University, Al Leiden, The Netherlands.</p><p class="abstract">Using exon trapping, we have identified a new human gene in Xp22 encoding a 3-kb mRNA. Expression of this RNA is detectable in a range of tissues but is most pronounced in skeletal muscle and heart. The gene, designated &quot;sex comb on midleg-like-1&quot; (SCML1), maps 14 kb centromeric of marker DXS418, between DXS418 and DXS7994, and is transcribed from telomere to centromere. SCML1 spans 18 kb of genomic DNA, consists of six exons, and has a 624-bp open reading frame. The predicted 27-kDa SCML1 protein contains two domains that each have a high homology to two Drosophila transcriptional repressors of the polycomb group (PcG) genes and their homologues in mouse and human. PcG genes are known to be involved in the regulation of homeotic genes, and the mammalian homologues of the PcG genes repress the expression of Hox genes. SCML1 appears to be a new human member of this gene group and may play an important role in the control of embryonal development.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1998</span>;<span class="volume">49</span>;<span class="number">1</span>;<span class="pagerange">96-102</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9570953" rel="external">9570953</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1998.5224" rel="external">10.1006/geno.1998.5224</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Localization of HuC (ELAVL3) to chromosome 19p13.2 by fluorescence in situ hybridization utilizing a novel tyramide labeling technique.</h4>
<p class="authors"><span class="author">Van Tine BA</span>, <span class="author">Knops JF</span>, <span class="author">Butler A</span>, <span class="author">Deloukas P</span>, <span class="author">Shaw GM</span> and <span class="author">King PH</span></p>
<p class="affiliation">Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama, 35294, USA.</p><p class="abstract">HuC is a neural-specific member of the Elav family of RNA-binding proteins. This highly conserved gene family plays a crucial role in neurogenesis, and HuC (HGMW-approved symbol ELAVL3) is expressed at an early stage of neural development. Using a novel tyramide fluorescence in situ hybridization (T-FISH) technique, we localized HuC to chromosome 19p13.2. This localization was confirmed by radiation hybrid mapping and coincides with that of HuR (HGMW-approved symbol ELAVL1), another elav family member. Dual T-FISH analysis with HuC and HuR probes, however, indicated distinct loci, with HuC being centromeric to HuR. This study demonstrates the utility of T-FISH in colocalizing two genes on the same chromosomal preparation using only biotinylated probes.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">NIHUO1-AI32775</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1998</span>;<span class="volume">53</span>;<span class="number">3</span>;<span class="pagerange">296-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9799595" rel="external">9799595</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1998.5468" rel="external">10.1006/geno.1998.5468</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing and analysis of genes involved in the biosynthesis of a vancomycin group antibiotic.</h4>
<p class="authors"><span class="author">van Wageningen AM</span>, <span class="author">Kirkpatrick PN</span>, <span class="author">Williams DH</span>, <span class="author">Harris BR</span>, <span class="author">Kershaw JK</span>, <span class="author">Lennard NJ</span>, <span class="author">Jones M</span>, <span class="author">Jones SJ</span> and <span class="author">Solenberg PJ</span></p>
<p class="affiliation">Cambridge Centre for Molecular Recognition, Department of Chemistry, Cambridge, UK.</p><p class="abstract"><strong>Background</strong>: The emergence of resistance to vancomycin, the drug of choice against methicillin-resistant Staphylococcus aureus, in enterococci has increased the need for new antibiotics. As chemical modification of the antibiotic structure is not trivial, we have initiated studies towards enzymatic modification by sequencing the DNA coding for the biosynthesis of chloroeremomycin (also known as A82846B and LY264826).</p><p class="abstract"><strong>Results</strong>: Analysis of 72 kilobases of genomic DNA from Amycolatopsis orientalis, the organism that produces chloroeremomycin, revealed the presence of 39 putative genes, including those coding for the biosynthesis of the antibiotic. Translation and subsequent comparison with known proteins in public databases identified enzymes responsible for the biosynthesis of the heptapeptide backbone and 4-epi-vancosamine, as well as those for chlorination and oxidation reactions involved in the biosynthesis of chloroeremomycin.</p><p class="abstract"><strong>Conclusions</strong>: The genes responsible for the biosynthesis of chloroeremomycin have been identified, and selective expression of these genes could lead to the synthesis of new potent glycopeptide antibiotics.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Chemistry &amp; biology</span>&nbsp;<span class="year">1998</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">155-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9545426" rel="external">9545426</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Human Genome Project: reaching the finish line.</h4>
<p class="authors"><span class="author">Waterston R</span> and <span class="author">Sulston JE</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, St. Louis, MO 63108, USA. <a href="mailto:%72%77%40%67%65%6E%65%74%69%63%73.%77%75%73%74%6C.%65%64%75">&#114;&#119;&#64;&#103;&#101;&#110;&#101;&#116;&#105;&#99;&#115;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">1998</span>;<span class="volume">282</span>;<span class="number">5386</span>;<span class="pagerange">53-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9786797">9786797</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automated sequence preprocessing in a large-scale sequencing environment.</h4>
<p class="authors"><span class="author">Wendl MC</span>, <span class="author">Dear S</span>, <span class="author">Hodgson D</span> and <span class="author">Hillier L</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University, St. Louis, Missouri 63108 USA. <a href="mailto:%6D%77%65%6E%64%6C%40%77%61%74%73%6F%6E.%77%75%73%74%6C.%65%64%75">&#109;&#119;&#101;&#110;&#100;&#108;&#64;&#119;&#97;&#116;&#115;&#111;&#110;&#46;&#119;&#117;&#115;&#116;&#108;&#46;&#101;&#100;&#117;</a></p><p class="abstract">A software system for transforming fragments from four-color fluorescence-based gel electrophoresis experiments into assembled sequence is described. It has been developed for large-scale processing of all trace data, including shotgun and finishing reads, regardless of clone origin. Design considerations are discussed in detail, as are programming implementation and graphic tools. The importance of input validation, record tracking, and use of base quality values is emphasized. Several quality analysis metrics are proposed and applied to sample results from recently sequenced clones. Such quantities prove to be a valuable aid in evaluating modifications of sequencing protocol. The system is in full production use at both the Genome Sequencing Center and the Sanger Centre, for which combined weekly production is approximately 100, 000 sequencing reads per week.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG00956</span>, <span class="grant">HG01458</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1998</span>;<span class="volume">8</span>;<span class="number">9</span>;<span class="pagerange">975-84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9750196" rel="external">9750196</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC310779" rel="external">310779</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Physical mapping of the CA6, ENO1, and SLC2A5 (GLUT5) genes and reassignment of SLC2A5 to 1p36.2.</h4>
<p class="authors"><span class="author">White PS</span>, <span class="author">Jensen SJ</span>, <span class="author">Rajalingam V</span>, <span class="author">Stairs D</span>, <span class="author">Sulman EP</span>, <span class="author">Maris JM</span>, <span class="author">Biegel JA</span>, <span class="author">Wooster R</span> and <span class="author">Brodeur GM</span></p>
<p class="affiliation">Oncology, The Children&#39;s Hospital of Philadelphia, Philadelphia, PA (USA). <a href="mailto:%77%68%69%74%65%40%6B%65%72%6D%69%74.%6F%6E%63%6F%6C.%63%68%6F%70.%65%64%75">&#119;&#104;&#105;&#116;&#101;&#64;&#107;&#101;&#114;&#109;&#105;&#116;&#46;&#111;&#110;&#99;&#111;&#108;&#46;&#99;&#104;&#111;&#112;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Several human malignancies frequently exhibit deletions or rearrangements of the distal short arm of chromosome 1 (1p36), and a number of genetic diseases also map to this region. The carbonic anhydrase (CA6) and alpha-enolase (ENO1) genes, previously mapped to 1p36, were physically linked in yeast- and P1-artificial chromosome (YAC and PAC) contigs. PACs from the contig were mapped to 1p36.2 by fluorescence in situ hybridization. The ESTs D1S2068, D1S274E, D1S3275, and stSG4370 were also placed in the same contig. The physical map was integrated with the genetic map of chromosome 1 by assignment of genetic markers D1S160, D1S1615, and D1S503 to the contig. Sequencing of the EST clone representing D1S274E indicated that it was derived from the same transcript as D1S2068E and corresponded to the SLC2A5 (GLUT5) gene, previously assigned to 1p31. Reassignment of SLC2A5 to 1p36.2 was confirmed by somatic cell and radiation hybrid mapping panels and was consistent with previous EST mapping data. Sequencing of the EST clone for D1S274E revealed the presence of intronic sequences, suggesting that the clone was derived from an unprocessed message. The presence of unprocessed and/or alternatively spliced EST clones has potential ramifications for EST-based genomic projects. This information should facilitate the mapping of tumor suppressor and genetic disease loci that have been localized to this region.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">CA39771</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Cytogenetics and cell genetics</span>&nbsp;<span class="year">1998</span>;<span class="volume">81</span>;<span class="number">1</span>;<span class="pagerange">60-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9691177" rel="external">9691177</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/hh3o2ctm"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/hh3o2ctm" src="/qr/hh3o2ctm.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:55:34 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
