<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1995 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1995.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1995</h1><p>Number of papers published in 1995: <strong>30</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Meiotic recombination, noncoding DNA and genomic organization in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Barnes TM</span>, <span class="author">Kohara Y</span>, <span class="author">Coulson A</span> and <span class="author">Hekimi S</span></p>
<p class="affiliation">Department of Biology, McGill University, Montr&eacute;al, Quebec, Canada.</p><p class="abstract">The genetic map of each Caenorhabditis elegans chromosome has a central gene cluster (less pronounced on the X chromosome) that contains most of the mutationally defined genes. Many linkage group termini also have clusters, though involving fewer loci. We examine the factors shaping the genetic map by analyzing the rate of recombination and gene density across the genome using the positions of cloned genes and random cDNA clones from the physical map. Each chromosome has a central gene-dense region (more diffuse on the X) with discrete boundaries, flanked by gene-poor regions. Only autosomes have reduced rates of recombination in these gene-dense regions. Cluster boundaries appear discrete also by recombination rate, and the boundaries defined by recombination rate and gene density mostly, but not always, coincide. Terminal clusters have greater gene densities than the adjoining arm but similar recombination rates. Thus, unlike in other species, most exchange in C. elegans occurs in gene-poor regions. The recombination rate across each cluster is constant and similar; and cluster size and gene number per chromosome are independent of the physical size of chromosomes. We propose a model of how this genome organization arose.</p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">1995</span>;<span class="volume">141</span>;<span class="number">1</span>;<span class="pagerange">159-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8536965" rel="external">8536965</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1206715" rel="external">1206715</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative sequence analysis of the human and pufferfish Huntington&#39;s disease genes.</h4>
<p class="authors"><span class="author">Baxendale S</span>, <span class="author">Abdulla S</span>, <span class="author">Elgar G</span>, <span class="author">Buck D</span>, <span class="author">Berks M</span>, <span class="author">Micklem G</span>, <span class="author">Durbin R</span>, <span class="author">Bates G</span>, <span class="author">Brenner S</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Genome Analysis Laboratory, ICRF, London, UK.</p><p class="abstract">The Huntington&#39;s disease (HD) gene encodes a novel protein with as yet no known function. In order to identify the functionally important domains of this protein, we have cloned and sequenced the homologue of the HD gene in the pufferfish, Fugu rubripes. The Fugu HD gene spans only 23 kb of genomic DNA, compared to the 170 kb human gene, and yet all 67 exons are conserved. The first coding exon, the site of the disease-causing triplet repeat, is highly conserved. However, the glutamine repeat in Fugu consists of just four residues. We also show that gene order may be conserved over longer stretches of the two genomes. Our work describes a detailed example of sequence comparison between human and Fugu, and illustrates the power of the pufferfish genome as a model system in the analysis of human genes.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">1995</span>;<span class="volume">10</span>;<span class="number">1</span>;<span class="pagerange">67-76</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7647794">7647794</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0595-67">10.1038/ng0595-67</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The C. elegans genome sequencing project. C. elegans Genome Mapping and Sequencing Consortium.</h4>
<p class="authors"><span class="author">Berks M</span></p>
<p class="affiliation">The Sanger Centre, Hinxton, Cambridgeshire, UK, mb 1 @sanger.ac.uk.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">1995</span>;<span class="volume">5</span>;<span class="number">2</span>;<span class="pagerange">99-104</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/9132275" rel="external">9132275</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A YAC contig encompassing the XRCC5 (Ku80) DNA repair gene and complementation of defective cells by YAC protoplast fusion.</h4>
<p class="authors"><span class="author">Blunt T</span>, <span class="author">Taccioli GE</span>, <span class="author">Priestley A</span>, <span class="author">Hafezparast M</span>, <span class="author">McMillan T</span>, <span class="author">Liu J</span>, <span class="author">Cole CC</span>, <span class="author">White J</span>, <span class="author">Alt FW</span>, <span class="author">Jackson SP</span> <em>et al.</em></p>
<p class="affiliation">MRC Cell Mutation Unit, University of Sussex, Brighton, United Kingdom.</p><p class="abstract">The Chinese hamster ovary xrs mutants are sensitive to ionizing radiation, defective in DNA double-strand break rejoining, and unable to carry out V(D)J recombination effectively. Recently, the gene defective in these mutants, XRCC5, has been shown to encode Ku80, a component of the Ku protein and DNA-dependent protein kinase. We present here a YAC contig involving 25 YACs mapping to the region 2q33-q34, which encompasses the XRCC5 gene. Eight new markers for this region of chromosome 2 are identified. YACs encoding the Ku80 gene were transferred to xrs cells by protoplast fusion, and complementation of all the defective phenotypes has been obtained with two YACs. We discuss the advantages and disadvantages of this approach as a strategy for cloning human genes complementing defective rodent cell lines.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1995</span>;<span class="volume">30</span>;<span class="number">2</span>;<span class="pagerange">320-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8586433" rel="external">8586433</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A high-density YAC contig map of human chromosome 22.</h4>
<p class="authors"><span class="author">Collins JE</span>, <span class="author">Cole CG</span>, <span class="author">Smink LJ</span>, <span class="author">Garrett CL</span>, <span class="author">Leversha MA</span>, <span class="author">Soderlund CA</span>, <span class="author">Maslen GL</span>, <span class="author">Everett LA</span>, <span class="author">Rice KM</span>, <span class="author">Coffey AJ</span> <em>et al.</em></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">We have constructed a high-resolution clone map of human chromosome 22 which integrates the available physical and genetic information, establishing a single consensus. The map consists of all classes of DNA landmarks ordered on 705 yeast artificial chromosomes (YACs) at an average landmark density of more than one per 70 kilobases. This map represents the practical limits of currently available YAC resources and provides the basis for determination of the entire gene content and genomic DNA sequence of human chromosome 22.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1995</span>;<span class="volume">377</span>;<span class="number">6547 Suppl</span>;<span class="pagerange">367-79</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7566101">7566101</a>; DOI: <a href="http://dx.doi.org/10.1038%2F377367a0">10.1038/377367a0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The physical map of the Caenorhabditis elegans genome.</h4>
<p class="authors"><span class="author">Coulson A</span>, <span class="author">Huynh C</span>, <span class="author">Kozono Y</span> and <span class="author">Shownkeen R</span></p>
<p class="affiliation">Sanger Centre, Cambridge, United Kingdom.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">1995</span>;<span class="volume">48</span>;<span class="pagerange">533-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8531741" rel="external">8531741</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Distribution of mRNA encoding the inwardly rectifying K+ channel, BIR1 in rat tissues.</h4>
<p class="authors"><span class="author">Dixon AK</span>, <span class="author">Gubitz AK</span>, <span class="author">Ashford ML</span>, <span class="author">Richardson PJ</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">Human Genetics Group, Sanger Centre, Hinxton, Cambs, UK.</p><p class="abstract">The distribution of mRNA encoding the inwardly rectifying K+ channel, BIR1 [1] was investigated in rat tissues, and a comparison made with the expression of related genes rcKATP and GIRK1 using the reverse transcription-polymerase chain reaction (RT-PCR). This showed BIR1 to be expressed in all areas of the brain examined, in the eye but not in any other peripheral tissue. This pattern was distinct from rcKATP and GIRK1. Additional in situ hybridisation studies of the central expression of BIR1 demonstrated high levels of BIR1 mRNA in the hippocampus, dentate gyrus, taenia tecta and cerebellum and at lower levels in the cortex, habenular nucleus, olfactory bulb, primary olfactory cortex, thalamus, pontine nucleus and amygdaloid nucleus.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">FEBS letters</span>&nbsp;<span class="year">1995</span>;<span class="volume">374</span>;<span class="number">1</span>;<span class="pagerange">135-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7589501" rel="external">7589501</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tropomyosin is essential in yeast, yet the TPM1 and TPM2 products perform distinct functions.</h4>
<p class="authors"><span class="author">Drees B</span>, <span class="author">Brown C</span>, <span class="author">Barrell BG</span> and <span class="author">Bretscher A</span></p>
<p class="affiliation">Section of Biochemistry, Molecular and Cell Biology, Cornell University, Ithaca, New York 14853.</p><p class="abstract">Sequence analysis of chromosome IX of Saccharomyces cerevisiae revealed an open reading frame of 166 residues, designated TPM2, having 64.5% sequence identity to TPM1, that encodes the major form of tropomyosin in yeast. Purification and characterization of Tpm2p revealed a protein with the characteristics of a bona fide tropomyosin; it is present in vivo at about one sixth the abundance of Tpm1p. Biochemical and sequence analysis indicates that Tpm2p spans four actin monomers along a filament, whereas Tpmlp spans five. Despite its shorter length, Tpm2p can compete with Tpm1p for binding to F-actin. Over-expression of Tpm2p in vivo alters the axial budding of haploids to a bipolar pattern, and this can be partially suppressed by co-over-expression of Tpm1p. This suggests distinct functions for the two tropomyosins, and indicates that the ratio between them is important for correct morphogenesis. Loss of Tpm2p has no detectable phenotype in otherwise wild type cells, but is lethal in combination with tpm1 delta. Over-expression of Tpm2p does not suppress the growth or cell surface targeting defects associated with tpm1 delta, so the two tropomyosins must perform an essential function, yet are not functionally interchangeable. S. cerevisiae therefore provides a simple system for the study of two tropomyosins having distinct yet overlapping functions.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM36552</span></p>
<p class="publication"><span class="title">The Journal of cell biology</span>&nbsp;<span class="year">1995</span>;<span class="volume">128</span>;<span class="number">3</span>;<span class="pagerange">383-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7844152" rel="external">7844152</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2120352" rel="external">2120352</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Maximum discrimination hidden Markov models of sequence consensus.</h4>
<p class="authors"><span class="author">Eddy SR</span>, <span class="author">Mitchison G</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Department of Genetics, Washington University School of Medicine, St. Louis, MO 63110, USA.</p><p class="abstract">We introduce a maximum discrimination method for building hidden Markov models (HMMs) of protein or nucleic acid primary sequence consensus. The method compensates for biased representation in sequence data sets, superseding the need for sequence weighting methods. Maximum discrimination HMMs are more sensitive for detecting distant sequence homologs than various other HMM methods or BLAST when tested on globin and protein kinase catalytic domain sequences.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">1995</span>;<span class="volume">2</span>;<span class="number">1</span>;<span class="pagerange">9-23</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7497123" rel="external">7497123</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ACeDB and macace.</h4>
<p class="authors"><span class="author">Eeckman FH</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">LBL Human Genome Center, Berkeley, California, USA.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">1995</span>;<span class="volume">48</span>;<span class="pagerange">583-605</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8531744" rel="external">8531744</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parallel patterns of cell-specific gene expression during enterocyte differentiation and maturation in the small intestine of the rabbit.</h4>
<p class="authors"><span class="author">Freeman TC</span></p>
<p class="affiliation">Department of Cellular Physiology, AFRC Babraham Institute, Cambridge, UK.</p><p class="abstract">Enterocytes are the major epithelial cell type of the small intestine. Their capacity to secret, absorb and digest specific ions and nutrients is dependent on their position along the length of the small intestine as well as their stage of development as they migrate and differentiate along the crypt-villus axis. In order to further understand the molecular processes that regulate enterocyte differentiation and function, this study has compared the levels of six mRNA species produced by genes expressed in rabbit enterocytes; specifically, the multidrug resistance (MDR1) gene encoding the 170-kDa P-glycoprotein, CaBP 9k, which encodes a putative intracellular calcium buffer, calbindin, LPH, APN, and AP which encode the brush-border hydrolases lactase-phlorizin hydrolase, aminopeptidase N and alkaline phosphatase, respectively, and SGLT1, encoding the brush border Na(+)-glucose cotransporter. The level of each mRNA species has been mapped along the small intestine using quantitative in situ hybridisation. This has revealed characteristic regional variations in the abundance of each of the mRNAs, supporting the opinion that there is a strong genetic component to the maintenance of gradients in epithelial function along the length of the small intestine. Analysis of the cellular accumulation of mRNA during enterocyte migration along the crypt-villus axis, over gut-associated lymphoid tissue, and at epithelial boundaries, has, by contrast, established a clear correlation in the expression of these genes. These data illustrate the dynamics of enterocyte gene expression, thereby providing an insight into the molecular mechanisms which co-ordinate the events of cell transformation that underlie functional differences between the epithelial populations of the small intestine.</p>
<p class="publication"><span class="title">Differentiation; research in biological diversity</span>&nbsp;<span class="year">1995</span>;<span class="volume">59</span>;<span class="number">3</span>;<span class="pagerange">179-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7589902" rel="external">7589902</a>; DOI: <a href="http://dx.doi.org/10.1046%2Fj.1432-0436.1995.5930179.x" rel="external">10.1046/j.1432-0436.1995.5930179.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">H+/di-tripeptide transporter (PepT1) expression in the rabbit intestine.</h4>
<p class="authors"><span class="author">Freeman TC</span>, <span class="author">Bentsen BS</span>, <span class="author">Thwaites DT</span> and <span class="author">Simmons NL</span></p>
<p class="affiliation">S&aacute;nger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">In order to examine the intestinal expression of the recently cloned H+/di-tripeptide transporter (PepT1), oligonucleotide probes were synthesized and their specificity confirmed by Northern blot analysis of rabbit jejunal RNA. In situ hybridization studies, using these probes, show that PepT1 is expressed all along the small intestine and at a very much reduced level in the colon. In contrast, PepT1 mRNA was not detected in the stomach, sacculus rotundus or caecum. Microscopic examination of tissue sections showed PepT1 expression to be restricted to intestinal epithelium with no detectable expression in the lamina propria, muscularis mucosae, muscularis or serosa. The accumulation of PepT1 mRNA along the crypt-villus axis was also investigated. In all regions of the small intestine (in duodenum, jejunum and ileum), PepT1 mRNA was undetectable in deeper epithelial cells of the crypts. Expression was first detectable at or near the crypt-villus junction, the amount of PepT1 mRNA increasing rapidly in the lower villus to a maximum approximately 100-200 microns from this point. Along the length of the small intestine PepT1 mRNA was most abundant in duodenal and jejunal enterocytes, with lower levels in the ileal epithelium. PepT1 expression is greatly depressed in the follicle-associated epithelium of the Peyer&#39;s patch relative to both interfollicular and adjacent &quot;normal&quot; villi. These data are discussed in the context of the known physiological role of PepT1 in the gastrointestinal tract.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Pfl&uuml;gers Archiv : European journal of physiology</span>&nbsp;<span class="year">1995</span>;<span class="volume">430</span>;<span class="number">3</span>;<span class="pagerange">394-400</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7491264" rel="external">7491264</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cellular and regional expression of transcripts of the plasma membrane calcium pump PMCA1 in rabbit intestine.</h4>
<p class="authors"><span class="author">Freeman TC</span>, <span class="author">Howard A</span>, <span class="author">Bentsen BS</span>, <span class="author">Legon S</span> and <span class="author">Walters JR</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The plasma membrane Ca(2+)-pumping adenosinetriphosphatase (PMCA) is the energy-dependent step in the active vitamin D-dependent absorption of dietary Ca2+ by the enterocyte. Studies of the various PMCA genes and splicing variants in humans and rats have indicated that the isoform known as PMCA1b is the predominant form expressed in small intestine. Using an oligonucleotide probe, we have studied the regional and cellular distribution of PMCA1 transcripts in rabbit intestinal tissues by in situ hybridization. On small intestinal RNA blots, this hybridized to species similar in size to those detected by PMCA1-specific cDNA probes; an additional larger transcript was present in rabbit than in rat or human. In situ hybridization signals were principally in the enterocyte population of the mucosa and were maximal in differentiating enterocytes on the lower part of the villus, a pattern similar to that previously demonstrated for other nutrient transporters. Reflecting the capacity of the different small intestinal segments to transport Ca2+, much higher levels of transcript were detected by both methods proximally (in duodenum) than distally (in jejunum and ileum) and were also higher in cecum and ascending colon mucosa than in descending colon. We conclude that as enterocytes differentiate in regions that absorb Ca2+, they express high levels of mRNA for PMCA1. These results confirm the importance of transcriptional regulation of this gene for active Ca2+ absorption.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">The American journal of physiology</span>&nbsp;<span class="year">1995</span>;<span class="volume">269</span>;<span class="number">1 Pt 1</span>;<span class="pagerange">G126-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7631790" rel="external">7631790</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Organization of the human immunoglobulin lambda light-chain locus on chromosome 22q11.2.</h4>
<p class="authors"><span class="author">Frippiat JP</span>, <span class="author">Williams SC</span>, <span class="author">Tomlinson IM</span>, <span class="author">Cook GP</span>, <span class="author">Cherif D</span>, <span class="author">Le Paslier D</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Winter G</span> and <span class="author">Lefranc MP</span></p>
<p class="affiliation">Laboratoire d&#39;ImmunoG&eacute;n&eacute;tique Mol&eacute;culaire, LIGM, Institut de G&eacute;n&eacute;tique Mol&eacute;culaire, UMR 9942 CNRS, Universit&eacute;s Montpellier I et II, France.</p><p class="abstract">The maps of the human immunoglobulin heavy-chain and kappa light-chain loci have recently been completed. We have now completed a map of the human lambda locus (IGL) located on chromosome 22q11.2. We mapped 52 V lambda genes from 10 V lambda families and 7 J lambda and C lambda genes on a 1140 kb contig constructed from eight YACs and 129 cosmid clones. The V lambda genes are arranged within 800 kb. Genes of the different V lambda families are organized in three clusters, V lambda II and III families (cluster A); V lambda I, V, VII and IX families (cluster B); V lambda IV, VI, VIII and X families (cluster C), in contrast to the dispersed organization of the different VH and V kappa families within the human VH and V kappa loci. We note that the most frequently used V lambda families (V lambda II and III) are proximal to the J lambda and C lambda genes. The VpreB gene, encoding part of the surrogate light chain, the GGT2 gene and the BCRL4 pseudogene were also mapped within the lambda locus.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1995</span>;<span class="volume">4</span>;<span class="number">6</span>;<span class="pagerange">983-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7655473" rel="external">7655473</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Assignment of the beta B1 crystallin gene (CRYBB1) to human chromosome 22 and mouse chromosome 5.</h4>
<p class="authors"><span class="author">Hulsebos TJ</span>, <span class="author">Gilbert DJ</span>, <span class="author">Delattre O</span>, <span class="author">Smink LJ</span>, <span class="author">Dunham I</span>, <span class="author">Westerveld A</span>, <span class="author">Thomas G</span>, <span class="author">Jenkins NA</span> and <span class="author">Copeland NG</span></p>
<p class="affiliation">Institute of Human Genetics, Academic Medical Center, University of Amsterdam, The Netherlands.</p><p class="abstract">By using primers complementary to the rat beta B1 crystallin gene sequence, we amplified exons 5 and 6 of the orthologous human gene (CRYBB1). The amplified human segments displayed greater than 88% sequence homology to the corresponding rat and bovine sequences. CRYBB1 was assigned to the group 5 region in 22q11.2-q12.1 by hybridizing the exon 6 PCR product to somatic cell hybrids containing defined portions of human chromosome 22. The exon 5 and exon 6 PCR products of CRYBB1 were used to localize, by interspecific backcross mapping, the mouse gene (Crybb1) to the central portion of chromosome 5. Three other beta crystallin genes (beta B2(-1), beta B3, and beta A4) have previously been mapped to the same regions in human and mouse. We demonstrate that the beta B1 and beta A4 crystallin genes are very closely linked in the two species. These assignments complete the mapping and identification of the human and mouse homologues of the major beta crystallins genes that are expressed in the bovine lens.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">N01-C0-46000</span></p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1995</span>;<span class="volume">29</span>;<span class="number">3</span>;<span class="pagerange">712-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8575764" rel="external">8575764</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1995.9947" rel="external">10.1006/geno.1995.9947</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An update and lessons from whole-genome sequencing projects.</h4>
<p class="authors"><span class="author">Jones SJ</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK.</p><p class="abstract">A number of prokaryotic and eukaryotic genomes are currently being sequenced. Already, the nucleotide sequences of four yeast chromosomes and of 2.2 Mb from Caenorhabditis elegans have been reported. Human genomic sequences have also been used in comparative studies with both mouse and Fugu rubripes.</p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">1995</span>;<span class="volume">5</span>;<span class="number">3</span>;<span class="pagerange">349-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7549430" rel="external">7549430</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the let-653 gene in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Jones SJ</span> and <span class="author">Baillie DL</span></p>
<p class="affiliation">Institute of Molecular Biology and Biochemistry, Simon Fraser University, Burnaby, B.C., Canada.</p><p class="abstract">A mutation in the let-653 gene of Caenorhabditis elegans results in larval death. The lethal arrest is concurrent with the appearance of a vacuole anterior to the lower pharyngeal bulb. The position of the vacuole is consistent with a dysfunction of the secretory/excretory apparatus. Germline transformation rescue experiments were able to position the let-653 gene to two overlapping cosmid subclones. Sequence data generated from both cDNA and genomic DNA subclones indicated that let-653 encodes a mucin-like protein. Our characterization suggests that a mucin-like protein is essential for effective functioning of the secretory/excretory apparatus within C. elegans.</p>
<p class="publication"><span class="title">Molecular &amp; general genetics : MGG</span>&nbsp;<span class="year">1995</span>;<span class="volume">248</span>;<span class="number">6</span>;<span class="pagerange">719-26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7476875" rel="external">7476875</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosomal localization of the mitochondrial phosphate carrier gene PHC to 12q23.</h4>
<p class="authors"><span class="author">Marsh S</span>, <span class="author">Carter NP</span>, <span class="author">Dolce V</span>, <span class="author">Iacobazzi V</span> and <span class="author">Palmieri F</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridgeshire, United Kingdom.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1995</span>;<span class="volume">29</span>;<span class="number">3</span>;<span class="pagerange">814-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8575787" rel="external">8575787</a>; DOI: <a href="http://dx.doi.org/10.1006%2Fgeno.1995.9924" rel="external">10.1006/geno.1995.9924</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular definition of the 22q11 deletions in velo-cardio-facial syndrome.</h4>
<p class="authors"><span class="author">Morrow B</span>, <span class="author">Goldberg R</span>, <span class="author">Carlson C</span>, <span class="author">Das Gupta R</span>, <span class="author">Sirotkin H</span>, <span class="author">Collins J</span>, <span class="author">Dunham I</span>, <span class="author">O&#39;Donnell H</span>, <span class="author">Scambler P</span>, <span class="author">Shprintzen R</span> <em>et al.</em></p>
<p class="affiliation">Department of Molecular Genetics, Albert Einstein College of Medicine, Bronx, NY 10461, USA.</p><p class="abstract">Velo-cardio-facial syndrome (VCFS) is a common genetic disorder among individuals with cleft palate and is associated with hemizygous deletions in human chromosome 22q11. Toward the molecular definition of the deletions, we constructed a physical map of 22q11 in the form of overlapping YACs. The physical map covers &gt; 9 cM of genetic distance, estimated to span 5 Mb of DNA, and contains a total of 64 markers. Eleven highly polymorphic short tandem-repeat polymorphic (STRP) markers were placed on the physical map, and 10 of these were unambiguously ordered. The 11 polymorphic markers were used to type the DNA from a total of 61 VCFS patients and 49 unaffected relatives. Comparison of levels of heterozygosity of these markers in VCFS patients and their unaffected relatives revealed that four of these markers are commonly hemizygous among VCFS patients. To confirm these results and to define further the breakpoints in VCFS patients, 15 VCFS individuals and their unaffected parents were genotyped for the 11 STRP markers. Haplotypes generated from this study revealed that 82% of the patients have deletions that can be defined by the STRP markers. The results revealed that all patients who have a deletion share a common proximal breakpoint, while there are two distinct distal breakpoints. Markers D22S941 and D22S944 appear to be consistently hemizygous in patients with deletions. Both of these markers are located on a single nonchimeric YAC that is 400 kb long. The results also show that the parental origin of the deleted chromosome does not have any effect on the phenotypic manifestation.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">R01HD31601</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">1995</span>;<span class="volume">56</span>;<span class="number">6</span>;<span class="pagerange">1391-403</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7762562" rel="external">7762562</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1801093" rel="external">1801093</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Investigation of the factor VIII intron 22 repeated region (int22h) and the associated inversion junctions.</h4>
<p class="authors"><span class="author">Naylor JA</span>, <span class="author">Buck D</span>, <span class="author">Green P</span>, <span class="author">Williamson H</span>, <span class="author">Bentley D</span> and <span class="author">Giannelli F</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, United Medical School of Guy&#39;s Hospital, London, UK.</p><p class="abstract">A region of intron 22 of the factor VIII gene, which contains factor VIII-associated gene A (F8A), is repeated twice more nearer the Xq telomere. It has been proposed that intrachromosomal homologous recombination occurs between the intron 22 repeat and either of the two extragenic copies, resulting in the recurrent inversions that cause almost half of all cases of severe haemophilia A. We have precisely defined the repeated region as 9.5 kb of DNA which we have termed int22h (intron 22 homologous region). The junctions of the inversions examined were shown to represent precise exchanges between the int22h repeats, thus providing conclusive evidence for homologous recombination. The three copies of int22h were compared along 8 kb of their length, using chemical mismatch analysis, and found to be 99.9% similar. The presence of such long, almost identical inverted repeats near the Xq telomere could account for the high frequency at which the inversions occur.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1995</span>;<span class="volume">4</span>;<span class="number">7</span>;<span class="pagerange">1217-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8528212" rel="external">8528212</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Method for calculation of probability of matching a bounded regular expression in a random data string.</h4>
<p class="authors"><span class="author">Sewell RF</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">A method is presented for determining within strict bounds the probability of matching a regular expression with a match start point in a given section of a random data string. The method in general requires time and space exponential in the number of optional characters in the regular expression, but in practice was used to determine bounds for probabilities of matching all the ProSite patterns without difficulty.</p>
<p class="publication"><span class="title">Journal of computational biology : a journal of computational molecular cell biology</span>&nbsp;<span class="year">1995</span>;<span class="volume">2</span>;<span class="number">1</span>;<span class="pagerange">25-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7497117" rel="external">7497117</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SAM: a system for iteratively building marker maps.</h4>
<p class="authors"><span class="author">Soderlund C</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">Sanger Centre, Cambridge, UK. <a href="mailto:%63%61%72%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#114;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">SAM (system for assembling markers) is a system which supports man-machine problem solving for iteratively ordering a set of markers. SAM aids the user in partially ordering a set of markers based on incomplete and uncertain data. As data is added and modified, SAM aids the user in updating the previously assembled maps. The input is a file of clones and for each clone, a list of the markers contained within it. The objective is to order the set of markers such that the markers contained in each clone are consecutive. The user directs the map building by selecting functions to assemble a region of markers, order the clones to fit the order of the markers and position new markers within an ordered set of markers. The user can edit the input data, edit the assembled map and add clones to the map based on their marker content. The results are displayed graphically and can be saved in a solution file. Based on the partial map, the user designs new experiments or edits the existing data to fill gaps and resolve ambiguities. When a previously assembled map is loaded into SAM, it is automatically updated with the new or altered data. SAM treats all markers as points, but has special features for multiple copy and long markers so that they can be used in the map building process. This system has supported the building of a YAC map of human chromosome 22 at the Sanger Centre, where use of Alu-PCR product markers is a major component in determining clone overlap and where we have an on-going effort to accumulate data from various sources. SAM is also being used at various other laboratories.</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1995</span>;<span class="volume">11</span>;<span class="number">6</span>;<span class="pagerange">645-55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8808581" rel="external">8808581</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A dot-matrix program with dynamic threshold control suited for genomic DNA and protein sequence analysis.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Hinxton Hall, Cambridge, UK.</p><p class="abstract">Graphical dot-matrix plots can provide the most complete and detailed comparison of two sequences. Presented here is DOTTER2, a dot-plot program for X-windows which can compare DNA or protein sequences, and also DNA versus protein. The main novel feature of DOTTER is that the user can vary the stringency cutoffs interactively, so that the dot-matrix only needs to be calculated once. This is possible thanks to a &#39;Greyramp tool&#39; that was developed to change the displayed stringency of the matrix by dynamically changing the greyscale rendering of the dots. The Greyramp tool allows the user to interactively change the lower and upper score limit for the greyscale rendering. This allows exploration of the separation between signal and noise, and fine-grained visualisation of different score levels in the dot-matrix. Other useful features are dot-matrix compression, mouse-controlled zooming, sequence alignment display and saving/loading of dot-matrices. Since the matrix only has to be calculated once and since the algorithm is fast and linear in space, DOTTER is practical to use even for sequences as long as cosmids. DOTTER was integrated in the gene-modelling module of the genomic database system ACEDB3. This was done via the homology viewer BLIXEM in a way that also allows segments from the BLAST suite of searching programs to be superimposed on top of the full dot-matrix. This feature can also be used for very quick finding of the strongest matches. As examples, we analyse a Caenorhabditis elegans cosmid with several tandem repeat families, and illustrate how DOTTER can improve gene modelling.</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">1995</span>;<span class="volume">167</span>;<span class="number">1-2</span>;<span class="pagerange">GC1-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8566757">8566757</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The construction and analysis of M13 libraries prepared from YAC DNA.</h4>
<p class="authors"><span class="author">Vaudin M</span>, <span class="author">Roopra A</span>, <span class="author">Hillier L</span>, <span class="author">Brinkman R</span>, <span class="author">Sulston J</span>, <span class="author">Wilson RK</span> and <span class="author">Waterston RH</span></p>
<p class="affiliation">Department of Genetics, Washington University School of Medicine, St. Louis, MO 63108.</p><p class="abstract">Yeast artificial chromosomes (YACs) provide a powerful way to isolate and map large regions of genomic DNA and their use in genome analysis is now extensive. We modified a series of procedures to produce high quality shotgun libraries from small amounts of YAC DNA. Clones from several different libraries have been sequenced and analyzed for distribution, sequence integrity and degree of contamination from yeast DNA. We describe these procedures and analyses and show that sequencing at about 1-fold coverage, followed by database comparison (survey sequencing) offers a relatively quick method to determine the nature of previously uncharacterized cosmid or YAC clones.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG00956</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1995</span>;<span class="volume">23</span>;<span class="number">4</span>;<span class="pagerange">670-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7899089" rel="external">7899089</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC306736" rel="external">306736</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparison of the human germline and rearranged VH repertoire reveals complementarity between germline variability and somatic mutation.</h4>
<p class="authors"><span class="author">Walter G</span>, <span class="author">Tomlinson IM</span>, <span class="author">Dear PH</span>, <span class="author">Sonnhammer EL</span>, <span class="author">Cook GP</span> and <span class="author">Winter G</span></p>
<p class="affiliation">Cambridge Antibody Technology Ltd., Melbourn, United Kingdom.</p>
<p class="publication"><span class="title">Annals of the New York Academy of Sciences</span>&nbsp;<span class="year">1995</span>;<span class="volume">764</span>;<span class="pagerange">180-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7486518" rel="external">7486518</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Free exchange.</h4>
<p class="authors"><span class="author">Waterston B</span> and <span class="author">Sulston J</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1995</span>;<span class="volume">376</span>;<span class="number">6536</span>;<span class="pagerange">111</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7603552">7603552</a>; DOI: <a href="http://dx.doi.org/10.1038%2F376111d0">10.1038/376111d0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The genome of Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Waterston R</span> and <span class="author">Sulston J</span></p>
<p class="affiliation">Department of Genetics and Genome Sequencing Center, Washington University School of Medicine, St. Louis, MO 63110, USA.</p><p class="abstract">The physical map of the 100-Mb Caenorhabditis elegans genome consists of 17,500 cosmids and 3500 yeast artificial chromosomes (YACs). A total of 22.5 Mb has been sequenced, with the remainder expected by 1998. A further 15.5 Mb of unfinished sequence is freely available online: because the areas sequenced so far are relatively gene rich, about half the 13,000 genes can now be scanned. More than a quarter of the genes are represented by expressed sequence tags (ESTs). All information pertaining to the genome is publicly available in the ACeDB data base.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1995</span>;<span class="volume">92</span>;<span class="number">24</span>;<span class="pagerange">10836-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7479894" rel="external">7479894</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC40526" rel="external">40526</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">41 kilobases of analyzed sequence from the pseudoautosomal and sex-determining regions of the short arm of the human Y chromosome.</h4>
<p class="authors"><span class="author">Whitfield LS</span>, <span class="author">Hawkins TL</span>, <span class="author">Goodfellow PN</span> and <span class="author">Sulston J</span></p>
<p class="affiliation">Department of Genetics, University of Cambridge, United Kingdom.</p><p class="abstract">Determination of 41.2 kb of Y chromosome genomic sequence has been made from a cosmid that spans the Yp pseudoautosomal boundary and includes 18.5 kb of sequence from the patient-defined sex-determining region of the Y chromosome. An AceDB database of the sequence and the analysis data have been produced as a resource for studies of the evolution and population genetics of the Y chromosome. Comparison of the 18.5 kb from the sex determining region to the sex determining region of mouse does not locate any areas of similarity outside SRY/Sry. Indeed, no coding regions other than those previously reported can be detected anywhere in the 41 kb. The Y-specific and pseudoautosomal portions of this sequence have different repeat sequence and GC contents: this may have relevance both to the events defining the pseudoautosomal boundary and to the course of sequence evolution in the absence of recombination.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1995</span>;<span class="volume">27</span>;<span class="number">2</span>;<span class="pagerange">306-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7557997" rel="external">7557997</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence variation of the human Y chromosome.</h4>
<p class="authors"><span class="author">Whitfield LS</span>, <span class="author">Sulston JE</span> and <span class="author">Goodfellow PN</span></p>
<p class="affiliation">Department of Genetics, University of Cambridge, UK.</p><p class="abstract">We have generated over 100 kilobases of sequence from the nonrecombining portion of the Y chromosomes from five humans and one common chimpanzee. The human subjects were chosen to match the earliest branches of the human mitochondrial tree. The survey of 18.3 kilobases from each human detected only three sites at which substitutions were present, whereas the human and chimpanzee sequences showed 1.3% divergence. The coalescence time estimated from our Y chromosome sample is more recent than that of the mitochondrial genome. A recent coalescence time for the Y chromosome could have been caused by the selected sweep of an advantageous Y chromosome or extensive migration of human males.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1995</span>;<span class="volume">378</span>;<span class="number">6555</span>;<span class="pagerange">379-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7477372">7477372</a>; DOI: <a href="http://dx.doi.org/10.1038%2F378379a0">10.1038/378379a0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of the breast cancer susceptibility gene BRCA2.</h4>
<p class="authors"><span class="author">Wooster R</span>, <span class="author">Bignell G</span>, <span class="author">Lancaster J</span>, <span class="author">Swift S</span>, <span class="author">Seal S</span>, <span class="author">Mangion J</span>, <span class="author">Collins N</span>, <span class="author">Gregory S</span>, <span class="author">Gumbs C</span> and <span class="author">Micklem G</span></p>
<p class="affiliation">Section of Molecular Carcinogenesis, Haddow Laboratories, Sutton Surrey, UK.</p><p class="abstract">In Western Europe and the United States approximately 1 in 12 women develop breast cancer. A small proportion of breast cancer cases, in particular those arising at a young age, are attributable to a highly penetrant, autosomal dominant predisposition to the disease. The breast cancer susceptibility gene, BRCA2, was recently localized to chromosome 13q12-q13. Here we report the identification of a gene in which we have detected six different germline mutations in breast cancer families that are likely to be due to BRCA2. Each mutation causes serious disruption to the open reading frame of the transcriptional unit. The results indicate that this is the BRCA2 gene.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1995</span>;<span class="volume">378</span>;<span class="number">6559</span>;<span class="pagerange">789-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8524414">8524414</a>; DOI: <a href="http://dx.doi.org/10.1038%2F378789a0">10.1038/378789a0</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/igspk1h6"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/igspk1h6" src="/qr/igspk1h6.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:47 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
