<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 2004 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 2004.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 2004</h1><p>Number of papers published in 2004: <strong>224</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Regions syntenic to human 17q are gained in mouse and rat neuroblastoma.</h4>
<p class="authors"><span class="author">&#x141;astowska M</span>, <span class="author">Chung YJ</span>, <span class="author">Cheng Ching N</span>, <span class="author">Haber M</span>, <span class="author">Norris MD</span>, <span class="author">Kees UR</span>, <span class="author">Pearson AD</span> and <span class="author">Jackson MS</span></p>
<p class="affiliation">Institute of Human Genetics, International Centre for Life, University of Newcastle upon Tyne, Newcastle upon Tyne, United Kingdom. <a href="mailto:%6D.%61.%6C%61%73%74%6F%77%73%6B%61%40%6E%63%6C.%61%63.%75%6B">&#109;&#46;&#97;&#46;&#108;&#97;&#115;&#116;&#111;&#119;&#115;&#107;&#97;&#64;&#110;&#99;&#108;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Gain of chromosome arm 17q is the most frequent chromosomal change in human neuroblastoma and is a powerful predictor of adverse outcome of disease. This suggests that the region of gain includes a gene or genes critical for tumor pathogenesis. Analyses of breakpoint positions have revealed that the shortest region of gain (SRG) extends from MPO (17q23.1) to 17qter. Because this encompasses &gt;300 genes, it precludes the identification of candidate genes from human breakpoint data alone. However, mouse chromosome 11, which is syntenic to human chromosome 17, is gained in up to 30% of neuroblastoma tumors developed in a murine MYCN transgenic model of this disease. To confirm that this key genetic change indicates the involvement of a molecular pathway conserved between mouse and man and is not occurring coincidentally in the transgenic model, we used fluorescence in situ hybridization to analyze sporadic cases of both mouse and rat neuroblastoma. Our results confirmed the presence of chromosome 11 gain in all three of the mouse cell lines we analyzed, with the SRG extending from Stat5b (101.6 Mb) to tel. In addition, the rat neuroblastoma cell line harbors an extra copy of distal chromosome 10, extending from 92.8 to 109.3 Mb, which is also syntenic to human 17q. Comparison of the regions gained in all three species has excluded 4.2 Mb from the previously defined region of 17q gain in humans as a likely location of the candidate gene or genes, and strongly suggests that the molecular etiology of neuroblastoma is similar in all three species.</p>
<p class="publication"><span class="title">Genes, chromosomes &amp; cancer</span>&nbsp;<span class="year">2004</span>;<span class="volume">40</span>;<span class="number">2</span>;<span class="pagerange">158-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15101050" rel="external">15101050</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fgcc.20031" rel="external">10.1002/gcc.20031</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microevolution and history of the plague bacillus, Yersinia pestis.</h4>
<p class="authors"><span class="author">Achtman M</span>, <span class="author">Morelli G</span>, <span class="author">Zhu P</span>, <span class="author">Wirth T</span>, <span class="author">Diehl I</span>, <span class="author">Kusecek B</span>, <span class="author">Vogler AJ</span>, <span class="author">Wagner DM</span>, <span class="author">Allender CJ</span>, <span class="author">Easterday WR</span>, <span class="author">Chenal-Francisque V</span>, <span class="author">Worsham P</span>, <span class="author">Thomson NR</span>, <span class="author">Parkhill J</span>, <span class="author">Lindler LE</span>, <span class="author">Carniel E</span> and <span class="author">Keim P</span></p>
<p class="affiliation">Department of Molecular Biology, Max-Planck Institut f&uuml;r Infektionsbiologie, D-10117 Berlin, Germany. <a href="mailto:%61%63%68%74%6D%61%6E%40%6D%70%6C%69%62-%62%65%72%6C%69%6E.%6D%70%67.%64%65">&#97;&#99;&#104;&#116;&#109;&#97;&#110;&#64;&#109;&#112;&#108;&#105;&#98;&#45;&#98;&#101;&#114;&#108;&#105;&#110;&#46;&#109;&#112;&#103;&#46;&#100;&#101;</a></p><p class="abstract">The association of historical plague pandemics with Yersinia pestis remains controversial, partly because the evolutionary history of this largely monomorphic bacterium was unknown. The microevolution of Y. pestis was therefore investigated by three different multilocus molecular methods, targeting genomewide synonymous SNPs, variation in number of tandem repeats, and insertion of IS100 insertion elements. Eight populations were recognized by the three methods, and we propose an evolutionary tree for these populations, rooted on Yersinia pseudotuberculosis. The tree invokes microevolution over millennia, during which enzootic pestoides isolates evolved. This initial phase was followed by a binary split 6,500 years ago, which led to populations that are more frequently associated with human disease. These populations do not correspond directly to classical biovars that are based on phenotypic properties. Thus, we recommend that henceforth groupings should be based on molecular signatures. The age of Y. pestis inferred here is compatible with the dates of historical pandemic plague. However, it is premature to infer an association between any modern molecular grouping and a particular pandemic wave that occurred before the 20th century.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01-GM060795</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">51</span>;<span class="pagerange">17837-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15598742" rel="external">15598742</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC535704" rel="external">535704</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0408026101" rel="external">10.1073/pnas.0408026101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutagenic insertion and chromosome engineering resource (MICER).</h4>
<p class="authors"><span class="author">Adams DJ</span>, <span class="author">Biggs PJ</span>, <span class="author">Cox T</span>, <span class="author">Davies R</span>, <span class="author">van der Weyden L</span>, <span class="author">Jonkers J</span>, <span class="author">Smith J</span>, <span class="author">Plumb B</span>, <span class="author">Taylor R</span>, <span class="author">Nishijima I</span>, <span class="author">Yu Y</span>, <span class="author">Rogers J</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambs, CB10 1SA, UK.</p><p class="abstract">Embryonic stem cell technology revolutionized biology by providing a means to assess mammalian gene function in vivo. Although it is now routine to generate mice from embryonic stem cells, one of the principal methods used to create mutations, gene targeting, is a cumbersome process. Here we describe the indexing of 93,960 ready-made insertional targeting vectors from two libraries. 5,925 of these vectors can be used directly to inactivate genes with an average targeting efficiency of 28%. Combinations of vectors from the two libraries can be used to disrupt both alleles of a gene or engineer larger genomic changes such as deletions, duplications, translocations or inversions. These indexed vectors constitute a public resource (Mutagenic Insertion and Chromosome Engineering Resource; MICER) for high-throughput, targeted manipulation of the mouse genome.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">8</span>;<span class="pagerange">867-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15235602">15235602</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1388">10.1038/ng1388</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An ENU-induced mutation in AP-2alpha leads to middle ear and ocular defects in Doarad mice.</h4>
<p class="authors"><span class="author">Ahituv N</span>, <span class="author">Erven A</span>, <span class="author">Fuchs H</span>, <span class="author">Guy K</span>, <span class="author">Ashery-Padan R</span>, <span class="author">Williams T</span>, <span class="author">de Angelis MH</span>, <span class="author">Avraham KB</span> and <span class="author">Steel KP</span></p>
<p class="affiliation">Department of Human Genetics and Molecular Medicine, Sackler School of Medicine, Tel Aviv University, 69978, Tel Aviv, Israel.</p><p class="abstract">One of the advantages of N-ethyl- N-nitrosourea (ENU)-induced mutagenesis is that, after randomly causing point mutations, a variety of alleles can be generated in genes leading to diverse phenotypes. For example, transcription factor AP-2alpha ( Tcfap2a) null homozygote mice show a large spectrum of developmental defects, among them missing middle ear bones and tympanic ring. This is the usual occurrence, where mutations causing middle ear anomalies usually coincide with other abnormalities. Using ENU-induced mutagenesis, we discovered a new dominant Tcfap2a mutant named Doarad ( Dor) that has a missense mutation in the PY motif of its transactivation domain, leading to a misshapen malleus, incus, and stapes without any other observable phenotype. Dor homozygous mice die perinatally, showing prominent abnormal facial structures and ocular defects. In vitro assays suggest that this mutation causes a &quot;gain of function&quot; in the transcriptional activation of AP-2alpha. These mice enable us to address more specifically the developmental role of Tcfap2a in the eye and middle ear and are the first report of a mutation in a gene specifically causing middle ear abnormalities, leading to conductive hearing loss.</p>
<p class="grants">Funded by: NIDCR NIH HHS: <span class="grant">DE12728</span></p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2004</span>;<span class="volume">15</span>;<span class="number">6</span>;<span class="pagerange">424-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15181535" rel="external">15181535</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-004-2334-z" rel="external">10.1007/s00335-004-2334-z</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Expression profiling of the Leishmania life cycle: cDNA arrays identify developmentally regulated genes present but not annotated in the genome.</h4>
<p class="authors"><span class="author">Almeida R</span>, <span class="author">Gilmartin BJ</span>, <span class="author">McCann SH</span>, <span class="author">Norrish A</span>, <span class="author">Ivens AC</span>, <span class="author">Lawson D</span>, <span class="author">Levick MP</span>, <span class="author">Smith DF</span>, <span class="author">Dyall SD</span>, <span class="author">Vetrie D</span>, <span class="author">Freeman TC</span>, <span class="author">Coulson RM</span>, <span class="author">Sampaio I</span>, <span class="author">Schneider H</span> and <span class="author">Blackwell JM</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Wellcome Trust/MRC Building, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB2 2XY, UK.</p><p class="abstract">As genomic sequencing of Leishmania nears completion, functional analyses that provide a global genetic perspective on biological processes are important. Despite polycistronic transcription, RNA transcript abundance can be measured using microarrays. To provide a resource to evaluate cDNA arrays, we undertook 5&#39; expressed sequence tag analysis of 2183 full-length randomly selected cDNAs from Leishmania major promastigote (days 3, 7, 10 of culture in vitro), and lesion-derived amastigote libraries. PCR-amplified inserts from 1830 of these cDNA representing 1001 unique genes were spotted onto microarrays, and compared internally with PCR-amplified open reading frames (ORFs) from 904 genes representing 842 unique genes annotated in the L. major genome. Microarrays were screened with RNA from procyclic, metacyclic and amastigote populations of L. major. Redundant clones on the array gave highly reproducible results, providing confidence in identification of stage-specific gene expression. Four hundred and thirty unique (i.e. non-redundant) stage-specific genes were identified. A higher percentage of stage-specific gene expression was observed in amastigotes ( approximately 35%) compared to metacyclics ( approximately 12%) for both cDNAs and ORFs, but cDNAs provided a richer source of regulated genes than currently annotated ORFs from the Leishmania genome. In mapping cDNAs onto the Leishmania genome, we noted that approximately 42% aligned to regions not recognised as genes using current predictive annotation tools. These genes are highly represented in our stage-specific genes, and therefore represent important drug targets and vaccine candidates. Careful annotation of cDNAs onto the Leishmania genome will be important before producing the next generation of oligonucleotide arrays based on annotated genes of the genomic sequencing project.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">061343</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">136</span>;<span class="number">1</span>;<span class="pagerange">87-100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15138070" rel="external">15138070</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2004.03.004" rel="external">10.1016/j.molbiopara.2004.03.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCOP database in 2004: refinements integrate structure and sequence family data.</h4>
<p class="authors"><span class="author">Andreeva A</span>, <span class="author">Howorth D</span>, <span class="author">Brenner SE</span>, <span class="author">Hubbard TJ</span>, <span class="author">Chothia C</span> and <span class="author">Murzin AG</span></p>
<p class="affiliation">MRC Centre for Protein Engineering, Hills Road, Cambridge CB2 2QH, UK.</p><p class="abstract">The Structural Classification of Proteins (SCOP) database is a comprehensive ordering of all proteins of known structure, according to their evolutionary and structural relationships. Protein domains in SCOP are hierarchically classified into families, superfamilies, folds and classes. The continual accumulation of sequence and structural data allows more rigorous analysis and provides important information for understanding the protein world and its evolutionary repertoire. SCOP participates in a project that aims to rationalize and integrate the data on proteins held in several sequence and structure databases. As part of this project, starting with release 1.63, we have initiated a refinement of the SCOP classification, which introduces a number of changes mostly at the levels below superfamily. The pending SCOP reclassification will be carried out gradually through a number of future releases. In addition to the expanded set of static links to external resources, available at the level of domain entries, we have started modernization of the interface capabilities of SCOP allowing more dynamic links with other databases. SCOP can be accessed at http://scop.mrc-lmb.cam.ac.uk/scop.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D226-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681400">14681400</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308773">308773</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh039">10.1093/nar/gkh039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Strong positive selection and recombination drive the antigenic variation of the PilE protein of the human pathogen Neisseria meningitidis.</h4>
<p class="authors"><span class="author">Andrews TD</span> and <span class="author">Gojobori T</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, CB10 1SA, United Kingdom.</p><p class="abstract">The PilE protein is the major component of the Neisseria meningitidis pilus, which is encoded by the pilE/pilS locus that includes an expressed gene and eight homologous silent fragments. The silent gene fragments have been shown to recombine through gene conversion with the expressed gene and thereby provide a means by which novel antigenic variants of the PilE protein can be generated. We have analyzed the evolutionary rate of the pilE gene using the nucleotide sequence of two complete pilE/pilS loci. The very high rate of evolution displayed by the PilE protein appears driven by both recombination and positive selection. Within the semivariable region of the pilE and pilS genes, recombination appears to occur within multiple small sequence blocks that lie between conserved sequence elements. Within the hypervariable region, positive selection was identified from comparison of the silent and expressed genes. The unusual gene conversion mechanism that operates at the pilE/pilS locus is a strategy employed by N. meningitidis to enhance mutation of certain regions of the PilE protein. The silent copies of the gene effectively allow &quot;parallelized&quot; evolution of pilE, thus enabling the encoded protein to rapidly explore a large area of sequence space in an effort to find novel antigenic variants.</p>
<p class="publication"><span class="title">Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">166</span>;<span class="number">1</span>;<span class="pagerange">25-32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15020403" rel="external">15020403</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1470718" rel="external">1470718</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome 21 and down syndrome: from genomics to pathophysiology.</h4>
<p class="authors"><span class="author">Antonarakis SE</span>, <span class="author">Lyle R</span>, <span class="author">Dermitzakis ET</span>, <span class="author">Reymond A</span> and <span class="author">Deutsch S</span></p>
<p class="affiliation">Department of Genetic Medicine and Development, University of Geneva Medical School and University Hospitals of Geneva, 1 rue Michel-Servet, 1211 Geneva, Switzerland. <a href="mailto:%53%74%79%6C%69%61%6E%6F%73.%41%6E%74%6F%6E%61%72%61%6B%69%73%40%6D%65%64%65%63%69%6E%65.%75%6E%69%67%65.%63%68">&#83;&#116;&#121;&#108;&#105;&#97;&#110;&#111;&#115;&#46;&#65;&#110;&#116;&#111;&#110;&#97;&#114;&#97;&#107;&#105;&#115;&#64;&#109;&#101;&#100;&#101;&#99;&#105;&#110;&#101;&#46;&#117;&#110;&#105;&#103;&#101;&#46;&#99;&#104;</a></p><p class="abstract">The sequence of chromosome 21 was a turning point for the understanding of Down syndrome. Comparative genomics is beginning to identify the functional components of the chromosome and that in turn will set the stage for the functional characterization of the sequences. Animal models combined with genome-wide analytical methods have proved indispensable for unravelling the mysteries of gene dosage imbalance.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">725-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15510164">15510164</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1448">10.1038/nrg1448</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Domain insertions in protein structures.</h4>
<p class="authors"><span class="author">Aroul-Selvam R</span>, <span class="author">Hubbard T</span> and <span class="author">Sasidharan R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Domains are the structural, functional or evolutionary units of proteins. Proteins can comprise a single domain or a combination of domains. In multi-domain proteins, the domains almost always occur end-to-end, i.e., one domain follows the C-terminal end of another domain. However, there are exceptions to this common pattern, where multi-domain proteins are formed by insertion of one domain (insert) into another domain (parent). Here, we provide a quantitative description of known insertions in the Protein Data Bank (PDB). We found that 9% of domain combinations observed in non-redundant PDB are insertions. Although 90% of all insertions involve only one insert, proteins can clearly have multiple (nested, two-domain and three-domain) inserts. We also observed correlations between the structure and function of a domain and its tendency to be found as a parent or an insert. There is a bias in insert position towards the C terminus of parents. We observed that the atomic distance between the N and C terminus of an insert is significantly smaller when compared to the N-to-C distance in a parent context or a single domain context. Insertions are found always to occur in loop regions of parent domains. Our observations regarding the relationship between domain insertions and the structure, function and evolution of proteins have implications for protein engineering.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">338</span>;<span class="number">4</span>;<span class="pagerange">633-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15099733">15099733</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665287">2665287</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jmb.2004.03.039">10.1016/j.jmb.2004.03.039</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A predominantly neolithic origin for Y-chromosomal DNA variation in North Africa.</h4>
<p class="authors"><span class="author">Arredi B</span>, <span class="author">Poloni ES</span>, <span class="author">Paracchini S</span>, <span class="author">Zerjal T</span>, <span class="author">Fathallah DM</span>, <span class="author">Makrelouf M</span>, <span class="author">Pascali VL</span>, <span class="author">Novelletto A</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Istituto di Medicina Legale, Universit&agrave; Cattolica del Sacro Cuore di Roma, Rome, Italy. <a href="mailto:%62%5F%61%72%72%65%64%69%40%6C%69%62%65%72%6F.%69%74">&#98;&#95;&#97;&#114;&#114;&#101;&#100;&#105;&#64;&#108;&#105;&#98;&#101;&#114;&#111;&#46;&#105;&#116;</a></p><p class="abstract">We have typed 275 men from five populations in Algeria, Tunisia, and Egypt with a set of 119 binary markers and 15 microsatellites from the Y chromosome, and we have analyzed the results together with published data from Moroccan populations. North African Y-chromosomal diversity is geographically structured and fits the pattern expected under an isolation-by-distance model. Autocorrelation analyses reveal an east-west cline of genetic variation that extends into the Middle East and is compatible with a hypothesis of demic expansion. This expansion must have involved relatively small numbers of Y chromosomes to account for the reduction in gene diversity towards the West that accompanied the frequency increase of Y haplogroup E3b2, but gene flow must have been maintained to explain the observed pattern of isolation-by-distance. Since the estimates of the times to the most recent common ancestor (TMRCAs) of the most common haplogroups are quite recent, we suggest that the North African pattern of Y-chromosomal variation is largely of Neolithic origin. Thus, we propose that the Neolithic transition in this part of the world was accompanied by demic diffusion of Afro-Asiatic-speaking pastoralists from the Middle East.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">75</span>;<span class="number">2</span>;<span class="pagerange">338-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15202071" rel="external">15202071</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1216069" rel="external">1216069</a>; DOI: <a href="http://dx.doi.org/10.1086%2F423147" rel="external">10.1086/423147</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Caenorhabditis elegans functional genomics: omic resonance.</h4>
<p class="authors"><span class="author">Astin J</span>, <span class="author">Merry A</span>, <span class="author">Rajan J</span> and <span class="author">Kuwabara PE</span></p>
<p class="affiliation">Department of Biochemistry, University of Bristol, The School of Medical Sciences, University Walk, Bristol BS8 1TD, UK.</p><p class="abstract">The nematode Caenorhabditis elegans is widely used as a model organism for studying many fundamental aspects of development and cell biology, including processes underlying human disease. The genome of C. elegans encodes over 19,000 protein-coding genes and hundreds of non-coding RNAs. The availability of whole genome sequence has facilitated the development of high throughput techniques for elucidating the function of individual genes and gene products. Furthermore, attempts can now be made to integrate these substantial functional genomics data collections and to understand at a global level how the flow of genomic information that is at the core of the central dogma leads to the development of a multicellular organism.</p>
<p class="publication"><span class="title">Briefings in functional genomics &amp; proteomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">26-34</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15163357" rel="external">15163357</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The knockout mouse project.</h4>
<p class="authors"><span class="author">Austin CP</span>, <span class="author">Battey JF</span>, <span class="author">Bradley A</span>, <span class="author">Bucan M</span>, <span class="author">Capecchi M</span>, <span class="author">Collins FS</span>, <span class="author">Dove WF</span>, <span class="author">Duyk G</span>, <span class="author">Dymecki S</span>, <span class="author">Eppig JT</span>, <span class="author">Grieder FB</span>, <span class="author">Heintz N</span>, <span class="author">Hicks G</span>, <span class="author">Insel TR</span>, <span class="author">Joyner A</span>, <span class="author">Koller BH</span>, <span class="author">Lloyd KC</span>, <span class="author">Magnuson T</span>, <span class="author">Moore MW</span>, <span class="author">Nagy A</span>, <span class="author">Pollock JD</span>, <span class="author">Roses AD</span>, <span class="author">Sands AT</span>, <span class="author">Seed B</span>, <span class="author">Skarnes WC</span>, <span class="author">Snoddy J</span>, <span class="author">Soriano P</span>, <span class="author">Stewart DJ</span>, <span class="author">Stewart F</span>, <span class="author">Stillman B</span>, <span class="author">Varmus H</span>, <span class="author">Varticovski L</span>, <span class="author">Verma IM</span>, <span class="author">Vogt TF</span>, <span class="author">von Melchner H</span>, <span class="author">Witkowski J</span>, <span class="author">Woychik RP</span>, <span class="author">Wurst W</span>, <span class="author">Yancopoulos GD</span>, <span class="author">Young SG</span> and <span class="author">Zambrowicz B</span></p>
<p class="affiliation">National Human Genome Research Institute, National Institutes of Health, Building 31, Room 4B09, 31 Center Drive, Bethesda, Maryland 20892, USA. <a href="mailto:%61%75%73%74%69%6E%63%40%6D%61%69%6C.%6E%69%68.%67%6F%76">&#97;&#117;&#115;&#116;&#105;&#110;&#99;&#64;&#109;&#97;&#105;&#108;&#46;&#110;&#105;&#104;&#46;&#103;&#111;&#118;</a></p><p class="abstract">Mouse knockout technology provides a powerful means of elucidating gene function in vivo, and a publicly available genome-wide collection of mouse knockouts would be significantly enabling for biomedical discovery. To date, published knockouts exist for only about 10% of mouse genes. Furthermore, many of these are limited in utility because they have not been made or phenotyped in standardized ways, and many are not freely available to researchers. It is time to harness new technologies and efficiencies of production to mount a high-throughput international effort to produce and phenotype knockouts for all mouse genes, and place these resources into the public domain.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077188</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">9</span>;<span class="pagerange">921-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15340423">15340423</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716027">2716027</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0904-921">10.1038/ng0904-921</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The European dimension for the mouse genome mutagenesis program.</h4>
<p class="authors"><span class="author">Auwerx J</span>, <span class="author">Avner P</span>, <span class="author">Baldock R</span>, <span class="author">Ballabio A</span>, <span class="author">Balling R</span>, <span class="author">Barbacid M</span>, <span class="author">Berns A</span>, <span class="author">Bradley A</span>, <span class="author">Brown S</span>, <span class="author">Carmeliet P</span>, <span class="author">Chambon P</span>, <span class="author">Cox R</span>, <span class="author">Davidson D</span>, <span class="author">Davies K</span>, <span class="author">Duboule D</span>, <span class="author">Forejt J</span>, <span class="author">Granucci F</span>, <span class="author">Hastie N</span>, <span class="author">de Angelis MH</span>, <span class="author">Jackson I</span>, <span class="author">Kioussis D</span>, <span class="author">Kollias G</span>, <span class="author">Lathrop M</span>, <span class="author">Lendahl U</span>, <span class="author">Malumbres M</span>, <span class="author">von Melchner H</span>, <span class="author">M&uuml;ller W</span>, <span class="author">Partanen J</span>, <span class="author">Ricciardi-Castagnoli P</span>, <span class="author">Rigby P</span>, <span class="author">Rosen B</span>, <span class="author">Rosenthal N</span>, <span class="author">Skarnes B</span>, <span class="author">Stewart AF</span>, <span class="author">Thornton J</span>, <span class="author">Tocchini-Valentini G</span>, <span class="author">Wagner E</span>, <span class="author">Wahli W</span> and <span class="author">Wurst W</span></p>
<p class="affiliation">Mouse Clinical Institute (MCI), Illkirch, Strasbourg, France [corrected].</p><p class="abstract">The European Mouse Mutagenesis Consortium is the European initiative contributing to the international effort on functional annotation of the mouse genome. Its objectives are to establish and integrate mutagenesis platforms, gene expression resources, phenotyping units, storage and distribution centers and bioinformatics resources. The combined efforts will accelerate our understanding of gene function and of human health and disease.</p>
<p class="grants">Funded by: Medical Research Council: <span class="grant">MC_U127527203</span>; Telethon: <span class="grant">TGM03S01</span>, <span class="grant">TGM06S01</span>; Wellcome Trust: <span class="grant">077188</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">9</span>;<span class="pagerange">925-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15340424">15340424</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716028">2716028</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0904-925">10.1038/ng0904-925</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Burkholderia cepacia epidemic strain marker is part of a novel genomic island encoding both virulence and metabolism-associated genes in Burkholderia cenocepacia.</h4>
<p class="authors"><span class="author">Baldwin A</span>, <span class="author">Sokol PA</span>, <span class="author">Parkhill J</span> and <span class="author">Mahenthiralingam E</span></p>
<p class="affiliation">Cardiff School of Biosciences, Cardiff University, Cardiff CF10 3TL, Wales, Canada TN2 4N1.</p><p class="abstract">The Burkholderia cepacia epidemic strain marker (BCESM) is a useful epidemiological marker for virulent B. cenocepacia strains that infect patients with cystic fibrosis. However, there was no evidence that the original marker, identified by random amplified polymorphic DNA fingerprinting, contributed to pathogenicity. Here we demonstrate that the BCESM is part of a novel genomic island encoding genes linked to both virulence and metabolism. The BCESM was present on a 31.7-kb low-GC-content island that encoded 35 predicted coding sequences (CDSs): an N-acyl homoserine lactone (AHL) synthase gene (cciI) and corresponding transcriptional regulator (cciR), representing the first time cell signaling genes have been found on a genomic island; fatty acid biosynthesis genes; an IS66 family transposase; transcriptional regulator CDSs; amino acid metabolism genes; and a group of hypothetical genes. Mutagenesis of the AHL synthase, amidase (amiI), and porin (opcI) genes on the island was carried out. Testing of the isogenic mutants in a rat model of chronic lung infection demonstrated that the amidase played a role in persistence, while the AHL synthase and porin were both involved in virulence. The island, designated the B. cenocepacia island (cci), is the first genomic island to be defined in the B. cepacia complex and its discovery validates the original epidemiological correlation of the BCESM with virulent CF strains. The features of the cci, which overlap both pathogenicity and metabolism, expand the concept of bacterial pathogenicity islands and illustrate the diversity of accessory functions that can be acquired by lateral gene transfer in bacteria.</p>
<p class="publication"><span class="title">Infection and immunity</span>&nbsp;<span class="year">2004</span>;<span class="volume">72</span>;<span class="number">3</span>;<span class="pagerange">1537-47</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14977960" rel="external">14977960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC356040" rel="external">356040</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website.</h4>
<p class="authors"><span class="author">Bamford S</span>, <span class="author">Dawson E</span>, <span class="author">Forbes S</span>, <span class="author">Clements J</span>, <span class="author">Pettett R</span>, <span class="author">Dogan A</span>, <span class="author">Flanagan A</span>, <span class="author">Teague J</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The discovery of mutations in cancer genes has advanced our understanding of cancer. These results are dispersed across the scientific literature and with the availability of the human genome sequence will continue to accrue. The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website have been developed to store somatic mutation data in a single location and display the data and other information related to human cancer. To populate this resource, data has currently been extracted from reports in the scientific literature for somatic mutations in four genes, BRAF, HRAS, KRAS2 and NRAS. At present, the database holds information on 66 634 samples and reports a total of 10 647 mutations. Through the web pages, these data can be queried, displayed as figures or tables and exported in a number of formats. COSMIC is an ongoing project that will continue to curate somatic mutation data and release it through the website.</p>
<p class="publication"><span class="title">British journal of cancer</span>&nbsp;<span class="year">2004</span>;<span class="volume">91</span>;<span class="number">2</span>;<span class="pagerange">355-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15188009">15188009</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2409828">2409828</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.bjc.6601894">10.1038/sj.bjc.6601894</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bioinformatics of proteases in the MEROPS database.</h4>
<p class="authors"><span class="author">Barrett AJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 ISA, UK. <a href="mailto:%61%6C%61%6E.%62%61%72%72%65%74%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#108;&#97;&#110;&#46;&#98;&#97;&#114;&#114;&#101;&#116;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Proteolytic enzymes represent approximately approximately 2% of the total number of proteins present in all types of organisms. Many of these enzymes are of medical importance, and those that are of potential interest as drug targets can be divided into the endogenous enzymes encoded in the human genome, and the exogenous proteases encoded in the genomes of disease-causing organisms. There are also naturally occurring inhibitors of proteases, some of which have pharmaceutical relevance. The MEROPS database provides a rich source of information on proteases and their inhibitors. Storage and retrieval of this information is facilitated by the use of a hierarchical classification system (which was pioneered by the compilers of the database) in which homologous proteases and their inhibitors are divided into clans and families.</p>
<p class="publication"><span class="title">Current opinion in drug discovery &amp; development</span>&nbsp;<span class="year">2004</span>;<span class="volume">7</span>;<span class="number">3</span>;<span class="pagerange">334-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15216937">15216937</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Pfam protein families database.</h4>
<p class="authors"><span class="author">Bateman A</span>, <span class="author">Coin L</span>, <span class="author">Durbin R</span>, <span class="author">Finn RD</span>, <span class="author">Hollich V</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Khanna A</span>, <span class="author">Marshall M</span>, <span class="author">Moxon S</span>, <span class="author">Sonnhammer EL</span>, <span class="author">Studholme DJ</span>, <span class="author">Yeats C</span> and <span class="author">Eddy SR</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Pfam is a large collection of protein families and domains. Over the past 2 years the number of families in Pfam has doubled and now stands at 6190 (version 10.0). Methodology improvements for searching the Pfam collection locally as well as via the web are described. Other recent innovations include modelling of discontinuous domains allowing Pfam domain definitions to be closer to those found in structure databases. Pfam is available on the web in the UK (http://www.sanger.ac.uk/Software/Pfam/), the USA (http://pfam.wustl.edu/), France (http://pfam.jouy.inra.fr/) and Sweden (http://Pfam.cgb.ki.se/).</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D138-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681378" rel="external">14681378</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308855" rel="external">308855</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh121" rel="external">10.1093/nar/gkh121</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of the enterobacterial phytopathogen Erwinia carotovora subsp. atroseptica and characterization of virulence factors.</h4>
<p class="authors"><span class="author">Bell KS</span>, <span class="author">Sebaihia M</span>, <span class="author">Pritchard L</span>, <span class="author">Holden MT</span>, <span class="author">Hyman LJ</span>, <span class="author">Holeva MC</span>, <span class="author">Thomson NR</span>, <span class="author">Bentley SD</span>, <span class="author">Churcher LJ</span>, <span class="author">Mungall K</span>, <span class="author">Atkin R</span>, <span class="author">Bason N</span>, <span class="author">Brooks K</span>, <span class="author">Chillingworth T</span>, <span class="author">Clark K</span>, <span class="author">Doggett J</span>, <span class="author">Fraser A</span>, <span class="author">Hance Z</span>, <span class="author">Hauser H</span>, <span class="author">Jagels K</span>, <span class="author">Moule S</span>, <span class="author">Norbertczak H</span>, <span class="author">Ormond D</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Sanders M</span>, <span class="author">Walker D</span>, <span class="author">Whitehead S</span>, <span class="author">Salmond GP</span>, <span class="author">Birch PR</span>, <span class="author">Parkhill J</span> and <span class="author">Toth IK</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The bacterial family Enterobacteriaceae is notable for its well studied human pathogens, including Salmonella, Yersinia, Shigella, and Escherichia spp. However, it also contains several plant pathogens. We report the genome sequence of a plant pathogenic enterobacterium, Erwinia carotovora subsp. atroseptica (Eca) strain SCRI1043, the causative agent of soft rot and blackleg potato diseases. Approximately 33% of Eca genes are not shared with sequenced enterobacterial human pathogens, including some predicted to facilitate unexpected metabolic traits, such as nitrogen fixation and opine catabolism. This proportion of genes also contains an overrepresentation of pathogenicity determinants, including possible horizontally acquired gene clusters for putative type IV secretion and polyketide phytotoxin synthesis. To investigate whether these gene clusters play a role in the disease process, an arrayed set of insertional mutants was generated, and mutations were identified. Plant bioassays showed that these mutants were significantly reduced in virulence, demonstrating both the presence of novel pathogenicity determinants in Eca, and the impact of functional genomics in expanding our understanding of phytopathogenicity in the Enterobacteriaceae.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">30</span>;<span class="pagerange">11105-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15263089">15263089</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC503747">503747</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0402424101">10.1073/pnas.0402424101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of the imprinted polycomb gene L3MBTL, a candidate 20q tumour suppressor gene, in patients with myeloid malignancies.</h4>
<p class="authors"><span class="author">Bench AJ</span>, <span class="author">Li J</span>, <span class="author">Huntly BJ</span>, <span class="author">Delabesse E</span>, <span class="author">Fourouclas N</span>, <span class="author">Hunt AR</span>, <span class="author">Deloukas P</span> and <span class="author">Green AR</span></p>
<p class="affiliation">Department of Haematology, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 2XY, UK.</p><p class="abstract">Chromosome 20q deletion is a recurrent chromosomal abnormality associated with myeloid malignancies. L3MBTL represents a strong candidate tumour suppressor gene since it lies within the common deleted region, is a member of the Polycomb-like family, encodes the human homologue of a Drosophila tumour suppressor and is expressed within haematopoietic progenitor cells. We describe the structure of L3MBTL, identify two putative promoters each associated with two CpG islands and characterize a complex pattern of alternative splicing events. Mutation analysis of the gene in patients with and without a 20q deletion identified several polymorphisms but no acquired mutations. The two CpG islands spanning promoter 2 undergo monoallelic methylation in normal haematopoietic cells consistent with imprinting of L3MBTL. Samples from patients with a 20q deletion retained either the methylated or unmethylated allele but retention of the methylated allele did not correlate with reduction in L3MBTL mRNA levels. The absence of a correlation between L3MBTL methylation and transcription could be shown to reflect loss of imprinting in one patient. In addition, our results demonstrate that inactivation of L3MBTL is not a common occurrence in patients with a 20q deletion or in cytogenetically normal patients with polycythaemia vera.</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2004</span>;<span class="volume">127</span>;<span class="number">5</span>;<span class="pagerange">509-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15566354" rel="external">15566354</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2141.2004.05278.x" rel="external">10.1111/j.1365-2141.2004.05278.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomes for medicine.</h4>
<p class="authors"><span class="author">Bentley DR</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%72%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#114;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have the human genome sequence. It is freely available, accurate and nearly complete. But is the genome ready for medicine? The new resource is already changing genetic research strategies to find information of medical value. Now we need high-quality annotation of all the functionally important sequences and the variations within them that contribute to health and disease. To achieve this, we need more genome sequences, systematic experimental analyses, and extensive information on human phenotypes. Flexible and user-friendly access to well-annotated genomes will create an environment for innovation, and the potential for unlimited use of sequencing in biomedical research and practice.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">429</span>;<span class="number">6990</span>;<span class="pagerange">440-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15164068">15164068</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02622">10.1038/nature02622</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic pot pourri.</h4>
<p class="authors"><span class="author">Bentley S</span>, <span class="author">Crossman L</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">12</span>;<span class="pagerange">928-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15580690">15580690</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1051">10.1038/nrmicro1051</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic structure of prokaryotes.</h4>
<p class="authors"><span class="author">Bentley SD</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%73%64%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#100;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Recent advances in DNA-sequencing technologies have made available an enormous resource of data for the study of bacterial genomes. The broad sample of complete genomes currently available allows us to look at variation in the gross features and characteristics of genomes while the detail of the sequences reveal some of the mechanisms by which these genomes evolve. This review aims to describe bacterial genome structures according to current knowledge and proposed hypotheses. We also describe examples where mechanisms of genome evolution have acted in the adaptation of bacterial species to particular niches.</p>
<p class="publication"><span class="title">Annual review of genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">38</span>;<span class="pagerange">771-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15568993" rel="external">15568993</a>; DOI: <a href="http://dx.doi.org/10.1146%2Fannurev.genet.38.072902.094318" rel="external">10.1146/annurev.genet.38.072902.094318</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SCP1, a 356,023 bp linear plasmid adapted to the ecology and developmental biology of its host, Streptomyces coelicolor A3(2).</h4>
<p class="authors"><span class="author">Bentley SD</span>, <span class="author">Brown S</span>, <span class="author">Murphy LD</span>, <span class="author">Harris DE</span>, <span class="author">Quail MA</span>, <span class="author">Parkhill J</span>, <span class="author">Barrell BG</span>, <span class="author">McCormick JR</span>, <span class="author">Santamaria RI</span>, <span class="author">Losick R</span>, <span class="author">Yamasaki M</span>, <span class="author">Kinashi H</span>, <span class="author">Chen CW</span>, <span class="author">Chandra G</span>, <span class="author">Jakimowicz D</span>, <span class="author">Kieser HM</span>, <span class="author">Kieser T</span> and <span class="author">Chater KF</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The sequencing of the entire genetic complement of Streptomyces coelicolor A3(2) has been completed with the determination of the 365,023 bp sequence of the linear plasmid SCP1. Remarkably, the functional distribution of SCP1 genes somewhat resembles that of the chromosome: predicted gene products/functions include ECF sigma factors, antibiotic biosynthesis, a gamma-butyrolactone signalling system, members of the actinomycete-specific Wbl class of regulatory proteins and 14 secreted proteins. Some of these genes are among the 18 that contain a TTA codon, making them targets for the developmentally important tRNA encoded by the bldA gene. RNA analysis and gene fusions showed that one of the TTA-containing genes is part of a large bldA-dependent operon, the gene products of which include three proteins isolated from the spore surface by detergent washing (SapC, D and E), and several probable metabolic enzymes. SCP1 shows much evidence of recombinational interactions with other replicons and transposable elements during its history. For example, it has two sets of partitioning genes (which may explain why an integrated copy of SCP1 partially suppressed the defective partitioning of a parAB-deleted chromosome during sporulation). SCP1 carries a cluster of probable transfer determinants and genes encoding likely DNA polymerase III subunits, but it lacks an obvious candidate gene for the terminal protein associated with its ends. This may be related to atypical features of its end sequences.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">F32 G12961</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">51</span>;<span class="number">6</span>;<span class="pagerange">1615-28</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15009889" rel="external">15009889</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Data mining parasite genomes.</h4>
<p class="authors"><span class="author">Berriman M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, CB10 ISA, UK. <a href="mailto:%6D%62%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#98;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The term &#39;data mining&#39; can be used to describe any process where useful information is extracted from data with a large background of &#39;noise&#39;. In the context of a genome project, several stages involve data mining. Amongst the sequence data, &#39;signals&#39; need to be detected that indicate the presence of interesting features. Often this involves differentiating between transcribed and non-transcribed bases to predict coding regions. After detection, defining the roles of these sequences involves sifting through multiple lines of evidence. If these roles are accurately reflected in genome annotation, they can be used by researchers to frame queries and interrogate the data further.</p>
<p class="publication"><span class="title">Parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">128 Suppl 1</span>;<span class="pagerange">S23-31</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16454895" rel="external">16454895</a>; DOI: <a href="http://dx.doi.org/10.1017%2FS0031182004006857" rel="external">10.1017/S0031182004006857</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Annotation of parasite genomes.</h4>
<p class="authors"><span class="author">Berriman M</span> and <span class="author">Harris M</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract">Genome annotation is the application of useful biological descriptions to sequence data. Different levels of time and effort can be invested to produce correspondingly different depths of annotation depending on what methods are employed. Researchers using genome data should, therefore, understand how annotations are generated to assess their validity correctly and to determine what level of inferences can be made accordingly. Thorough annotation requires a large range of procedures, most of which involve manual reviews of all available evidence. First, gene structures often are computed algorithmically and edited based on in-depth analyses of the underlying sequence data. Second, functional predictions draw on data from various sources. Finally, the use of structured and controlled descriptions, such as those provided by gene ontology, can be used so that final descriptions are not only consistent and unambiguous, but capable of being used in further downstream analyses such as cross-species comparisons.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">270</span>;<span class="pagerange">17-44</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153621" rel="external">15153621</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-793-9%3A017" rel="external">10.1385/1-59259-793-9:017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Curation of the Plasmodium falciparum genome.</h4>
<p class="authors"><span class="author">Berry AE</span>, <span class="author">Gardner MJ</span>, <span class="author">Caspers GJ</span>, <span class="author">Roos DS</span> and <span class="author">Berriman M</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Hinxton Hall, Cambridge CB10 1SA, UK. <a href="mailto:%61%65%62%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#101;&#98;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The malaria genome has proved invaluable to researchers worldwide in the continuing fight against malaria by stimulating and underpinning molecular approaches in gene expression studies, vaccine and drug discovery research, and by providing data to facilitate hypothesis-driven research. The combination of in silico and experimental investigations has already yielded dividends by strengthening our understanding of the many facets of the malaria parasite Plasmodium falciparum. The recently initiated curation of the genome resource is a vital investment for maintaining and enhancing the use of this genomic information in the post-genomic era.</p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">12</span>;<span class="pagerange">548-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15522662" rel="external">15522662</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2004.09.003" rel="external">10.1016/j.pt.2004.09.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution analysis of DNA copy number using oligonucleotide microarrays.</h4>
<p class="authors"><span class="author">Bignell GR</span>, <span class="author">Huang J</span>, <span class="author">Greshock J</span>, <span class="author">Watt S</span>, <span class="author">Butler A</span>, <span class="author">West S</span>, <span class="author">Grigorova M</span>, <span class="author">Jones KW</span>, <span class="author">Wei W</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Weber B</span>, <span class="author">Shapero MH</span> and <span class="author">Wooster R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK.</p><p class="abstract">Genomic copy number alterations are a feature of many human diseases including cancer. We have evaluated the effectiveness of an oligonucleotide array, originally designed to detect single-nucleotide polymorphisms, to assess DNA copy number. We first showed that fluorescent signal from the oligonucleotide array varies in proportion to both decreases and increases in copy number. Subsequently we applied the system to a series of 20 cancer cell lines. All of the putative homozygous deletions (10) and high-level amplifications (12; putative copy number &gt;4) tested were confirmed by PCR (either qPCR or normal PCR) analysis. Low-level copy number changes for two of the lines under analysis were compared with BAC array CGH; 77% (n = 44) of the autosomal chromosomes used in the comparison showed consistent patterns of LOH (loss of heterozygosity) and low-level amplification. Of the remaining 10 comparisons that were discordant, eight were caused by low SNP densities and failed in both lines. The studies demonstrate that combining the genotype and copy number analyses gives greater insight into the underlying genetic alterations in cancer cells with identification of complex events including loss and reduplication of loci.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">2</span>;<span class="pagerange">287-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14762065" rel="external">14762065</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC327104" rel="external">327104</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2012304" rel="external">10.1101/gr.2012304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">BAC finishing strategies.</h4>
<p class="authors"><span class="author">Bird C</span> and <span class="author">Grafham D</span></p>
<p class="affiliation">Welcome Trust Genome Campus, The Sanger Institute, Cambridge, UK.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">255</span>;<span class="pagerange">255-77</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15020831" rel="external">15020831</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-752-1%3A255" rel="external">10.1385/1-59259-752-1:255</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biological database design and implementation.</h4>
<p class="authors"><span class="author">Birney E</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">EBI, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK. <a href="mailto:%62%69%72%6E%65%79%40%65%62%69.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present our experience of building biological databases. Such databases have most aspects in common with other complex databases in other fields. We do not believe that biological data are that different from complex data in other fields. Our experience has led us to emphasise simplicity and conservative technology choices when building these databases. This is a short paper of advice that we hope is useful to people designing their own biological database.</p>
<p class="publication"><span class="title">Briefings in bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">1</span>;<span class="pagerange">31-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153304" rel="external">15153304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Ensembl 2004.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Andrews D</span>, <span class="author">Bevan P</span>, <span class="author">Caccamo M</span>, <span class="author">Cameron G</span>, <span class="author">Chen Y</span>, <span class="author">Clarke L</span>, <span class="author">Coates G</span>, <span class="author">Cox T</span>, <span class="author">Cuff J</span>, <span class="author">Curwen V</span>, <span class="author">Cutts T</span>, <span class="author">Down T</span>, <span class="author">Durbin R</span>, <span class="author">Eyras E</span>, <span class="author">Fernandez-Suarez XM</span>, <span class="author">Gane P</span>, <span class="author">Gibbins B</span>, <span class="author">Gilbert J</span>, <span class="author">Hammond M</span>, <span class="author">Hotz H</span>, <span class="author">Iyer V</span>, <span class="author">Kahari A</span>, <span class="author">Jekosch K</span>, <span class="author">Kasprzyk A</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Lehvaslaiho H</span>, <span class="author">McVicker G</span>, <span class="author">Melsopp C</span>, <span class="author">Meidl P</span>, <span class="author">Mongin E</span>, <span class="author">Pettett R</span>, <span class="author">Potter S</span>, <span class="author">Proctor G</span>, <span class="author">Rae M</span>, <span class="author">Searle S</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Stabenau A</span>, <span class="author">Stalker J</span>, <span class="author">Storey R</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Woodwark C</span>, <span class="author">Clamp M</span> and <span class="author">Hubbard T</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The Ensembl (http://www.ensembl.org/) database project provides a bioinformatics framework to organize biology around the sequences of large genomes. It is a comprehensive and integrated source of annotation of large genome sequences, available via interactive website, web services or flat files. As well as being one of the leading sources of genome annotation, Ensembl is an open source software engineering project to develop a portable system able to handle very large genomes and associated requirements. The facilities of the system range from sequence analysis to data storage and visualization and installations exist around the world both in companies and at academic sites. With a total of nine genome sequences available from Ensembl and more genomes to follow, recent developments have focused mainly on closer integration between genomes and external data.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D468-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681459">14681459</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308772">308772</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh038">10.1093/nar/gkh038</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An overview of Ensembl.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Andrews TD</span>, <span class="author">Bevan P</span>, <span class="author">Caccamo M</span>, <span class="author">Chen Y</span>, <span class="author">Clarke L</span>, <span class="author">Coates G</span>, <span class="author">Cuff J</span>, <span class="author">Curwen V</span>, <span class="author">Cutts T</span>, <span class="author">Down T</span>, <span class="author">Eyras E</span>, <span class="author">Fernandez-Suarez XM</span>, <span class="author">Gane P</span>, <span class="author">Gibbins B</span>, <span class="author">Gilbert J</span>, <span class="author">Hammond M</span>, <span class="author">Hotz HR</span>, <span class="author">Iyer V</span>, <span class="author">Jekosch K</span>, <span class="author">Kahari A</span>, <span class="author">Kasprzyk A</span>, <span class="author">Keefe D</span>, <span class="author">Keenan S</span>, <span class="author">Lehvaslaiho H</span>, <span class="author">McVicker G</span>, <span class="author">Melsopp C</span>, <span class="author">Meidl P</span>, <span class="author">Mongin E</span>, <span class="author">Pettett R</span>, <span class="author">Potter S</span>, <span class="author">Proctor G</span>, <span class="author">Rae M</span>, <span class="author">Searle S</span>, <span class="author">Slater G</span>, <span class="author">Smedley D</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Stabenau A</span>, <span class="author">Stalker J</span>, <span class="author">Storey R</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Woodwark KC</span>, <span class="author">Cameron G</span>, <span class="author">Durbin R</span>, <span class="author">Cox A</span>, <span class="author">Hubbard T</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">EMBL European Bioinformatics Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%62%69%72%6E%65%79%40%65%62%69.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Ensembl (http://www.ensembl.org/) is a bioinformatics project to organize biological information around the sequences of large genomes. It is a comprehensive source of stable automatic annotation of individual genomes, and of the synteny and orthology relationships between them. It is also a framework for integration of any biological data that can be mapped onto features derived from the genomic sequence. Ensembl is available as an interactive Web site, a set of flat files, and as a complete, portable open source software system for handling genomes. All data are provided without restriction, and code is freely available. Ensembl&#39;s aims are to continue to &quot;widen&quot; this biological integration to include other model organisms relevant to understanding human biology as they become available; to &quot;deepen&quot; this integration to provide an ever more seamless linkage between equivalent components in different species; and to provide further classification of functional elements in the genome that have been previously elusive.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">062023</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">925-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15078858">15078858</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479121">479121</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1860604">10.1101/gr.1860604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GeneWise and Genomewise.</h4>
<p class="authors"><span class="author">Birney E</span>, <span class="author">Clamp M</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">The European Bioinformatics Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%62%69%72%6E%65%79%40%65%62%69.%61%63.%75%6B">&#98;&#105;&#114;&#110;&#101;&#121;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We present two algorithms in this paper: GeneWise, which predicts gene structure using similar protein sequences, and Genomewise, which provides a gene structure final parse across cDNA- and EST-defined spliced structure. Both algorithms are heavily used by the Ensembl annotation system. The GeneWise algorithm was developed from a principled combination of hidden Markov models (HMMs). Both algorithms are highly accurate and can provide both accurate and complete gene structures when used with the correct evidence.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">988-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123596" rel="external">15123596</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479130" rel="external">479130</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1865504" rel="external">10.1101/gr.1865504</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A survey of RNA editing in human brain.</h4>
<p class="authors"><span class="author">Blow M</span>, <span class="author">Futreal PA</span>, <span class="author">Wooster R</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">We have conducted a survey of RNA editing in human brain by comparing sequences of clones from a human brain cDNA library to the reference human genome sequence and to genomic DNA from the same individual. In the RNA sample from which the library was constructed, approximately 1:2000 nucleotides were edited out of &gt;3 Mb surveyed. All edits were adenosine to inosine (A--&gt;I) and were predominantly in intronic and in intergenic RNAs. No edits were found in translated exons and few in untranslated exons. Most edits were in high-copy-number repeats, usually Alus. Analysis of the genome in the vicinity of edited sequences strongly supports the idea that formation of intramolecular double-stranded RNA with an inverted copy underlies most A--&gt;I editing. The likelihood of editing is increased by the presence of two inverted copies of a sequence within the same intron, proximity of the two sequences to each other (preferably within 2 kb), and by a high density of inverted copies in the vicinity. Editing exhibits sequence preferences and is less likely at an adenosine 3&#39; to a guanosine and more likely at an adenosine 5&#39; to a guanosine. Simulation by BLAST alignment of the double-stranded RNA molecules that underlie known edits indicates that there is a greater likelihood of A--&gt;I editing at A:C mismatches than editing at other mismatches or at A:U matches. However, because A:U matches in double-stranded RNA are more common than all mismatches, overall the likely effect of editing is to increase the number of mismatches in double-stranded RNA.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">12</span>;<span class="pagerange">2379-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15545495" rel="external">15545495</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC534661" rel="external">534661</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2951204" rel="external">10.1101/gr.2951204</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NIPBL mutations and genetic heterogeneity in Cornelia de Lange syndrome.</h4>
<p class="authors"><span class="author">Borck G</span>, <span class="author">Redon R</span>, <span class="author">Sanlaville D</span>, <span class="author">Rio M</span>, <span class="author">Prieur M</span>, <span class="author">Lyonnet S</span>, <span class="author">Vekemans M</span>, <span class="author">Carter NP</span>, <span class="author">Munnich A</span>, <span class="author">Colleaux L</span> and <span class="author">Cormier-Daire V</span></p>
<p class="affiliation">INSERM U393 and D&eacute;partement de G&eacute;n&eacute;tique M&eacute;dicale, H&ocirc;pital Necker - Enfants Malades, Paris, France.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">12</span>;<span class="pagerange">e128</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15591270" rel="external">15591270</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735640" rel="external">1735640</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2004.026666" rel="external">10.1136/jmg.2004.026666</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide screening using automated fluorescent genotyping to detect cryptic cytogenetic abnormalities in children with idiopathic syndromic mental retardation.</h4>
<p class="authors"><span class="author">Borck G</span>, <span class="author">Rio M</span>, <span class="author">Sanlaville D</span>, <span class="author">Redon R</span>, <span class="author">Molinari F</span>, <span class="author">Bacq D</span>, <span class="author">Raoul O</span>, <span class="author">Cormier-Daire V</span>, <span class="author">Lyonnet S</span>, <span class="author">Amiel J</span>, <span class="author">Le Merrer M</span>, <span class="author">de Blois MC</span>, <span class="author">Prieur M</span>, <span class="author">Vekemans M</span>, <span class="author">Carter NP</span>, <span class="author">Munnich A</span> and <span class="author">Colleaux L</span></p>
<p class="affiliation">INSERM U393 et D&eacute;partement de G&eacute;n&eacute;tique, H&ocirc;pital Necker-Enfants Malades, Paris, France.</p><p class="abstract">Mental retardation (MR) is the most common developmental disability, affecting approximately 2% of the population. The causes of MR are diverse and poorly understood, but chromosomal rearrangements account for 4-28% of cases, and duplications/deletions smaller than 5 Mb are known to cause syndromic MR. We have previously developed a strategy based on automated fluorescent microsatellite genotyping to test for telomere integrity. This strategy detected about 10% of cryptic subtelomeric rearrangements in patients with idiopathic syndromic MR. Because telomere screening is a first step toward the goal of analyzing the entire genome for chromosomal rearrangements in MR, we have extended our strategy to 400 markers evenly distributed along the chromosomes to detect interstitial anomalies. Among 97 individuals tested, three anomalies were found: two deletions (one in three siblings) and one parental disomy. These results emphasize the value of a genome-wide microsatellite scan for the detection of interstitial aberrations and demonstrate that automated genotyping is a sensitive method that not only detects small interstitial rearrangements and their parental origin but also provides a unique opportunity to detect uniparental disomies. This study will hopefully contribute to the delineation of new contiguous gene syndromes and the identification of new imprinted regions.</p>
<p class="publication"><span class="title">Clinical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">66</span>;<span class="number">2</span>;<span class="pagerange">122-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15253762" rel="external">15253762</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1399-0004.2004.00288.x" rel="external">10.1111/j.1399-0004.2004.00288.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The ingi and RIME non-LTR retrotransposons are not randomly distributed in the genome of Trypanosoma brucei.</h4>
<p class="authors"><span class="author">Bringaud F</span>, <span class="author">Biteau N</span>, <span class="author">Zuiderwijk E</span>, <span class="author">Berriman M</span>, <span class="author">El-Sayed NM</span>, <span class="author">Ghedin E</span>, <span class="author">Melville SE</span>, <span class="author">Hall N</span> and <span class="author">Baltz T</span></p>
<p class="affiliation">Laboratoire de G&eacute;nomique Fonctionnelle des Trypanosomatides, UMR-5162 CNRS, Universit&eacute; Victor Segalen Bordeaux II, Bordeaux, France. <a href="mailto:%62%72%69%6E%67%61%75%64%40%75-%62%6F%72%64%65%61%75%78%32.%66%72">&#98;&#114;&#105;&#110;&#103;&#97;&#117;&#100;&#64;&#117;&#45;&#98;&#111;&#114;&#100;&#101;&#97;&#117;&#120;&#50;&#46;&#102;&#114;</a></p><p class="abstract">The ingi (long and autonomous) and RIME (short and nonautonomous) non--long-terminal repeat retrotransposons are the most abundant mobile elements characterized to date in the genome of the African trypanosome Trypanosoma brucei. These retrotransposons were thought to be randomly distributed, but a detailed and comprehensive analysis of their genomic distribution had not been performed until now. To address this question, we analyzed the ingi/RIME sequences and flanking sequences from the ongoing T. brucei genome sequencing project (TREU927/4 strain). Among the 81 ingi/RIME elements analyzed, 60% are complete, and 7% of the ingi elements (approximately 15 copies per haploid genome) appear to encode for their own transposition. The size of the direct repeat flanking the ingi/RIME retrotransposons is conserved (i.e., 12-bp), and a strong 11-bp consensus pattern precedes the 5&#39;-direct repeat. The presence of a consensus pattern upstream of the retroelements was confirmed by the analysis of the base occurrence in 294 GSS containing 5&#39;-adjacent ingi/RIME sequences. The conserved sequence is present upstream of ingis and RIMEs, suggesting that ingi-encoded enzymatic activities are used for retrotransposition of RIMEs, which are short nonautonomous retroelements. In conclusion, the ingi and RIME retroelements are not randomly distributed in the genome of T. brucei and are preceded by a conserved sequence, which may be the recognition site of the ingi-encoded endonuclease.</p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2004</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">520-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14694076" rel="external">14694076</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsh045" rel="external">10.1093/molbev/msh045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Micro-geographical differentiation in Northern Iberia revealed by Y-chromosomal DNA analysis.</h4>
<p class="authors"><span class="author">Brion M</span>, <span class="author">Quintans B</span>, <span class="author">Zarrabeitia M</span>, <span class="author">Gonzalez-Neira A</span>, <span class="author">Salas A</span>, <span class="author">Lareu V</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Carracedo A</span></p>
<p class="affiliation">Institute of Legal Medicine, University of Santiago de Compostela, San Francisco s/n., 15782 Santiago de Compostela, Spain.</p><p class="abstract">Y-chromosome diversity has been analyzed at a micro-geographical level, examining 10 binary polymorphisms and 7 short tandem repeats (STRs) in 443 samples belonging to 11 populations from two regions of Northern Spain, Galicia and Cantabria. Both regions, as a whole, cluster with other Iberian populations. However, some individual populations, particularly that from the Pas Valley in Cantabria, depart markedly from this general pattern, with higher genetic distances and reduced diversity. This unusual population is even more distinct than the Basques from their Iberian neighbors. Genetic drift in a small isolated population could explain this special behavior, and in addition to its anthropological interest, this finding has important forensic implications.</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">2004</span>;<span class="volume">329</span>;<span class="pagerange">17-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15033525" rel="external">15033525</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gene.2003.12.035" rel="external">10.1016/j.gene.2003.12.035</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Variation in the effectors of the type III secretion system among Photorhabdus species as revealed by genomic analysis.</h4>
<p class="authors"><span class="author">Brugirard-Ricaud K</span>, <span class="author">Givaudan A</span>, <span class="author">Parkhill J</span>, <span class="author">Boemare N</span>, <span class="author">Kunst F</span>, <span class="author">Zumbihl R</span> and <span class="author">Duchaud E</span></p>
<p class="affiliation">Laboratoire EMIP Ecologie Microbienne des Insectes et Interaction H&ocirc;te-Pathog&egrave;ne, Universit&eacute; de Montpellier II, UMR1133 INRA-UMII, 34095 Montpellier 5, France.</p><p class="abstract">Entomopathogenic bacteria of the genus Photorhabdus harbor a type III secretion system. This system was probably acquired prior to the separation of the species within this genus. Furthermore, the core components of the secretion machinery are highly conserved but the predicted effectors differ between Photorhabdus luminescens and P. asymbiotica, two highly related species with different hosts.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2004</span>;<span class="volume">186</span>;<span class="number">13</span>;<span class="pagerange">4376-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15205440" rel="external">15205440</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC421592" rel="external">421592</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.186.13.4376-4381.2004" rel="external">10.1128/JB.186.13.4376-4381.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Neurospheres: insights into neural stem cell biology.</h4>
<p class="authors"><span class="author">Campos LS</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom. <a href="mailto:%6C%73%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#115;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Neural stem cells (NSC) are a tissue-specific subtype of self-renewing and multipotent cells that can give rise to all neural populations. In this review, the importance of maintaining cell-cell contacts in the study of NSC is highlighted, and data obtained from some crucial single-cell studies is compared to results obtained from neurospheres, where aggregates of NSC are grown in suspension. In particular, results that indicate how this culture system may be well suited to analyze NSC plasticity, cell-cell, and cell-extracellular matrix (ECM) interactions are pointed out, and the hypothesis that cell-cell and cell-ECM contacts may be essential for NSC maintenance, survival, and proliferation is highlighted. Finally, it is suggested that neurospheres might play a role in the study of context-dependent behavior of NSC in niches by providing a system where NSC can be challenged chemically or biologically and analyzed in vitro, in a time- and context-dependent manner.</p>
<p class="publication"><span class="title">Journal of neuroscience research</span>&nbsp;<span class="year">2004</span>;<span class="volume">78</span>;<span class="number">6</span>;<span class="pagerange">761-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15505793" rel="external">15505793</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fjnr.20333" rel="external">10.1002/jnr.20333</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">As normal as normal can be?</h4>
<p class="authors"><span class="author">Carter NP</span></p><p class="abstract">Two papers report that large-scale copy-number variations, ranging in size from 100 kb to 2 Mb, are distributed widely throughout the human genome, and that a high proportion of them encompass known genes. This unexpected level of genome variation has implications for our view of human genetic diversity and phenotypic variation.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">9</span>;<span class="pagerange">931-2</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15340426">15340426</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0904-931">10.1038/ng0904-931</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Applications of genomic microarrays to explore human chromosome structure and function.</h4>
<p class="authors"><span class="author">Carter NP</span> and <span class="author">Vetrie D</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK. <a href="mailto:%6E%70%63%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#112;&#99;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The combination of genomic microarrays with comparative genomic hybridization and with chromatin immunoprecipitation is providing an increasingly detailed view of the way in which the human genome is organized and functions and how disorganization and disfunction can lead to disease. These studies are enhanced by the flexibility of array technology, allowing resolutions from coverage of the whole genome using 200 kb cloned DNA inserts to detailed analysis using PCR products or oligonucleotides of 100 bp or less. In particular, the use of chromatin immunoprecipitation is providing new insights into chromosome structure and gene regulation and control through the analysis of protein--DNA interactions.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13 Spec No 2</span>;<span class="pagerange">R297-302</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15358737" rel="external">15358737</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh230" rel="external">10.1093/hmg/ddh230</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Molecular characterization and evolution of X and Y-borne ATRX homologues in American marsupials.</h4>
<p class="authors"><span class="author">Carvalho-Silva DR</span>, <span class="author">O&#39;Neill RJ</span>, <span class="author">Brown JD</span>, <span class="author">Huynh K</span>, <span class="author">Waters PD</span>, <span class="author">Pask AJ</span>, <span class="author">Delbridge ML</span> and <span class="author">Graves JA</span></p>
<p class="affiliation">Research School of Biological Science, Australian National University, ACT 0200, Canberra, Australia. <a href="mailto:%64%73%38%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#56;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In eutherians, the sex-reversing ATRX gene on the X has no homologue on the Y chromosome. However, testis-specific and ubiquitously expressed X-borne genes have been identified in Australian marsupials. We studied nucleotide sequence and chromosomal location of ATRX homologues in two American marsupials, the opossums Didelphis virginiana and Monodelphis domestica. A PCR fragment of M. domestica ATRX was used to probe Southern blots and to screen male genomic libraries. Southern analysis demonstrated ATRX homologues on both X and Y in D. virginiana, and two clones were isolated which hybridized to a single position on the Y chromosome in male-derived cells but to multiple sites of the X in female cells. In M. domestica, there was a single clone that mapped to the X but not to the Y, suggesting that it represents the M. domestica ATRX. However a male-specific band was detected in Southern blots probed with the D. virginiana ATRY and with a mouse ATRX clone, which implies that the Y copy in M. domestica has diverged further from other ATRX homologues. Thus there appears to be a Y-borne copy of ATRY in American, as well as Australian marsupials, although it has diverged in sequence, as have other Y genes that are testis-specific in both eutherian and marsupial lineages.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">12</span>;<span class="number">8</span>;<span class="pagerange">795-804</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15702418" rel="external">15702418</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs10577-005-5376-5" rel="external">10.1007/s10577-005-5376-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">New environments, versatile genomes.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Crossman L</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">6</span>;<span class="pagerange">446-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15181901">15181901</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro910">10.1038/nrmicro910</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pathogens in decay.</h4>
<p class="authors"><span class="author">Cerde&ntilde;o-T&aacute;rraga A</span>, <span class="author">Thomson N</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">10</span>;<span class="pagerange">774-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15468460">15468460</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1009">10.1038/nrmicro1009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Knock-in human rhodopsin-GFP fusions as mouse models for human disease and targets for gene therapy.</h4>
<p class="authors"><span class="author">Chan F</span>, <span class="author">Bradley A</span>, <span class="author">Wensel TG</span> and <span class="author">Wilson JH</span></p>
<p class="affiliation">Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, TX 77030, USA.</p><p class="abstract">The human rhodopsin gene is the locus for numerous alleles linked to the neurodegenerative disease retinitis pigmentosa. To facilitate the study of retinal degeneration and to test reagents designed to alter the structure and function of this gene, we have developed strains of mice whose native rhodopsin gene has been replaced with the corresponding human DNA modified to encode an enhanced GFP fusion at the C terminus of rhodopsin. The human rhodopsin-GFP fusion faithfully mimics the expression and distribution of wild-type rhodopsin in heterozygotes and serves as a sensitive reporter of rod-cell structure and integrity. In homozygotes, however, the gene induces progressive retinal degeneration bearing many of the hallmarks of recessive retinitis pigmentosa. When the gene is flanked by recognition sites for Cre recombinase, protein expression is reduced approximately 5-fold despite undiminished mRNA levels, suggesting translation inhibition. GFP-tagged human rhodopsin provides a sensitive method to monitor the development of normal and diseased retinas in dissected samples, and it offers a noninvasive means to observe the progress of retinal degeneration and the efficacy of gene-based therapies in whole animals.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY002520</span>, <span class="grant">EY11731</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">24</span>;<span class="pagerange">9109-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15184660" rel="external">15184660</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC428481" rel="external">428481</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0403149101" rel="external">10.1073/pnas.0403149101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Analysis of multiple genomic sequence alignments: a web resource, online tools, and lessons learned from analysis of mammalian SCL loci.</h4>
<p class="authors"><span class="author">Chapman MA</span>, <span class="author">Donaldson IJ</span>, <span class="author">Gilbert J</span>, <span class="author">Grafham D</span>, <span class="author">Rogers J</span>, <span class="author">Green AR</span> and <span class="author">G&ouml;ttgens B</span></p>
<p class="affiliation">Cambridge Institute for Medical Research, Cambridge, CB2 2XY, UK.</p><p class="abstract">Comparative analysis of genomic sequences is becoming a standard technique for studying gene regulation. However, only a limited number of tools are currently available for the analysis of multiple genomic sequences. An extensive data set for the testing and training of such tools is provided by the SCL gene locus. Here we have expanded the data set to eight vertebrate species by sequencing the dog SCL locus and by annotating the dog and rat SCL loci. To provide a resource for the bioinformatics community, all SCL sequences and functional annotations, comprising a collation of the extensive experimental evidence pertaining to SCL regulation, have been made available via a Web server. A Web interface to new tools specifically designed for the display and analysis of multiple sequence alignments was also implemented. The unique SCL data set and new sequence comparison tools allowed us to perform a rigorous examination of the true benefits of multiple sequence comparisons. We demonstrate that multiple sequence alignments are, overall, superior to pairwise alignments for identification of mammalian regulatory regions. In the search for individual transcription factor binding sites, multiple alignments markedly increase the signal-to-noise ratio compared to pairwise alignments.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">2</span>;<span class="pagerange">313-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14718377" rel="external">14718377</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC327107" rel="external">327107</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1759004" rel="external">10.1101/gr.1759004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase as an integrated platform for the C. elegans ORFeome.</h4>
<p class="authors"><span class="author">Chen N</span>, <span class="author">Lawson D</span>, <span class="author">Bradnam K</span>, <span class="author">Harris TW</span> and <span class="author">Stein LD</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, Cold Spring Harbor, New York 11724, USA. <a href="mailto:%63%68%65%6E%6E%40%63%73%68%6C.%6F%72%67">&#99;&#104;&#101;&#110;&#110;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">The ORFeome project has validated and corrected a large number of predicted gene models in the nematode C. elegans, and has provided an enormous resource for proteome-scale studies. To make the resource useful to the research and teaching community, it needs to be integrated with other large-scale data sets, including the C. elegans genome, cell lineage, neurological wiring diagram, transcriptome, and gene expression map. This integration is also critical because the ORFeome data sets, like other &#39;omics&#39; data sets, have significant false-positive and false-negative rates, and comparison to related data is necessary to make confidence judgments in any given data point. WormBase, the central data repository for information about C. elegans and related nematodes, provides such a platform for integration. In this report, we will describe how C. elegans ORFeome data are deposited in the database, how they are used to correct gene models, how they are integrated and displayed in the context of other data sets at the WormBase Web site, and how WormBase establishes connection with the reagent-based resources at the ORFeome project Web site.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">10B</span>;<span class="pagerange">2155-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15489338" rel="external">15489338</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC528932" rel="external">528932</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2521304" rel="external">10.1101/gr.2521304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bigenic Cre/loxP, puDeltatk conditional genetic ablation.</h4>
<p class="authors"><span class="author">Chen YT</span>, <span class="author">Levasseur R</span>, <span class="author">Vaishnav S</span>, <span class="author">Karsenty G</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Program in Developmental Biology, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA.</p><p class="abstract">Genetic ablation experiments are used to resolve problems regarding cell lineages and the in vivo function of certain groups of cells. We describe a two-component conditional ablation technology using a mouse carrying an X-linked puDeltatk transgene, which is only activated in cells expressing Cre. Ablation of the Cre-expressing cells can be temporally regulated by the time of ganciclovir (GCV) administration. This strategy was demonstrated using a Col2Cre transgenic line. Differentiating chondrocytes in bigenic animals could be ablated at different developmental stages resulting in disorganized growth plates and dwarfism. Macrocephaly, macroglossia and umbilical hernia were also observed in ablated 18.5 dpc embryos. Crosses between the puDeltatk selector transgenic line and existing cre lines will facilitate numerous temporally regulated tissue-specific ablation experiments.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">20</span>;<span class="pagerange">e161</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15561996" rel="external">15561996</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC534639" rel="external">534639</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgnh158" rel="external">10.1093/nar/gnh158</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Inducible gene trapping with drug-selectable markers and Cre/loxP to identify developmentally regulated genes.</h4>
<p class="authors"><span class="author">Chen YT</span>, <span class="author">Liu P</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Program in Developmental Biology, Baylor College of Medicine, Houston, Texas, USA.</p><p class="abstract">Gene trapping in mouse embryonic stem cells is an important genetic approach that allows simultaneous mutation of genes and generation of corresponding mutant mice. We designed a selection scheme with drug selection markers and Cre/loxP technology which allows screening of gene trap events that responded to a signaling molecule in a 96-well format. Nine hundred twenty gene trap clones were assayed, and 258 were classified as gene traps induced by in vitro differentiation. Sixty-five of the in vitro differentiation-inducible gene traps were also responsive to retinoic acid treatment. In vivo analysis revealed that 85% of the retinoic acid-inducible gene traps trapped developmentally regulated genes, consistent with the observation that genes induced by retinoic acid treatment are likely to be developmentally regulated. Our results demonstrate that the inducible gene trapping system described here can be used to enrich in vitro for traps in genes of interest. Furthermore, we demonstrate that the cre reporter is extremely sensitive and can be used to explore chromosomal regions that are not detectable with neo as a selection cassette.</p>
<p class="publication"><span class="title">Molecular and cellular biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">24</span>;<span class="number">22</span>;<span class="pagerange">9930-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15509795" rel="external">15509795</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC525470" rel="external">525470</a>; DOI: <a href="http://dx.doi.org/10.1128%2FMCB.24.22.9930-9941.2004" rel="external">10.1128/MCB.24.22.9930-9941.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The apical caspase dronc governs programmed and unprogrammed cell death in Drosophila.</h4>
<p class="authors"><span class="author">Chew SK</span>, <span class="author">Akdemir F</span>, <span class="author">Chen P</span>, <span class="author">Lu WJ</span>, <span class="author">Mills K</span>, <span class="author">Daish T</span>, <span class="author">Kumar S</span>, <span class="author">Rodriguez A</span> and <span class="author">Abrams JM</span></p>
<p class="affiliation">Department of Cell Biology, UT Southwestern Medical Center, Dallas, TX 75390, USA.</p><p class="abstract">Among the seven caspases encoded in the fly genome, only dronc contains a caspase recruitment domain. To assess the function of this gene in development, we produced a null mutation in dronc. Animals lacking zygotic dronc are defective for programmed cell death (PCD) and arrest as early pupae. These mutants present a range of defects, including extensive hyperplasia of hematopoietic tissues, supernumerary neuronal cells, and head involution failure. dronc genetically interacts with the Ced4/Apaf1 counterpart, Dark, and adult structures lacking dronc are disrupted for fine patterning. Furthermore, in diverse models of metabolic injury, dronc- cells are completely insensitive to induction of cell killing. These findings establish dronc as an essential regulator of cell number in development and illustrate broad requirements for this apical caspase in adaptive responses during stress-induced apoptosis.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">R01 GM072124-14A1</span>, <span class="grant">R01GM072124</span></p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">7</span>;<span class="number">6</span>;<span class="pagerange">897-907</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15572131" rel="external">15572131</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2004.09.016" rel="external">10.1016/j.devcel.2004.09.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Proteomics in postgenomic neuroscience: the end of the beginning.</h4>
<p class="authors"><span class="author">Choudhary J</span> and <span class="author">Grant SG</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK.</p><p class="abstract">Proteomics is complementary to genomic approaches anchored in DNA and RNA. Global characterization of proteins is providing new insights into general biological structures as well as synapses, receptor complexes and other neuronal and glial features. Current challenges for proteomics of the nervous system include problems relating to sample preparation, brain complexity, limited databases and informatics tools. The combination of proteomics with other global functional genomic approaches at the levels of genome and transcriptome, together with network biology, will provide important bridges between genes, physiology and pathology.</p>
<p class="publication"><span class="title">Nature neuroscience</span>&nbsp;<span class="year">2004</span>;<span class="volume">7</span>;<span class="number">5</span>;<span class="pagerange">440-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15114355" rel="external">15114355</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnn1240" rel="external">10.1038/nn1240</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A whole-genome mouse BAC microarray with 1-Mb resolution for analysis of DNA copy number changes by array comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Chung YJ</span>, <span class="author">Jonkers J</span>, <span class="author">Kitson H</span>, <span class="author">Fiegler H</span>, <span class="author">Humphray S</span>, <span class="author">Scott C</span>, <span class="author">Hunt S</span>, <span class="author">Yu Y</span>, <span class="author">Nishijima I</span>, <span class="author">Velds A</span>, <span class="author">Holstege H</span>, <span class="author">Carter N</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Microarray-based comparative genomic hybridization (CGH) has become a powerful method for the genome-wide detection of chromosomal imbalances. Although BAC microarrays have been used for mouse CGH studies, the resolving power of these analyses was limited because high-density whole-genome mouse BAC microarrays were not available. We therefore developed a mouse BAC microarray containing 2803 unique BAC clones from mouse genomic libraries at 1-Mb intervals. For the general amplification of BAC clone DNA prior to spotting, we designed a set of three novel degenerate oligonucleotide-primed (DOP) PCR primers that preferentially amplify mouse genomic sequences while minimizing unwanted amplification of contaminating Escherichia coli DNA. The resulting 3K mouse BAC microarrays reproducibly identified DNA copy number alterations in cell lines and primary tumors, such as single-copy deletions, regional amplifications, and aneuploidy.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">188-96</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14707179" rel="external">14707179</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC314296" rel="external">314296</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1878804" rel="external">10.1101/gr.1878804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Jalview Java alignment editor.</h4>
<p class="authors"><span class="author">Clamp M</span>, <span class="author">Cuff J</span>, <span class="author">Searle SM</span> and <span class="author">Barton GJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, UK. <a href="mailto:%6D%69%63%68%65%6C%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#104;&#101;&#108;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Multiple sequence alignment remains a crucial method for understanding the function of groups of related nucleic acid and protein sequences. However, it is known that automatic multiple sequence alignments can often be improved by manual editing. Therefore, tools are needed to view and edit multiple sequence alignments. Due to growth in the sequence databases, multiple sequence alignments can often be large and difficult to view efficiently. The Jalview Java alignment editor is presented here, which enables fast viewing and editing of large multiple sequence alignments.</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">3</span>;<span class="pagerange">426-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14960472" rel="external">14960472</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtg430" rel="external">10.1093/bioinformatics/btg430</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Binary and microsatellite polymorphisms of the Y-chromosome in the Mbenzele pygmies from the Central African Republic.</h4>
<p class="authors"><span class="author">Coia V</span>, <span class="author">Cagli&agrave; A</span>, <span class="author">Arredi B</span>, <span class="author">Donati F</span>, <span class="author">Santos FR</span>, <span class="author">Pandya A</span>, <span class="author">Taglioli L</span>, <span class="author">Paoli G</span>, <span class="author">Pascali V</span>, <span class="author">Spedini G</span>, <span class="author">Destro-Bisol G</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Department of Animal and Human Biology, University La Sapienza, Rome, Italy.</p><p class="abstract">This study analyzes the variation of six binary polymorphisms and six microsatellites in the Mbenzele Pygmies from the Central African Republic. Five different haplogroups (B2b, E(xE3a), E3a, P and BR(xB2b,DE,P)) were observed, with frequencies ranging from 0.022 (haplogroup P) to 0.609 (haplogroup E3a). A comparison of haplogroup frequencies indicates a close genetic affinity between the Mbenzele and the Biaka Pygmies, a finding consistent with the common origin and the geographical proximity of the two populations. The haplogroups P, BR(xB2b,DE,P) and E(xE3a), which are rare in sub-Saharan Africa but common in western Eurasia, were observed with frequencies ranging from 0.022 (haplogroup P) to 0.087 (haplogroup E(xE3a)). Thirty different microsatellite haplotypes were detected, with frequencies ranging from 0.022 to 0.152. The Mbenzele share the highest percent of microsatellite haplotypes with the Biaka Pygmies. Five out seven haplotypes which are shared by the Mbenzele and Biaka Pygmies belong to haplogroup E3a, which suggests that they are of Bantu origin. The plot based on F(st) genetic distances calculated using microsatellite data provides a picture of population relationships which is in part congruent and in part complementary to that obtained using haplogroup frequencies. Finally, the Mbenzele and Biaka Pygmies were found to be markedly more genetically similar using Y-chromosomal than autosomal microsatellites. We suggest that this could be due to the higher phylogenetic stability of Y-chromosome and to the effect of the male-biased gene flow during the Bantu expansion.</p>
<p class="publication"><span class="title">American journal of human biology : the official journal of the Human Biology Council</span>&nbsp;<span class="year">2004</span>;<span class="volume">16</span>;<span class="number">1</span>;<span class="pagerange">57-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14689516" rel="external">14689516</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fajhb.10236" rel="external">10.1002/ajhb.10236</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Improved techniques for the identification of pseudogenes.</h4>
<p class="authors"><span class="author">Coin L</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6C%63%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#99;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Motivation</strong>: Pseudogenes are the remnants of genomic sequences of genes which are no longer functional. They are frequent in most eukaryotic genomes, and an important resource for comparative genomics. However, pseudogenes are often mis-annotated as functional genes in sequence databases. Current methods for identifying pseudogenes include methods which rely on the presence of stop codons and frameshifts, as well as methods based on the ratio of non-silent to silent nucleotide substitution rates (dN/dS). A recent survey concluded that 50% of human pseudogenes have no detectable truncation in their pseudo-coding regions, indicating that the former methods lack sensitivity. The latter methods have been used to find sets of genes enriched for pseudogenes, but are not specific enough to accurately separate pseudogenes from expressed genes.</p><p class="abstract"><strong>Results</strong>: We introduce a program called pseudogene inference from loss of constraint (PSILC) which incorporates novel methods for separating pseudogenes from functional genes. The methods calculate the log-odds score that evolution along the final branch of the gene tree to the query gene has been according to the following constraints: A neutral nucleotide model compared to a Pfam domain encoding model (PSILC(nuc/dom)); A protein coding model compared to a Pfam domain encoding model (PSILC(prot/dom)). Using the manual annotation of human chromosome 6, we show that both these methods result in a more accurate classification of pseudogenes than dN/dS when a Pfam domain alignment is available.</p><p class="abstract"><strong>Availability</strong>: PSILC is available from http://www.sanger.ac.uk/Software/PSILC</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Bioinformatics (Oxford, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">20 Suppl 1</span>;<span class="pagerange">i94-100</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15262786">15262786</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbth942">10.1093/bioinformatics/bth942</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Enhanced protein domain discovery using taxonomy.</h4>
<p class="authors"><span class="author">Coin L</span>, <span class="author">Bateman A</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6C%63%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#108;&#99;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: It is well known that different species have different protein domain repertoires, and indeed that some protein domains are kingdom specific. This information has not yet been incorporated into statistical methods for finding domains in sequences of amino acids.</p><p class="abstract"><strong>Results</strong>: We show that by incorporating our understanding of the taxonomic distribution of specific protein domains, we can enhance domain recognition in protein sequences. We identify 4447 new instances of Pfam domains in the SP-TREMBL database using this technique, equivalent to the coverage increase given by the last 8.3% of Pfam families and to a 0.7% increase in the number of domain predictions. We use PSI-BLAST to cross-validate our new predictions. We also benchmark our approach using a SCOP test set of proteins of known structure, and demonstrate improvements relative to standard Hidden Markov model techniques.</p><p class="abstract"><strong>Conclusions</strong>: Explicitly including knowledge about the taxonomic distribution of protein domains can enhance protein domain recognition. Our method can also incorporate other context-specific domain distributions - such as domain co-occurrence and protein localisation.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="pagerange">56</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15137915" rel="external">15137915</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC434490" rel="external">434490</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-5-56" rel="external">10.1186/1471-2105-5-56</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome annotation-driven approach to cloning the human ORFeome.</h4>
<p class="authors"><span class="author">Collins JE</span>, <span class="author">Wright CL</span>, <span class="author">Edwards CA</span>, <span class="author">Davis MP</span>, <span class="author">Grinham JA</span>, <span class="author">Cole CG</span>, <span class="author">Goward ME</span>, <span class="author">Aguado B</span>, <span class="author">Mallya M</span>, <span class="author">Mokrab Y</span>, <span class="author">Huckle EJ</span>, <span class="author">Beare DM</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">We have developed a systematic approach to generating cDNA clones containing full-length open reading frames (ORFs), exploiting knowledge of gene structure from genomic sequence. Each ORF was amplified by PCR from a pool of primary cDNAs, cloned and confirmed by sequencing. We obtained clones representing 70% of genes on human chromosome 22, whereas searching available cDNA clone collections found at best 48% from a single collection and 60% for all collections combined.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">R84</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15461802">15461802</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC545604">545604</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2004-5-10-r84">10.1186/gb-2004-5-10-r84</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomics of transcriptional control in the human malaria parasite Plasmodium falciparum.</h4>
<p class="authors"><span class="author">Coulson RM</span>, <span class="author">Hall N</span> and <span class="author">Ouzounis CA</span></p>
<p class="affiliation">Computational Genomics Group, The European Bioinformatics Institute, European Molecular Biology Laboratory Cambridge Outstation, Cambridge CB10 1SD, United Kingdom.</p><p class="abstract">The life cycle of the parasite Plasmodium falciparum, responsible for the most deadly form of human malaria, requires specialized protein expression for survival in the mammalian host and insect vector. To identify components of processes controlling gene expression during its life cycle, the malarial genome--along with seven crown eukaryote group genomes--was queried with a reference set of transcription-associated proteins (TAPs). Following clustering on the basis of sequence similarity of the TAPs with their homologs, and together with hidden Markov model profile searches, 156 P. falciparum TAPs were identified. This represents about a third of the number of TAPs usually found in the genome of a free-living eukaryote. Furthermore, the P. falciparum genome appears to contain a low number of sequences, which are highly conserved and abundant within the kingdoms of free-living eukaryotes, that contribute to gene-specific transcriptional regulation. However, in comparison with these other eukaryotic genomes, the CCCH-type zinc finger (common in proteins modulating mRNA decay and translation rates) was found to be the most abundant in the P. falciparum genome. This observation, together with the paucity of malarial transcriptional regulators identified, suggests Plasmodium protein levels are primarily determined by posttranscriptional mechanisms.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">8</span>;<span class="pagerange">1548-54</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15256513" rel="external">15256513</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC509263" rel="external">509263</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2218604" rel="external">10.1101/gr.2218604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Differential requirements for COPI transport during vertebrate early development.</h4>
<p class="authors"><span class="author">Coutinho P</span>, <span class="author">Parsons MJ</span>, <span class="author">Thomas KA</span>, <span class="author">Hirst EM</span>, <span class="author">Sa&uacute;de L</span>, <span class="author">Campos I</span>, <span class="author">Williams PH</span> and <span class="author">Stemple DL</span></p>
<p class="affiliation">Division of Developmental Biology, National Institute for Medical Research, The Ridgeway, Mill Hill, London NW7 1AA, United Kingdom.</p><p class="abstract">The coatomer vesicular coat complex is essential for normal Golgi and secretory activities in eukaryotic cells. Through positional cloning of genes controlling zebrafish notochord development, we found that the sneezy, happy, and dopey loci encode the alpha, beta, and beta&#39; subunits of the coatomer complex. Export from mutant endoplasmic reticulum is blocked, Golgi structure is disrupted, and mutant embryos eventually degenerate due to widespread apoptosis. The early embryonic phenotype, however, demonstrates that despite its &quot;housekeeping&quot; functions, coatomer activity is specifically and cell autonomously required for normal chordamesoderm differentiation, perinotochordal basement membrane formation, and melanophore pigmentation. Hence, differential requirements for coatomer activity among embryonic tissues lead to tissue-specific developmental defects. Moreover, we note that the mRNA encoding alpha coatomer is strikingly upregulated in notochord progenitors, and we present data suggesting that alpha coatomer transcription is tuned to activity- and cell type-specific secretory loads.</p>
<p class="publication"><span class="title">Developmental cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">7</span>;<span class="number">4</span>;<span class="pagerange">547-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15469843">15469843</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.devcel.2004.07.020">10.1016/j.devcel.2004.07.020</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Premature termination codons enhance mRNA decapping in human cells.</h4>
<p class="authors"><span class="author">Couttet P</span> and <span class="author">Grange T</span></p>
<p class="affiliation">Institut Jacques Monod du CNRS, Universit&eacute;s Paris 6-7, Tour 43, 2 Place Jussieu, 75251 Paris Cedex 05, France.</p><p class="abstract">Nonsense-mediated mRNA decay (NMD) is a eukaryotic surveillance process that promotes selective degradation of imperfect messages containing premature translation termination codons (PTCs). In yeast, PTCs trigger both deadenylylation-independent mRNA decapping, thereby allowing their rapid degradation by a 5&#39; to 3&#39; exonuclease, and to a smaller extent accelerated deadenylylation. It is not clear to what extent this decay pathway is conserved in higher eukaryotes. We used a transcriptional pulse strategy relying on a tetracycline-regulated promoter to study the decay of a PTC- containing beta-globin mRNA in human cells. We show that a PTC destabilizes the mRNA and decreases its half-life from &gt;16 h to 3 h. The deadenylylation rate is increased, but not sufficiently to account for the decreased half-life on its own. Using a circularization RT-PCR (cRT-PCR) strategy, we could detect decapped degradation intermediates and measure simultaneously their poly(A) tail length. This allowed us to show that a PTC enhances the rate of mRNA decapping and that decapped products have been deadenylylated to a certain extent. Thus the major feature of the NMD pathway, enhanced decapping, is conserved from yeast to man even though the kinetic details might differ between various mRNAs and/or species.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">2</span>;<span class="pagerange">488-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14742663" rel="external">14742663</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC373342" rel="external">373342</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh218" rel="external">10.1093/nar/gkh218</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Biofilm formation and dispersal in Xanthomonas campestris.</h4>
<p class="authors"><span class="author">Crossman L</span> and <span class="author">Dow JM</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Xanthomonas campestris pathovar campestris is the causal agent of black rot disease of cruciferous plants. A cell-cell signalling system encoded by genes within the rpf cluster is required for the full virulence of this plant pathogen. This system has recently been implicated in regulation of the formation and dispersal of Xanthomonas biofilms.</p>
<p class="publication"><span class="title">Microbes and infection / Institut Pasteur</span>&nbsp;<span class="year">2004</span>;<span class="volume">6</span>;<span class="number">6</span>;<span class="pagerange">623-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15158198" rel="external">15158198</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.micinf.2004.01.013" rel="external">10.1016/j.micinf.2004.01.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chalk and cheese.</h4>
<p class="authors"><span class="author">Crossman L</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Thomson NR</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">7</span>;<span class="pagerange">528-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15248310">15248310</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro932">10.1038/nrmicro932</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequencing the environment.</h4>
<p class="authors"><span class="author">Crossman L</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">3</span>;<span class="pagerange">184-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15751186">15751186</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro847">10.1038/nrmicro847</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ensembl computing architecture.</h4>
<p class="authors"><span class="author">Cuff JA</span>, <span class="author">Coates GM</span>, <span class="author">Cutts TJ</span> and <span class="author">Rae M</span></p>
<p class="affiliation">The Broad Institute, Cambridge, Massachusetts 02141, USA. <a href="mailto:%6A%63%75%66%66%40%62%72%6F%61%64.%6D%69%74.%65%64%75">&#106;&#99;&#117;&#102;&#102;&#64;&#98;&#114;&#111;&#97;&#100;&#46;&#109;&#105;&#116;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Ensembl is a software project to automatically annotate large eukaryotic genomes and release them freely into the public domain. The project currently automatically annotates 10 complete genomes. This makes very large demands on compute resources, due to the vast number of sequence comparisons that need to be executed. To circumvent the financial outlay often associated with classical supercomputing environments, farms of multiple, lower-cost machines have now become the norm and have been deployed successfully with this project. The architecture and design of farms containing hundreds of compute nodes is complex and nontrivial to implement. This study will define and explain some of the essential elements to consider when designing such systems. Server architecture and network infrastructure are discussed with a particular emphasis on solutions that worked and those that did not (often with fairly spectacular consequences). The aim of the study is to give the reader, who may be implementing a large-scale biocompute project, an insight into some of the pitfalls that may be waiting ahead.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">971-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123594" rel="external">15123594</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479128" rel="external">479128</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1866304" rel="external">10.1101/gr.1866304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ensembl automatic gene annotation system.</h4>
<p class="authors"><span class="author">Curwen V</span>, <span class="author">Eyras E</span>, <span class="author">Andrews TD</span>, <span class="author">Clarke L</span>, <span class="author">Mongin E</span>, <span class="author">Searle SM</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">As more genomes are sequenced, there is an increasing need for automated first-pass annotation which allows timely access to important genomic information. The Ensembl gene-building system enables fast automated annotation of eukaryotic genomes. It annotates genes based on evidence derived from known protein, cDNA, and EST sequences. The gene-building system rests on top of the core Ensembl (MySQL) database schema and Perl Application Programming Interface (API), and the data generated are accessible through the Ensembl genome browser (http://www.ensembl.org). To date, the Ensembl predicted gene sets are available for the A. gambiae, C. briggsae, zebrafish, mouse, rat, and human genomes and have been heavily relied upon in the publication of the human, mouse, rat, and A. gambiae genome sequence analysis. Here we describe in detail the gene-building system and the algorithms involved. All code and data are freely available from http://www.ensembl.org.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">942-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123590">15123590</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479124">479124</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1858004">10.1101/gr.1858004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The HapMap project and its application to genetic studies of drug response.</h4>
<p class="authors"><span class="author">Deloukas P</span> and <span class="author">Bentley D</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK. <a href="mailto:%70%61%6E%6F%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#110;&#111;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">The pharmacogenomics journal</span>&nbsp;<span class="year">2004</span>;<span class="volume">4</span>;<span class="number">2</span>;<span class="pagerange">88-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14676823" rel="external">14676823</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.tpj.6500226" rel="external">10.1038/sj.tpj.6500226</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence and comparative analysis of human chromosome 10.</h4>
<p class="authors"><span class="author">Deloukas P</span>, <span class="author">Earthrowl ME</span>, <span class="author">Grafham DV</span>, <span class="author">Rubenfield M</span>, <span class="author">French L</span>, <span class="author">Steward CA</span>, <span class="author">Sims SK</span>, <span class="author">Jones MC</span>, <span class="author">Searle S</span>, <span class="author">Scott C</span>, <span class="author">Howe K</span>, <span class="author">Hunt SE</span>, <span class="author">Andrews TD</span>, <span class="author">Gilbert JG</span>, <span class="author">Swarbreck D</span>, <span class="author">Ashurst JL</span>, <span class="author">Taylor A</span>, <span class="author">Battles J</span>, <span class="author">Bird CP</span>, <span class="author">Ainscough R</span>, <span class="author">Almeida JP</span>, <span class="author">Ashwell RI</span>, <span class="author">Ambrose KD</span>, <span class="author">Babbage AK</span>, <span class="author">Bagguley CL</span>, <span class="author">Bailey J</span>, <span class="author">Banerjee R</span>, <span class="author">Bates K</span>, <span class="author">Beasley H</span>, <span class="author">Bray-Allen S</span>, <span class="author">Brown AJ</span>, <span class="author">Brown JY</span>, <span class="author">Burford DC</span>, <span class="author">Burrill W</span>, <span class="author">Burton J</span>, <span class="author">Cahill P</span>, <span class="author">Camire D</span>, <span class="author">Carter NP</span>, <span class="author">Chapman JC</span>, <span class="author">Clark SY</span>, <span class="author">Clarke G</span>, <span class="author">Clee CM</span>, <span class="author">Clegg S</span>, <span class="author">Corby N</span>, <span class="author">Coulson A</span>, <span class="author">Dhami P</span>, <span class="author">Dutta I</span>, <span class="author">Dunn M</span>, <span class="author">Faulkner L</span>, <span class="author">Frankish A</span>, <span class="author">Frankland JA</span>, <span class="author">Garner P</span>, <span class="author">Garnett J</span>, <span class="author">Gribble S</span>, <span class="author">Griffiths C</span>, <span class="author">Grocock R</span>, <span class="author">Gustafson E</span>, <span class="author">Hammond S</span>, <span class="author">Harley JL</span>, <span class="author">Hart E</span>, <span class="author">Heath PD</span>, <span class="author">Ho TP</span>, <span class="author">Hopkins B</span>, <span class="author">Horne J</span>, <span class="author">Howden PJ</span>, <span class="author">Huckle E</span>, <span class="author">Hynds C</span>, <span class="author">Johnson C</span>, <span class="author">Johnson D</span>, <span class="author">Kana A</span>, <span class="author">Kay M</span>, <span class="author">Kimberley AM</span>, <span class="author">Kershaw JK</span>, <span class="author">Kokkinaki M</span>, <span class="author">Laird GK</span>, <span class="author">Lawlor S</span>, <span class="author">Lee HM</span>, <span class="author">Leongamornlert DA</span>, <span class="author">Laird G</span>, <span class="author">Lloyd C</span>, <span class="author">Lloyd DM</span>, <span class="author">Loveland J</span>, <span class="author">Lovell J</span>, <span class="author">McLaren S</span>, <span class="author">McLay KE</span>, <span class="author">McMurray A</span>, <span class="author">Mashreghi-Mohammadi M</span>, <span class="author">Matthews L</span>, <span class="author">Milne S</span>, <span class="author">Nickerson T</span>, <span class="author">Nguyen M</span>, <span class="author">Overton-Larty E</span>, <span class="author">Palmer SA</span>, <span class="author">Pearce AV</span>, <span class="author">Peck AI</span>, <span class="author">Pelan S</span>, <span class="author">Phillimore B</span>, <span class="author">Porter K</span>, <span class="author">Rice CM</span>, <span class="author">Rogosin A</span>, <span class="author">Ross MT</span>, <span class="author">Sarafidou T</span>, <span class="author">Sehra HK</span>, <span class="author">Shownkeen R</span>, <span class="author">Skuce CD</span>, <span class="author">Smith M</span>, <span class="author">Standring L</span>, <span class="author">Sycamore N</span>, <span class="author">Tester J</span>, <span class="author">Thorpe A</span>, <span class="author">Torcasso W</span>, <span class="author">Tracey A</span>, <span class="author">Tromans A</span>, <span class="author">Tsolas J</span>, <span class="author">Wall M</span>, <span class="author">Walsh J</span>, <span class="author">Wang H</span>, <span class="author">Weinstock K</span>, <span class="author">West AP</span>, <span class="author">Willey DL</span>, <span class="author">Whitehead SL</span>, <span class="author">Wilming L</span>, <span class="author">Wray PW</span>, <span class="author">Young L</span>, <span class="author">Chen Y</span>, <span class="author">Lovering RC</span>, <span class="author">Moschonas NK</span>, <span class="author">Siebert R</span>, <span class="author">Fechtel K</span>, <span class="author">Bentley D</span>, <span class="author">Durbin R</span>, <span class="author">Hubbard T</span>, <span class="author">Doucette-Stamm L</span>, <span class="author">Beck S</span>, <span class="author">Smith DR</span> and <span class="author">Rogers J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SA, UK. <a href="mailto:%70%61%6E%6F%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#112;&#97;&#110;&#111;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The finished sequence of human chromosome 10 comprises a total of 131,666,441 base pairs. It represents 99.4% of the euchromatic DNA and includes one megabase of heterochromatic sequence within the pericentromeric region of the short and long arm of the chromosome. Sequence annotation revealed 1,357 genes, of which 816 are protein coding, and 430 are pseudogenes. We observed widespread occurrence of overlapping coding genes (either strand) and identified 67 antisense transcripts. Our analysis suggests that both inter- and intrachromosomal segmental duplications have impacted on the gene count on chromosome 10. Multispecies comparative analysis indicated that we can readily annotate the protein-coding genes with current resources. We estimate that over 95% of all coding exons were identified in this study. Assessment of single base changes between the human chromosome 10 and chimpanzee sequence revealed nonsense mutations in only 21 coding genes with respect to the human sequence.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">429</span>;<span class="number">6990</span>;<span class="pagerange">375-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15164054">15164054</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02462">10.1038/nature02462</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Hotdog fold: wrapping up a superfamily of thioesterases and dehydratases.</h4>
<p class="authors"><span class="author">Dillon SC</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%73%63%64%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#99;&#100;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The Hotdog fold was initially identified in the structure of Escherichia coli FabA and subsequently in 4-hydroxybenzoyl-CoA thioesterase from Pseudomonas sp. strain CBS. Since that time structural determinations have shown a number of other apparently unrelated proteins also share the Hotdog fold.</p><p class="abstract"><strong>Results</strong>: Using sequence analysis we unify a large superfamily of HotDog domains. Membership includes numerous prokaryotic, archaeal and eukaryotic proteins involved in several related, but distinct, catalytic activities, from metabolic roles such as thioester hydrolysis in fatty acid metabolism, to degradation of phenylacetic acid and the environmental pollutant 4-chlorobenzoate. The superfamily also includes FapR, a non-catalytic bacterial homologue that is involved in transcriptional regulation of fatty acid biosynthesis. We have defined 17 subfamilies, with some characterisation. Operon analysis has revealed numerous HotDog domain-containing proteins to be fusion proteins, where two genes, once separate but adjacent open-reading frames, have been fused into one open-reading frame to give a protein with two functional domains. Finally we have generated a Hidden Markov Model library from our analysis, which can be used as a tool for predicting the occurrence of HotDog domains in any protein sequence.</p><p class="abstract"><strong>Conclusions</strong>: The HotDog domain is both an ancient and ubiquitous motif, with members found in the three branches of life.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="pagerange">109</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15307895">15307895</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC516016">516016</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-5-109">10.1186/1471-2105-5-109</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Array comparative genomic hybridization analysis of colorectal cancer cell lines and primary carcinomas.</h4>
<p class="authors"><span class="author">Douglas EJ</span>, <span class="author">Fiegler H</span>, <span class="author">Rowan A</span>, <span class="author">Halford S</span>, <span class="author">Bicknell DC</span>, <span class="author">Bodmer W</span>, <span class="author">Tomlinson IP</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Array comparative genomic hybridization, with a genome-wide resolution of approximately 1 Mb, has been used to investigate copy number changes in 48 colorectal cancer (CRC) cell lines and 37 primary CRCs. The samples were divided for analysis according to the type of genomic instability that they exhibit, microsatellite instability (MSI) or chromosomal instability (CIN). Consistent copy number changes were identified, including gain of chromosomes 20, 13, and 8q and smaller regions of amplification such as chromosome 17q11.2-q12. Loss of chromosome 18q was a recurrent finding along with deletion of discrete regions such as chromosome 4q34-q35. The overall pattern of copy number change was strikingly similar between cell lines and primary cancers with a few obvious exceptions such as loss of chromosome 6 and gain of chromosomes 15 and 12p in the former. A greater number of aberrations were detected in CIN+ than MSI+ samples as well as differences in the type and extent of change reported. For example, loss of chromosome 8p was a common event in CIN+ cell lines and cancers but was often found to be gained in MSI+ cancers. In addition, the target of amplification on chromosome 8q appeared to differ, with 8q24.21 amplified frequently in CIN+ samples but 8q24.3 amplification a common finding in MSI+ samples. A number of genes of interest are located within the frequently aberrated regions, which are likely to be of importance in the development and progression of CRC.</p>
<p class="publication"><span class="title">Cancer research</span>&nbsp;<span class="year">2004</span>;<span class="volume">64</span>;<span class="number">14</span>;<span class="pagerange">4817-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15256451" rel="external">15256451</a>; DOI: <a href="http://dx.doi.org/10.1158%2F0008-5472.CAN-04-0328" rel="external">10.1158/0008-5472.CAN-04-0328</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative cell wall core biosynthesis in the mycolated pathogens, Mycobacterium tuberculosis and Corynebacterium diphtheriae.</h4>
<p class="authors"><span class="author">Dover LG</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Pallen MJ</span>, <span class="author">Parkhill J</span> and <span class="author">Besra GS</span></p>
<p class="affiliation">School of Biosciences, The University of Birmingham, Edgbaston, Birmingham B15 2TT, UK. <a href="mailto:%6C.%67.%64%6F%76%65%72%40%62%68%61%6D.%61%63.%75%6B">&#108;&#46;&#103;&#46;&#100;&#111;&#118;&#101;&#114;&#64;&#98;&#104;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The recent determination of the complete genome sequence of Corynebacterium diphtheriae, the aetiological agent of diphtheria, has allowed a detailed comparison of its physiology with that of its closest sequenced pathogenic relative Mycobacterium tuberculosis. Of major importance to the pathogenicity and resilience of the latter is its particularly complex cell envelope. The corynebacteria share many of the features of this extraordinary structure although to a lesser level of complexity. The cell envelope of M. tuberculosis has provided the molecular targets for several of the major anti-tubercular drugs. Given a backdrop of emerging multi-drug resistant strains of the organism (MDR-TB) and its continuing global threat to human health, the search for novel anti-tubercular agents is of paramount importance. The unique structure of this cell wall and the importance of its integrity to the viability of the organism suggest that the search for novel drug targets within the array of enzymes responsible for its construction may prove fruitful. Although the application of modern bioinformatics techniques to the &#39;mining&#39; of the M. tuberculosis genome has already increased our knowledge of the biosynthesis and assembly of the mycobacterial cell wall, several issues remain uncertain. Further analysis by comparison with its relatives may bring clarity and aid the early identification of novel cellular targets for new anti-tuberculosis drugs. In order to facilitate this aim, this review intends to illustrate the broad similarities and highlight the structural differences between the two bacterial envelopes and discuss the genetics of their biosynthesis.</p>
<p class="publication"><span class="title">FEMS microbiology reviews</span>&nbsp;<span class="year">2004</span>;<span class="volume">28</span>;<span class="number">2</span>;<span class="pagerange">225-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15109786" rel="external">15109786</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.femsre.2003.10.001" rel="external">10.1016/j.femsre.2003.10.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">What can we learn from noncoding regions of similarity between genomes?</h4>
<p class="authors"><span class="author">Down TA</span> and <span class="author">Hubbard TJ</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%74%64%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#116;&#100;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: In addition to known protein-coding genes, large amounts of apparently non-coding sequence are conserved between the human and mouse genomes. It seems reasonable to assume that these conserved regions are more likely to contain functional elements than less-conserved portions of the genome.</p><p class="abstract"><strong>Methods</strong>: Here we used a motif-oriented machine learning method based on the Relevance Vector Machine algorithm to extract the strongest signal from a set of non-coding conserved sequences.</p><p class="abstract"><strong>Results</strong>: We successfully fitted models to reflect the non-coding sequences, and showed that the results were quite consistent for repeated training runs. Using the learned models to scan genomic sequence, we found that they often made predictions close to the start of annotated genes. We compared this method with other published promoter-prediction systems, and showed that the set of promoters which are detected by this method is substantially similar to that detected by existing methods.</p><p class="abstract"><strong>Conclusions</strong>: The results presented here indicate that the promoter signal is the strongest single motif-based signal in the non-coding functional fraction of the genome. They also lend support to the belief that there exists a substantial subset of promoter regions which share several common features including, but not restricted to, a relative abundance of CpG dinucleotides. This subset is detectable by a variety of distinct computational methods.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="pagerange">131</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15369604">15369604</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC523850">523850</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-5-131">10.1186/1471-2105-5-131</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">SNP allele frequency estimation in DNA pools and variance components analysis.</h4>
<p class="authors"><span class="author">Downes K</span>, <span class="author">Barratt BJ</span>, <span class="author">Akan P</span>, <span class="author">Bumpstead SJ</span>, <span class="author">Taylor SD</span>, <span class="author">Clayton DG</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><p class="abstract">The estimation of single nucleotide polymorphism (SNP) allele frequency in pooled DNA samples has been proposed as a cost-effective approach to whole genome association studies. However, the key issue is the allele frequency window in which a genotyping method operates and provides a statistically reliable answer. We assessed the homogeneous mass extend assay and estimated the variance associated with each experimental stage. We report that a relationship between estimated allele frequency and variance might exist, suggesting that high statistical power can be retained at low, as well as high, allele frequencies. Assuming this relationship, the formation of subpools consisting of 100 samples retains an effective sample size greater than 70% of the true sample size, with a savings of 11-fold the cost of an individual genotyping study, regardless of allele frequency.</p>
<p class="publication"><span class="title">BioTechniques</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">5</span>;<span class="pagerange">840-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15152604" rel="external">15152604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The DNA sequence and analysis of human chromosome 13.</h4>
<p class="authors"><span class="author">Dunham A</span>, <span class="author">Matthews LH</span>, <span class="author">Burton J</span>, <span class="author">Ashurst JL</span>, <span class="author">Howe KL</span>, <span class="author">Ashcroft KJ</span>, <span class="author">Beare DM</span>, <span class="author">Burford DC</span>, <span class="author">Hunt SE</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Jones MC</span>, <span class="author">Keenan SJ</span>, <span class="author">Oliver K</span>, <span class="author">Scott CE</span>, <span class="author">Ainscough R</span>, <span class="author">Almeida JP</span>, <span class="author">Ambrose KD</span>, <span class="author">Andrews DT</span>, <span class="author">Ashwell RI</span>, <span class="author">Babbage AK</span>, <span class="author">Bagguley CL</span>, <span class="author">Bailey J</span>, <span class="author">Bannerjee R</span>, <span class="author">Barlow KF</span>, <span class="author">Bates K</span>, <span class="author">Beasley H</span>, <span class="author">Bird CP</span>, <span class="author">Bray-Allen S</span>, <span class="author">Brown AJ</span>, <span class="author">Brown JY</span>, <span class="author">Burrill W</span>, <span class="author">Carder C</span>, <span class="author">Carter NP</span>, <span class="author">Chapman JC</span>, <span class="author">Clamp ME</span>, <span class="author">Clark SY</span>, <span class="author">Clarke G</span>, <span class="author">Clee CM</span>, <span class="author">Clegg SC</span>, <span class="author">Cobley V</span>, <span class="author">Collins JE</span>, <span class="author">Corby N</span>, <span class="author">Coville GJ</span>, <span class="author">Deloukas P</span>, <span class="author">Dhami P</span>, <span class="author">Dunham I</span>, <span class="author">Dunn M</span>, <span class="author">Earthrowl ME</span>, <span class="author">Ellington AG</span>, <span class="author">Faulkner L</span>, <span class="author">Frankish AG</span>, <span class="author">Frankland J</span>, <span class="author">French L</span>, <span class="author">Garner P</span>, <span class="author">Garnett J</span>, <span class="author">Gilbert JG</span>, <span class="author">Gilson CJ</span>, <span class="author">Ghori J</span>, <span class="author">Grafham DV</span>, <span class="author">Gribble SM</span>, <span class="author">Griffiths C</span>, <span class="author">Hall RE</span>, <span class="author">Hammond S</span>, <span class="author">Harley JL</span>, <span class="author">Hart EA</span>, <span class="author">Heath PD</span>, <span class="author">Howden PJ</span>, <span class="author">Huckle EJ</span>, <span class="author">Hunt PJ</span>, <span class="author">Hunt AR</span>, <span class="author">Johnson C</span>, <span class="author">Johnson D</span>, <span class="author">Kay M</span>, <span class="author">Kimberley AM</span>, <span class="author">King A</span>, <span class="author">Laird GK</span>, <span class="author">Langford CJ</span>, <span class="author">Lawlor S</span>, <span class="author">Leongamornlert DA</span>, <span class="author">Lloyd DM</span>, <span class="author">Lloyd C</span>, <span class="author">Loveland JE</span>, <span class="author">Lovell J</span>, <span class="author">Martin S</span>, <span class="author">Mashreghi-Mohammadi M</span>, <span class="author">McLaren SJ</span>, <span class="author">McMurray A</span>, <span class="author">Milne S</span>, <span class="author">Moore MJ</span>, <span class="author">Nickerson T</span>, <span class="author">Palmer SA</span>, <span class="author">Pearce AV</span>, <span class="author">Peck AI</span>, <span class="author">Pelan S</span>, <span class="author">Phillimore B</span>, <span class="author">Porter KM</span>, <span class="author">Rice CM</span>, <span class="author">Searle S</span>, <span class="author">Sehra HK</span>, <span class="author">Shownkeen R</span>, <span class="author">Skuce CD</span>, <span class="author">Smith M</span>, <span class="author">Steward CA</span>, <span class="author">Sycamore N</span>, <span class="author">Tester J</span>, <span class="author">Thomas DW</span>, <span class="author">Tracey A</span>, <span class="author">Tromans A</span>, <span class="author">Tubby B</span>, <span class="author">Wall M</span>, <span class="author">Wallis JM</span>, <span class="author">West AP</span>, <span class="author">Whitehead SL</span>, <span class="author">Willey DL</span>, <span class="author">Wilming L</span>, <span class="author">Wray PW</span>, <span class="author">Wright MW</span>, <span class="author">Young L</span>, <span class="author">Coulson A</span>, <span class="author">Durbin R</span>, <span class="author">Hubbard T</span>, <span class="author">Sulston JE</span>, <span class="author">Beck S</span>, <span class="author">Bentley DR</span>, <span class="author">Rogers J</span> and <span class="author">Ross MT</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%61%64%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#100;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chromosome 13 is the largest acrocentric human chromosome. It carries genes involved in cancer including the breast cancer type 2 (BRCA2) and retinoblastoma (RB1) genes, is frequently rearranged in B-cell chronic lymphocytic leukaemia, and contains the DAOA locus associated with bipolar disorder and schizophrenia. We describe completion and analysis of 95.5 megabases (Mb) of sequence from chromosome 13, which contains 633 genes and 296 pseudogenes. We estimate that more than 95.4% of the protein-coding genes of this chromosome have been identified, on the basis of comparison with other vertebrate genome sequences. Additionally, 105 putative non-coding RNA genes were found. Chromosome 13 has one of the lowest gene densities (6.5 genes per Mb) among human chromosomes, and contains a central region of 38 Mb where the gene density drops to only 3.1 genes per Mb.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">428</span>;<span class="number">6982</span>;<span class="pagerange">522-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15057823">15057823</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665288">2665288</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02379">10.1038/nature02379</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Linkage disequilibrium mapping via cladistic analysis of single-nucleotide polymorphism haplotypes.</h4>
<p class="authors"><span class="author">Durrant C</span>, <span class="author">Zondervan KT</span>, <span class="author">Cardon LR</span>, <span class="author">Hunt S</span>, <span class="author">Deloukas P</span> and <span class="author">Morris AP</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom.</p><p class="abstract">We present a novel approach to disease-gene mapping via cladistic analysis of single-nucleotide polymorphism (SNP) haplotypes obtained from large-scale, population-based association studies, applicable to whole-genome screens, candidate-gene studies, or fine-scale mapping. Clades of haplotypes are tested for association with disease, exploiting the expected similarity of chromosomes with recent shared ancestry in the region flanking the disease gene. The method is developed in a logistic-regression framework and can easily incorporate covariates such as environmental risk factors or additional unlinked loci to allow for population structure. To evaluate the power of this approach to detect disease-marker association, we have developed a simulation algorithm to generate high-density SNP data with short-range linkage disequilibrium based on empirical patterns of haplotype diversity. The results of the simulation study highlight substantial gains in power over single-locus tests for a wide range of disease models, despite overcorrection for multiple testing.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">75</span>;<span class="number">1</span>;<span class="pagerange">35-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15148658" rel="external">15148658</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1182006" rel="external">1182006</a>; DOI: <a href="http://dx.doi.org/10.1086%2F422174" rel="external">10.1086/422174</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Production of soluble mammalian proteins in Escherichia coli: identification of protein features that correlate with successful expression.</h4>
<p class="authors"><span class="author">Dyson MR</span>, <span class="author">Shadbolt SP</span>, <span class="author">Vincent KJ</span>, <span class="author">Perera RL</span> and <span class="author">McCafferty J</span></p>
<p class="affiliation">The Atlas of Gene Expression Project, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%6D%72%64%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#114;&#100;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: In the search for generic expression strategies for mammalian protein families several bacterial expression vectors were examined for their ability to promote high yields of soluble protein. Proteins studied included cell surface receptors (Ephrins and Eph receptors, CD44), kinases (EGFR-cytoplasmic domain, CDK2 and 4), proteases (MMP1, CASP2), signal transduction proteins (GRB2, RAF1, HRAS) and transcription factors (GATA2, Fli1, Trp53, Mdm2, JUN, FOS, MAD, MAX). Over 400 experiments were performed where expression of 30 full-length proteins and protein domains were evaluated with 6 different N-terminal and 8 C-terminal fusion partners. Expression of an additional set of 95 mammalian proteins was also performed to test the conclusions of this study.</p><p class="abstract"><strong>Results</strong>: Several protein features correlated with soluble protein expression yield including molecular weight and the number of contiguous hydrophobic residues and low complexity regions. There was no relationship between successful expression and protein pI, grand average of hydropathicity (GRAVY), or sub-cellular location. Only small globular cytoplasmic proteins with an average molecular weight of 23 kDa did not require a solubility enhancing tag for high level soluble expression. Thioredoxin (Trx) and maltose binding protein (MBP) were the best N-terminal protein fusions to promote soluble expression, but MBP was most effective as a C-terminal fusion. 63 of 95 mammalian proteins expressed at soluble levels of greater than 1 mg/l as N-terminal H10-MBP fusions and those that failed possessed, on average, a higher molecular weight and greater number of contiguous hydrophobic amino acids and low complexity regions.</p><p class="abstract"><strong>Conclusions</strong>: By analysis of the protein features identified here, this study will help predict which mammalian proteins and domains can be successfully expressed in E. coli as soluble product and also which are best targeted for a eukaryotic expression system. In some cases proteins may be truncated to minimise molecular weight and the numbers of contiguous hydrophobic amino acids and low complexity regions to aid soluble expression in E. coli.</p>
<p class="publication"><span class="title">BMC biotechnology</span>&nbsp;<span class="year">2004</span>;<span class="volume">4</span>;<span class="pagerange">32</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15598350" rel="external">15598350</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC544853" rel="external">544853</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1472-6750-4-32" rel="external">10.1186/1472-6750-4-32</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Future potential of the Human Epigenome Project.</h4>
<p class="authors"><span class="author">Eckhardt F</span>, <span class="author">Beck S</span>, <span class="author">Gut IG</span> and <span class="author">Berlin K</span></p>
<p class="affiliation">Epigenomics AG, Kleine Pr&auml;sidentenstrasse 1, 10178 Berlin, Germany. <a href="mailto:%66%6C%6F%72%69%61%6E.%65%63%6B%68%61%72%64%74%40%65%70%69%67%65%6E%6F%6D%69%63%73.%63%6F%6D">&#102;&#108;&#111;&#114;&#105;&#97;&#110;&#46;&#101;&#99;&#107;&#104;&#97;&#114;&#100;&#116;&#64;&#101;&#112;&#105;&#103;&#101;&#110;&#111;&#109;&#105;&#99;&#115;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Deciphering the information encoded in the human genome is key for the further understanding of human biology, physiology and evolution. With the draft sequence of the human genome completed, elucidation of the epigenetic information layer of the human genome becomes accessible. Epigenetic mechanisms are mediated by either chemical modifications of the DNA itself or by modifications of proteins that are closely associated with DNA. Defects of the epigenetic regulation involved in processes such as imprinting, X chromosome inactivation, transcriptional control of genes, as well as mutations affecting DNA methylation enzymes, contribute fundamentally to the etiology of many human diseases. Headed by the Human Epigenome Consortium, the Human Epigenome Project is a joint effort by an international collaboration that aims to identify, catalog and interpret genome-wide DNA methylation patterns of all human genes in all major tissues. Methylation variable positions are thought to reflect gene activity, tissue type and disease state, and are useful epigenetic markers revealing the dynamic state of the genome. Like single nucleotide polymorphisms, methylation variable positions will greatly advance our ability to elucidate and diagnose the molecular basis of human diseases.</p>
<p class="publication"><span class="title">Expert review of molecular diagnostics</span>&nbsp;<span class="year">2004</span>;<span class="volume">4</span>;<span class="number">5</span>;<span class="pagerange">609-18</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15347255" rel="external">15347255</a>; DOI: <a href="http://dx.doi.org/10.1586%2F14737159.4.5.609" rel="external">10.1586/14737159.4.5.609</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Advances in schistosome genomics.</h4>
<p class="authors"><span class="author">El-Sayed NM</span>, <span class="author">Bartholomeu D</span>, <span class="author">Ivens A</span>, <span class="author">Johnston DA</span> and <span class="author">LoVerde PT</span></p>
<p class="affiliation">Department of Parasite Genomics, The Institute for Genomic Research, 9712 Medical Center Drive, Rockville, MD 20850, USA.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">U01 AI 48828</span></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">4</span>;<span class="pagerange">154-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15099549" rel="external">15099549</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2004.02.002" rel="external">10.1016/j.pt.2004.02.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative evolutionary genomics of androgen-binding protein genes.</h4>
<p class="authors"><span class="author">Emes RD</span>, <span class="author">Riley MC</span>, <span class="author">Laukaitis CM</span>, <span class="author">Goodstadt L</span>, <span class="author">Karn RC</span> and <span class="author">Ponting CP</span></p>
<p class="affiliation">MRC Functional Genetics Unit, Department of Human Anatomy and Genetics, University of Oxford, Oxford OX1 3QX, United Kingdom.</p><p class="abstract">Allelic variation within the mouse androgen-binding protein (ABP) alpha subunit gene (Abpa) has been suggested to promote assortative mating and thus prezygotic isolation. This is consistent with the elevated evolutionary rates observed for the Abpa gene, and the Abpb and Abpg genes whose products (ABPbeta and ABPgamma) form heterodimers with ABPalpha. We have investigated the mouse sequence that contains the three Abpa/b/g genes, and orthologous regions in rat, human, and chimpanzee genomes. Our studies reveal extensive &quot;remodeling&quot; of this region: Duplication rates of Abpa-like and Abpbg-like genes in mouse are &gt;2 orders of magnitude higher than the average rate for all mouse genes; synonymous nucleotide substitution rates are twofold higher; and the Abpabg genomic region has expanded nearly threefold since divergence of the rodents. During this time, one in six amino acid sites in ABPbetagamma-like proteins appear to have been subject to positive selection; these may constitute a site of interaction with receptors or ligands. Greater adaptive variation among Abpbg-like sequences than among Abpa-like sequences suggests that assortative mating preferences are more influenced by variation in Abpbg-like genes. We propose a role for ABPalpha/beta/gamma proteins as pheromones, or in modulating odorant detection. This would account for the extraordinary adaptive evolution of these genes, and surrounding genomic regions, in murid rodents.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">8</span>;<span class="pagerange">1516-29</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15256509" rel="external">15256509</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC509260" rel="external">509260</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2540304" rel="external">10.1101/gr.2540304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The ENCODE (ENCyclopedia Of DNA Elements) Project.</h4>
<p class="authors"><span class="author">ENCODE Project Consortium</span></p><p class="abstract">The ENCyclopedia Of DNA Elements (ENCODE) Project aims to identify all functional elements in the human genome sequence. The pilot phase of the Project is focused on a specified 30 megabases (approximately 1%) of the human genome sequence and is organized as an international consortium of computational and laboratory-based scientists working to develop and apply high-throughput approaches for detecting all sequence elements that confer biological function. The results of this pilot phase will guide future efforts to analyze the entire human genome.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG003143</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">306</span>;<span class="number">5696</span>;<span class="pagerange">636-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15499007">15499007</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1105136">10.1126/science.1105136</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ESTGenes: alternative splicing from ESTs in Ensembl.</h4>
<p class="authors"><span class="author">Eyras E</span>, <span class="author">Caccamo M</span>, <span class="author">Curwen V</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%65%61%65%40%73%61%6E%67%65%72.%61%63.%75%6B">&#101;&#97;&#101;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We describe a novel algorithm for deriving the minimal set of nonredundant transcripts compatible with the splicing structure of a set of ESTs mapped on a genome. Sets of ESTs with compatible splicing are represented by a special type of graph. We describe the algorithms for building the graphs and for deriving the minimal set of transcripts from the graphs that are compatible with the evidence. These algorithms are part of the Ensembl automatic gene annotation system, and its results, using ESTs, are provided at www.ensembl.org as ESTgenes for the mosquito, Caenorhabditis briggsae, C. elegans, zebrafish, human, mouse, and rat genomes. Here we also report on the results of this method applied to the human and mouse genomes.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">976-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123595" rel="external">15123595</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479129" rel="external">479129</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1862204" rel="external">10.1101/gr.1862204</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Amplification and overexpression of E2F3 in human bladder cancer.</h4>
<p class="authors"><span class="author">Feber A</span>, <span class="author">Clark J</span>, <span class="author">Goodwin G</span>, <span class="author">Dodson AR</span>, <span class="author">Smith PH</span>, <span class="author">Fletcher A</span>, <span class="author">Edwards S</span>, <span class="author">Flohr P</span>, <span class="author">Falconer A</span>, <span class="author">Roe T</span>, <span class="author">Kovacs G</span>, <span class="author">Dennis N</span>, <span class="author">Fisher C</span>, <span class="author">Wooster R</span>, <span class="author">Huddart R</span>, <span class="author">Foster CS</span> and <span class="author">Cooper CS</span></p>
<p class="affiliation">Section of Molecular Carcinogenesis and Male Urological Cancer Research, Centre, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK.</p><p class="abstract">We demonstrate that, in human bladder cancer, amplification of the E2F3 gene, located at 6p22, is associated with overexpression of its encoded mRNA transcripts and high levels of expression of E2F3 protein. Immunohistochemical analyses of E2F3 protein levels have established that around one-third (33/101) of primary transitional cell carcinomas of the bladder overexpress nuclear E2F3 protein, with the proportion of tumours containing overexpressed nuclear E2F3 increasing with tumour stage and grade. When considered together with the established role of E2F3 in cell cycle progression, these results suggest that the E2F3 gene represents a candidate bladder cancer oncogene that is activated by DNA amplification and overexpression.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2004</span>;<span class="volume">23</span>;<span class="number">8</span>;<span class="pagerange">1627-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14716298" rel="external">14716298</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1207274" rel="external">10.1038/sj.onc.1207274</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A large AZFc deletion removes DAZ3/DAZ4 and nearby genes from men in Y haplogroup N.</h4>
<p class="authors"><span class="author">Fernandes S</span>, <span class="author">Paracchini S</span>, <span class="author">Meyer LH</span>, <span class="author">Floridia G</span>, <span class="author">Tyler-Smith C</span> and <span class="author">Vogt PH</span></p>
<p class="affiliation">Section of Molecular Genetics &amp; Infertility, Department of Gynecological Endocrinology &amp; Reproductive Medicine, University of Heidelberg, Heidelberg, Germany.</p><p class="abstract">Deletion of the entire AZFc locus on the human Y chromosome leads to male infertility. The functional roles of the individual gene families mapped to AZFc are, however, still poorly understood, since the analysis of the region is complicated by its repeated structure. We have therefore used single-nucleotide variants (SNVs) across approximately 3 Mb of the AZFc sequence to identify 17 AZFc haplotypes and have examined them for deletion of individual AZFc gene copies. We found five individuals who lacked SNVs from a large segment of DNA containing the DAZ3/DAZ4 and BPY2.2/BPY2.3 gene doublets in distal AZFc. Southern blot analyses showed that the lack of these SNVs was due to deletion of the underlying DNA segment. Typing 118 binary Y markers showed that all five individuals belonged to Y haplogroup N, and 15 of 15 independently ascertained men in haplogroup N carried a similar deletion. Haplogroup N is known to be common and widespread in Europe and Asia, and there is no indication of reduced fertility in men with this Y chromosome. We therefore conclude that a common variant of the human Y chromosome lacks the DAZ3/DAZ4 and BPY2.2/BPY2.3 doublets in distal AZFc and thus that these genes cannot be required for male fertility; the gene content of the AZFc locus is likely to be genetically redundant. Furthermore, the observed deletions cannot be derived from the GenBank reference sequence by a single recombination event; an origin by homologous recombination from such a sequence organization must be preceded by an inversion event. These data confirm the expectation that the human Y chromosome sequence and gene complement may differ substantially between individuals and more variations are to be expected in different Y chromosomal haplogroups.</p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">74</span>;<span class="number">1</span>;<span class="pagerange">180-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14639527" rel="external">14639527</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1181906" rel="external">1181906</a>; DOI: <a href="http://dx.doi.org/10.1086%2F381132" rel="external">10.1086/381132</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic array technology.</h4>
<p class="authors"><span class="author">Fiegler H</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">75</span>;<span class="pagerange">769-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15603452" rel="external">15603452</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A central role for the notochord in vertebral patterning.</h4>
<p class="authors"><span class="author">Fleming A</span>, <span class="author">Keynes R</span> and <span class="author">Tannahill D</span></p>
<p class="affiliation">Department of Anatomy, University of Cambridge, Downing Street, Cambridge CB2 3DY, UK.</p><p class="abstract">The vertebrates are defined by their segmented vertebral column, and vertebral periodicity is thought to originate from embryonic segments, the somites. According to the widely accepted &#39;resegmentation&#39; model, a single vertebra forms from the recombination of the anterior and posterior halves of two adjacent sclerotomes on both sides of the embryo. Although there is supporting evidence for this model in amniotes, it remains uncertain whether it applies to all vertebrates. To explore this, we have investigated vertebral patterning in the zebrafish. Surprisingly, we find that vertebral bodies (centra) arise by secretion of bone matrix from the notochord rather than somites; centra do not form via a cartilage intermediate stage, nor do they contain osteoblasts. Moreover, isolated, cultured notochords secrete bone matrix in vitro, and ablation of notochord cells at segmentally reiterated positions in vivo prevents the formation of centra. Analysis of fss mutant embryos, in which sclerotome segmentation is disrupted, shows that whereas neural arch segmentation is also disrupted, centrum development proceeds normally. These findings suggest that the notochord plays a key, perhaps ancient, role in the segmental patterning of vertebrae.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">131</span>;<span class="number">4</span>;<span class="pagerange">873-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14736741" rel="external">14736741</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.00952" rel="external">10.1242/dev.00952</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Nodal/activin signaling establishes oral-aboral polarity in the early sea urchin embryo.</h4>
<p class="authors"><span class="author">Flowers VL</span>, <span class="author">Courteau GR</span>, <span class="author">Poustka AJ</span>, <span class="author">Weng W</span> and <span class="author">Venuti JM</span></p>
<p class="affiliation">Department of Cell Biology and Anatomy, Louisiana State University Health Sciences Center, New Orleans, Louisiana 70112-1393, USA.</p><p class="abstract">Components of the Wnt signaling pathway are involved in patterning the sea urchin primary or animal-vegetal (AV) axis, but the molecular cues that pattern the secondary embryonic axis, the aboral/oral (AO) axis, are not known. In an analysis of signaling molecules that influence patterning along the sea urchin embryonic axes, we found that members of the activin subfamily of transforming growth factor-beta (TGF-beta) signaling molecules influence the establishment of AO polarities in the early embryo. Injection of activin mRNAs into fertilized eggs or treatment with exogenously applied recombinant activin altered the allocation of ectodermal fates and ventralized the embryo. The phenotypes observed resemble the ventralized phenotype previously reported for NiCl2, a known disrupter of AO patterning. Sensitivity to exogenous activin occurs between fertilization and the late blastula stage, which is also the time of highest NiCl2 sensitivity. These results argue that specification of fates along the embryonic AO axis involves TGF-beta signaling. To further examine TGF-beta signaling in these embryos, we cloned an endogenous TGF-beta from sea urchin embryos that is a member of the activin subfamily, SpNodal, and show through gain of function analysis that it recapitulates results obtained with exogenous activins and NiCl2. The expression pattern of SpNodal is consistent with a role for nodal signaling in the establishment of fates along the AO axis. Loss of function experiments using SpNodal antisense morpholinos also support a role for SpNodal in the establishment of the AO axis.</p>
<p class="publication"><span class="title">Developmental dynamics : an official publication of the American Association of Anatomists</span>&nbsp;<span class="year">2004</span>;<span class="volume">231</span>;<span class="number">4</span>;<span class="pagerange">727-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15517584" rel="external">15517584</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fdvdy.20194" rel="external">10.1002/dvdy.20194</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNA interference: human genes hit the big screen.</h4>
<p class="authors"><span class="author">Fraser A</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">428</span>;<span class="number">6981</span>;<span class="pagerange">375-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15042071">15042071</a>; DOI: <a href="http://dx.doi.org/10.1038%2F428375a">10.1038/428375a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Towards full employment: using RNAi to find roles for the redundant.</h4>
<p class="authors"><span class="author">Fraser A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Cancer is a genetic disease that ultimately results from the failure of cells to respond correctly to diverse signals. Signal transduction and signal integration are highly complex, requiring the combinatorial interaction of multiple genes. Classical genetics in model organisms including Caenorhabditis elegans has been of immense use in identifying nonredundant components of conserved signalling pathways. However, it is likely that there is much functional redundancy in the informational processing machinery of metazoan cells; we therefore need to develop methods for uncovering such redundant functions in model organisms if we are to use them to understand complex gene interactions and oncogene cooperation. RNAi may provide a powerful tool to probe redundancy in informational networks. In this review, I set out some of the progress made so far by classical genetics in understanding redundancy in gene networks, and outline how RNAi may allow us to approach this problem more systematically in C. elegans. In particular, I discuss the use of genome-wide RNAi screens in C. elegans to identify synthetic lethal interactions and compare this with synthetic lethal interaction analysis in Saccharomyces cerevisiae.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2004</span>;<span class="volume">23</span>;<span class="number">51</span>;<span class="pagerange">8346-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15517015" rel="external">15517015</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1208044" rel="external">10.1038/sj.onc.1208044</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A probabilistic view of gene function.</h4>
<p class="authors"><span class="author">Fraser AG</span> and <span class="author">Marcotte EM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%61%67%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Cells are controlled by the complex and dynamic actions of thousands of genes. With the sequencing of many genomes, the key problem has shifted from identifying genes to knowing what the genes do; we need a framework for expressing that knowledge. Even the most rigorous attempts to construct ontological frameworks describing gene function (e.g., the Gene Ontology project) ultimately rely on manual curation and are thus labor-intensive and subjective. But an alternative exists: the field of functional genomics is piecing together networks of gene interactions, and although these data are currently incomplete and error-prone, they provide a glimpse of a new, probabilistic view of gene function. We outline such a framework, which revolves around a statistical description of gene interactions derived from large, systematically compiled data sets. In this probabilistic view, pleiotropy is implicit, all data have errors and the definition of gene function is an iterative process that ultimately converges on the correct functions. The relationships between the genes are defined by the data, not by hand. Even this comprehensive view fails to capture key aspects of gene function, not least their dynamics in time and space, showing that there are limitations to the model that must ultimately be addressed.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">6</span>;<span class="pagerange">559-64</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15167932" rel="external">15167932</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1370" rel="external">10.1038/ng1370</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Development through the eyes of functional genomics.</h4>
<p class="authors"><span class="author">Fraser AG</span> and <span class="author">Marcotte EM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%67%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#103;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">In many of the model organisms used to study development, it is becoming relatively routine to carry out global analyses of gene function. These analyses take many forms, from microarray analyses to the construction of physical interaction maps to the systematic analyses of loss-of-function phenotypes. Such large-scale datasets can be integrated to generate complex gene networks, and we explore how these gene networks can contribute to an understanding of developmental pathways. In particular, we examine how combining large-scale expression experiments and gene networks may move us towards a molecular description of the events of development, embodied in a succession of stage-specific subnetworks sampled from an organism&#39;s overall gene network.</p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">336-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15261648" rel="external">15261648</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2004.06.015" rel="external">10.1016/j.gde.2004.06.015</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A census of human cancer genes.</h4>
<p class="authors"><span class="author">Futreal PA</span>, <span class="author">Coin L</span>, <span class="author">Marshall M</span>, <span class="author">Down T</span>, <span class="author">Hubbard T</span>, <span class="author">Wooster R</span>, <span class="author">Rahman N</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Human Genome Analysis Group and Pfam Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton Cambs, CB10 1SA, UK.</p>
<p class="publication"><span class="title">Nature reviews. Cancer</span>&nbsp;<span class="year">2004</span>;<span class="volume">4</span>;<span class="number">3</span>;<span class="pagerange">177-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14993899">14993899</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2665285">2665285</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrc1299">10.1038/nrc1299</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global gene expression of fission yeast in response to cisplatin.</h4>
<p class="authors"><span class="author">Gatti L</span>, <span class="author">Chen D</span>, <span class="author">Beretta GL</span>, <span class="author">Rustici G</span>, <span class="author">Carenini N</span>, <span class="author">Corna E</span>, <span class="author">Colangelo D</span>, <span class="author">Zunino F</span>, <span class="author">B&auml;hler J</span> and <span class="author">Perego P</span></p>
<p class="affiliation">Istituto Nazionale Tumori, 20133, Milan, Italy.</p><p class="abstract">The cellular response to the antitumor drug cisplatin is complex, and resistance is widespread. To gain insights into the global transcriptional response and mechanisms of resistance, we used microarrays to examine the fission yeast cell response to cisplatin. In two isogenic strains with differing drug sensitivity, cisplatin activated a stress response involving glutathione-S-transferase, heat shock, and recombinational repair genes. Genes required for proteasome-mediated protein degradation were up-regulated in the sensitive strain, whereas genes for DNA damage recognition/repair and for mitotic progression were induced in the resistant strain. The response to cisplatin overlaps in part with the responses to cadmium and the DNA-damaging agent methylmethane sulfonate. The different gene groups involved in the cellular response to cisplatin help the cells to tolerate and repair DNA damage and to overcome cell cycle blocks. These findings are discussed with respect to known cisplatin response pathways in human cells.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Cellular and molecular life sciences : CMLS</span>&nbsp;<span class="year">2004</span>;<span class="volume">61</span>;<span class="number">17</span>;<span class="pagerange">2253-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15338055" rel="external">15338055</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00018-004-4218-5" rel="external">10.1007/s00018-004-4218-5</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A family with severe insulin resistance and diabetes due to a mutation in AKT2.</h4>
<p class="authors"><span class="author">George S</span>, <span class="author">Rochford JJ</span>, <span class="author">Wolfrum C</span>, <span class="author">Gray SL</span>, <span class="author">Schinner S</span>, <span class="author">Wilson JC</span>, <span class="author">Soos MA</span>, <span class="author">Murgatroyd PR</span>, <span class="author">Williams RM</span>, <span class="author">Acerini CL</span>, <span class="author">Dunger DB</span>, <span class="author">Barford D</span>, <span class="author">Umpleby AM</span>, <span class="author">Wareham NJ</span>, <span class="author">Davies HA</span>, <span class="author">Schafer AJ</span>, <span class="author">Stoffel M</span>, <span class="author">O&#39;Rahilly S</span> and <span class="author">Barroso I</span></p>
<p class="affiliation">Department of Clinical Biochemistry, University of Cambridge, Addenbrooke&#39;s Hospital, Hills Road, Cambridge CB2 2QQ, UK.</p><p class="abstract">Inherited defects in signaling pathways downstream of the insulin receptor have long been suggested to contribute to human type 2 diabetes mellitus. Here we describe a mutation in the gene encoding the protein kinase AKT2/PKBbeta in a family that shows autosomal dominant inheritance of severe insulin resistance and diabetes mellitus. Expression of the mutant kinase in cultured cells disrupted insulin signaling to metabolic end points and inhibited the function of coexpressed, wild-type AKT. These findings demonstrate the central importance of AKT signaling to insulin sensitivity in humans.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">078986</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">304</span>;<span class="number">5675</span>;<span class="pagerange">1325-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15166380">15166380</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2258004">2258004</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1096706">10.1126/science.1096706</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene synteny and evolution of genome architecture in trypanosomatids.</h4>
<p class="authors"><span class="author">Ghedin E</span>, <span class="author">Bringaud F</span>, <span class="author">Peterson J</span>, <span class="author">Myler P</span>, <span class="author">Berriman M</span>, <span class="author">Ivens A</span>, <span class="author">Andersson B</span>, <span class="author">Bontempi E</span>, <span class="author">Eisen J</span>, <span class="author">Angiuoli S</span>, <span class="author">Wanless D</span>, <span class="author">Von Arx A</span>, <span class="author">Murphy L</span>, <span class="author">Lennard N</span>, <span class="author">Salzberg S</span>, <span class="author">Adams MD</span>, <span class="author">White O</span>, <span class="author">Hall N</span>, <span class="author">Stuart K</span>, <span class="author">Fraser CM</span> and <span class="author">El-Sayed NM</span></p>
<p class="affiliation">Parasity Genomics, The Institute for Genomics Research, 9712 Medical Center Dr. Rockville, MD 20850, USA.</p><p class="abstract">The trypanosomatid protozoa Trypanosoma brucei, Trypanosoma cruzi and Leishmania major are related human pathogens that cause markedly distinct diseases. Using information from genome sequencing projects currently underway, we have compared the sequences of large chromosomal fragments from each species. Despite high levels of divergence at the sequence level, these three species exhibit a striking conservation of gene order, suggesting that selection has maintained gene order among the trypanosomatids over hundreds of millions of years of evolution. The few sites of genome rearrangement between these species are marked by the presence of retrotransposon-like elements, suggesting that retrotransposons may have played an important role in shaping trypanosomatid genome organization. A degenerate retroelement was identified in L. major by examining the regions near breakage points of the synteny. This is the first such element found in L. major suggesting that retroelements were found in the common ancestor of all three species.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI43062</span>, <span class="grant">AI45038</span>, <span class="grant">AI45039</span>, <span class="grant">AI45061</span>, <span class="grant">AI49599</span></p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">134</span>;<span class="number">2</span>;<span class="pagerange">183-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15003838" rel="external">15003838</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2003.11.012" rel="external">10.1016/j.molbiopara.2003.11.012</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome sequence of the Brown Norway rat yields insights into mammalian evolution.</h4>
<p class="authors"><span class="author">Gibbs RA</span>, <span class="author">Weinstock GM</span>, <span class="author">Metzker ML</span>, <span class="author">Muzny DM</span>, <span class="author">Sodergren EJ</span>, <span class="author">Scherer S</span>, <span class="author">Scott G</span>, <span class="author">Steffen D</span>, <span class="author">Worley KC</span>, <span class="author">Burch PE</span>, <span class="author">Okwuonu G</span>, <span class="author">Hines S</span>, <span class="author">Lewis L</span>, <span class="author">DeRamo C</span>, <span class="author">Delgado O</span>, <span class="author">Dugan-Rocha S</span>, <span class="author">Miner G</span>, <span class="author">Morgan M</span>, <span class="author">Hawes A</span>, <span class="author">Gill R</span>, <span class="author">Celera</span>, <span class="author">Holt RA</span>, <span class="author">Adams MD</span>, <span class="author">Amanatides PG</span>, <span class="author">Baden-Tillson H</span>, <span class="author">Barnstead M</span>, <span class="author">Chin S</span>, <span class="author">Evans CA</span>, <span class="author">Ferriera S</span>, <span class="author">Fosler C</span>, <span class="author">Glodek A</span>, <span class="author">Gu Z</span>, <span class="author">Jennings D</span>, <span class="author">Kraft CL</span>, <span class="author">Nguyen T</span>, <span class="author">Pfannkoch CM</span>, <span class="author">Sitter C</span>, <span class="author">Sutton GG</span>, <span class="author">Venter JC</span>, <span class="author">Woodage T</span>, <span class="author">Smith D</span>, <span class="author">Lee HM</span>, <span class="author">Gustafson E</span>, <span class="author">Cahill P</span>, <span class="author">Kana A</span>, <span class="author">Doucette-Stamm L</span>, <span class="author">Weinstock K</span>, <span class="author">Fechtel K</span>, <span class="author">Weiss RB</span>, <span class="author">Dunn DM</span>, <span class="author">Green ED</span>, <span class="author">Blakesley RW</span>, <span class="author">Bouffard GG</span>, <span class="author">De Jong PJ</span>, <span class="author">Osoegawa K</span>, <span class="author">Zhu B</span>, <span class="author">Marra M</span>, <span class="author">Schein J</span>, <span class="author">Bosdet I</span>, <span class="author">Fjell C</span>, <span class="author">Jones S</span>, <span class="author">Krzywinski M</span>, <span class="author">Mathewson C</span>, <span class="author">Siddiqui A</span>, <span class="author">Wye N</span>, <span class="author">McPherson J</span>, <span class="author">Zhao S</span>, <span class="author">Fraser CM</span>, <span class="author">Shetty J</span>, <span class="author">Shatsman S</span>, <span class="author">Geer K</span>, <span class="author">Chen Y</span>, <span class="author">Abramzon S</span>, <span class="author">Nierman WC</span>, <span class="author">Havlak PH</span>, <span class="author">Chen R</span>, <span class="author">Durbin KJ</span>, <span class="author">Egan A</span>, <span class="author">Ren Y</span>, <span class="author">Song XZ</span>, <span class="author">Li B</span>, <span class="author">Liu Y</span>, <span class="author">Qin X</span>, <span class="author">Cawley S</span>, <span class="author">Worley KC</span>, <span class="author">Cooney AJ</span>, <span class="author">D&#39;Souza LM</span>, <span class="author">Martin K</span>, <span class="author">Wu JQ</span>, <span class="author">Gonzalez-Garay ML</span>, <span class="author">Jackson AR</span>, <span class="author">Kalafus KJ</span>, <span class="author">McLeod MP</span>, <span class="author">Milosavljevic A</span>, <span class="author">Virk D</span>, <span class="author">Volkov A</span>, <span class="author">Wheeler DA</span>, <span class="author">Zhang Z</span>, <span class="author">Bailey JA</span>, <span class="author">Eichler EE</span>, <span class="author">Tuzun E</span>, <span class="author">Birney E</span>, <span class="author">Mongin E</span>, <span class="author">Ureta-Vidal A</span>, <span class="author">Woodwark C</span>, <span class="author">Zdobnov E</span>, <span class="author">Bork P</span>, <span class="author">Suyama M</span>, <span class="author">Torrents D</span>, <span class="author">Alexandersson M</span>, <span class="author">Trask BJ</span>, <span class="author">Young JM</span>, <span class="author">Huang H</span>, <span class="author">Wang H</span>, <span class="author">Xing H</span>, <span class="author">Daniels S</span>, <span class="author">Gietzen D</span>, <span class="author">Schmidt J</span>, <span class="author">Stevens K</span>, <span class="author">Vitt U</span>, <span class="author">Wingrove J</span>, <span class="author">Camara F</span>, <span class="author">Mar Alb&agrave; M</span>, <span class="author">Abril JF</span>, <span class="author">Guigo R</span>, <span class="author">Smit A</span>, <span class="author">Dubchak I</span>, <span class="author">Rubin EM</span>, <span class="author">Couronne O</span>, <span class="author">Poliakov A</span>, <span class="author">H&uuml;bner N</span>, <span class="author">Ganten D</span>, <span class="author">Goesele C</span>, <span class="author">Hummel O</span>, <span class="author">Kreitler T</span>, <span class="author">Lee YA</span>, <span class="author">Monti J</span>, <span class="author">Schulz H</span>, <span class="author">Zimdahl H</span>, <span class="author">Himmelbauer H</span>, <span class="author">Lehrach H</span>, <span class="author">Jacob HJ</span>, <span class="author">Bromberg S</span>, <span class="author">Gullings-Handley J</span>, <span class="author">Jensen-Seaman MI</span>, <span class="author">Kwitek AE</span>, <span class="author">Lazar J</span>, <span class="author">Pasko D</span>, <span class="author">Tonellato PJ</span>, <span class="author">Twigger S</span>, <span class="author">Ponting CP</span>, <span class="author">Duarte JM</span>, <span class="author">Rice S</span>, <span class="author">Goodstadt L</span>, <span class="author">Beatson SA</span>, <span class="author">Emes RD</span>, <span class="author">Winter EE</span>, <span class="author">Webber C</span>, <span class="author">Brandt P</span>, <span class="author">Nyakatura G</span>, <span class="author">Adetobi M</span>, <span class="author">Chiaromonte F</span>, <span class="author">Elnitski L</span>, <span class="author">Eswara P</span>, <span class="author">Hardison RC</span>, <span class="author">Hou M</span>, <span class="author">Kolbe D</span>, <span class="author">Makova K</span>, <span class="author">Miller W</span>, <span class="author">Nekrutenko A</span>, <span class="author">Riemer C</span>, <span class="author">Schwartz S</span>, <span class="author">Taylor J</span>, <span class="author">Yang S</span>, <span class="author">Zhang Y</span>, <span class="author">Lindpaintner K</span>, <span class="author">Andrews TD</span>, <span class="author">Caccamo M</span>, <span class="author">Clamp M</span>, <span class="author">Clarke L</span>, <span class="author">Curwen V</span>, <span class="author">Durbin R</span>, <span class="author">Eyras E</span>, <span class="author">Searle SM</span>, <span class="author">Cooper GM</span>, <span class="author">Batzoglou S</span>, <span class="author">Brudno M</span>, <span class="author">Sidow A</span>, <span class="author">Stone EA</span>, <span class="author">Venter JC</span>, <span class="author">Payseur BA</span>, <span class="author">Bourque G</span>, <span class="author">L&oacute;pez-Ot&iacute;n C</span>, <span class="author">Puente XS</span>, <span class="author">Chakrabarti K</span>, <span class="author">Chatterji S</span>, <span class="author">Dewey C</span>, <span class="author">Pachter L</span>, <span class="author">Bray N</span>, <span class="author">Yap VB</span>, <span class="author">Caspi A</span>, <span class="author">Tesler G</span>, <span class="author">Pevzner PA</span>, <span class="author">Haussler D</span>, <span class="author">Roskin KM</span>, <span class="author">Baertsch R</span>, <span class="author">Clawson H</span>, <span class="author">Furey TS</span>, <span class="author">Hinrichs AS</span>, <span class="author">Karolchik D</span>, <span class="author">Kent WJ</span>, <span class="author">Rosenbloom KR</span>, <span class="author">Trumbower H</span>, <span class="author">Weirauch M</span>, <span class="author">Cooper DN</span>, <span class="author">Stenson PD</span>, <span class="author">Ma B</span>, <span class="author">Brent M</span>, <span class="author">Arumugam M</span>, <span class="author">Shteynberg D</span>, <span class="author">Copley RR</span>, <span class="author">Taylor MS</span>, <span class="author">Riethman H</span>, <span class="author">Mudunuri U</span>, <span class="author">Peterson J</span>, <span class="author">Guyer M</span>, <span class="author">Felsenfeld A</span>, <span class="author">Old S</span>, <span class="author">Mockrin S</span>, <span class="author">Collins F</span> and <span class="author">Rat Genome Sequencing Project Consortium</span></p>
<p class="affiliation">Human Genome Sequencing Center, Department of Molecular and Human Genetics, Baylor College of Medicine, MS BCM226, One Baylor Plaza, Houston, Texas 77030, USA &lt;http://www.hgsc.bcm.tmc.edu&gt;.</p><p class="abstract">The laboratory rat (Rattus norvegicus) is an indispensable tool in experimental medicine and drug development, having made inestimable contributions to human health. We report here the genome sequence of the Brown Norway (BN) rat strain. The sequence represents a high-quality &#39;draft&#39; covering over 90% of the genome. The BN rat sequence is the third complete mammalian genome to be deciphered, and three-way comparisons with the human and mouse genomes resolve details of mammalian evolution. This first comprehensive analysis includes genes and proteins and their relation to human disease, repeated sequences, comparative genome-wide studies of mammalian orthologous chromosomal regions and rearrangement breakpoints, reconstruction of ancestral karyotypes and the events leading to existing species, rates of variation, and lineage-specific and lineage-independent evolutionary events such as expansion of gene families, orthology relations and protein evolution.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">U01 HG002137-02S2</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">428</span>;<span class="number">6982</span>;<span class="pagerange">493-521</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15057822">15057822</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02426">10.1038/nature02426</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromatin architecture of the human genome: gene-rich domains are enriched in open chromatin fibers.</h4>
<p class="authors"><span class="author">Gilbert N</span>, <span class="author">Boyle S</span>, <span class="author">Fiegler H</span>, <span class="author">Woodfine K</span>, <span class="author">Carter NP</span> and <span class="author">Bickmore WA</span></p>
<p class="affiliation">MRC Human Genetics Unit, Edinburgh, EH4 2XU, Scotland.</p><p class="abstract">We present an analysis of chromatin fiber structure across the human genome. Compact and open chromatin fiber structures were separated by sucrose sedimentation and their distributions analyzed by hybridization to metaphase chromosomes and genomic microarrays. We show that compact chromatin fibers originate from some sites of heterochromatin (C-bands), and G-bands (euchromatin). Open chromatin fibers correlate with regions of highest gene density, but not with gene expression since inactive genes can be in domains of open chromatin, and active genes in regions of low gene density can be embedded in compact chromatin fibers. Moreover, we show that chromatin fiber structure impacts on further levels of chromatin condensation. Regions of open chromatin fibers are cytologically decondensed and have a distinctive nuclear organization. We suggest that domains of open chromatin may create an environment that facilitates transcriptional activation and could provide an evolutionary constraint to maintain clusters of genes together along chromosomes.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">118</span>;<span class="number">5</span>;<span class="pagerange">555-66</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15339661">15339661</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2004.08.011">10.1016/j.cell.2004.08.011</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The complete nucleotide sequence of the resistance plasmid R478: defining the backbone components of incompatibility group H conjugative plasmids through comparative genomics.</h4>
<p class="authors"><span class="author">Gilmour MW</span>, <span class="author">Thomson NR</span>, <span class="author">Sanders M</span>, <span class="author">Parkhill J</span> and <span class="author">Taylor DE</span></p>
<p class="affiliation">Department of Medical Microbiology and Immunology, University of Alberta, Edmonton, Alta., Canada T6G 2R3.</p><p class="abstract">Horizontal transfer of resistance determinants amongst bacteria can be achieved by conjugative plasmid DNA elements. We have determined the complete 274,762 bp sequence of the incompatibility group H (IncH) plasmid R478, originally isolated from the Gram negative opportunistic pathogen Serratia marcescens. This self-transferable extrachromosomal genetic element contains 295 predicted genes, of which 144 are highly similar to coding sequences of IncH plasmids R27 and pHCM1. The regions of similarity among these three IncH plasmids principally encode core plasmid determinants (i.e., replication, partitioning and stability, and conjugative transfer) and we conducted a comparative analysis to define the minimal IncHI plasmid backbone determinants. No resistance determinants are included in the backbone and most of the sequences unique to R478 were contained in a large contiguous region between the two transfer regions. These findings indicate that plasmid evolution occurs through gene acquisition/loss predominantly in regions outside of the core determinants. Furthermore, a modular evolution for R478 was signified by the presence of gene neighbors or operons that were highly related to sequences from a wide range of chromosomal, transposon, and plasmid elements. The conjugative transfer regions are most similar to sequences encoded on SXT, Rts1, pCAR1, R391, and pRS241d. The dual partitioning modules encoded on R478 resemble numerous sequences; including pMT1, pCTX-M3, pCP301, P1, P7, and pB171. R478 also codes for resistance to tetracycline (Tn10), chloramphenicol (cat), kanamycin (aphA), mercury (similar to Tn21), silver (similar to pMG101), copper (similar to pRJ1004), arsenic (similar to pYV), and tellurite (two separate regions similar to IncHI2 ter determinants and IncP kla determinants). Other R478-encoded sequences are related to Tn7, IS26, tus, mucAB, and hok, where the latter is surrounded by insLKJ, and could potentially be involved in post-segregation killing. The similarity to a diverse set of bacterial sequences highlights the ability of horizontally transferable DNA elements to acquire and disseminate genetic traits through the bacterial gene pool.</p>
<p class="publication"><span class="title">Plasmid</span>&nbsp;<span class="year">2004</span>;<span class="volume">52</span>;<span class="number">3</span>;<span class="pagerange">182-202</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15518875" rel="external">15518875</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.plasmid.2004.06.006" rel="external">10.1016/j.plasmid.2004.06.006</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Coping with cold: An integrative, multitissue analysis of the transcriptome of a poikilothermic vertebrate.</h4>
<p class="authors"><span class="author">Gracey AY</span>, <span class="author">Fraser EJ</span>, <span class="author">Li W</span>, <span class="author">Fang Y</span>, <span class="author">Taylor RR</span>, <span class="author">Rogers J</span>, <span class="author">Brass A</span> and <span class="author">Cossins AR</span></p>
<p class="affiliation">School of Biological Sciences, University of Liverpool, Biosciences Building, Crown Street, Liverpool L39 7ZB, United Kindgom. <a href="mailto:%61%67%72%61%63%65%79%40%73%74%61%6E%66%6F%72%64.%65%64%75">&#97;&#103;&#114;&#97;&#99;&#101;&#121;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></p><p class="abstract">How do organisms respond adaptively to environmental stress? Although some gene-specific responses have been explored, others remain to be identified, and there is a very poor understanding of the system-wide integration of response, particularly in complex, multitissue animals. Here, we adopt a transcript screening approach to explore the mechanisms underpinning a major, whole-body phenotypic transition in a vertebrate animal that naturally experiences extreme environmental stress. Carp were exposed to increasing levels of cold, and responses across seven tissues were assessed by using a microarray composed of 13,440 cDNA probes. A large set of unique cDNAs (approximately 3,400) were affected by cold. These cDNAs included an expression signature common to all tissues of 252 up-regulated genes involved in RNA processing, translation initiation, mitochondrial metabolism, proteasomal function, and modification of higher-order structures of lipid membranes and chromosomes. Also identified were large numbers of transcripts with highly tissue-specific patterns of regulation. By unbiased profiling of gene ontologies, we have identified the distinctive functional features of each tissue&#39;s response and integrate them into a comprehensive view of the whole-body transition from one strongly adaptive phenotype to another. This approach revealed an expression signature suggestive of atrophy in cooled skeletal muscle. This environmental genomics approach by using a well studied but nongenomic species has identified a range of candidate genes endowing thermotolerance and reveals a previously unrecognized scale and complexity of responses that impacts at the level of cellular and tissue function.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">48</span>;<span class="pagerange">16970-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15550548" rel="external">15550548</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC534716" rel="external">534716</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0403627101" rel="external">10.1073/pnas.0403627101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">1-Mb resolution array-based comparative genomic hybridization using a BAC clone set optimized for cancer gene analysis.</h4>
<p class="authors"><span class="author">Greshock J</span>, <span class="author">Naylor TL</span>, <span class="author">Margolin A</span>, <span class="author">Diskin S</span>, <span class="author">Cleaver SH</span>, <span class="author">Futreal PA</span>, <span class="author">deJong PJ</span>, <span class="author">Zhao S</span>, <span class="author">Liebman M</span> and <span class="author">Weber BL</span></p>
<p class="affiliation">Abramson Family Cancer Research Institute, University of Pennsylvania, Philadelphia, Pennsylvania 19104, USA.</p><p class="abstract">Array-based comparative genomic hybridization (aCGH) is a recently developed tool for genome-wide determination of DNA copy number alterations. This technology has tremendous potential for disease-gene discovery in cancer and developmental disorders as well as numerous other applications. However, widespread utilization of a CGH has been limited by the lack of well characterized, high-resolution clone sets optimized for consistent performance in aCGH assays and specifically designed analytic software. We have assembled a set of approximately 4100 publicly available human bacterial artificial chromosome (BAC) clones evenly spaced at approximately 1-Mb resolution across the genome, which includes direct coverage of approximately 400 known cancer genes. This aCGH-optimized clone set was compiled from five existing sets, experimentally refined, and supplemented for higher resolution and enhancing mapping capabilities. This clone set is associated with a public online resource containing detailed clone mapping data, protocols for the construction and use of arrays, and a suite of analytical software tools designed specifically for aCGH analysis. These resources should greatly facilitate the use of aCGH in gene discovery.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">179-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14672980" rel="external">14672980</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC314295" rel="external">314295</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1847304" rel="external">10.1101/gr.1847304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome paints from single copies of chromosomes.</h4>
<p class="authors"><span class="author">Gribble S</span>, <span class="author">Ng BL</span>, <span class="author">Prigmore E</span>, <span class="author">Burford DC</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">We have used OmniPlex library technology to construct chromosome painting probes from single copies of flow sorted chromosomes. We show that this whole genome amplification technology is particularly efficient at amplifying single copies of chromosomes for the production of paints and that single aberrant chromosomes can be analysed in this way using reverse chromosome painting. The efficient generation of painting probes from single copies of sorted chromosomes has the advantage that the probe must be specific for the chromosome sorted and will not suffer from contamination from other chromosomes particularly in situations where flow karyotype peaks are poorly resolved. These initial results suggest that OmniPlex whole genome amplification will be equally effective in other cytogenetic applications where only small amounts of DNA are available, i.e. from single cells or from small pieces of microdissected tissue.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">12</span>;<span class="number">2</span>;<span class="pagerange">143-51</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15053484">15053484</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Applications of combined DNA microarray and chromosome sorting technologies.</h4>
<p class="authors"><span class="author">Gribble SM</span>, <span class="author">Fiegler H</span>, <span class="author">Burford DC</span>, <span class="author">Prigmore E</span>, <span class="author">Yang F</span>, <span class="author">Carr P</span>, <span class="author">Ng BL</span>, <span class="author">Sun T</span>, <span class="author">Kamberov ES</span>, <span class="author">Makarov VL</span>, <span class="author">Langmore JP</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">The sequencing of the human genome has led to the availability of an extensive mapped clone resource that is ideal for the construction of DNA microarrays. These genomic clone microarrays have largely been used for comparative genomic hybridisation studies of tumours to enable accurate measurement of copy number changes (array-CGH) at increased resolution. We have utilised these microarrays as the target for chromosome painting and reverse chromosome painting to provide a similar improvement in analysis resolution for these studies in a process we have termed array painting. In array painting, chromosomes are flow sorted, fluorescently labelled and hybridised to the microarray. The complete composition and the breakpoints of aberrant chromosomes can be analysed at high resolution in this way with a considerable reduction in time, effort and cytogenetic expertise required for conventional analysis using fluorescence in situ hybridisation. In a similar way, the resolution of cross-species chromosome painting can be improved and we present preliminary observations of the organisation of homologous DNA blocks between the white cheeked gibbon chromosome 14 and human chromosomes 2 and 17.</p>
<p class="publication"><span class="title">Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">12</span>;<span class="number">1</span>;<span class="pagerange">35-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14984100">14984100</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The microRNA Registry.</h4>
<p class="authors"><span class="author">Griffiths-Jones S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 9SA, UK. <a href="mailto:%73%67%6A%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#103;&#106;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The miRNA Registry provides a service for the assignment of miRNA gene names prior to publication. A comprehensive and searchable database of published miRNA sequences is accessible via a web interface (http://www.sanger.ac.uk/Software/Rfam/mirna/), and all sequence and annotation data are freely available for download. Release 2.0 of the database contains 506 miRNA entries from six organisms.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D109-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681370" rel="external">14681370</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308757" rel="external">308757</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh023" rel="external">10.1093/nar/gkh023</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mismatch repair genes identified using genetic screens in Blm-deficient embryonic stem cells.</h4>
<p class="authors"><span class="author">Guo G</span>, <span class="author">Wang W</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Phenotype-driven recessive genetic screens in diploid organisms require a strategy to render the mutation homozygous. Although homozygous mutant mice can be generated by breeding, a reliable method to make homozygous mutations in cultured cells has not been available, limiting recessive screens in culture. Cultured embryonic stem (ES) cells provide access to all of the genes required to elaborate the fundamental components and physiological systems of a mammalian cell. Here we have exploited the high rate of mitotic recombination in Bloom&#39;s syndrome protein (Blm)-deficient ES cells to generate a genome-wide library of homozygous mutant cells from heterozygous mutations induced with a revertible gene trap retrovirus. We have screened this library for cells with defects in DNA mismatch repair (MMR), a system that detects and repairs base-base mismatches. We demonstrate the recovery of cells with homozygous mutations in known and novel MMR genes. We identified Dnmt1(ref. 5) as a novel MMR gene and confirmed that Dnmt1-deficient ES cells exhibit micro-satellite instability, providing a mechanistic explanation for the role of Dnmt1 in cancer. The combination of insertional mutagenesis in Blm-deficient ES cells establishes a new approach for phenotype-based recessive genetic screens in ES cells.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">429</span>;<span class="number">6994</span>;<span class="pagerange">891-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15215866">15215866</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02653">10.1038/nature02653</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MANSC: a seven-cysteine-containing domain present in animal membrane and extracellular proteins.</h4>
<p class="authors"><span class="author">Guo J</span>, <span class="author">Chen S</span>, <span class="author">Huang C</span>, <span class="author">Chen L</span>, <span class="author">Studholme DJ</span>, <span class="author">Zhao S</span> and <span class="author">Yu L</span></p>
<p class="affiliation">State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Handan Road 220, Shanghai 200-433, China.</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2004</span>;<span class="volume">29</span>;<span class="number">4</span>;<span class="pagerange">172-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15124631" rel="external">15124631</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tibs.2004.02.007" rel="external">10.1016/j.tibs.2004.02.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A probabilistic model of 3&#39; end formation in Caenorhabditis elegans.</h4>
<p class="authors"><span class="author">Hajarnavis A</span>, <span class="author">Korf I</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.</p><p class="abstract">The 3&#39; ends of mRNAs terminate with a poly(A) tail. This post-transcriptional modification is directed by sequence features present in the 3&#39;-untranslated region (3&#39;-UTR). We have undertaken a computational analysis of 3&#39; end formation in Caenorhabditis elegans. By aligning cDNAs that diverge from genomic sequence at the poly(A) tract, we accurately identified a large set of true cleavage sites. When there are many transcripts aligned to a particular locus, local variation of the cleavage site over a span of a few bases is frequently observed. We find that in addition to the well-known AAUAAA motif there are several regions with distinct nucleotide compositional biases. We propose a generalized hidden Markov model that describes sequence features in C.elegans 3&#39;-UTRs. We find that a computer program employing this model accurately predicts experimentally observed 3&#39; ends even when there are multiple AAUAAA motifs and multiple cleavage sites. We have made available a complete set of polyadenylation site predictions for the C.elegans genome, including a subset of 6570 supported by aligned transcripts.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">11</span>;<span class="pagerange">3392-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15247332" rel="external">15247332</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC443532" rel="external">443532</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh656" rel="external">10.1093/nar/gkh656</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Accelerated screening of phage-display output with alkaline phosphatase fusions.</h4>
<p class="authors"><span class="author">Han Z</span>, <span class="author">Karatan E</span>, <span class="author">Scholle MD</span>, <span class="author">McCafferty J</span> and <span class="author">Kay BK</span></p>
<p class="affiliation">Combinatorial Biology Unit, Biosciences Division, Argonne National Laboratory, 9700 South Cass Avenue, Argonne, IL 60439, USA.</p><p class="abstract">When using multiple targets and libraries, selection of affinity reagents from phage-displayed libraries is a relatively time-consuming process. Herein, we describe an automation-amenable approach to accelerate the process by using alkaline phosphatase (AP) fusion proteins in place of the phage ELISA screening and subsequent confirmation steps with purified protein. After two or three rounds of affinity selection, the open reading frames that encode the affinity selected molecules (i.e., antibody fragments, engineered scaffold proteins, combinatorial peptides) are amplified from the phage or phagemid DNA molecules by PCR and cloned en masse by a Ligation Independent Cloning (LIC) method into a plasmid encoding a highly active variant of E. coli AP. This time-saving process identifies affinity reagents that work out of context of the phage and that can be used in various downstream enzyme linked binding assays. The utility of this approach was demonstrated by analyzing single-chain antibodies (scFvs), engineered fibronectin type III domains (FN3), and combinatorial peptides that were selected for binding to the Epsin N-terminal Homology (ENTH) domain of epsin 1, the c-Src SH3 domain, and the appendage domain of the gamma subunit of the clathrin adaptor complex, AP-1, respectively.</p>
<p class="publication"><span class="title">Combinatorial chemistry &amp; high throughput screening</span>&nbsp;<span class="year">2004</span>;<span class="volume">7</span>;<span class="number">1</span>;<span class="pagerange">55-62</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14965261" rel="external">14965261</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic equity.</h4>
<p class="authors"><span class="author">Harris J</span> and <span class="author">Sulston J</span></p>
<p class="affiliation">Institute of Medicine, Law and Bioethics, School of Law, University of Manchester, Oxford Road, Manchester M13 9PL, UK.</p><p class="abstract">This paper proposes, elaborates and defends a principle of genetic equity. In doing so it articulates, explains and justifies what might be meant by the concept of &#39;human dignity&#39; in a way that is clear, defensible and consistent with, but by no means the same as, the plethora of appeals to human dignity found in contemporary bioethics, and more particularly in international instruments on bioethics. We propose the following principle of genetic equity: humans are born equal; they are entitled to freedom from discrimination and equality of opportunity to flourish; genetic information may not be used to limit that equality.</p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">796-800</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15510171">15510171</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1454">10.1038/nrg1454</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Gene Ontology (GO) database and informatics resource.</h4>
<p class="authors"><span class="author">Harris MA</span>, <span class="author">Clark J</span>, <span class="author">Ireland A</span>, <span class="author">Lomax J</span>, <span class="author">Ashburner M</span>, <span class="author">Foulger R</span>, <span class="author">Eilbeck K</span>, <span class="author">Lewis S</span>, <span class="author">Marshall B</span>, <span class="author">Mungall C</span>, <span class="author">Richter J</span>, <span class="author">Rubin GM</span>, <span class="author">Blake JA</span>, <span class="author">Bult C</span>, <span class="author">Dolan M</span>, <span class="author">Drabkin H</span>, <span class="author">Eppig JT</span>, <span class="author">Hill DP</span>, <span class="author">Ni L</span>, <span class="author">Ringwald M</span>, <span class="author">Balakrishnan R</span>, <span class="author">Cherry JM</span>, <span class="author">Christie KR</span>, <span class="author">Costanzo MC</span>, <span class="author">Dwight SS</span>, <span class="author">Engel S</span>, <span class="author">Fisk DG</span>, <span class="author">Hirschman JE</span>, <span class="author">Hong EL</span>, <span class="author">Nash RS</span>, <span class="author">Sethuraman A</span>, <span class="author">Theesfeld CL</span>, <span class="author">Botstein D</span>, <span class="author">Dolinski K</span>, <span class="author">Feierbach B</span>, <span class="author">Berardini T</span>, <span class="author">Mundodi S</span>, <span class="author">Rhee SY</span>, <span class="author">Apweiler R</span>, <span class="author">Barrell D</span>, <span class="author">Camon E</span>, <span class="author">Dimmer E</span>, <span class="author">Lee V</span>, <span class="author">Chisholm R</span>, <span class="author">Gaudet P</span>, <span class="author">Kibbe W</span>, <span class="author">Kishore R</span>, <span class="author">Schwarz EM</span>, <span class="author">Sternberg P</span>, <span class="author">Gwinn M</span>, <span class="author">Hannick L</span>, <span class="author">Wortman J</span>, <span class="author">Berriman M</span>, <span class="author">Wood V</span>, <span class="author">de la Cruz N</span>, <span class="author">Tonellato P</span>, <span class="author">Jaiswal P</span>, <span class="author">Seigfried T</span>, <span class="author">White R</span> and <span class="author">Gene Ontology Consortium</span></p>
<p class="affiliation">GO-EBI, Hinxton, UK.</p><p class="abstract">The Gene Ontology (GO) project (http://www. geneontology.org/) provides structured, controlled vocabularies and classifications that cover several domains of molecular and cellular biology and are freely available for community use in the annotation of genes, gene products and sequences. Many model organism databases and genome annotation groups use the GO and contribute their annotation sets to the GO resource. The GO database integrates the vocabularies and contributed annotations and provides full access to this information in several formats. Members of the GO Consortium continually work collectively, involving outside experts as needed, to expand and update the GO vocabularies. The GO Web resource also provides access to extensive documentation about the GO project and links to applications that use GO data for functional analyses.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG02273</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D258-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681407" rel="external">14681407</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308770" rel="external">308770</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh036" rel="external">10.1093/nar/gkh036</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">WormBase: a multi-species resource for nematode biology and genomics.</h4>
<p class="authors"><span class="author">Harris TW</span>, <span class="author">Chen N</span>, <span class="author">Cunningham F</span>, <span class="author">Tello-Ruiz M</span>, <span class="author">Antoshechkin I</span>, <span class="author">Bastiani C</span>, <span class="author">Bieri T</span>, <span class="author">Blasiar D</span>, <span class="author">Bradnam K</span>, <span class="author">Chan J</span>, <span class="author">Chen CK</span>, <span class="author">Chen WJ</span>, <span class="author">Davis P</span>, <span class="author">Kenny E</span>, <span class="author">Kishore R</span>, <span class="author">Lawson D</span>, <span class="author">Lee R</span>, <span class="author">Muller HM</span>, <span class="author">Nakamura C</span>, <span class="author">Ozersky P</span>, <span class="author">Petcherski A</span>, <span class="author">Rogers A</span>, <span class="author">Sabo A</span>, <span class="author">Schwarz EM</span>, <span class="author">Van Auken K</span>, <span class="author">Wang Q</span>, <span class="author">Durbin R</span>, <span class="author">Spieth J</span>, <span class="author">Sternberg PW</span> and <span class="author">Stein LD</span></p>
<p class="affiliation">Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724, USA. <a href="mailto:%68%61%72%72%69%73%40%63%73%68%6C.%6F%72%67">&#104;&#97;&#114;&#114;&#105;&#115;&#64;&#99;&#115;&#104;&#108;&#46;&#111;&#114;&#103;</a></p><p class="abstract">WormBase (http://www.wormbase.org/) is the central data repository for information about Caenorhabditis elegans and related nematodes. As a model organism database, WormBase extends beyond the genomic sequence, integrating experimental results with extensively annotated views of the genome. The WormBase Consortium continues to expand the biological scope and utility of WormBase with the inclusion of large-scale genomic analyses, through active data and literature curation, through new analysis and visualization tools, and through refinement of the user interface. Over the past year, the nearly complete genomic sequence and comparative analyses of the closely related species Caenorhabditis briggsae have been integrated into WormBase, including gene predictions, ortholog assignments and a new synteny viewer to display the relationships between the two species. Extensive site-wide refinement of the user interface now provides quick access to the most frequently accessed resources and a consistent browsing experience across the site. Unified single-page views now provide complete summaries of commonly accessed entries like genes. These advances continue to increase the utility of WormBase for C.elegans researchers, as well as for those researchers exploring problems in functional and comparative genomics in the context of a powerful genetic system.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41-HG02223</span></p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D411-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681445" rel="external">14681445</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308800" rel="external">308800</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh066" rel="external">10.1093/nar/gkh066</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Continuing tsetse and Trypanosoma genome sequencing projects.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span> and <span class="author">Berriman M</span></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">7</span>;<span class="pagerange">308-9; author reply 309-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15193559" rel="external">15193559</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2004.04.013" rel="external">10.1016/j.pt.2004.04.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Parasite genome databases and web-based resources.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span> and <span class="author">Hall N</span></p>
<p class="affiliation">Pathogen Sequencing Unit, Wellcome Trust Sanger Institute, Hinxton, Cambridgeshire, UK.</p><p class="abstract">In the last decade, high-throughput genome sequencing and complementary techniques such as microarray and proteomics have generated, and will continue to generate, ever-increasing amounts of data. These technologies of gene discovery, expression, and functional analysis have been applied to a vast array of organisms, including parasites. In most instances, the data are freely available via the Internet, and researchers are becoming increasingly reliant on up-to-date, centralized data repositories to complement wet bench science. This chapter presents an overview of resources relevant to researchers with an interest in para-site genomics and biology. After briefly touching on some of the publicly available nucleotide and protein sequence as well as domain databases, the focus turns to parasite genome projects and associated Web-based resources. A list of parasite sequencing projects current at the time of writing, including relevant Web site addresses, is provided. The available resources range from network sites and project pages at sequencing institutes to databases that integrate and curate sequence data and associated annotation with diverse biological datasets. Particular attention is given to three databases, GeneDB (http://www.genedb.org/), PlasmoDB (http://plasmodb. org/), and tigr db, detailing the scope of each database and the tools available for data querying and retrieval.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">270</span>;<span class="pagerange">45-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153622" rel="external">15153622</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-793-9%3A045" rel="external">10.1385/1-59259-793-9:045</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GeneDB: a resource for prokaryotic and eukaryotic organisms.</h4>
<p class="authors"><span class="author">Hertz-Fowler C</span>, <span class="author">Peacock CS</span>, <span class="author">Wood V</span>, <span class="author">Aslett M</span>, <span class="author">Kerhornou A</span>, <span class="author">Mooney P</span>, <span class="author">Tivey A</span>, <span class="author">Berriman M</span>, <span class="author">Hall N</span>, <span class="author">Rutherford K</span>, <span class="author">Parkhill J</span>, <span class="author">Ivens AC</span>, <span class="author">Rajandream MA</span> and <span class="author">Barrell B</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%63%68%66%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#104;&#102;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">GeneDB (http://www.genedb.org/) is a genome database for prokaryotic and eukaryotic organisms. The resource provides a portal through which data generated by the Pathogen Sequencing Unit at the Wellcome Trust Sanger Institute and other collaborating sequencing centres can be made publicly available. It combines data from finished and ongoing genome and expressed sequence tag (EST) projects with curated annotation, that can be searched, sorted and downloaded, using a single web based resource. The current release stores 11 datasets of which six are curated and maintained by biologists, who review and incorporate information from the scientific literature, public databases and the respective research communities.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D339-43</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681429" rel="external">14681429</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308742" rel="external">308742</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh007" rel="external">10.1093/nar/gkh007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pathogenomics of non-pathogens.</h4>
<p class="authors"><span class="author">Holden M</span>, <span class="author">Crossman L</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Analysing the genomes of non-pathogenic microorganisms, in addition to its basic and applied scientific interest, can also shed considerable light on the study of pathogenic microorganisms. Two of the three microorganisms described here are rarely pathogenic, but carry genetic determinants that have previously been identified as being important for the pathogenicity of other microorganisms. This underlines the growing understanding that many so-called &#39;virulence genes&#39; are probably involved in more general interactions between the microorganism and the host or the environment.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">2</span>;<span class="pagerange">91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15040257">15040257</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro825">10.1038/nrmicro825</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete genomes of two clinical Staphylococcus aureus strains: evidence for the rapid evolution of virulence and drug resistance.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Feil EJ</span>, <span class="author">Lindsay JA</span>, <span class="author">Peacock SJ</span>, <span class="author">Day NP</span>, <span class="author">Enright MC</span>, <span class="author">Foster TJ</span>, <span class="author">Moore CE</span>, <span class="author">Hurst L</span>, <span class="author">Atkin R</span>, <span class="author">Barron A</span>, <span class="author">Bason N</span>, <span class="author">Bentley SD</span>, <span class="author">Chillingworth C</span>, <span class="author">Chillingworth T</span>, <span class="author">Churcher C</span>, <span class="author">Clark L</span>, <span class="author">Corton C</span>, <span class="author">Cronin A</span>, <span class="author">Doggett J</span>, <span class="author">Dowd L</span>, <span class="author">Feltwell T</span>, <span class="author">Hance Z</span>, <span class="author">Harris B</span>, <span class="author">Hauser H</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">James KD</span>, <span class="author">Lennard N</span>, <span class="author">Line A</span>, <span class="author">Mayes R</span>, <span class="author">Moule S</span>, <span class="author">Mungall K</span>, <span class="author">Ormond D</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rutherford K</span>, <span class="author">Sanders M</span>, <span class="author">Sharp S</span>, <span class="author">Simmonds M</span>, <span class="author">Stevens K</span>, <span class="author">Whitehead S</span>, <span class="author">Barrell BG</span>, <span class="author">Spratt BG</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Staphylococcus aureus is an important nosocomial and community-acquired pathogen. Its genetic plasticity has facilitated the evolution of many virulent and drug-resistant strains, presenting a major and constantly changing clinical challenge. We sequenced the approximately 2.8-Mbp genomes of two disease-causing S. aureus strains isolated from distinct clinical settings: a recent hospital-acquired representative of the epidemic methicillin-resistant S. aureus EMRSA-16 clone (MRSA252), a clinically important and globally prevalent lineage; and a representative of an invasive community-acquired methicillin-susceptible S. aureus clone (MSSA476). A comparative-genomics approach was used to explore the mechanisms of evolution of clinically important S. aureus genomes and to identify regions affecting virulence and drug resistance. The genome sequences of MRSA252 and MSSA476 have a well conserved core region but differ markedly in their accessory genetic elements. MRSA252 is the most genetically diverse S. aureus strain sequenced to date: approximately 6% of the genome is novel compared with other published genomes, and it contains several unique genetic elements. MSSA476 is methicillin-susceptible, but it contains a novel Staphylococcal chromosomal cassette (SCC) mec-like element (designated SCC(476)), which is integrated at the same site on the chromosome as SCCmec elements in MRSA strains but encodes a putative fusidic acid resistance protein. The crucial role that accessory elements play in the rapid evolution of S. aureus is clearly illustrated by comparing the MSSA476 genome with that of an extremely closely related MRSA community-acquired strain; the differential distribution of large mobile elements carrying virulence and drug-resistance determinants may be responsible for the clinically important phenotypic differences in these strains.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">26</span>;<span class="pagerange">9786-91</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15213324">15213324</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC470752">470752</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0402521101">10.1073/pnas.0402521101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic plasticity of the causative agent of melioidosis, Burkholderia pseudomallei.</h4>
<p class="authors"><span class="author">Holden MT</span>, <span class="author">Titball RW</span>, <span class="author">Peacock SJ</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Atkins T</span>, <span class="author">Crossman LC</span>, <span class="author">Pitt T</span>, <span class="author">Churcher C</span>, <span class="author">Mungall K</span>, <span class="author">Bentley SD</span>, <span class="author">Sebaihia M</span>, <span class="author">Thomson NR</span>, <span class="author">Bason N</span>, <span class="author">Beacham IR</span>, <span class="author">Brooks K</span>, <span class="author">Brown KA</span>, <span class="author">Brown NF</span>, <span class="author">Challis GL</span>, <span class="author">Cherevach I</span>, <span class="author">Chillingworth T</span>, <span class="author">Cronin A</span>, <span class="author">Crossett B</span>, <span class="author">Davis P</span>, <span class="author">DeShazer D</span>, <span class="author">Feltwell T</span>, <span class="author">Fraser A</span>, <span class="author">Hance Z</span>, <span class="author">Hauser H</span>, <span class="author">Holroyd S</span>, <span class="author">Jagels K</span>, <span class="author">Keith KE</span>, <span class="author">Maddison M</span>, <span class="author">Moule S</span>, <span class="author">Price C</span>, <span class="author">Quail MA</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rutherford K</span>, <span class="author">Sanders M</span>, <span class="author">Simmonds M</span>, <span class="author">Songsivilai S</span>, <span class="author">Stevens K</span>, <span class="author">Tumapa S</span>, <span class="author">Vesaratchavest M</span>, <span class="author">Whitehead S</span>, <span class="author">Yeats C</span>, <span class="author">Barrell BG</span>, <span class="author">Oyston PC</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">Burkholderia pseudomallei is a recognized biothreat agent and the causative agent of melioidosis. This Gram-negative bacterium exists as a soil saprophyte in melioidosis-endemic areas of the world and accounts for 20% of community-acquired septicaemias in northeastern Thailand where half of those affected die. Here we report the complete genome of B. pseudomallei, which is composed of two chromosomes of 4.07 megabase pairs and 3.17 megabase pairs, showing significant functional partitioning of genes between them. The large chromosome encodes many of the core functions associated with central metabolism and cell growth, whereas the small chromosome carries more accessory functions associated with adaptation and survival in different niches. Genomic comparisons with closely and more distantly related bacteria revealed a greater level of gene order conservation and a greater number of orthologous genes on the large chromosome, suggesting that the two replicons have distinct evolutionary origins. A striking feature of the genome was the presence of 16 genomic islands (GIs) that together made up 6.1% of the genome. Further analysis revealed these islands to be variably present in a collection of invasive and soil isolates but entirely absent from the clonally related organism B. mallei. We propose that variable horizontal gene acquisition by B. pseudomallei is an important feature of recent genetic evolution and that this has resulted in a genetically diverse pathogenic species.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">39</span>;<span class="pagerange">14240-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15377794">15377794</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC521101">521101</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0403302101">10.1073/pnas.0403302101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene map of the extended human MHC.</h4>
<p class="authors"><span class="author">Horton R</span>, <span class="author">Wilming L</span>, <span class="author">Rand V</span>, <span class="author">Lovering RC</span>, <span class="author">Bruford EA</span>, <span class="author">Khodiyar VK</span>, <span class="author">Lush MJ</span>, <span class="author">Povey S</span>, <span class="author">Talbot CC</span>, <span class="author">Wright MW</span>, <span class="author">Wain HM</span>, <span class="author">Trowsdale J</span>, <span class="author">Ziegler A</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">The major histocompatibility complex (MHC) is the most important region in the vertebrate genome with respect to infection and autoimmunity, and is crucial in adaptive and innate immunity. Decades of biomedical research have revealed many MHC genes that are duplicated, polymorphic and associated with more diseases than any other region of the human genome. The recent completion of several large-scale studies offers the opportunity to assimilate the latest data into an integrated gene map of the extended human MHC. Here, we present this map and review its content in relation to paralogy, polymorphism, immune function and disease.</p>
<p class="grants">Funded by: Multiple Sclerosis Society: <span class="grant">588</span></p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">12</span>;<span class="pagerange">889-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15573121">15573121</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1489">10.1038/nrg1489</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Whole genome DNA copy number changes identified by high density oligonucleotide arrays.</h4>
<p class="authors"><span class="author">Huang J</span>, <span class="author">Wei W</span>, <span class="author">Zhang J</span>, <span class="author">Liu G</span>, <span class="author">Bignell GR</span>, <span class="author">Stratton MR</span>, <span class="author">Futreal PA</span>, <span class="author">Wooster R</span>, <span class="author">Jones KW</span> and <span class="author">Shapero MH</span></p>
<p class="affiliation">Affymetrix, Inc., 3380 Central Expressway, Santa Clara, CA 95051, USA. <a href="mailto:%6A%69%6E%67%5F%68%75%61%6E%67%40%61%66%66%79%6D%65%74%72%69%78.%63%6F%6D">&#106;&#105;&#110;&#103;&#95;&#104;&#117;&#97;&#110;&#103;&#64;&#97;&#102;&#102;&#121;&#109;&#101;&#116;&#114;&#105;&#120;&#46;&#99;&#111;&#109;</a></p><p class="abstract">Changes in DNA copy number are one of the hallmarks of the genetic instability common to most human cancers. Previous microarray-based methods have been used to identify chromosomal gains and losses; however, they are unable to genotype alleles at the level of single nucleotide polymorphisms (SNPs). Here we describe a novel algorithm that uses a recently developed high-density oligonucleotide array-based SNP genotyping method, whole genome sampling analysis (WGSA), to identify genome-wide chromosomal gains and losses at high resolution. WGSA simultaneously genotypes over 10,000 SNPs by allele-specific hybridisation to perfect match (PM) and mismatch (MM) probes synthesised on a single array. The copy number algorithm jointly uses PM intensity and discrimination ratios between paired PM and MM intensity values to identify and estimate genetic copy number changes. Values from an experimental sample are compared with SNP-specific distributions derived from a reference set containing over 100 normal individuals to gain statistical power. Genomic regions with statistically significant copy number changes can be identified using both single point analysis and contiguous point analysis of SNP intensities. We identified multiple regions of amplification and deletion using a panel of human breast cancer cell lines. We verified these results using an independent method based on quantitative polymerase chain reaction and found that our approach is both sensitive and specific and can tolerate samples which contain a mixture of both tumour and normal DNA. In addition, by using known allele frequencies from the reference set, statistically significant genomic intervals can be identified containing contiguous stretches of homozygous markers, potentially allowing the detection of regions undergoing loss of heterozygosity (LOH) without the need for a matched normal control sample. The coupling of LOH analysis, via SNP genotyping, with copy number estimations using a single array provides additional insight into the structure of genomic alterations. With mean and median inter-SNP euchromatin distances of 244 kilobases (kb) and 119 kb, respectively, this method affords a resolution that is not easily achievable with non-oligonucleotide-based experimental approaches.</p>
<p class="publication"><span class="title">Human genomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">1</span>;<span class="number">4</span>;<span class="pagerange">287-99</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15588488" rel="external">15588488</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A new trade framework for global healthcare R&amp;D.</h4>
<p class="authors"><span class="author">Hubbard T</span> and <span class="author">Love J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute in Hinxton, United Kingdom.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">2</span>;<span class="pagerange">E52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14966544">14966544</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC340954">340954</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0020052">10.1371/journal.pbio.0020052</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA sequence and analysis of human chromosome 9.</h4>
<p class="authors"><span class="author">Humphray SJ</span>, <span class="author">Oliver K</span>, <span class="author">Hunt AR</span>, <span class="author">Plumb RW</span>, <span class="author">Loveland JE</span>, <span class="author">Howe KL</span>, <span class="author">Andrews TD</span>, <span class="author">Searle S</span>, <span class="author">Hunt SE</span>, <span class="author">Scott CE</span>, <span class="author">Jones MC</span>, <span class="author">Ainscough R</span>, <span class="author">Almeida JP</span>, <span class="author">Ambrose KD</span>, <span class="author">Ashwell RI</span>, <span class="author">Babbage AK</span>, <span class="author">Babbage S</span>, <span class="author">Bagguley CL</span>, <span class="author">Bailey J</span>, <span class="author">Banerjee R</span>, <span class="author">Barker DJ</span>, <span class="author">Barlow KF</span>, <span class="author">Bates K</span>, <span class="author">Beasley H</span>, <span class="author">Beasley O</span>, <span class="author">Bird CP</span>, <span class="author">Bray-Allen S</span>, <span class="author">Brown AJ</span>, <span class="author">Brown JY</span>, <span class="author">Burford D</span>, <span class="author">Burrill W</span>, <span class="author">Burton J</span>, <span class="author">Carder C</span>, <span class="author">Carter NP</span>, <span class="author">Chapman JC</span>, <span class="author">Chen Y</span>, <span class="author">Clarke G</span>, <span class="author">Clark SY</span>, <span class="author">Clee CM</span>, <span class="author">Clegg S</span>, <span class="author">Collier RE</span>, <span class="author">Corby N</span>, <span class="author">Crosier M</span>, <span class="author">Cummings AT</span>, <span class="author">Davies J</span>, <span class="author">Dhami P</span>, <span class="author">Dunn M</span>, <span class="author">Dutta I</span>, <span class="author">Dyer LW</span>, <span class="author">Earthrowl ME</span>, <span class="author">Faulkner L</span>, <span class="author">Fleming CJ</span>, <span class="author">Frankish A</span>, <span class="author">Frankland JA</span>, <span class="author">French L</span>, <span class="author">Fricker DG</span>, <span class="author">Garner P</span>, <span class="author">Garnett J</span>, <span class="author">Ghori J</span>, <span class="author">Gilbert JG</span>, <span class="author">Glison C</span>, <span class="author">Grafham DV</span>, <span class="author">Gribble S</span>, <span class="author">Griffiths C</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Grocock R</span>, <span class="author">Guy J</span>, <span class="author">Hall RE</span>, <span class="author">Hammond S</span>, <span class="author">Harley JL</span>, <span class="author">Harrison ES</span>, <span class="author">Hart EA</span>, <span class="author">Heath PD</span>, <span class="author">Henderson CD</span>, <span class="author">Hopkins BL</span>, <span class="author">Howard PJ</span>, <span class="author">Howden PJ</span>, <span class="author">Huckle E</span>, <span class="author">Johnson C</span>, <span class="author">Johnson D</span>, <span class="author">Joy AA</span>, <span class="author">Kay M</span>, <span class="author">Keenan S</span>, <span class="author">Kershaw JK</span>, <span class="author">Kimberley AM</span>, <span class="author">King A</span>, <span class="author">Knights A</span>, <span class="author">Laird GK</span>, <span class="author">Langford C</span>, <span class="author">Lawlor S</span>, <span class="author">Leongamornlert DA</span>, <span class="author">Leversha M</span>, <span class="author">Lloyd C</span>, <span class="author">Lloyd DM</span>, <span class="author">Lovell J</span>, <span class="author">Martin S</span>, <span class="author">Mashreghi-Mohammadi M</span>, <span class="author">Matthews L</span>, <span class="author">McLaren S</span>, <span class="author">McLay KE</span>, <span class="author">McMurray A</span>, <span class="author">Milne S</span>, <span class="author">Nickerson T</span>, <span class="author">Nisbett J</span>, <span class="author">Nordsiek G</span>, <span class="author">Pearce AV</span>, <span class="author">Peck AI</span>, <span class="author">Porter KM</span>, <span class="author">Pandian R</span>, <span class="author">Pelan S</span>, <span class="author">Phillimore B</span>, <span class="author">Povey S</span>, <span class="author">Ramsey Y</span>, <span class="author">Rand V</span>, <span class="author">Scharfe M</span>, <span class="author">Sehra HK</span>, <span class="author">Shownkeen R</span>, <span class="author">Sims SK</span>, <span class="author">Skuce CD</span>, <span class="author">Smith M</span>, <span class="author">Steward CA</span>, <span class="author">Swarbreck D</span>, <span class="author">Sycamore N</span>, <span class="author">Tester J</span>, <span class="author">Thorpe A</span>, <span class="author">Tracey A</span>, <span class="author">Tromans A</span>, <span class="author">Thomas DW</span>, <span class="author">Wall M</span>, <span class="author">Wallis JM</span>, <span class="author">West AP</span>, <span class="author">Whitehead SL</span>, <span class="author">Willey DL</span>, <span class="author">Williams SA</span>, <span class="author">Wilming L</span>, <span class="author">Wray PW</span>, <span class="author">Young L</span>, <span class="author">Ashurst JL</span>, <span class="author">Coulson A</span>, <span class="author">Bl&ouml;cker H</span>, <span class="author">Durbin R</span>, <span class="author">Sulston JE</span>, <span class="author">Hubbard T</span>, <span class="author">Jackson MJ</span>, <span class="author">Bentley DR</span>, <span class="author">Beck S</span>, <span class="author">Rogers J</span> and <span class="author">Dunham I</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%73%6A%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#115;&#106;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Chromosome 9 is highly structurally polymorphic. It contains the largest autosomal block of heterochromatin, which is heteromorphic in 6-8% of humans, whereas pericentric inversions occur in more than 1% of the population. The finished euchromatic sequence of chromosome 9 comprises 109,044,351 base pairs and represents &gt;99.6% of the region. Analysis of the sequence reveals many intra- and interchromosomal duplications, including segmental duplications adjacent to both the centromere and the large heterochromatic block. We have annotated 1,149 genes, including genes implicated in male-to-female sex reversal, cancer and neurodegenerative disease, and 426 pseudogenes. The chromosome contains the largest interferon gene cluster in the human genome. There is also a region of exceptionally high gene and G + C content including genes paralogous to those in the major histocompatibility complex. We have also detected recently duplicated genes that exhibit different rates of sequence divergence, presumably reflecting natural selection.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">429</span>;<span class="number">6990</span>;<span class="pagerange">369-74</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15164053">15164053</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2734081">2734081</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature02465">10.1038/nature02465</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene duplication: the genomic trade in spare parts.</h4>
<p class="authors"><span class="author">Hurles M</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute near Cambridge in the United Kingdom. <a href="mailto:%6D%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">7</span>;<span class="pagerange">E206</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15252449" rel="external">15252449</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC449868" rel="external">449868</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0020206" rel="external">10.1371/journal.pbio.0020206</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Origins of chromosomal rearrangement hotspots in the human genome: evidence from the AZFa deletion hotspots.</h4>
<p class="authors"><span class="author">Hurles ME</span>, <span class="author">Willey D</span>, <span class="author">Matthews L</span> and <span class="author">Hussain SS</span></p>
<p class="affiliation">Molecular Genetics Laboratory, McDonald Institute for Archaeological Research, University of Cambridge, Downing Street, Cambridge, CB2 3ER, UK. <a href="mailto:%6D%65%68%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#101;&#104;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The origins of the recombination hotspots that are a common feature of both allelic and non-allelic homologous recombination in the human genome are poorly understood. We have investigated, by comparative sequencing, the evolution of two hotspots of non-allelic homologous recombination on the Y chromosome that lie within paralogous sequences known to sponsor deletions resulting in male infertility.</p><p class="abstract"><strong>Results</strong>: These recombination hotspots are characterized by signatures of concerted evolution, which indicate that gene conversion between paralogs has been predominant in shaping their recent evolution. By contrast, the paralogous sequences that surround the hotspots exhibit little evidence of gene conversion. A second feature of these rearrangement hotspots is the extreme interspecific sequence divergence (around 2.5%) that places them among the most divergent orthologous sequences between humans and chimpanzees.</p><p class="abstract"><strong>Conclusions</strong>: Several hominid-specific gene conversion events have rendered these hotspots better substrates for chromosomal rearrangements in humans than in chimpanzees or gorillas. Monte Carlo simulations of sequence evolution suggest that extreme sequence divergence is a direct consequence of gene conversion between paralogs. We propose that the coincidence of signatures of concerted evolution and recurrent breakpoints of chromosomal rearrangement (mapped at the sequence level) may enable the identification of putative rearrangement hotspots from analysis of comparative sequences from great apes.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">8</span>;<span class="pagerange">R55</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15287977">15287977</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC507880">507880</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2004-5-8-r55">10.1186/gb-2004-5-8-r55</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">High-resolution analysis of genomic copy number alterations in bladder cancer by microarray-based comparative genomic hybridization.</h4>
<p class="authors"><span class="author">Hurst CD</span>, <span class="author">Fiegler H</span>, <span class="author">Carr P</span>, <span class="author">Williams S</span>, <span class="author">Carter NP</span> and <span class="author">Knowles MA</span></p>
<p class="affiliation">Cancer Research UK Clinical Centre, St James&#39;s University Hospital, Beckett St, Leeds LS9 7TF, UK.</p><p class="abstract">We have screened 22 bladder tumour-derived cell lines and one normal urothelium-derived cell line for genome-wide copy number changes using array comparative genomic hybridization (CGH). Comparison of array CGH with existing multiplex-fluorescence in situ hybridization (M-FISH) results revealed excellent concordance. Regions of gain and loss were defined more accurately by array CGH, and several small regions of deletion were detected that were not identified by M-FISH. Numerous genetic changes were identified, many of which were compatible with previous results from conventional CGH and loss of heterozygosity analyses on bladder tumours. The most frequent changes involved complete or partial loss of 4q (83%) and gain of 20q (78%). Other frequent losses were of 18q (65%), 8p (65%), 2q (61%), 6q (61%), 3p (56%), 13q (56%), 4p (52%), 6p (52%), 10p (52%), 10q (52%) and 5p (43%). We have refined the localization of a region of deletion at 8p21.2-p21.3 to an interval of approximately 1 Mb. Five homozygous deletions of tumour suppressor genes were confirmed, and several potentially novel homozygous deletions were identified. In all, 15 high-level amplifications were detected, with a previously reported amplification at 6p22.3 being the most frequent. Real-time PCR analysis revealed a novel candidate gene with consistent overexpression in all cell lines with the 6p22.3 amplicon.</p>
<p class="publication"><span class="title">Oncogene</span>&nbsp;<span class="year">2004</span>;<span class="volume">23</span>;<span class="number">12</span>;<span class="pagerange">2250-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14968109" rel="external">14968109</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.onc.1207260" rel="external">10.1038/sj.onc.1207260</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrative annotation of 21,037 human genes validated by full-length cDNA clones.</h4>
<p class="authors"><span class="author">Imanishi T</span>, <span class="author">Itoh T</span>, <span class="author">Suzuki Y</span>, <span class="author">O&#39;Donovan C</span>, <span class="author">Fukuchi S</span>, <span class="author">Koyanagi KO</span>, <span class="author">Barrero RA</span>, <span class="author">Tamura T</span>, <span class="author">Yamaguchi-Kabata Y</span>, <span class="author">Tanino M</span>, <span class="author">Yura K</span>, <span class="author">Miyazaki S</span>, <span class="author">Ikeo K</span>, <span class="author">Homma K</span>, <span class="author">Kasprzyk A</span>, <span class="author">Nishikawa T</span>, <span class="author">Hirakawa M</span>, <span class="author">Thierry-Mieg J</span>, <span class="author">Thierry-Mieg D</span>, <span class="author">Ashurst J</span>, <span class="author">Jia L</span>, <span class="author">Nakao M</span>, <span class="author">Thomas MA</span>, <span class="author">Mulder N</span>, <span class="author">Karavidopoulou Y</span>, <span class="author">Jin L</span>, <span class="author">Kim S</span>, <span class="author">Yasuda T</span>, <span class="author">Lenhard B</span>, <span class="author">Eveno E</span>, <span class="author">Suzuki Y</span>, <span class="author">Yamasaki C</span>, <span class="author">Takeda J</span>, <span class="author">Gough C</span>, <span class="author">Hilton P</span>, <span class="author">Fujii Y</span>, <span class="author">Sakai H</span>, <span class="author">Tanaka S</span>, <span class="author">Amid C</span>, <span class="author">Bellgard M</span>, <span class="author">Bonaldo Mde F</span>, <span class="author">Bono H</span>, <span class="author">Bromberg SK</span>, <span class="author">Brookes AJ</span>, <span class="author">Bruford E</span>, <span class="author">Carninci P</span>, <span class="author">Chelala C</span>, <span class="author">Couillault C</span>, <span class="author">de Souza SJ</span>, <span class="author">Debily MA</span>, <span class="author">Devignes MD</span>, <span class="author">Dubchak I</span>, <span class="author">Endo T</span>, <span class="author">Estreicher A</span>, <span class="author">Eyras E</span>, <span class="author">Fukami-Kobayashi K</span>, <span class="author">Gopinath GR</span>, <span class="author">Graudens E</span>, <span class="author">Hahn Y</span>, <span class="author">Han M</span>, <span class="author">Han ZG</span>, <span class="author">Hanada K</span>, <span class="author">Hanaoka H</span>, <span class="author">Harada E</span>, <span class="author">Hashimoto K</span>, <span class="author">Hinz U</span>, <span class="author">Hirai M</span>, <span class="author">Hishiki T</span>, <span class="author">Hopkinson I</span>, <span class="author">Imbeaud S</span>, <span class="author">Inoko H</span>, <span class="author">Kanapin A</span>, <span class="author">Kaneko Y</span>, <span class="author">Kasukawa T</span>, <span class="author">Kelso J</span>, <span class="author">Kersey P</span>, <span class="author">Kikuno R</span>, <span class="author">Kimura K</span>, <span class="author">Korn B</span>, <span class="author">Kuryshev V</span>, <span class="author">Makalowska I</span>, <span class="author">Makino T</span>, <span class="author">Mano S</span>, <span class="author">Mariage-Samson R</span>, <span class="author">Mashima J</span>, <span class="author">Matsuda H</span>, <span class="author">Mewes HW</span>, <span class="author">Minoshima S</span>, <span class="author">Nagai K</span>, <span class="author">Nagasaki H</span>, <span class="author">Nagata N</span>, <span class="author">Nigam R</span>, <span class="author">Ogasawara O</span>, <span class="author">Ohara O</span>, <span class="author">Ohtsubo M</span>, <span class="author">Okada N</span>, <span class="author">Okido T</span>, <span class="author">Oota S</span>, <span class="author">Ota M</span>, <span class="author">Ota T</span>, <span class="author">Otsuki T</span>, <span class="author">Piatier-Tonneau D</span>, <span class="author">Poustka A</span>, <span class="author">Ren SX</span>, <span class="author">Saitou N</span>, <span class="author">Sakai K</span>, <span class="author">Sakamoto S</span>, <span class="author">Sakate R</span>, <span class="author">Schupp I</span>, <span class="author">Servant F</span>, <span class="author">Sherry S</span>, <span class="author">Shiba R</span>, <span class="author">Shimizu N</span>, <span class="author">Shimoyama M</span>, <span class="author">Simpson AJ</span>, <span class="author">Soares B</span>, <span class="author">Steward C</span>, <span class="author">Suwa M</span>, <span class="author">Suzuki M</span>, <span class="author">Takahashi A</span>, <span class="author">Tamiya G</span>, <span class="author">Tanaka H</span>, <span class="author">Taylor T</span>, <span class="author">Terwilliger JD</span>, <span class="author">Unneberg P</span>, <span class="author">Veeramachaneni V</span>, <span class="author">Watanabe S</span>, <span class="author">Wilming L</span>, <span class="author">Yasuda N</span>, <span class="author">Yoo HS</span>, <span class="author">Stodolsky M</span>, <span class="author">Makalowski W</span>, <span class="author">Go M</span>, <span class="author">Nakai K</span>, <span class="author">Takagi T</span>, <span class="author">Kanehisa M</span>, <span class="author">Sakaki Y</span>, <span class="author">Quackenbush J</span>, <span class="author">Okazaki Y</span>, <span class="author">Hayashizaki Y</span>, <span class="author">Hide W</span>, <span class="author">Chakraborty R</span>, <span class="author">Nishikawa K</span>, <span class="author">Sugawara H</span>, <span class="author">Tateno Y</span>, <span class="author">Chen Z</span>, <span class="author">Oishi M</span>, <span class="author">Tonellato P</span>, <span class="author">Apweiler R</span>, <span class="author">Okubo K</span>, <span class="author">Wagner L</span>, <span class="author">Wiemann S</span>, <span class="author">Strausberg RL</span>, <span class="author">Isogai T</span>, <span class="author">Auffray C</span>, <span class="author">Nomura N</span>, <span class="author">Gojobori T</span> and <span class="author">Sugano S</span></p>
<p class="affiliation">Integrated Database Group, Biological Information Research Center, National Institute of Advanced Industrial Science and Technology, Tokyo, Japan.</p><p class="abstract">The human genome sequence defines our inherent biological potential; the realization of the biology encoded therein requires knowledge of the function of each gene. Currently, our knowledge in this area is still limited. Several lines of investigation have been used to elucidate the structure and function of the genes in the human genome. Even so, gene prediction remains a difficult task, as the varieties of transcripts of a gene may vary to a great extent. We thus performed an exhaustive integrative characterization of 41,118 full-length cDNAs that capture the gene transcripts as complete functional cassettes, providing an unequivocal report of structural and functional diversity at the gene level. Our international collaboration has validated 21,037 human gene candidates by analysis of high-quality full-length cDNA clones through curation using unified criteria. This led to the identification of 5,155 new gene candidates. It also manifested the most reliable way to control the quality of the cDNA clones. We have developed a human gene database, called the H-Invitational Database (H-InvDB; http://www.h-invitational.jp/). It provides the following: integrative annotation of human genes, description of gene structures, details of novel alternative splicing isoforms, non-protein-coding RNAs, functional domains, subcellular localizations, metabolic pathways, predictions of protein three-dimensional structure, mapping of known single nucleotide polymorphisms (SNPs), identification of polymorphic microsatellite repeats within human genes, and comparative results with mouse full-length cDNAs. The H-InvDB analysis has shown that up to 4% of the human genome sequence (National Center for Biotechnology Information build 34 assembly) may contain misassembled or missing regions. We found that 6.5% of the human gene candidates (1,377 loci) did not have a good protein-coding open reading frame, of which 296 loci are strong candidates for non-protein-coding RNA genes. In addition, among 72,027 uniquely mapped SNPs and insertions/deletions localized within human genes, 13,215 nonsynonymous SNPs, 315 nonsense SNPs, and 452 indels occurred in coding regions. Together with 25 polymorphic microsatellite repeats present in coding regions, they may alter protein structure, causing phenotypic effects or resulting in disease. The H-InvDB platform represents a substantial contribution to resources needed for the exploration of human biology and pathology.</p>
<p class="grants">Funded by: NHLBI NIH HHS: <span class="grant">R01 HL064541</span></p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">6</span>;<span class="pagerange">e162</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15103394">15103394</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC393292">393292</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0020162">10.1371/journal.pbio.0020162</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.</h4>
<p class="authors"><span class="author">International Chicken Genome Sequencing Consortium</span></p>
<p class="affiliation">Genome Sequencing Center, Washington University School of Medicine, Campus Box 8501, 4444 Forest Park Avenue, St Louis, Missouri 63108, USA.</p><p class="abstract">We present here a draft genome sequence of the red jungle fowl, Gallus gallus. Because the chicken is a modern descendant of the dinosaurs and the first non-mammalian amniote to have its genome sequenced, the draft sequence of its genome--composed of approximately one billion base pairs of sequence and an estimated 20,000-23,000 genes--provides a new perspective on vertebrate genome evolution, while also improving the annotation of mammalian genomes. For example, the evolutionary distance between chicken and human provides high specificity in detecting functional elements, both non-coding and coding. Notably, many conserved non-coding sequences are far from genes and cannot be assigned to defined functional classes. In coding regions the evolutionary dynamics of protein domains and orthologous groups illustrate processes that distinguish the lineages leading to birds and mammals. The distinctive properties of avian microchromosomes, together with the inferred patterns of conserved synteny, provide additional insights into vertebrate chromosome architecture.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">432</span>;<span class="number">7018</span>;<span class="pagerange">695-716</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15592404">15592404</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03154">10.1038/nature03154</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Integrating ethics and science in the International HapMap Project.</h4>
<p class="authors"><span class="author">International HapMap Consortium</span></p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">R01 HG002189-01</span>, <span class="grant">R01 HG002189-02</span>, <span class="grant">R01 HG002189-03</span></p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">6</span>;<span class="pagerange">467-75</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153999" rel="external">15153999</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2271136" rel="external">2271136</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1351" rel="external">10.1038/nrg1351</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Finishing the euchromatic sequence of the human genome.</h4>
<p class="authors"><span class="author">International Human Genome Sequencing Consortium</span></p><p class="abstract">The sequence of the human genome encodes the genetic instructions for human physiology, as well as rich information about human evolution. In 2001, the International Human Genome Sequencing Consortium reported a draft sequence of the euchromatic portion of the human genome. Since then, the international collaboration has worked to convert this draft into a genome sequence with high accuracy and nearly complete coverage. Here, we report the result of this finishing process. The current genome sequence (Build 35) contains 2.85 billion nucleotides interrupted by only 341 gaps. It covers approximately 99% of the euchromatic genome and is accurate to an error rate of approximately 1 event per 100,000 bases. Many of the remaining euchromatic gaps are associated with segmental duplications and will require focused work with new methods. The near-complete sequence, the first for a vertebrate, greatly improves the precision of biological analyses of the human genome including studies of gene number, birth and death. Notably, the human genome seems to encode only 20,000-25,000 protein-coding genes. The genome sequence reported here should serve as a firm foundation for biomedical research in the decades ahead.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">431</span>;<span class="number">7011</span>;<span class="pagerange">931-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15496913">15496913</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03001">10.1038/nature03001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The zebrafish genome project: sequence analysis and annotation.</h4>
<p class="authors"><span class="author">Jekosch K</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Cambridge CB10 1SA, United Kingdom.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">P41 HG002659</span></p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">77</span>;<span class="pagerange">225-39</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15602914" rel="external">15602914</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Human MicroRNA targets.</h4>
<p class="authors"><span class="author">John B</span>, <span class="author">Enright AJ</span>, <span class="author">Aravin A</span>, <span class="author">Tuschl T</span>, <span class="author">Sander C</span> and <span class="author">Marks DS</span></p>
<p class="affiliation">Computational Biology Center, Memorial Sloan-Kettering Cancer Center, New York, New York, USA.</p><p class="abstract">MicroRNAs (miRNAs) interact with target mRNAs at specific sites to induce cleavage of the message or inhibit translation. The specific function of most mammalian miRNAs is unknown. We have predicted target sites on the 3&#39; untranslated regions of human gene transcripts for all currently known 218 mammalian miRNAs to facilitate focused experiments. We report about 2,000 human genes with miRNA target sites conserved in mammals and about 250 human genes conserved as targets between mammals and fish. The prediction algorithm optimizes sequence complementarity using position-specific rules and relies on strict requirements of interspecies conservation. Experimental support for the validity of the method comes from known targets and from strong enrichment of predicted targets in mRNAs associated with the fragile X mental retardation protein in mammals. This is consistent with the hypothesis that miRNAs act as sequence-specific adaptors in the interaction of ribonuclear particles with translationally regulated messages. Overrepresented groups of targets include mRNAs coding for transcription factors, components of the miRNA machinery, and other proteins involved in translational regulation, as well as components of the ubiquitin machinery, representing novel feedback loops in gene regulation. Detailed information about target genes, target processes, and open-source software for target prediction (miRanda) is available at http://www.microrna.org. Our analysis suggests that miRNA genes, which are about 1% of all human genes, regulate protein production for 10% or more of all human genes.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">11</span>;<span class="pagerange">e363</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15502875" rel="external">15502875</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC521178" rel="external">521178</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0020363" rel="external">10.1371/journal.pbio.0020363</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene array analysis of Yersinia enterocolitica FlhD and FlhC: regulation of enzymes affecting synthesis and degradation of carbamoylphosphate.</h4>
<p class="authors"><span class="author">Kapatral V</span>, <span class="author">Campbell JW</span>, <span class="author">Minnich SA</span>, <span class="author">Thomson NR</span>, <span class="author">Matsumura P</span> and <span class="author">Pr&uuml;ss BM</span></p>
<p class="affiliation">Integrated Genomics, Inc., 2201 West Campbell Park Dr., Chicago, IL 60612, USA.</p><p class="abstract">This paper focuses on global gene regulation by FlhD/FlhC in enteric bacteria. Even though Yersinia enterocolitica FlhD/FlhC can complement an Escherichia coli flhDC mutant for motility, it is not known if the Y. enterocolitica FlhD/FlhC complex has an effect on metabolism similar to E. coli. To study metabolic gene regulation, a partial Yersinia enterocolitica 8081c microarray was constructed and the expression patterns of wild-type cells were compared to an flhDC mutant strain at 25 and 37 degrees C. The overlap between the E. coli and Y. enterocolitica FlhD/FlhC regulated genes was 25 %. Genes that were regulated at least fivefold by FlhD/FlhC in Y. enterocolitica are genes encoding urocanate hydratase (hutU), imidazolone propionase (hutI), carbamoylphosphate synthetase (carAB) and aspartate carbamoyltransferase (pyrBI). These enzymes are part of a pathway that is involved in the degradation of L-histidine to L-glutamate and eventually leads into purine/pyrimidine biosynthesis via carbamoylphosphate and carbamoylaspartate. A number of other genes were regulated at a lower rate. In two additional experiments, the expression of wild-type cells grown at 4 or 25 degrees C was compared to the same strain grown at 37 degrees C. The expression of the flagella master operon flhD was not affected by temperature, whereas the flagella-specific sigma factor fliA was highly expressed at 25 degrees C and reduced at 4 and 37 degrees C. Several other flagella genes, all of which are under the control of FliA, exhibited a similar temperature profile. These data are consistent with the hypothesis that temperature regulation of flagella genes might be mediated by the flagella-specific sigma factor FliA and not the flagella master regulator FlhD/FlhC.</p>
<p class="grants">Funded by: NCRR NIH HHS: <span class="grant">P20 RR16454</span>; NIGMS NIH HHS: <span class="grant">GM59484</span></p>
<p class="publication"><span class="title">Microbiology (Reading, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">150</span>;<span class="number">Pt 7</span>;<span class="pagerange">2289-300</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15256571" rel="external">15256571</a>; DOI: <a href="http://dx.doi.org/10.1099%2Fmic.0.26814-0" rel="external">10.1099/mic.0.26814-0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">EnsMart: a generic system for fast and flexible access to biological data.</h4>
<p class="authors"><span class="author">Kasprzyk A</span>, <span class="author">Keefe D</span>, <span class="author">Smedley D</span>, <span class="author">London D</span>, <span class="author">Spooner W</span>, <span class="author">Melsopp C</span>, <span class="author">Hammond M</span>, <span class="author">Rocca-Serra P</span>, <span class="author">Cox T</span> and <span class="author">Birney E</span></p>
<p class="affiliation">European Bioinformatics Institute (EBI), Hinxton, Cambridge CB10 1SH, UK. <a href="mailto:%61%72%65%6B%40%65%62%69.%61%63.%75%6B">&#97;&#114;&#101;&#107;&#64;&#101;&#98;&#105;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The EnsMart system (www.ensembl.org/EnsMart) provides a generic data warehousing solution for fast and flexible querying of large biological data sets and integration with third-party data and tools. The system consists of a query-optimized database and interactive, user-friendly interfaces. EnsMart has been applied to Ensembl, where it extends its genomic browser capabilities, facilitating rapid retrieval of customized data sets. A wide variety of complex queries, on various types of annotations, for numerous species are supported. These can be applied to many research problems, ranging from SNP selection for candidate gene screening, through cross-species evolutionary comparisons, to microarray annotation. Users can group and refine biological data according to many criteria, including cross-species analyses, disease links, sequence variations, and expression patterns. Both tabulated list data and biological sequence output can be generated dynamically, in HTML, text, Microsoft Excel, and compressed formats. A wide range of sequence types, such as cDNA, peptides, coding regions, UTRs, and exons, with additional upstream and downstream regions, can be retrieved. The EnsMart database can be accessed via a public Web site, or through a Java application suite. Both implementations and the database are freely available for local installation, and can be extended or adapted to &#39;non-Ensembl&#39; data sets.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">1</span>;<span class="pagerange">160-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14707178" rel="external">14707178</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC314293" rel="external">314293</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1645104" rel="external">10.1101/gr.1645104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A comprehensive survey of human Y-chromosomal microsatellites.</h4>
<p class="authors"><span class="author">Kayser M</span>, <span class="author">Kittler R</span>, <span class="author">Erler A</span>, <span class="author">Hedman M</span>, <span class="author">Lee AC</span>, <span class="author">Mohyuddin A</span>, <span class="author">Mehdi SQ</span>, <span class="author">Rosser Z</span>, <span class="author">Stoneking M</span>, <span class="author">Jobling MA</span>, <span class="author">Sajantila A</span> and <span class="author">Tyler-Smith C</span></p>
<p class="affiliation">Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany. <a href="mailto:%6D.%6B%61%79%73%65%72%40%65%72%61%73%6D%75%73%6D%63.%6E%6C">&#109;&#46;&#107;&#97;&#121;&#115;&#101;&#114;&#64;&#101;&#114;&#97;&#115;&#109;&#117;&#115;&#109;&#99;&#46;&#110;&#108;</a></p><p class="abstract">We have screened the nearly complete DNA sequence of the human Y chromosome for microsatellites (short tandem repeats) that meet the criteria of having a repeat-unit size of &gt; or = 3 and a repeat count of &gt; or = 8 and thus are likely to be easy to genotype accurately and to be polymorphic. Candidate loci were tested in silico for novelty and for probable Y specificity, and then they were tested experimentally to identify Y-specific loci and to assess their polymorphism. This yielded 166 useful new Y-chromosomal microsatellites, 139 of which were polymorphic, in a sample of eight diverse Y chromosomes representing eight Y-SNP haplogroups. This large sample of microsatellites, together with 28 previously known markers analyzed here--all sharing a common evolutionary history--allowed us to investigate the factors influencing their variation. For simple microsatellites, the average repeat count accounted for the highest proportion of repeat variance (approximately 34%). For complex microsatellites, the largest proportion of the variance (again, approximately 34%) was explained by the average repeat count of the longest homogeneous array, which normally is variable. In these complex microsatellites, the additional repeats outside the longest homogeneous array significantly increased the variance, but this was lower than the variance of a simple microsatellite with the same total repeat count. As a result of this work, a large number of new, highly polymorphic Y-chromosomal microsatellites are now available for population-genetic, evolutionary, genealogical, and forensic investigations.</p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">057559</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">74</span>;<span class="number">6</span>;<span class="pagerange">1183-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15195656" rel="external">15195656</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1182082" rel="external">1182082</a>; DOI: <a href="http://dx.doi.org/10.1086%2F421531" rel="external">10.1086/421531</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Efficiency and consistency of haplotype tagging of dense SNP maps in multiple samples.</h4>
<p class="authors"><span class="author">Ke X</span>, <span class="author">Durrant C</span>, <span class="author">Morris AP</span>, <span class="author">Hunt S</span>, <span class="author">Bentley DR</span>, <span class="author">Deloukas P</span> and <span class="author">Cardon LR</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK.</p><p class="abstract">Haplotype tagging is a means of retaining most of the information in high density marker maps, while reducing genotyping requirements. Estimates of the numbers of tagging SNPs required to cover the human genome have varied widely, ranging from 100,000 to 1,000,000. Tagging has been applied to a number of gene-based datasets but has not been evaluated in contexts reflecting those of genome-wide association studies--large chromosome regions and multiple samples drawn from the same population. We analysed 5000 common markers across a 10 Mb segment of human chromosome 20 in three samples (UK Caucasian, CEPH Caucasian, African American) to evaluate tagging efficiency and consistency. Overall, the results indicate a high degree of efficiency, yielding 3-5-fold savings in Caucasians and 2-3-fold savings in African Americans. These levels varied according to linkage disequilibrium (LD) levels, tagging thresholds and allele frequencies, but in high LD regions they did not vary markedly due to marker density. However, a strong positive relationship between marker density and tagging was observed, relating to the fact that increasing marker density yields greater sequence coverage in high LD, thus requiring more tag SNPs to cover a greater fraction of the genome. Encouragingly, whatever the density employed, a high level of robustness was observed between UK and CEPH samples, as most of the htSNPs selected in one sample were also appropriate as tags in the other.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">NEI-12562</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13</span>;<span class="number">21</span>;<span class="pagerange">2557-65</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15367493" rel="external">15367493</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh294" rel="external">10.1093/hmg/ddh294</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The impact of SNP density on fine-scale patterns of linkage disequilibrium.</h4>
<p class="authors"><span class="author">Ke X</span>, <span class="author">Hunt S</span>, <span class="author">Tapper W</span>, <span class="author">Lawrence R</span>, <span class="author">Stavrides G</span>, <span class="author">Ghori J</span>, <span class="author">Whittaker P</span>, <span class="author">Collins A</span>, <span class="author">Morris AP</span>, <span class="author">Bentley D</span>, <span class="author">Cardon LR</span> and <span class="author">Deloukas P</span></p>
<p class="affiliation">Wellcome Trust Centre for Human Genetics, University of Oxford, UK.</p><p class="abstract">Linkage disequilibrium (LD) is a measure of the degree of association between alleles in a population. The detection of disease-causing variants by association with neighbouring single nucleotide polymorphisms (SNPs) depends on the existence of strong LD between them. Previous studies have indicated that the extent of LD is highly variable in different chromosome regions and different populations, demonstrating the importance of genome-wide accurate measurement of LD at high resolution throughout the human genome. A uniform feature of these studies has been the inability to detect LD in regions of low marker density. To investigate the dependence of LD patterns on marker selection we performed a high-resolution study in African-American, Asian and UK Caucasian populations. We selected over 5000 SNPs with an average spacing of approximately 1 SNP per 2 kb after validating ca 12 000 SNPs derived from a dense SNP collection (1 SNP per 0.3 kb on average). Applications of different statistical methods of LD assessment highlight similar areas of high and low LD. However, at high resolution, features such as overall sequence coverage in LD blocks and block boundaries vary substantially with respect to marker density. Model-based linkage disequilibrium unit (LDU) maps appear robust to marker density and consistently influenced by marker allele frequency. The results suggest that very dense marker sets will be required to yield stable views of fine-scale LD in the human genome.</p>
<p class="grants">Funded by: NEI NIH HHS: <span class="grant">EY-126562</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13</span>;<span class="number">6</span>;<span class="pagerange">577-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14734624">14734624</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh060">10.1093/hmg/ddh060</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Wnt co-receptors Lrp5 and Lrp6 are essential for gastrulation in mice.</h4>
<p class="authors"><span class="author">Kelly OG</span>, <span class="author">Pinson KI</span> and <span class="author">Skarnes WC</span></p>
<p class="affiliation">Department of Molecular and Cell Biology, University of California at Berkeley, Berkeley, CA 94720-3200, USA.</p><p class="abstract">Recent work has identified LDL receptor-related family members, Lrp5 and Lrp6, as co-receptors for the transduction of Wnt signals. Our analysis of mice carrying mutations in both Lrp5 and Lrp6 demonstrates that the functions of these genes are redundant and are essential for gastrulation. Lrp5;Lrp6 double homozygous mutants fail to establish a primitive streak, although the anterior visceral endoderm and anterior epiblast fates are specified. Thus, Lrp5 and Lrp6 are required for posterior patterning of the epiblast, consistent with a role in transducing Wnt signals in the early embryo. Interestingly, Lrp5(+/-);Lrp6(-/-) embryos die shortly after gastrulation and exhibit an accumulation of cells at the primitive streak and a selective loss of paraxial mesoderm. A similar phenotype is observed in Fgf8 and Fgfr1 mutant embryos and provides genetic evidence in support of a molecular link between the Fgf and Wnt signaling pathways in patterning nascent mesoderm. Lrp5(+/-);Lrp6(-/-) embryos also display an expansion of anterior primitive streak derivatives and anterior neurectoderm that correlates with increased Nodal expression in these embryos. The effect of reducing, but not eliminating, Wnt signaling in Lrp5(+/-);Lrp6(-/-) mutant embryos provides important insight into the interplay between Wnt, Fgf and Nodal signals in patterning the early mouse embryo.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">131</span>;<span class="number">12</span>;<span class="pagerange">2803-15</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15142971">15142971</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.01137">10.1242/dev.01137</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fibronectin binding to the Salmonella enterica serotype Typhimurium ShdA autotransporter protein is inhibited by a monoclonal antibody recognizing the A3 repeat.</h4>
<p class="authors"><span class="author">Kingsley RA</span>, <span class="author">Abi Ghanem D</span>, <span class="author">Puebla-Osorio N</span>, <span class="author">Keestra AM</span>, <span class="author">Berghman L</span> and <span class="author">B&auml;umler AJ</span></p>
<p class="affiliation">Department of Medical Microbiology and Immunology, College of Medicine, College Station, TX 77843, USA. <a href="mailto:%72%61%6B%40%73%61%6E%67%65%72.%61%63.%75%6B">&#114;&#97;&#107;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">ShdA is a large outer membrane protein of the autotransporter family whose passenger domain binds the extracellular matrix proteins fibronectin and collagen I, possibly by mimicking the host ligand heparin. The ShdA passenger domain consists of approximately 1,500 amino acid residues that can be divided into two regions based on features of the primary amino acid sequence: an N-terminal nonrepeat region followed by a repeat region composed of two types of imperfect direct amino acid repeats, called type A and type B. The repeat region bound bovine fibronectin with an affinity similar to that for the complete ShdA passenger domain, while the nonrepeat region exhibited comparatively low fibronectin-binding activity. A number of fusion proteins containing truncated fragments of the repeat region did not bind bovine fibronectin. However, binding of the passenger domain to fibronectin was inhibited in the presence of immune serum raised to one truncated fragment of the repeat region that contained repeats A2, B8, A3, and B9. Furthermore, a monoclonal antibody that specifically recognized an epitope in a recombinant protein containing the A3 repeat inhibited binding of ShdA to fibronectin.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI40124</span>, <span class="grant">AI44170</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2004</span>;<span class="volume">186</span>;<span class="number">15</span>;<span class="pagerange">4931-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15262930" rel="external">15262930</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC451617" rel="external">451617</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.186.15.4931-4939.2004" rel="external">10.1128/JB.186.15.4931-4939.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Two new mouse chromosome 11 balancers.</h4>
<p class="authors"><span class="author">Klysik J</span>, <span class="author">Dinh C</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA. <a href="mailto:%4A%61%6E%5F%4B%6C%79%73%69%6B%40%42%72%6F%77%6E.%65%64%75">&#74;&#97;&#110;&#95;&#75;&#108;&#121;&#115;&#105;&#107;&#64;&#66;&#114;&#111;&#119;&#110;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Segmental inversions causing recombination suppression are an essential feature of balancer chromosomes. Meiotic crossing over between homologous chromosomes within an inversion interval will lead to nonviable gametes, while gametes generated from recombination events elsewhere on the chromosome will be unaffected. This apparent recombination suppression has been widely exploited in genetic studies in Drosophila to maintain and analyze stocks carrying recessive lethal mutations. Balancers are particularly useful in mutagenesis screens since they help to establish the approximate genomic location of alleles of genes causing phenotypes. Using the Cre-loxP recombination system, we have constructed two mouse balancer chromosomes carrying 8- and 30-cM inversions between Wnt3 and D11Mit69 and between Trp53 and EgfR loci, respectively. The Wnt3-D11Mit69 inversion mutates the Wnt3 locus and is therefore homozygous lethal. The Trp53-EgfR inversion is homozygous viable, since the EgfR locus is intact and mutations in p53 are homozygous viable. A dominantly acting K14-agouti minigene tags both rearrangements, which enables these balancer chromosomes to be visibly tracked in mouse stocks. With the addition of these balancers to the previously reported Trp53-Wnt3 balancer, most of mouse chromosome 11 is now available in balancer stocks.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">83</span>;<span class="number">2</span>;<span class="pagerange">303-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14706459" rel="external">14706459</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bacterial artificial chromosome (BAC) clones and the current clone map of the zebrafish genome.</h4>
<p class="authors"><span class="author">Koch R</span>, <span class="author">Rauch GJ</span>, <span class="author">Humphray S</span>, <span class="author">Geisler TR</span> and <span class="author">Plasterk R</span></p>
<p class="affiliation">Hubrecht Laboratory, Uppsalalaan 8, 3584 CT Utrecht, The Netherlands.</p>
<p class="publication"><span class="title">Methods in cell biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">77</span>;<span class="pagerange">295-304</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15602918" rel="external">15602918</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene finding in novel genomes.</h4>
<p class="authors"><span class="author">Korf I</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, CB10 1SA, UK. <a href="mailto:%69%6B%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#105;&#107;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Computational gene prediction continues to be an important problem, especially for genomes with little experimental data.</p><p class="abstract"><strong>Results</strong>: I introduce the SNAP gene finder which has been designed to be easily adaptable to a variety of genomes. In novel genomes without an appropriate gene finder, I demonstrate that employing a foreign gene finder can produce highly inaccurate results, and that the most compatible parameters may not come from the nearest phylogenetic neighbor. I find that foreign gene finders are more usefully employed to bootstrap parameter estimation and that the resulting parameters can be highly accurate.</p><p class="abstract"><strong>Conclusion</strong>: Since gene prediction is sensitive to species-specific parameters, every genome needs a dedicated gene finder.</p>
<p class="grants">Funded by: PHS HHS: <span class="grant">K22-00064-01</span></p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="pagerange">59</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15144565" rel="external">15144565</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC421630" rel="external">421630</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-5-59" rel="external">10.1186/1471-2105-5-59</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Somite polarity and segmental patterning of the peripheral nervous system.</h4>
<p class="authors"><span class="author">Kuan CY</span>, <span class="author">Tannahill D</span>, <span class="author">Cook GM</span> and <span class="author">Keynes RJ</span></p>
<p class="affiliation">Department of Anatomy, University of Cambridge, Downing Street, Cambridge CB2 3DY, UK.</p><p class="abstract">The analysis of the outgrowth pattern of spinal axons in the chick embryo has shown that somites are polarized into anterior and posterior halves. This polarity dictates the segmental development of the peripheral nervous system: migrating neural crest cells and outgrowing spinal axons traverse exclusively the anterior halves of the somite-derived sclerotomes, ensuring a proper register between spinal axons, their ganglia and the segmented vertebral column. Much progress has been made recently in understanding the molecular basis for somite polarization, and its linkage with Notch/Delta, Wnt and Fgf signalling. Contact-repulsive molecules expressed by posterior half-sclerotome cells provide critical guidance cues for axons and neural crest cells along the anterior-posterior axis. Diffusible repellents from surrounding tissues, particularly the dermomyotome and notochord, orient outgrowing spinal axons in the dorso-ventral axis (&#39;surround repulsion&#39;). Repulsive forces therefore guide axons in three dimensions. Although several molecular systems have been identified that may guide neural crest cells and axons in the sclerotome, it remains unclear whether these operate together with considerable overall redundancy, or whether any one system predominates in vivo.</p>
<p class="publication"><span class="title">Mechanisms of development</span>&nbsp;<span class="year">2004</span>;<span class="volume">121</span>;<span class="number">9</span>;<span class="pagerange">1055-68</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15296971" rel="external">15296971</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.mod.2004.05.001" rel="external">10.1016/j.mod.2004.05.001</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel microsatellite markers and single nucleotide polymorphisms refine the tylosis with oesophageal cancer (TOC) minimal region on 17q25 to 42.5 kb: sequencing does not identify the causative gene.</h4>
<p class="authors"><span class="author">Langan JE</span>, <span class="author">Cole CG</span>, <span class="author">Huckle EJ</span>, <span class="author">Byrne S</span>, <span class="author">McRonald FE</span>, <span class="author">Rowbottom L</span>, <span class="author">Ellis A</span>, <span class="author">Shaw JM</span>, <span class="author">Leigh IM</span>, <span class="author">Kelsell DP</span>, <span class="author">Dunham I</span>, <span class="author">Field JK</span> and <span class="author">Risk JM</span></p>
<p class="affiliation">Molecular Genetics and Oncology Group, Department of Clinical Dental Sciences, University of Liverpool, Edward&#39;s Building, Daulby Street, L69 3GN, Liverpool, UK.</p><p class="abstract">Tylosis (focal non-epidermolytic palmoplantar keratoderma) is associated with the early onset of squamous cell oesophageal cancer in three families. Linkage and haplotype analyses have previously mapped the tylosis with oesophageal cancer ( TOC) locus to a 500-kb region on chromosome 17q25 that has also been implicated in sporadically occurring squamous cell oesophageal cancer. In the current study, 17 additional putative microsatellite markers were identified within this 500-kb region by using sequence data and seven of these were shown to be polymorphic in the UK and US families. In addition, our complete sequence analysis of the non-repetitive parts of the TOC minimal region identified 53 novel and six known single nucleotide polymorphisms (SNPs) in one or both of these families. Further fine mapping of the TOC disease locus by haplotype analysis of the seven polymorphic markers and 21 of the 59 SNPs allowed the reduction of the minimal region to 42.5 kb. One known and two putative genes are located within this region but none of these genes shows tylosis-specific mutations within their protein-coding regions. Alternative mechanisms of disease gene action must therefore be considered.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">114</span>;<span class="number">6</span>;<span class="pagerange">534-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15007728" rel="external">15007728</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-004-1100-3" rel="external">10.1007/s00439-004-1100-3</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Separation, digestion, and cloning of intact parasite chromosomes embedded in agarose.</h4>
<p class="authors"><span class="author">Leech V</span>, <span class="author">Quail MA</span> and <span class="author">Melville SE</span></p>
<p class="affiliation">Department of Pathology, University of Cambridge, UK.</p><p class="abstract">The chromosomes of most protozoan parasites cannot be visualized using conventional microscopy because they are too small and do not condense sufficiently at metaphase. Therefore, the development of pulsed field gel electrophoresis allowed the resolution of many parasite karyotypes for the first time. The ability to prepare intact chromosomes in agarose plugs and to isolate individual homologs by electrophoresis has led to many new applications in parasite genomic analysis. This chapter describes the preparation of chromosome plugs from single-celled protozoan parasites, providing numerous tips on how to achieve the highest-quality preparations that will last for years. We also provide detailed protocols for the manipulation of individual excised chromosomes, including restriction mapping and preparation of chromosome shotgun libraries as used in many of the genomic sequencing projects. The protocols provided here underpin several of the advanced methods of genomic analysis and manipulation described in this volume of parasite genomics protocols.</p>
<p class="publication"><span class="title">Methods in molecular biology (Clifton, N.J.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">270</span>;<span class="pagerange">335-52</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153638" rel="external">15153638</a>; DOI: <a href="http://dx.doi.org/10.1385%2F1-59259-793-9%3A335" rel="external">10.1385/1-59259-793-9:335</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A first-draft human protein-interaction map.</h4>
<p class="authors"><span class="author">Lehner B</span> and <span class="author">Fraser AG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%62%6C%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#108;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Protein-interaction maps are powerful tools for suggesting the cellular functions of genes. Although large-scale protein-interaction maps have been generated for several invertebrate species, projects of a similar scale have not yet been described for any mammal. Because many physical interactions are conserved between species, it should be possible to infer information about human protein interactions (and hence protein function) using model organism protein-interaction datasets.</p><p class="abstract"><strong>Results</strong>: Here we describe a network of over 70,000 predicted physical interactions between around 6,200 human proteins generated using the data from lower eukaryotic protein-interaction maps. The physiological relevance of this network is supported by its ability to preferentially connect human proteins that share the same functional annotations, and we show how the network can be used to successfully predict the functions of human proteins. We find that combining interaction datasets from a single organism (but generated using independent assays) and combining interaction datasets from two organisms (but generated using the same assay) are both very effective ways of further improving the accuracy of protein-interaction maps.</p><p class="abstract"><strong>Conclusions</strong>: The complete network predicts interactions for a third of human genes, including 448 human disease genes and 1,482 genes of unknown function, and so provides a rich framework for biomedical research.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">9</span>;<span class="pagerange">R63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15345047" rel="external">15345047</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC522870" rel="external">522870</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2004-5-9-r63" rel="external">10.1186/gb-2004-5-9-r63</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">5,000 RNAi experiments on a chip.</h4>
<p class="authors"><span class="author">Lehner B</span> and <span class="author">Fraser AG</span></p>
<p class="publication"><span class="title">Nature methods</span>&nbsp;<span class="year">2004</span>;<span class="volume">1</span>;<span class="number">2</span>;<span class="pagerange">103-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15782169">15782169</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnmeth1104-103">10.1038/nmeth1104-103</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Protein domains enriched in mammalian tissue-specific or widely expressed genes.</h4>
<p class="authors"><span class="author">Lehner B</span> and <span class="author">Fraser AG</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">In multicellular organisms some genes are expressed in essentially all tissues, whereas others are expressed predominantly in only one or a few tissues. In this study, we investigate the relationship between the tissue-specificity of gene expression and the type of protein encoded. We find that many protein domains are found to be enriched in either tissue-specific or widely expressed genes. Domains enriched in tissue-specific genes tend to be metazoan-specific; these same domains are also enriched in genes that are not essential for cell viability. These findings identify families of proteins that are probably used in the development or terminal differentiation of many different tissue types.</p>
<p class="publication"><span class="title">Trends in genetics : TIG</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">10</span>;<span class="pagerange">468-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15363898" rel="external">15363898</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tig.2004.08.002" rel="external">10.1016/j.tig.2004.08.002</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Technique review: how to use RNA interference.</h4>
<p class="authors"><span class="author">Lehner B</span>, <span class="author">Fraser AG</span> and <span class="author">Sanderson CM</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%62%6C%6C%40%73%61%6E%67%65%72.%61%63.%75%6B">&#98;&#108;&#108;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">RNA interference (RNAi) has been rapidly adopted as a general method for inhibiting gene expression in most laboratory organisms. This paper discusses how libraries of RNAi reagents are being used to perform genome-wide reverse genetic screens in both model organisms and mammalian cells. Guidelines for designing effective small interfering RNAs and appropriate controls for mammalian RNAi experiments will also be discussed.</p>
<p class="publication"><span class="title">Briefings in functional genomics &amp; proteomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">3</span>;<span class="number">1</span>;<span class="pagerange">68-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15163361" rel="external">15163361</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genome-wide RNAi identifies p53-dependent and -independent regulators of germ cell apoptosis in C. elegans.</h4>
<p class="authors"><span class="author">Lettre G</span>, <span class="author">Kritikou EA</span>, <span class="author">Jaeggi M</span>, <span class="author">Calixto A</span>, <span class="author">Fraser AG</span>, <span class="author">Kamath RS</span>, <span class="author">Ahringer J</span> and <span class="author">Hengartner MO</span></p>
<p class="affiliation">Institute for Molecular Biology, University of Zurich, Winterthurerstrasse 190, 8057 Zurich, Switzerland.</p><p class="abstract">We used genome-wide RNA interference (RNAi) to identify genes that affect apoptosis in the C. elegans germ line. RNAi-mediated knockdown of 21 genes caused a moderate to strong increase in germ cell death. Genetic epistasis studies with these RNAi candidates showed that a large subset (16/21) requires p53 to activate germ cell apoptosis. Apoptosis following knockdown of the genes in the p53-dependent class also depended on a functional DNA damage response pathway, suggesting that these genes might function in DNA repair or to maintain genome integrity. As apoptotic pathways are conserved, orthologues of the worm germline apoptosis genes presented here could be involved in the maintenance of genomic stability, p53 activation, and fertility in mammals.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM52240</span>; Wellcome Trust: <span class="grant">054523</span></p>
<p class="publication"><span class="title">Cell death and differentiation</span>&nbsp;<span class="year">2004</span>;<span class="volume">11</span>;<span class="number">11</span>;<span class="pagerange">1198-203</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15272318" rel="external">15272318</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fsj.cdd.4401488" rel="external">10.1038/sj.cdd.4401488</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A map of the interactome network of the metazoan C. elegans.</h4>
<p class="authors"><span class="author">Li S</span>, <span class="author">Armstrong CM</span>, <span class="author">Bertin N</span>, <span class="author">Ge H</span>, <span class="author">Milstein S</span>, <span class="author">Boxem M</span>, <span class="author">Vidalain PO</span>, <span class="author">Han JD</span>, <span class="author">Chesneau A</span>, <span class="author">Hao T</span>, <span class="author">Goldberg DS</span>, <span class="author">Li N</span>, <span class="author">Martinez M</span>, <span class="author">Rual JF</span>, <span class="author">Lamesch P</span>, <span class="author">Xu L</span>, <span class="author">Tewari M</span>, <span class="author">Wong SL</span>, <span class="author">Zhang LV</span>, <span class="author">Berriz GF</span>, <span class="author">Jacotot L</span>, <span class="author">Vaglio P</span>, <span class="author">Reboul J</span>, <span class="author">Hirozane-Kishikawa T</span>, <span class="author">Li Q</span>, <span class="author">Gabel HW</span>, <span class="author">Elewa A</span>, <span class="author">Baumgartner B</span>, <span class="author">Rose DJ</span>, <span class="author">Yu H</span>, <span class="author">Bosak S</span>, <span class="author">Sequerra R</span>, <span class="author">Fraser A</span>, <span class="author">Mango SE</span>, <span class="author">Saxton WM</span>, <span class="author">Strome S</span>, <span class="author">Van Den Heuvel S</span>, <span class="author">Piano F</span>, <span class="author">Vandenhaute J</span>, <span class="author">Sardet C</span>, <span class="author">Gerstein M</span>, <span class="author">Doucette-Stamm L</span>, <span class="author">Gunsalus KC</span>, <span class="author">Harper JW</span>, <span class="author">Cusick ME</span>, <span class="author">Roth FP</span>, <span class="author">Hill DE</span> and <span class="author">Vidal M</span></p>
<p class="affiliation">Dana-Farber Cancer Institute and Department of Genetics, Harvard Medical School, 44 Binney Street, Boston, MA 02115, USA.</p><p class="abstract">To initiate studies on how protein-protein interaction (or &quot;interactome&quot;) networks relate to multicellular functions, we have mapped a large fraction of the Caenorhabditis elegans interactome network. Starting with a subset of metazoan-specific proteins, more than 4000 interactions were identified from high-throughput, yeast two-hybrid (HT=Y2H) screens. Independent coaffinity purification assays experimentally validated the overall quality of this Y2H data set. Together with already described Y2H interactions and interologs predicted in silico, the current version of the Worm Interactome (WI5) map contains approximately 5500 interactions. Topological and biological features of this interactome network, as well as its integration with phenome and transcriptome data sets, lead to numerous biological hypotheses.</p>
<p class="grants">Funded by: NIA NIH HHS: <span class="grant">R01 AG011085</span>; NIGMS NIH HHS: <span class="grant">R01 GM034059</span>, <span class="grant">R01 GM034059-18</span></p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">303</span>;<span class="number">5657</span>;<span class="pagerange">540-3</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14704431">14704431</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1698949">1698949</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1091403">10.1126/science.1091403</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetics of the DST-mediated mRNA decay pathway using a transgene-based selection.</h4>
<p class="authors"><span class="author">Lidder P</span>, <span class="author">Johnson MA</span>, <span class="author">Sullivan ML</span>, <span class="author">Thompson DM</span>, <span class="author">P&eacute;rez-Amador MA</span>, <span class="author">Howard CJ</span> and <span class="author">Green PJ</span></p>
<p class="affiliation">Delaware Biotechnology Institute, University of Delaware, 15 Innovation Way, Newark, DE 19711, USA.</p><p class="abstract">mRNA sequences that control abundance, localization and translation initiation have been identified, yet the factors that recognize these sequences are largely unknown. In this report, a transgene-based strategy designed to isolate mutants of Arabidopsis thaliana that fail to recognize these sequences is described. In this strategy, a selectable gene and a screenable marker gene are put under the control of the sequence element being analysed and mutants are selected with altered abundance of the corresponding marker RNAs. The selection of mutants deficient in recognition of the DST (downstream) mRNA degradation signal is used as a test-case to illustrate some of the technical aspects that have facilitated success. Using this strategy, we report the isolation of a new mutant, dst3, deficient in the DST-mediated mRNA decay pathway. The targeted genetic strategy described circumvents certain technical limitations of biochemical approaches. Hence, it provides a means to investigate a variety of other mechanisms responsible for post-transcriptional regulation.</p>
<p class="publication"><span class="title">Biochemical Society transactions</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Pt 4</span>;<span class="pagerange">575-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15270679" rel="external">15270679</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBST0320575" rel="external">10.1042/BST0320575</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Staphylococcus aureus: superbug, super genome?</h4>
<p class="authors"><span class="author">Lindsay JA</span> and <span class="author">Holden MT</span></p>
<p class="affiliation">Department of Cellular &amp; Molecular Medicine, St George&#39;s Hospital Medical School, Cranmer Terrace, London, UK.</p><p class="abstract">Staphylococcus aureus is a common cause of infection in both hospitals and the community, and it is becoming increasingly virulent and resistant to antibiotics. The recent sequencing of seven strains of S. aureus provides unprecedented information about its genome diversity. Subtle differences in core (stable) regions of the genome have been exploited by multi-locus sequence typing (MLST) to understand S. aureus population structure. Dramatic differences in the carriage and spread of accessory genes, including those involved in virulence and resistance, contribute to the emergence of new strains with healthcare implications. Understanding the differences between S. aureus genomes and the controls that govern these changes is helping to improve our knowledge of S. aureus pathogenicity and to predict the evolution of super-superbugs.</p>
<p class="publication"><span class="title">Trends in microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">12</span>;<span class="number">8</span>;<span class="pagerange">378-85</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15276614" rel="external">15276614</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tim.2004.06.004" rel="external">10.1016/j.tim.2004.06.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genomic and genetic analysis of Bordetella bacteriophages encoding reverse transcriptase-mediated tropism-switching cassettes.</h4>
<p class="authors"><span class="author">Liu M</span>, <span class="author">Gingery M</span>, <span class="author">Doulatov SR</span>, <span class="author">Liu Y</span>, <span class="author">Hodes A</span>, <span class="author">Baker S</span>, <span class="author">Davis P</span>, <span class="author">Simmonds M</span>, <span class="author">Churcher C</span>, <span class="author">Mungall K</span>, <span class="author">Quail MA</span>, <span class="author">Preston A</span>, <span class="author">Harvill ET</span>, <span class="author">Maskell DJ</span>, <span class="author">Eiserling FA</span>, <span class="author">Parkhill J</span> and <span class="author">Miller JF</span></p>
<p class="affiliation">Department of Microbiology, Immunology, and Molecular Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA.</p><p class="abstract">Liu et al. recently described a group of related temperate bacteriophages that infect Bordetella subspecies and undergo a unique template-dependent, reverse transcriptase-mediated tropism switching phenomenon (Liu et al., Science 295: 2091-2094, 2002). Tropism switching results from the introduction of single nucleotide substitutions at defined locations in the VR1 (variable region 1) segment of the mtd (major tropism determinant) gene, which determines specificity for receptors on host bacteria. In this report, we describe the complete nucleotide sequences of the 42.5- to 42.7-kb double-stranded DNA genomes of three related phage isolates and characterize two additional regions of variability. Forty-nine coding sequences were identified. Of these coding sequences, bbp36 contained VR2 (variable region 2), which is highly dynamic and consists of a variable number of identical 19-bp repeats separated by one of three 5-bp spacers, and bpm encodes a DNA adenine methylase with unusual site specificity and a homopolymer tract that functions as a hotspot for frameshift mutations. Morphological and sequence analysis suggests that these Bordetella phage are genetic hybrids of P22 and T7 family genomes, lending further support to the idea that regions encoding protein domains, single genes, or blocks of genes are readily exchanged between bacterial and phage genomes. Bordetella bacteriophages are capable of transducing genetic markers in vitro, and by using animal models, we demonstrated that lysogenic conversion can take place in the mouse respiratory tract during infection.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">2-T32-AI07323</span>, <span class="grant">AI38417</span>; NIGMS NIH HHS: <span class="grant">GM-08042</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2004</span>;<span class="volume">186</span>;<span class="number">5</span>;<span class="pagerange">1503-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14973019">14973019</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC344406">344406</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NCD3G: a novel nine-cysteine domain in family 3 GPCRs.</h4>
<p class="authors"><span class="author">Liu X</span>, <span class="author">He Q</span>, <span class="author">Studholme DJ</span>, <span class="author">Wu Q</span>, <span class="author">Liang S</span> and <span class="author">Yu L</span></p>
<p class="affiliation">State Key Laboratory of Genetic Engineering, School of Life Sciences, Fudan University, Handan Road 220, Shanghai 200433, P.R. China.</p><p class="abstract">The NCD3G [for nine-cysteine domain of family 3 G-protein-coupled receptors (GPCRs)] domain is a novel protein domain that is conserved in family 3 GPCRs, including metabotropic glutamate receptors, calcium-sensing receptors, pheromone receptors and taste receptors, with the exception of GABA(B) receptors. The NCD3G domain contains nine highly conserved cysteine residues. Structural predictions suggest that NCD3G might possess four beta strands and three disulfide bridges. The structural model of NCD3G highlights the conserved residues co-segregated with certain familial diseases.</p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2004</span>;<span class="volume">29</span>;<span class="number">9</span>;<span class="pagerange">458-61</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15337118" rel="external">15337118</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tibs.2004.07.009" rel="external">10.1016/j.tibs.2004.07.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Hotspots of homologous recombination in the human genome: not all homologous sequences are equal.</h4>
<p class="authors"><span class="author">Lupski JR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA. <a href="mailto:%6A%6C%75%70%73%6B%69%40%73%61%6E%67%65%72.%61%63.%75%6B">&#106;&#108;&#117;&#112;&#115;&#107;&#105;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Homologous recombination between alleles or non-allelic paralogous sequences does not occur uniformly but is concentrated in &#39;hotspots&#39; with high recombination rates. Recent studies of these hotspots show that they do not share common sequence motifs, but they do have other features in common.</p>
<p class="publication"><span class="title">Genome biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">10</span>;<span class="pagerange">242</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15461806" rel="external">15461806</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC545587" rel="external">545587</a>; DOI: <a href="http://dx.doi.org/10.1186%2Fgb-2004-5-10-242" rel="external">10.1186/gb-2004-5-10-242</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Organization and evolution of a gene-rich region of the mouse genome: a 12.7-Mb region deleted in the Del(13)Svea36H mouse.</h4>
<p class="authors"><span class="author">Mallon AM</span>, <span class="author">Wilming L</span>, <span class="author">Weekes J</span>, <span class="author">Gilbert JG</span>, <span class="author">Ashurst J</span>, <span class="author">Peyrefitte S</span>, <span class="author">Matthews L</span>, <span class="author">Cadman M</span>, <span class="author">McKeone R</span>, <span class="author">Sellick CA</span>, <span class="author">Arkell R</span>, <span class="author">Botcherby MR</span>, <span class="author">Strivens MA</span>, <span class="author">Campbell RD</span>, <span class="author">Gregory S</span>, <span class="author">Denny P</span>, <span class="author">Hancock JM</span>, <span class="author">Rogers J</span> and <span class="author">Brown SD</span></p>
<p class="affiliation">Medical Research Council Mammalian Genetics Unit, Harwell, Oxfordshire, United Kingdom.</p><p class="abstract">Del(13)Svea36H (Del36H) is a deletion of approximately 20% of mouse chromosome 13 showing conserved synteny with human chromosome 6p22.1-6p22.3/6p25. The human region is lost in some deletion syndromes and is the site of several disease loci. Heterozygous Del36H mice show numerous phenotypes and may model aspects of human genetic disease. We describe 12.7 Mb of finished, annotated sequence from Del36H. Del36H has a higher gene density than the draft mouse genome, reflecting high local densities of three gene families (vomeronasal receptors, serpins, and prolactins) which are greatly expanded relative to human. Transposable elements are concentrated near these gene families. We therefore suggest that their neighborhoods are gene factories, regions of frequent recombination in which gene duplication is more frequent. The gene families show different proportions of pseudogenes, likely reflecting different strengths of purifying selection and/or gene conversion. They are also associated with relatively low simple sequence concentrations, which vary across the region with a periodicity of approximately 5 Mb. Del36H contains numerous evolutionarily conserved regions (ECRs). Many lie in noncoding regions, are detectable in species as distant as Ciona intestinalis, and therefore are candidate regulatory sequences. This analysis will facilitate functional genomic analysis of Del36H and provides insights into mouse genome evolution.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">10A</span>;<span class="pagerange">1888-901</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15364904">15364904</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC524412">524412</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2478604">10.1101/gr.2478604</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GOtcha: a new method for prediction of protein function assessed by the annotation of seven genomes.</h4>
<p class="authors"><span class="author">Martin DM</span>, <span class="author">Berriman M</span> and <span class="author">Barton GJ</span></p>
<p class="affiliation">Post-Genomics and Molecular Interactions Centre, School of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, UK. <a href="mailto:%64.%6D.%61.%6D%61%72%74%69%6E%40%64%75%6E%64%65%65.%61%63.%75%6B">&#100;&#46;&#109;&#46;&#97;&#46;&#109;&#97;&#114;&#116;&#105;&#110;&#64;&#100;&#117;&#110;&#100;&#101;&#101;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: The function of a novel gene product is typically predicted by transitive assignment of annotation from similar sequences. We describe a novel method, GOtcha, for predicting gene product function by annotation with Gene Ontology (GO) terms. GOtcha predicts GO term associations with term-specific probability (P-score) measures of confidence. Term-specific probabilities are a novel feature of GOtcha and allow the identification of conflicts or uncertainty in annotation.</p><p class="abstract"><strong>Results</strong>: The GOtcha method was applied to the recently sequenced genome for Plasmodium falciparum and six other genomes. GOtcha was compared quantitatively for retrieval of assigned GO terms against direct transitive assignment from the highest scoring annotated BLAST search hit (TOPBLAST). GOtcha exploits information deep into the &#39;twilight zone&#39; of similarity search matches, making use of much information that is otherwise discarded by more simplistic approaches. At a P-score cutoff of 50%, GOtcha provided 60% better recovery of annotation terms and 20% higher selectivity than annotation with TOPBLAST at an E-value cutoff of 10(-4).</p><p class="abstract"><strong>Conclusions</strong>: The GOtcha method is a useful tool for genome annotators. It has identified both errors and omissions in the original Plasmodium falciparum annotation and is being adopted by many other genome sequencing projects.</p>
<p class="publication"><span class="title">BMC bioinformatics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="pagerange">178</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15550167" rel="external">15550167</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC535938" rel="external">535938</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2105-5-178" rel="external">10.1186/1471-2105-5-178</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Automated comparative sequence analysis identifies mutations in 89% of NF1 patients and confirms a mutation cluster in exons 11-17 distinct from the GAP related domain.</h4>
<p class="authors"><span class="author">Mattocks C</span>, <span class="author">Baralle D</span>, <span class="author">Tarpey P</span>, <span class="author">ffrench-Constant C</span>, <span class="author">Bobrow M</span> and <span class="author">Whittaker J</span></p>
<p class="affiliation">Department of Medical Genetics, Box 134, Addenbrooke&#39;s Hospital, Cambridge, UK.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">e48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15060124" rel="external">15060124</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735749" rel="external">1735749</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Specific deletion of focal adhesion kinase suppresses tumor formation and blocks malignant progression.</h4>
<p class="authors"><span class="author">McLean GW</span>, <span class="author">Komiyama NH</span>, <span class="author">Serrels B</span>, <span class="author">Asano H</span>, <span class="author">Reynolds L</span>, <span class="author">Conti F</span>, <span class="author">Hodivala-Dilke K</span>, <span class="author">Metzger D</span>, <span class="author">Chambon P</span>, <span class="author">Grant SG</span> and <span class="author">Frame MC</span></p>
<p class="affiliation">The Beatson Institute for Cancer Research, Garscube Estate, Bearsden, Glasgow, G61 1BD, United Kingdom. <a href="mailto:%67.%6D%63%6C%65%61%6E%40%62%65%61%74%73%6F%6E.%67%6C%61.%61%63.%75%6B">&#103;&#46;&#109;&#99;&#108;&#101;&#97;&#110;&#64;&#98;&#101;&#97;&#116;&#115;&#111;&#110;&#46;&#103;&#108;&#97;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">We have generated mice with a floxed fak allele under the control of keratin-14-driven Cre fused to a modified estrogen receptor (CreER(T2)). 4-Hydroxy-tamoxifen treatment induced fak deletion in the epidermis, and suppressed chemically induced skin tumor formation. Loss of fak induced once benign tumors had formed inhibited malignant progression. Although fak deletion was associated with reduced migration of keratinocytes in vitro, we found no effect on wound re-epithelialization in vivo. However, increased keratinocyte cell death was observed after fak deletion in vitro and in vivo. Our work provides the first experimental proof implicating FAK in tumorigenesis, and this is associated with enhanced apoptosis.</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2004</span>;<span class="volume">18</span>;<span class="number">24</span>;<span class="pagerange">2998-3003</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15601818" rel="external">15601818</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC535910" rel="external">535910</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.316304" rel="external">10.1101/gad.316304</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The fine-scale structure of recombination rate variation in the human genome.</h4>
<p class="authors"><span class="author">McVean GA</span>, <span class="author">Myers SR</span>, <span class="author">Hunt S</span>, <span class="author">Deloukas P</span>, <span class="author">Bentley DR</span> and <span class="author">Donnelly P</span></p>
<p class="affiliation">Department of Statistics, University of Oxford, Oxford OX1 3TG, UK. <a href="mailto:%6D%63%76%65%61%6E%40%73%74%61%74%73.%6F%78.%61%63.%75%6B">&#109;&#99;&#118;&#101;&#97;&#110;&#64;&#115;&#116;&#97;&#116;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The nature and scale of recombination rate variation are largely unknown for most species. In humans, pedigree analysis has documented variation at the chromosomal level, and sperm studies have identified specific hotspots in which crossing-over events cluster. To address whether this picture is representative of the genome as a whole, we have developed and validated a method for estimating recombination rates from patterns of genetic variation. From extensive single-nucleotide polymorphism surveys in European and African populations, we find evidence for extreme local rate variation spanning four orders in magnitude, in which 50% of all recombination events take place in less than 10% of the sequence. We demonstrate that recombination hotspots are a ubiquitous feature of the human genome, occurring on average every 200 kilobases or less, but recombination occurs preferentially outside genes.</p>
<p class="publication"><span class="title">Science (New York, N.Y.)</span>&nbsp;<span class="year">2004</span>;<span class="volume">304</span>;<span class="number">5670</span>;<span class="pagerange">581-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15105499">15105499</a>; DOI: <a href="http://dx.doi.org/10.1126%2Fscience.1092500">10.1126/science.1092500</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene structure conservation aids similarity based gene prediction.</h4>
<p class="authors"><span class="author">Meyer IM</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6D%65%79%65%72%40%73%74%61%74%73.%6F%78.%61%63.%75%6B">&#109;&#101;&#121;&#101;&#114;&#64;&#115;&#116;&#97;&#116;&#115;&#46;&#111;&#120;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">One of the primary tasks in deciphering the functional contents of a newly sequenced genome is the identification of its protein coding genes. Existing computational methods for gene prediction include ab initio methods which use the DNA sequence itself as the only source of information, comparative methods using multiple genomic sequences, and similarity based methods which employ the cDNA or protein sequences of related genes to aid the gene prediction. We present here an algorithm implemented in a computer program called Projector which combines comparative and similarity approaches. Projector employs similarity information at the genomic DNA level by directly using known genes annotated on one DNA sequence to predict the corresponding related genes on another DNA sequence. It therefore makes explicit use of the conservation of the exon-intron structure between two related genes in addition to the similarity of their encoded amino acid sequences. We evaluate the performance of Projector by comparing it with the program Genewise on a test set of 491 pairs of independently confirmed mouse and human genes. It is more accurate than Genewise for genes whose proteins are &lt;80% identical, and is suitable for use in a combined gene prediction system where other methods identify well conserved and non-conserved genes, and pseudogenes.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">2</span>;<span class="pagerange">776-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14764925">14764925</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC373336">373336</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh211">10.1093/nar/gkh211</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Transferable antibiotic resistance elements in Haemophilus influenzae share a common evolutionary origin with a diverse family of syntenic genomic islands.</h4>
<p class="authors"><span class="author">Mohd-Zain Z</span>, <span class="author">Turner SL</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Lilley AK</span>, <span class="author">Inzana TJ</span>, <span class="author">Duncan AJ</span>, <span class="author">Harding RM</span>, <span class="author">Hood DW</span>, <span class="author">Peto TE</span> and <span class="author">Crook DW</span></p>
<p class="affiliation">Infectious Diseases and Clinical Microbiology, John Radcliffe Hospital, University of Oxford, Headington, Oxford, OX3 9DU, UK.</p><p class="abstract">Transferable antibiotic resistance in Haemophilus influenzae was first detected in the early 1970s. After this, resistance spread rapidly worldwide and was shown to be transferred by a large 40- to 60-kb conjugative element. Bioinformatics analysis of the complete sequence of a typical H. influenzae conjugative resistance element, ICEHin1056, revealed the shared evolutionary origin of this element. ICEHin1056 has homology to 20 contiguous sequences in the National Center for Biotechnology Information database. Systematic comparison of these homologous sequences resulted in identification of a conserved syntenic genomic island consisting of up to 33 core genes in 16 beta- and gamma-Proteobacteria. These diverse genomic islands shared a common evolutionary origin, insert into tRNA genes, and have diverged widely, with G+C contents ranging from 40 to 70% and amino acid homologies as low as 20 to 25% for shared core genes. These core genes are likely to account for the conjugative transfer of the genomic islands and may even encode autonomous replication. Accessory gene clusters were nestled among the core genes and encode the following diverse major attributes: antibiotic, metal, and antiseptic resistance; degradation of chemicals; type IV secretion systems; two-component signaling systems; Vi antigen capsule synthesis; toxin production; and a wide range of metabolic functions. These related genomic islands include the following well-characterized structures: SPI-7, found in Salmonella enterica serovar Typhi; PAP1 or pKLC102, found in Pseudomonas aeruginosa; and the clc element, found in Pseudomonas sp. strain B13. This is the first report of a diverse family of related syntenic genomic islands with a deep evolutionary origin, and our findings challenge the view that genomic islands consist only of independently evolving modules.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">R01-AI45091</span></p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2004</span>;<span class="volume">186</span>;<span class="number">23</span>;<span class="pagerange">8114-22</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15547285" rel="external">15547285</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC529066" rel="external">529066</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.186.23.8114-8122.2004" rel="external">10.1128/JB.186.23.8114-8122.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Position effect on PLP1 may cause a subset of Pelizaeus-Merzbacher disease symptoms.</h4>
<p class="authors"><span class="author">Muncke N</span>, <span class="author">Wogatzky BS</span>, <span class="author">Breuning M</span>, <span class="author">Sistermans EA</span>, <span class="author">Endris V</span>, <span class="author">Ross M</span>, <span class="author">Vetrie D</span>, <span class="author">Catsman-Berrevoets CE</span> and <span class="author">Rappold G</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">12</span>;<span class="pagerange">e121</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15591263" rel="external">15591263</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735635" rel="external">1735635</a>; DOI: <a href="http://dx.doi.org/10.1136%2Fjmg.2004.019141" rel="external">10.1136/jmg.2004.019141</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Interaction between differentially methylated regions partitions the imprinted genes Igf2 and H19 into parent-specific chromatin loops.</h4>
<p class="authors"><span class="author">Murrell A</span>, <span class="author">Heeson S</span> and <span class="author">Reik W</span></p>
<p class="affiliation">Laboratory of Developmental Genetics and Imprinting, Developmental Genetics Programme, The Babraham Institute, Cambridge CB2 4AT, UK. <a href="mailto:%61%6D%6D%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#109;&#109;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Imprinted genes are expressed from only one of the parental alleles and are marked epigenetically by DNA methylation and histone modifications. The paternally expressed gene insulin-like growth-factor 2 (Igf2) is separated by approximately 100 kb from the maternally expressed noncoding gene H19 on mouse distal chromosome 7. Differentially methylated regions in Igf2 and H19 contain chromatin boundaries, silencers and activators and regulate the reciprocal expression of the two genes in a methylation-sensitive manner by allowing them exclusive access to a shared set of enhancers. Various chromatin models have been proposed that separate Igf2 and H19 into active and silent domains. Here we used a GAL4 knock-in approach as well as the chromosome conformation capture technique to show that the differentially methylated regions in the imprinted genes Igf2 and H19 interact in mice. These interactions are epigenetically regulated and partition maternal and paternal chromatin into distinct loops. This generates a simple epigenetic switch for Igf2 through which it moves between an active and a silent chromatin domain.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">8</span>;<span class="pagerange">889-93</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15273689">15273689</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1402">10.1038/ng1402</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A clinical, microbiological, and pathological study of intestinal perforation associated with typhoid fever.</h4>
<p class="authors"><span class="author">Nguyen QC</span>, <span class="author">Everest P</span>, <span class="author">Tran TK</span>, <span class="author">House D</span>, <span class="author">Murch S</span>, <span class="author">Parry C</span>, <span class="author">Connerton P</span>, <span class="author">Phan VB</span>, <span class="author">To SD</span>, <span class="author">Mastroeni P</span>, <span class="author">White NJ</span>, <span class="author">Tran TH</span>, <span class="author">Vo VH</span>, <span class="author">Dougan G</span>, <span class="author">Farrar JJ</span> and <span class="author">Wain J</span></p>
<p class="affiliation">Dong Thap Provincial Hospital, Dong Thap, Ho Chi Minh City, Vietnam.</p><p class="abstract">One of the most serious complications of typhoid fever is intestinal perforation. Of 27 patients admitted to a provincial hospital in the Mekong Delta region of Vietnam who had gastrointestinal perforation secondary to suspected typhoid fever, 67% were male, with a median age of 23 years and a median duration of illness of 10 days. Salmonella enterica subspecies enterica serotype Typhi (S. Typhi) was isolated from 11 (41%) of 27 patients; of 27 patients, only 4 (15%) had positive cultures from gut biopsies. S. Typhi DNA was detected by polymerase chain reaction for all perforation biopsy samples. Detailed histological examination of the gastrointestinal mucosa at the site of perforation in all cases showed a combination of discrete acute and chronic inflammation. Acute inflammation at the serosal surface indicated additional tissue damage after perforation. Immunohistochemical results showed that the predominant infiltrating cell types at the site of perforation were CD68+ leukocytes (macrophages) or CD3+ leukocytes (T lymphocytes).</p>
<p class="publication"><span class="title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">39</span>;<span class="number">1</span>;<span class="pagerange">61-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15206054" rel="external">15206054</a>; DOI: <a href="http://dx.doi.org/10.1086%2F421555" rel="external">10.1086/421555</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Eukaryotes: not beyond compare.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">Bentley S</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">11</span>;<span class="pagerange">856-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15523762">15523762</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro1028">10.1038/nrmicro1028</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Strength in diversity.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">Crossman L</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga A</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK. <a href="mailto:%6D%69%63%72%6F%62%65%73%40%73%61%6E%67%65%72.%61%63.%75%6B">&#109;&#105;&#99;&#114;&#111;&#98;&#101;&#115;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">5</span>;<span class="pagerange">358-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15134078">15134078</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro889">10.1038/nrmicro889</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Insight into the genome of Aspergillus fumigatus: analysis of a 922 kb region encompassing the nitrate assimilation gene cluster.</h4>
<p class="authors"><span class="author">Pain A</span>, <span class="author">Woodward J</span>, <span class="author">Quail MA</span>, <span class="author">Anderson MJ</span>, <span class="author">Clark R</span>, <span class="author">Collins M</span>, <span class="author">Fosker N</span>, <span class="author">Fraser A</span>, <span class="author">Harris D</span>, <span class="author">Larke N</span>, <span class="author">Murphy L</span>, <span class="author">Humphray S</span>, <span class="author">O&#39;Neil S</span>, <span class="author">Pertea M</span>, <span class="author">Price C</span>, <span class="author">Rabbinowitsch E</span>, <span class="author">Rajandream MA</span>, <span class="author">Salzberg S</span>, <span class="author">Saunders D</span>, <span class="author">Seeger K</span>, <span class="author">Sharp S</span>, <span class="author">Warren T</span>, <span class="author">Denning DW</span>, <span class="author">Barrell B</span> and <span class="author">Hall N</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Aspergillus fumigatus is the most ubiquitous opportunistic filamentous fungal pathogen of human. As an initial step toward sequencing the entire genome of A. fumigatus, which is estimated to be approximately 30 Mb in size, we have sequenced a 922 kb region, contained within 16 overlapping bacterial artificial chromosome (BAC) clones. Fifty-four percent of the DNA is predicted to be coding with 341 putative protein coding genes. Functional classification of the proteins showed the presence of a higher proportion of enzymes and membrane transporters when compared to those of Saccharomyces cerevisiae. In addition to the nitrate assimilation gene cluster, the quinate utilisation gene cluster is also present on this 922 kb genomic sequence. We observed large scale synteny between A. fumigatus and Aspergillus nidulans by comparing this sequence to the A. nidulans genetic map of linkage group VIII.</p>
<p class="publication"><span class="title">Fungal genetics and biology : FG &amp; B</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">443-53</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14998527" rel="external">14998527</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.fgb.2003.12.003" rel="external">10.1016/j.fgb.2003.12.003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A transcriptomic analysis of the phylum Nematoda.</h4>
<p class="authors"><span class="author">Parkinson J</span>, <span class="author">Mitreva M</span>, <span class="author">Whitton C</span>, <span class="author">Thomson M</span>, <span class="author">Daub J</span>, <span class="author">Martin J</span>, <span class="author">Schmid R</span>, <span class="author">Hall N</span>, <span class="author">Barrell B</span>, <span class="author">Waterston RH</span>, <span class="author">McCarter JP</span> and <span class="author">Blaxter ML</span></p>
<p class="affiliation">Hospital for Sick Children, 555 University Avenue, Departments of Biochemistry and Medical Genetics and Microbiology, University of Toronto, Toronto, Ontario M5G 1X8, Canada. <a href="mailto:%6A%70%61%72%6B%69%6E%40%73%69%63%6B%6B%69%64%73.%63%61">&#106;&#112;&#97;&#114;&#107;&#105;&#110;&#64;&#115;&#105;&#99;&#107;&#107;&#105;&#100;&#115;&#46;&#99;&#97;</a></p><p class="abstract">The phylum Nematoda occupies a huge range of ecological niches, from free-living microbivores to human parasites. We analyzed the genomic biology of the phylum using 265,494 expressed-sequence tag sequences, corresponding to 93,645 putative genes, from 30 species, including 28 parasites. From 35% to 70% of each species&#39; genes had significant similarity to proteins from the model nematode Caenorhabditis elegans. More than half of the putative genes were unique to the phylum, and 23% were unique to the species from which they were derived. We have not yet come close to exhausting the genomic diversity of the phylum. We identified more than 2,600 different known protein domains, some of which had differential abundances between major taxonomic groups of nematodes. We also defined 4,228 nematode-specific protein families from nematode-restricted genes: this class of genes probably underpins species- and higher-level taxonomic disparity. Nematode-specific families are particularly interesting as drug and vaccine targets.</p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">12</span>;<span class="pagerange">1259-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15543149">15543149</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1472">10.1038/ng1472</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Gene expression profiling in the myelodysplastic syndromes using cDNA microarray technology.</h4>
<p class="authors"><span class="author">Pellagatti A</span>, <span class="author">Esoof N</span>, <span class="author">Watkins F</span>, <span class="author">Langford CF</span>, <span class="author">Vetrie D</span>, <span class="author">Campbell LJ</span>, <span class="author">Fidler C</span>, <span class="author">Cavenagh JD</span>, <span class="author">Eagleton H</span>, <span class="author">Gordon P</span>, <span class="author">Woodcock B</span>, <span class="author">Pushkaran B</span>, <span class="author">Kwan M</span>, <span class="author">Wainscoat JS</span> and <span class="author">Boultwood J</span></p>
<p class="affiliation">Leukaemia Research Fund Molecular Haematology Unit, Nuffield Department of Clinical Laboratory Sciences, John Radcliffe Hospital, Oxford, UK.</p><p class="abstract">The myelodysplastic syndromes (MDS) comprise a heterogeneous group of clonal disorders of the haematopoietic stem cell and primarily involve cells of the myeloid lineage. Using cDNA microarrays comprising 6000 human genes, we studied the gene expression profiles in the neutrophils of 21 MDS patients, seven of which had the 5q- syndrome, and two acute myeloid leukaemia (AML) patients when compared with the neutrophils from pooled healthy controls. Data analysis showed a high level of heterogeneity of gene expression between MDS patients, most probably reflecting the underlying karyotypic and genetic heterogeneity. Nevertheless, several genes were commonly up or down-regulated in MDS. The most up-regulated genes included RAB20, ARG1, ZNF183 and ACPL. The RAB20 gene is a member of the Ras gene superfamily and ARG1 promotes cellular proliferation. The most down-regulated genes include COX2, CD18, FOS and IL7R. COX2 is anti-apoptotic and promotes cell survival. Many genes were identified that are differentially expressed in the different MDS subtypes and AML. A subset of genes was able to discriminate patients with the 5q- syndrome from patients with refractory anaemia and a normal karyotype. The microarray expression results for several genes were confirmed by real-time quantitative polymerase chain reaction. The MDS-specific expression changes identified are likely to be biologically important in the pathophysiology of this disorder.</p>
<p class="publication"><span class="title">British journal of haematology</span>&nbsp;<span class="year">2004</span>;<span class="volume">125</span>;<span class="number">5</span>;<span class="pagerange">576-83</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15147372" rel="external">15147372</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2141.2004.04958.x" rel="external">10.1111/j.1365-2141.2004.04958.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ensembl analysis pipeline.</h4>
<p class="authors"><span class="author">Potter SC</span>, <span class="author">Clarke L</span>, <span class="author">Curwen V</span>, <span class="author">Keenan S</span>, <span class="author">Mongin E</span>, <span class="author">Searle SM</span>, <span class="author">Stabenau A</span>, <span class="author">Storey R</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK.</p><p class="abstract">The Ensembl pipeline is an extension to the Ensembl system which allows automated annotation of genomic sequence. The software comprises two parts. First, there is a set of Perl modules (&quot;Runnables&quot; and &quot;RunnableDBs&quot;) which are &#39;wrappers&#39; for a variety of commonly used analysis tools. These retrieve sequence data from a relational database, run the analysis, and write the results back to the database. They inherit from a common interface, which simplifies the writing of new wrapper modules. On top of this sits a job submission system (the &quot;RuleManager&quot;) which allows efficient and reliable submission of large numbers of jobs to a compute farm. Here we describe the fundamental software components of the pipeline, and we also highlight some features of the Sanger installation which were necessary to enable the pipeline to scale to whole-genome analysis.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">934-41</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123589" rel="external">15123589</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479123" rel="external">479123</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1859804" rel="external">10.1101/gr.1859804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The bordetellae: lessons from genomics.</h4>
<p class="authors"><span class="author">Preston A</span>, <span class="author">Parkhill J</span> and <span class="author">Maskell DJ</span></p>
<p class="affiliation">Department of Microbiology, University of Guelph, Guelph, Ontario N1G 2W1, Canada. <a href="mailto:%61%70%72%65%73%74%6F%6E%40%75%6F%67%75%65%6C%70%68.%63%61">&#97;&#112;&#114;&#101;&#115;&#116;&#111;&#110;&#64;&#117;&#111;&#103;&#117;&#101;&#108;&#112;&#104;&#46;&#99;&#97;</a></p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">5</span>;<span class="pagerange">379-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15100691">15100691</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro886">10.1038/nrmicro886</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Where west meets east: the complex mtDNA landscape of the southwest and Central Asian corridor.</h4>
<p class="authors"><span class="author">Quintana-Murci L</span>, <span class="author">Chaix R</span>, <span class="author">Wells RS</span>, <span class="author">Behar DM</span>, <span class="author">Sayar H</span>, <span class="author">Scozzari R</span>, <span class="author">Rengo C</span>, <span class="author">Al-Zahery N</span>, <span class="author">Semino O</span>, <span class="author">Santachiara-Benerecetti AS</span>, <span class="author">Coppa A</span>, <span class="author">Ayub Q</span>, <span class="author">Mohyuddin A</span>, <span class="author">Tyler-Smith C</span>, <span class="author">Qasim Mehdi S</span>, <span class="author">Torroni A</span> and <span class="author">McElreavey K</span></p>
<p class="affiliation">Centre National de la Recherche Scientifique (CNRS) URA 1961, Institut Pasteur, 75724 Paris Cedex 15, France. <a href="mailto:%71%75%69%6E%74%61%6E%61%40%70%61%73%74%65%75%72.%66%72">&#113;&#117;&#105;&#110;&#116;&#97;&#110;&#97;&#64;&#112;&#97;&#115;&#116;&#101;&#117;&#114;&#46;&#102;&#114;</a></p><p class="abstract">The southwestern and Central Asian corridor has played a pivotal role in the history of humankind, witnessing numerous waves of migration of different peoples at different times. To evaluate the effects of these population movements on the current genetic landscape of the Iranian plateau, the Indus Valley, and Central Asia, we have analyzed 910 mitochondrial DNAs (mtDNAs) from 23 populations of the region. This study has allowed a refinement of the phylogenetic relationships of some lineages and the identification of new haplogroups in the southwestern and Central Asian mtDNA tree. Both lineage geographical distribution and spatial analysis of molecular variance showed that populations located west of the Indus Valley mainly harbor mtDNAs of western Eurasian origin, whereas those inhabiting the Indo-Gangetic region and Central Asia present substantial proportions of lineages that can be allocated to three different genetic components of western Eurasian, eastern Eurasian, and south Asian origin. In addition to the overall composite picture of lineage clusters of different origin, we observed a number of deep-rooting lineages, whose relative clustering and coalescent ages suggest an autochthonous origin in the southwestern Asian corridor during the Pleistocene. The comparison with Y-chromosome data revealed a highly complex genetic and demographic history of the region, which includes sexually asymmetrical mating patterns, founder effects, and female-specific traces of the East African slave trade.</p>
<p class="grants">Funded by: Telethon: <span class="grant">E.0890</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">74</span>;<span class="number">5</span>;<span class="pagerange">827-45</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15077202" rel="external">15077202</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1181978" rel="external">1181978</a>; DOI: <a href="http://dx.doi.org/10.1086%2F383236" rel="external">10.1086/383236</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA methylation profiling of the human major histocompatibility complex: a pilot study for the human epigenome project.</h4>
<p class="authors"><span class="author">Rakyan VK</span>, <span class="author">Hildmann T</span>, <span class="author">Novik KL</span>, <span class="author">Lewin J</span>, <span class="author">Tost J</span>, <span class="author">Cox AV</span>, <span class="author">Andrews TD</span>, <span class="author">Howe KL</span>, <span class="author">Otto T</span>, <span class="author">Olek A</span>, <span class="author">Fischer J</span>, <span class="author">Gut IG</span>, <span class="author">Berlin K</span> and <span class="author">Beck S</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge, United Kingdom.</p><p class="abstract">The Human Epigenome Project aims to identify, catalogue, and interpret genome-wide DNA methylation phenomena. Occurring naturally on cytosine bases at cytosine-guanine dinucleotides, DNA methylation is intimately involved in diverse biological processes and the aetiology of many diseases. Differentially methylated cytosines give rise to distinct profiles, thought to be specific for gene activity, tissue type, and disease state. The identification of such methylation variable positions will significantly improve our understanding of genome biology and our ability to diagnose disease. Here, we report the results of the pilot study for the Human Epigenome Project entailing the methylation analysis of the human major histocompatibility complex. This study involved the development of an integrated pipeline for high-throughput methylation analysis using bisulphite DNA sequencing, discovery of methylation variable positions, epigenotyping by matrix-assisted laser desorption/ionisation mass spectrometry, and development of an integrated public database available at http://www.epigenome.org. Our analysis of DNA methylation levels within the major histocompatibility complex, including regulatory exonic and intronic regions associated with 90 genes in multiple tissues and individuals, reveals a bimodal distribution of methylation profiles (i.e., the vast majority of the analysed regions were either hypo- or hypermethylated), tissue specificity, inter-individual variation, and correlation with independent gene expression data.</p>
<p class="publication"><span class="title">PLoS biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">12</span>;<span class="pagerange">e405</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15550986">15550986</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC529316">529316</a>; DOI: <a href="http://dx.doi.org/10.1371%2Fjournal.pbio.0020405">10.1371/journal.pbio.0020405</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary pressures on apicoplast transit peptides.</h4>
<p class="authors"><span class="author">Ralph SA</span>, <span class="author">Foth BJ</span>, <span class="author">Hall N</span> and <span class="author">McFadden GI</span></p>
<p class="affiliation">Plant Cell Biology Research Centre, School of Botany, University of Melbourne, Parkville, Victoria, Australia.</p><p class="abstract">Malaria parasites (species of the genus Plasmodium) harbor a relict chloroplast (the apicoplast) that is the target of novel antimalarials. Numerous nuclear-encoded proteins are translocated into the apicoplast courtesy of a bipartite N-terminal extension. The first component of the bipartite leader resembles a standard signal peptide present at the N-terminus of secreted proteins that enter the endomembrane system. Analysis of the second portion of the bipartite leaders of P. falciparum, the so-called transit peptide, indicates similarities to plant transit peptides, although the amino acid composition of P. falciparum transit peptides shows a strong bias, which we rationalize by the extraordinarily high AT content of P. falciparum DNA. 786 plastid transit peptides were also examined from several other apicomplexan parasites, as well as from angiosperm plants. In each case, amino acid biases were correlated with nucleotide AT content. A comparison of a spectrum of organisms containing primary and secondary plastids also revealed features unique to secondary plastid transit peptides. These unusual features are explained in the context of secondary plastid trafficking via the endomembrane system.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI05093</span></p>
<p class="publication"><span class="title">Molecular biology and evolution</span>&nbsp;<span class="year">2004</span>;<span class="volume">21</span>;<span class="number">12</span>;<span class="pagerange">2183-94</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15317876" rel="external">15317876</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fmolbev%2Fmsh233" rel="external">10.1093/molbev/msh233</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Evolutionary families of peptidase inhibitors.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Tolle DP</span> and <span class="author">Barrett AJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The proteins that inhibit peptidases are of great importance in medicine and biotechnology, but there has never been a comprehensive system of classification for them. Some of the terminology currently in use is potentially confusing. In the hope of facilitating the exchange, storage and retrieval of information about this important group of proteins, we now describe a system wherein the inhibitor units of the peptidase inhibitors are assigned to 48 families on the basis of similarities detectable at the level of amino acid sequence. Then, on the basis of three-dimensional structures, 31 of the families are assigned to 26 clans. A simple system of nomenclature is introduced for reference to each clan, family and inhibitor. We briefly discuss the specificities and mechanisms of the interactions of the inhibitors in the various families with their target enzymes. The system of families and clans of inhibitors described has been implemented in the MEROPS peptidase database (http://merops.sanger.ac.uk/), and this will provide a mechanism for updating it as new information becomes available.</p>
<p class="publication"><span class="title">The Biochemical journal</span>&nbsp;<span class="year">2004</span>;<span class="volume">378</span>;<span class="number">Pt 3</span>;<span class="pagerange">705-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14705960" rel="external">14705960</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1224039" rel="external">1224039</a>; DOI: <a href="http://dx.doi.org/10.1042%2FBJ20031825" rel="external">10.1042/BJ20031825</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">MEROPS: the peptidase database.</h4>
<p class="authors"><span class="author">Rawlings ND</span>, <span class="author">Tolle DP</span> and <span class="author">Barrett AJ</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%6E%64%72%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#100;&#114;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Peptidases (proteolytic enzymes) are of great relevance to biology, medicine and biotechnology. This practical importance creates a need for an integrated source of information about them, and also about their natural inhibitors. The MEROPS database (http://merops.sanger.ac.uk) aims to fill this need. The organizational principle of the database is a hierarchical classification in which homologous sets of the proteins of interest are grouped in families and the homologous families are grouped in clans. Each peptidase, family and clan has a unique identifier. The database has recently been expanded to include the protein inhibitors of peptidases, and these are classified in much the same way as the peptidases. Forms of information recently added include new links to other databases, summary alignments for peptidase clans, displays to show the distribution of peptidases and inhibitors among organisms, substrate cleavage sites and indexes for expressed sequence tag libraries containing peptidases. A new way of making hyperlinks to the database has been devised and a BlastP search of our library of peptidase and inhibitor sequences has been added.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D160-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681384" rel="external">14681384</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308805" rel="external">308805</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh071" rel="external">10.1093/nar/gkh071</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Chromosome loops, insulators, and histone methylation: new insights into regulation of imprinting in clusters.</h4>
<p class="authors"><span class="author">Reik W</span>, <span class="author">Murrell A</span>, <span class="author">Lewis A</span>, <span class="author">Mitsuya K</span>, <span class="author">Umlauf D</span>, <span class="author">Dean W</span>, <span class="author">Higgins M</span> and <span class="author">Feil R</span></p>
<p class="affiliation">Laboratory of Developmental Genetics and Imprinting, The Babraham Institute, Cambridge CB2 4AT, United Kingdom. <a href="mailto:%77%6F%6C%66.%72%65%69%6B%40%62%62%73%72%63.%61%63.%75%6B">&#119;&#111;&#108;&#102;&#46;&#114;&#101;&#105;&#107;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Cold Spring Harbor symposia on quantitative biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">69</span>;<span class="pagerange">29-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/16117630" rel="external">16117630</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fsqb.2004.69.29" rel="external">10.1101/sqb.2004.69.29</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A Myo7a mutation cosegregates with stereocilia defects and low-frequency hearing impairment.</h4>
<p class="authors"><span class="author">Rhodes CR</span>, <span class="author">Hertzano R</span>, <span class="author">Fuchs H</span>, <span class="author">Bell RE</span>, <span class="author">de Angelis MH</span>, <span class="author">Steel KP</span> and <span class="author">Avraham KB</span></p>
<p class="affiliation">MRC Institute of Hearing Research, University Park, NG7 2RD, Nottingham, UK.</p><p class="abstract">A phenotype-driven approach was adopted in the mouse to identify molecules involved in ear development and function. Mutant mice were obtained using N-ethyl- N-nitrosourea (ENU) mutagenesis and were screened for dominant mutations that affect hearing and/or balance. Heterozygote headbanger ( Hdb/+) mutants display classic behavior indicative of vestibular dysfunction including hyperactivity and head bobbing, and they show a Preyer reflex in response to sound but have raised cochlear thresholds especially at low frequencies. Scanning electron microscopy of the surface of the organ of Corti revealed abnormal stereocilia bundle development from an early age that was more severe in the apex than the base. Utricular stereocilia were long, thin, and wispy. Homozygotes showed a similar but more severe phenotype. The headbanger mutation has been mapped to a 1.5-cM region on mouse Chromosome 7 in the region of the unconventional myosin gene Myo7a, and mutation screening revealed an A&gt;T transversion that is predicted to cause an isoleucine-to-phenylalanine amino acid substitution (I178F) in a conserved region in the motor-encoding domain of the gene. Protein analysis revealed reduced levels of myosin VIIa expression in inner ears of headbanger mice. Headbanger represents a novel inner ear phenotype and provides a potential model for low-frequency-type human hearing loss.</p>
<p class="publication"><span class="title">Mammalian genome : official journal of the International Mammalian Genome Society</span>&nbsp;<span class="year">2004</span>;<span class="volume">15</span>;<span class="number">9</span>;<span class="pagerange">686-97</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15389316" rel="external">15389316</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00335-004-2344-x" rel="external">10.1007/s00335-004-2344-x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete nucleotide sequence of the conjugative tetracycline resistance plasmid pFBAOT6, a member of a group of IncU plasmids with global ubiquity.</h4>
<p class="authors"><span class="author">Rhodes G</span>, <span class="author">Parkhill J</span>, <span class="author">Bird C</span>, <span class="author">Ambrose K</span>, <span class="author">Jones MC</span>, <span class="author">Huys G</span>, <span class="author">Swings J</span> and <span class="author">Pickup RW</span></p>
<p class="affiliation">Centre for Ecology and Hydrology, Lancaster, United Kingdom. <a href="mailto:%67%6C%65%6E%6E%40%63%65%68.%61%63.%75%6B">&#103;&#108;&#101;&#110;&#110;&#64;&#99;&#101;&#104;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">This study presents the first complete sequence of an IncU plasmid, pFBAOT6. This plasmid was originally isolated from a strain of Aeromonas caviae from hospital effluent (Westmorland General Hospital, Kendal, United Kingdom) in September 1997 (G. Rhodes, G. Huys, J. Swings, P. McGann, M. Hiney, P. Smith, and R. W. Pickup, Appl. Environ. Microbiol. 66:3883-3890, 2000) and belongs to a group of related plasmids with global ubiquity. pFBAOT6 is 84,748 bp long and has 94 predicted coding sequences, only 12 of which do not have a possible function that has been attributed. Putative replication, maintenance, and transfer functions have been identified and are located in a region in the first 31 kb of the plasmid. The replication region is poorly understood but exhibits some identity at the protein level with replication proteins from the gram-positive bacteria Bacillus and Clostridium. The mating pair formation system is a virB homologue, type IV secretory pathway that is similar in its structural organization to the mating pair formation systems of the related broad-host-range (BHR) environmental plasmids pIPO2, pXF51, and pSB102 from plant-associated bacteria. Partitioning and maintenance genes are homologues of genes in IncP plasmids. The DNA transfer genes and the putative oriT site also exhibit high levels of similarity with those of plasmids pIPO2, pXF51, and pSB102. The genetic load region encompasses 54 kb, comprises the resistance genes, and includes a class I integron, an IS630 relative, and other transposable elements in a 43-kb region that may be a novel Tn1721-flanked composite transposon. This region also contains 24 genes that exhibit the highest levels of identity to chromosomal genes of several plant-associated bacteria. The features of the backbone of pFBAOT6 that are shared with this newly defined group of environmental BHR plasmids suggest that pFBAOT6 may be a relative of this group, but a relative that was isolated from a clinical bacterial environment rather than a plant-associated bacterial environment.</p>
<p class="publication"><span class="title">Applied and environmental microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">70</span>;<span class="number">12</span>;<span class="pagerange">7497-510</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15574953" rel="external">15574953</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC535204" rel="external">535204</a>; DOI: <a href="http://dx.doi.org/10.1128%2FAEM.70.12.7497-7510.2004" rel="external">10.1128/AEM.70.12.7497-7510.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cloning of a new familial t(3;8) translocation associated with conventional renal cell carcinoma reveals a 5 kb microdeletion and no gene involved in the rearrangement.</h4>
<p class="authors"><span class="author">Rodr&iacute;guez-Perales S</span>, <span class="author">Mel&eacute;ndez B</span>, <span class="author">Gribble SM</span>, <span class="author">Valle L</span>, <span class="author">Carter NP</span>, <span class="author">Santamar&iacute;a I</span>, <span class="author">Conde L</span>, <span class="author">Urioste M</span>, <span class="author">Ben&iacute;tez J</span> and <span class="author">Cigudosa JC</span></p>
<p class="affiliation">Cytogenetics Unit, Biotechnology Programme, Centro Nacional de Investigaciones Oncol&oacute;gicas, Madrid, 28029, Spain.</p><p class="abstract">This study describes the molecular cloning of a familial translocation, t(3;8)(p14.2;q24.2), that segregates with the conventional renal cell carcinoma (conventional RCC). We had previously reported the family history and, through loss of heterozygosity and comparative genomic hybridization, detected the loss of the 3p chromosome arm and somatic mutation in the retained von Hippel-Lindau gene in some members of the family. With the help of array painting and sequence tagged site-PCR on flow-sorted derivative chromosomes, we have cloned the breakpoints of the translocation. We have studied the junctions on both derivative chromosomes at the genomic and expression levels. The analysis of the sequence revealed a 5 kb microdeletion at the chromosome 3 breakpoint together with a high density of repetitive motifs (Alu, short interspersed nuclear element) and an AT-rich region. Both chromosome 3 and 8 rearranged regions were very poor in gene content. We tested an expressed sequence tag, two predicted genes, one novel gene and LRIG1, a gene located more than 200 kb apart from the breakpoint on chromosome 3. None of these genes, except LRIG1, showed expression in any of the tested tissues (including normal adult and fetal kidney, sporadic kidney tumours and tumour samples from the proband&#39;s family). Taken together, all these data suggest that, rather than deregulation of specific genes that may be rearranged by the translocation, the proposed three-step model of tumour development (translocation, loss of the 3p chromosome, and mutation in a tumour suppressor gene located within that region) could be the biological mechanism that takes place in this familial form of conventional RCC.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13</span>;<span class="number">9</span>;<span class="pagerange">983-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15016767" rel="external">15016767</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh111" rel="external">10.1093/hmg/ddh111</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of mammalian microRNA host genes and transcription units.</h4>
<p class="authors"><span class="author">Rodriguez A</span>, <span class="author">Griffiths-Jones S</span>, <span class="author">Ashurst JL</span> and <span class="author">Bradley A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">To derive a global perspective on the transcription of microRNAs (miRNAs) in mammals, we annotated the genomic position and context of this class of noncoding RNAs (ncRNAs) in the human and mouse genomes. Of the 232 known mammalian miRNAs, we found that 161 overlap with 123 defined transcription units (TUs). We identified miRNAs within introns of 90 protein-coding genes with a broad spectrum of molecular functions, and in both introns and exons of 66 mRNA-like noncoding RNAs (mlncRNAs). In addition, novel families of miRNAs based on host gene identity were identified. The transcription patterns of all miRNA host genes were curated from a variety of sources illustrating spatial, temporal, and physiological regulation of miRNA expression. These findings strongly suggest that miRNAs are transcribed in parallel with their host transcripts, and that the two different transcription classes of miRNAs (&#39;exonic&#39; and &#39;intronic&#39;) identified here may require slightly different mechanisms of biogenesis.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">10A</span>;<span class="pagerange">1902-10</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15364901">15364901</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC524413">524413</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2722704">10.1101/gr.2722704</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tetrasomy 21pter--&gt;q21.2 in a male infant without typical Down&#39;s syndrome dysmorphic features but moderate mental retardation.</h4>
<p class="authors"><span class="author">Rost I</span>, <span class="author">Fiegler H</span>, <span class="author">Fauth C</span>, <span class="author">Carr P</span>, <span class="author">Bettecken T</span>, <span class="author">Kraus J</span>, <span class="author">Meyer C</span>, <span class="author">Enders A</span>, <span class="author">Wirtz A</span>, <span class="author">Meitinger T</span>, <span class="author">Carter NP</span> and <span class="author">Speicher MR</span></p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">3</span>;<span class="pagerange">e26</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14985397" rel="external">14985397</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735700" rel="external">1735700</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Periodic gene expression program of the fission yeast cell cycle.</h4>
<p class="authors"><span class="author">Rustici G</span>, <span class="author">Mata J</span>, <span class="author">Kivinen K</span>, <span class="author">Li&oacute; P</span>, <span class="author">Penkett CJ</span>, <span class="author">Burns G</span>, <span class="author">Hayles J</span>, <span class="author">Brazma A</span>, <span class="author">Nurse P</span> and <span class="author">B&auml;hler J</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Cell-cycle control of transcription seems to be universal, but little is known about its global conservation and biological significance. We report on the genome-wide transcriptional program of the Schizosaccharomyces pombe cell cycle, identifying 407 periodically expressed genes of which 136 show high-amplitude changes. These genes cluster in four major waves of expression. The forkhead protein Sep1p regulates mitotic genes in the first cluster, including Ace2p, which activates transcription in the second cluster during the M-G1 transition and cytokinesis. Other genes in the second cluster, which are required for G1-S progression, are regulated by the MBF complex independently of Sep1p and Ace2p. The third cluster coincides with S phase and a fourth cluster contains genes weakly regulated during G2 phase. Despite conserved cell-cycle transcription factors, differences in regulatory circuits between fission and budding yeasts are evident, revealing evolutionary plasticity of transcriptional control. Periodic transcription of most genes is not conserved between the two yeasts, except for a core set of approximately 40 genes that seem to be universally regulated during the eukaryotic cell cycle and may have key roles in cell-cycle progression.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">8</span>;<span class="pagerange">809-17</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15195092">15195092</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1377">10.1038/ng1377</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Methylation of histone H4 lysine 20 controls recruitment of Crb2 to sites of DNA damage.</h4>
<p class="authors"><span class="author">Sanders SL</span>, <span class="author">Portoso M</span>, <span class="author">Mata J</span>, <span class="author">B&auml;hler J</span>, <span class="author">Allshire RC</span> and <span class="author">Kouzarides T</span></p>
<p class="affiliation">The Wellcome Trust/Cancer Research UK Gurdon Institute and Department of Pathology, Tennis Court Road, Cambridge CB2 1QN, United Kingdom.</p><p class="abstract">Histone lysine methylation is a key regulator of gene expression and heterochromatin function, but little is known as to how this modification impinges on other chromatin activities. Here we demonstrate that a previously uncharacterized SET domain protein, Set9, is responsible for H4-K20 methylation in the fission yeast Schizosaccharomyces pombe. Surprisingly, H4-K20 methylation does not have any apparent role in the regulation of gene expression or heterochromatin function. Rather, we find the modification has a role in DNA damage response. Loss of Set9 activity or mutation of H4-K20 markedly impairs cell survival after genotoxic challenge and compromises the ability of cells to maintain checkpoint mediated cell cycle arrest. Genetic experiments link Set9 to Crb2, a homolog of the mammalian checkpoint protein 53BP1, and the enzyme is required for Crb2 localization to sites of DNA damage. These results argue that H4-K20 methylation functions as a &quot;histone mark&quot; required for the recruitment of the checkpoint protein Crb2.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">119</span>;<span class="number">5</span>;<span class="pagerange">603-14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15550243">15550243</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cell.2004.11.009">10.1016/j.cell.2004.11.009</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Folate-sensitive fragile site FRA10A is due to an expansion of a CGG repeat in a novel gene, FRA10AC1, encoding a nuclear protein.</h4>
<p class="authors"><span class="author">Sarafidou T</span>, <span class="author">Kahl C</span>, <span class="author">Martinez-Garay I</span>, <span class="author">Mangelsdorf M</span>, <span class="author">Gesk S</span>, <span class="author">Baker E</span>, <span class="author">Kokkinaki M</span>, <span class="author">Talley P</span>, <span class="author">Maltby EL</span>, <span class="author">French L</span>, <span class="author">Harder L</span>, <span class="author">Hinzmann B</span>, <span class="author">Nobile C</span>, <span class="author">Richkind K</span>, <span class="author">Finnis M</span>, <span class="author">Deloukas P</span>, <span class="author">Sutherland GR</span>, <span class="author">Kutsche K</span>, <span class="author">Moschonas NK</span>, <span class="author">Siebert R</span>, <span class="author">G&eacute;cz J</span> and <span class="author">European Collaborative Consortium for the Study of ADLTE</span></p>
<p class="affiliation">Department of Biology, University of Crete, and Institute of Molecular Biology and Biotechnology(IMBB), Foundation of Research and Technology (FORTH-GR), P.O. Box 2208, 714 09 Heraklion, Crete, Greece.</p><p class="abstract">Fragile sites appear visually as nonstaining gaps on chromosomes that are inducible by specific cell culture conditions. Expansion of CGG/CCG repeats has been shown to be the molecular basis of all five folate-sensitive fragile sites characterized molecularly so far, i.e., FRAXA, FRAXE, FRAXF, FRA11B, and FRA16A. In the present study we have refined the localization of the FRA10A folate-sensitive fragile site by fluorescence in situ hybridization. Sequence analysis of a BAC clone spanning FRA10A identified a single, imperfect, but polymorphic CGG repeat that is part of a CpG island in the 5&#39;UTR of a novel gene named FRA10AC1. The number of CGG repeats varied in the population from 8 to 13. Expansions exceeding 200 repeat units were methylated in all FRA10A fragile site carriers tested. The FRA10AC1 gene consists of 19 exons and is transcribed in the centromeric direction from the FRA10A repeat. The major transcript of approximately 1450 nt is ubiquitously expressed and codes for a highly conserved protein, FRA10AC1, of unknown function. Several splice variants leading to alternative 3&#39; ends were identified (particularly in testis). These give rise to FRA10AC1 proteins with altered COOH-termini. Immunofluorescence analysis of full-length, recombinant EGFP-tagged FRA10AC1 protein showed that it was present exclusively in the nucleoplasm. We show that the expression of FRA10A, in parallel to the other cloned folate-sensitive fragile sites, is caused by an expansion and subsequent methylation of an unstable CGG trinucleotide repeat. Taking advantage of three cSNPs within the FRA10AC1 gene we demonstrate that one allele of the gene is not transcribed in a FRA10A carrier. Our data also suggest that in the heterozygous state FRA10A is likely a benign folate-sensitive fragile site.</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">84</span>;<span class="number">1</span>;<span class="pagerange">69-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15203205" rel="external">15203205</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ygeno.2003.12.017" rel="external">10.1016/j.ygeno.2003.12.017</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The otter annotation system.</h4>
<p class="authors"><span class="author">Searle SM</span>, <span class="author">Gilbert J</span>, <span class="author">Iyer V</span> and <span class="author">Clamp M</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, The Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK.</p><p class="abstract">With the completion of the human genome sequence and genome sequence available for other vertebrate genomes, the task of manual annotation at the large genome scale has become a priority. Possibly even more important, is the requirement to curate and improve this annotation in the light of future data. For this to be possible, there is a need for tools to access and manage the annotation. Ensembl provides an excellent means for storing gene structures, genome features, and sequence, but it does not support the extra textual data necessary for manual annotation. We have extended Ensembl to create the Otter manual annotation system. This comprises a relational database schema for storing the manual annotation data, an application-programming interface (API) to access it, an extensible markup language (XML) format to allow transfer of the data, and a server to allow multiuser/multimachine access to the data. We have also written a data-adaptor plugin for the Apollo Browser/Editor to enable it to utilize an Otter server. The otter database is currently used by the Vertebrate Genome Annotation (VEGA) site (http://vega.sanger.ac.uk), which provides access to manually curated human chromosomes. Support is also being developed for using the AceDB annotation editor, FMap, via a perl wrapper called Lace. The Human and Vertebrate Annotation (HAVANA) group annotators at the Sanger center are using this to annotate human chromosomes 1 and 20.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">963-70</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123593">15123593</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479127">479127</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1864804">10.1101/gr.1864804</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DomIns: a web resource for domain insertions in known protein structures.</h4>
<p class="authors"><span class="author">Selvam RA</span> and <span class="author">Sasidharan R</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Proteins can be formed by single or multiple domains. The process of recombination at the molecular level has generated a wide variety of multi-domain proteins with specific domain organization to cater to the functional requirements of an organism. The functional and structural costs of inserting a domain into another means that multi-domain proteins are usually formed by covalently linking the N-terminus of one domain to the C-terminus of the preceding domain. While this is true in a large proportion of multi-domain proteins, we find a significant fraction of proteins that are the result of domain insertion. The inserted domain breaks the sequence contiguity of the domain into which it is inserted leading to a novel domain organization. This web resource aims to document domain insertions in known protein structures that are classified in the SCOP database. The web server can be accessed from http://stash.mrc-lmb.cam. ac.uk/DomIns/.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">2004</span>;<span class="volume">32</span>;<span class="number">Database issue</span>;<span class="pagerange">D193-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14681392" rel="external">14681392</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308781" rel="external">308781</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fnar%2Fgkh047" rel="external">10.1093/nar/gkh047</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Microarray based comparative genomic hybridisation (array-CGH) detects submicroscopic chromosomal deletions and duplications in patients with learning disability/mental retardation and dysmorphic features.</h4>
<p class="authors"><span class="author">Shaw-Smith C</span>, <span class="author">Redon R</span>, <span class="author">Rickman L</span>, <span class="author">Rio M</span>, <span class="author">Willatt L</span>, <span class="author">Fiegler H</span>, <span class="author">Firth H</span>, <span class="author">Sanlaville D</span>, <span class="author">Winter R</span>, <span class="author">Colleaux L</span>, <span class="author">Bobrow M</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">University of Cambridge Department of Medical Genetics, Addenbrooke&#39;s Hospital, Hills Road, Cambridge, UK.</p><p class="abstract">The underlying causes of learning disability and dysmorphic features in many patients remain unidentified despite extensive investigation. Routine karyotype analysis is not sensitive enough to detect subtle chromosome rearrangements (less than 5 Mb). The presence of subtle DNA copy number changes was investigated by array-CGH in 50 patients with learning disability and dysmorphism, employing a DNA microarray constructed from large insert clones spaced at approximately 1 Mb intervals across the genome. Twelve copy number abnormalities were identified in 12 patients (24% of the total): seven deletions (six apparently de novo and one inherited from a phenotypically normal parent) and five duplications (one de novo and four inherited from phenotypically normal parents). Altered segments ranged in size from those involving a single clone to regions as large as 14 Mb. No recurrent deletion or duplication was identified within this cohort of patients. On the basis of these results, we anticipate that array-CGH will become a routine method of genome-wide screening for imbalanced rearrangements in children with learning disability.</p>
<p class="publication"><span class="title">Journal of medical genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">241-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15060094">15060094</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1735726">1735726</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Comparative genomic analysis of two avian (quail and chicken) MHC regions.</h4>
<p class="authors"><span class="author">Shiina T</span>, <span class="author">Shimizu S</span>, <span class="author">Hosomichi K</span>, <span class="author">Kohara S</span>, <span class="author">Watanabe S</span>, <span class="author">Hanzawa K</span>, <span class="author">Beck S</span>, <span class="author">Kulski JK</span> and <span class="author">Inoko H</span></p>
<p class="affiliation">Department of Molecular Life Science, Division of Basic Medical Science and Molecular Medicine, Tokai University School of Medicine, Bohseidai, Isehara, Kanagawa, Japan.</p><p class="abstract">We mapped two different quail Mhc haplotypes and sequenced one of them (haplotype A) for comparative genomic analysis with a previously sequenced haplotype of the chicken Mhc. The quail haplotype A spans 180 kb of genomic sequence, encoding a total of 41 genes compared with only 19 genes within the 92-kb chicken Mhc. Except for two gene families (B30 and tRNA), both species have the same basic set of gene family members that were previously described in the chicken &quot;minimal essential&quot; Mhc. The two Mhc regions have a similar overall organization but differ markedly in that the quail has an expanded number of duplicated genes with 7 class I, 10 class IIB, 4 NK, 6 lectin, and 8 B-G genes. Comparisons between the quail and chicken Mhc class I and class II gene sequences by phylogenetic analysis showed that they were more closely related within species than between species, suggesting that the quail Mhc genes were duplicated after the separation of these two species from their common ancestor. The proteins encoded by the NK and class I genes are known to interact as ligands and receptors, but unlike in the quail and the chicken, the genes encoding these proteins in mammals are found on different chromosomes. The finding of NK-like genes in the quail Mhc strongly suggests an evolutionary connection between the NK C-type lectin-like superfamily and the Mhc, providing support for future studies on the NK, lectin, class I, and class II interaction in birds.</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2004</span>;<span class="volume">172</span>;<span class="number">11</span>;<span class="pagerange">6751-63</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15153492" rel="external">15153492</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Eimeria genome projects: a sequence of events.</h4>
<p class="authors"><span class="author">Shirley MW</span>, <span class="author">Ivens A</span>, <span class="author">Gruber A</span>, <span class="author">Madeira AM</span>, <span class="author">Wan KL</span>, <span class="author">Dear PH</span> and <span class="author">Tomley FM</span></p>
<p class="affiliation">Institute for Animal Health, Compton Laboratory, Compton, Nr Newbury, Berkshire RG20 7NN, UK. <a href="mailto:%73%68%69%72%6C%65%79%40%62%62%73%72%63.%61%63.%75%6B">&#115;&#104;&#105;&#114;&#108;&#101;&#121;&#64;&#98;&#98;&#115;&#114;&#99;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">20</span>;<span class="number">5</span>;<span class="pagerange">199-201</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15105014" rel="external">15105014</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.pt.2004.02.005" rel="external">10.1016/j.pt.2004.02.005</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A public gene trap resource for mouse functional genomics.</h4>
<p class="authors"><span class="author">Skarnes WC</span>, <span class="author">von Melchner H</span>, <span class="author">Wurst W</span>, <span class="author">Hicks G</span>, <span class="author">Nord AS</span>, <span class="author">Cox T</span>, <span class="author">Young SG</span>, <span class="author">Ruiz P</span>, <span class="author">Soriano P</span>, <span class="author">Tessier-Lavigne M</span>, <span class="author">Conklin BR</span>, <span class="author">Stanford WL</span>, <span class="author">Rossant J</span> and <span class="author">International Gene Trap Consortium</span></p>
<p class="grants">Funded by: Wellcome Trust: <span class="grant">077188</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">6</span>;<span class="pagerange">543-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15167922">15167922</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC2716026">2716026</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng0604-543">10.1038/ng0604-543</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The Ensembl Web site: mechanics of a genome browser.</h4>
<p class="authors"><span class="author">Stalker J</span>, <span class="author">Gibbins B</span>, <span class="author">Meidl P</span>, <span class="author">Smith J</span>, <span class="author">Spooner W</span>, <span class="author">Hotz HR</span> and <span class="author">Cox AV</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, CB10 1SD, UK. <a href="mailto:%4A%57%53%40%73%61%6E%67%65%72.%61%63.%75%6B">&#74;&#87;&#83;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Ensembl Web site (http://www.ensembl.org/) is the principal user interface to the data of the Ensembl project, and currently serves &gt;500,000 pages (approximately 2.5 million hits) per week, providing access to &gt;80 GB (gigabyte) of data to users in more than 80 countries. Built atop an open-source platform comprising Apache/mod_perl and the MySQL relational database management system, it is modular, extensible, and freely available. It is being actively reused and extended in several different projects, and has been downloaded and installed in companies and academic institutions worldwide. Here, we describe some of the technical features of the site, with particular reference to its dynamic configuration that enables it to handle disparate data from multiple species.</p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">5</span>;<span class="pagerange">951-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15123591">15123591</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC479125">479125</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.1863004">10.1101/gr.1863004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The notochord.</h4>
<p class="authors"><span class="author">Stemple DL</span></p>
<p class="affiliation">Vertebrate Development and Genetics (Team 31), Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%73%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">20</span>;<span class="pagerange">R873-4</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15498474" rel="external">15498474</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.cub.2004.09.065" rel="external">10.1016/j.cub.2004.09.065</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">TILLING--a high-throughput harvest for functional genomics.</h4>
<p class="authors"><span class="author">Stemple DL</span></p>
<p class="affiliation">Vertebrate Development and Genetics (Team 31), Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%73%34%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#52;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Nature reviews. Genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">5</span>;<span class="number">2</span>;<span class="pagerange">145-50</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14726927">14726927</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrg1273">10.1038/nrg1273</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Pattern formation and developmental mechanisms: super-models of development.</h4>
<p class="authors"><span class="author">Stemple DL</span> and <span class="author">Vincent JP</span></p>
<p class="publication"><span class="title">Current opinion in genetics &amp; development</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">4</span>;<span class="pagerange">325-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15261646" rel="external">15261646</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.gde.2004.06.016" rel="external">10.1016/j.gde.2004.06.016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Lung cancer: intragenic ERBB2 kinase mutations in tumours.</h4>
<p class="authors"><span class="author">Stephens P</span>, <span class="author">Hunter C</span>, <span class="author">Bignell G</span>, <span class="author">Edkins S</span>, <span class="author">Davies H</span>, <span class="author">Teague J</span>, <span class="author">Stevens C</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Smith R</span>, <span class="author">Parker A</span>, <span class="author">Barthorpe A</span>, <span class="author">Blow M</span>, <span class="author">Brackenbury L</span>, <span class="author">Butler A</span>, <span class="author">Clarke O</span>, <span class="author">Cole J</span>, <span class="author">Dicks E</span>, <span class="author">Dike A</span>, <span class="author">Drozd A</span>, <span class="author">Edwards K</span>, <span class="author">Forbes S</span>, <span class="author">Foster R</span>, <span class="author">Gray K</span>, <span class="author">Greenman C</span>, <span class="author">Halliday K</span>, <span class="author">Hills K</span>, <span class="author">Kosmidou V</span>, <span class="author">Lugg R</span>, <span class="author">Menzies A</span>, <span class="author">Perry J</span>, <span class="author">Petty R</span>, <span class="author">Raine K</span>, <span class="author">Ratford L</span>, <span class="author">Shepherd R</span>, <span class="author">Small A</span>, <span class="author">Stephens Y</span>, <span class="author">Tofts C</span>, <span class="author">Varian J</span>, <span class="author">West S</span>, <span class="author">Widaa S</span>, <span class="author">Yates A</span>, <span class="author">Brasseur F</span>, <span class="author">Cooper CS</span>, <span class="author">Flanagan AM</span>, <span class="author">Knowles M</span>, <span class="author">Leung SY</span>, <span class="author">Louis DN</span>, <span class="author">Looijenga LH</span>, <span class="author">Malkowicz B</span>, <span class="author">Pierotti MA</span>, <span class="author">Teh B</span>, <span class="author">Chenevix-Trench G</span>, <span class="author">Weber BL</span>, <span class="author">Yuen ST</span>, <span class="author">Harris G</span>, <span class="author">Goldstraw P</span>, <span class="author">Nicholson AG</span>, <span class="author">Futreal PA</span>, <span class="author">Wooster R</span> and <span class="author">Stratton MR</span></p>
<p class="affiliation">Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK.</p><p class="abstract">The protein-kinase family is the most frequently mutated gene family found in human cancer and faulty kinase enzymes are being investigated as promising targets for the design of antitumour therapies. We have sequenced the gene encoding the transmembrane protein tyrosine kinase ERBB2 (also known as HER2 or Neu) from 120 primary lung tumours and identified 4% that have mutations within the kinase domain; in the adenocarcinoma subtype of lung cancer, 10% of cases had mutations. ERBB2 inhibitors, which have so far proved to be ineffective in treating lung cancer, should now be clinically re-evaluated in the specific subset of patients with lung cancer whose tumours carry ERBB2 mutations.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">431</span>;<span class="number">7008</span>;<span class="pagerange">525-6</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15457249">15457249</a>; DOI: <a href="http://dx.doi.org/10.1038%2F431525b">10.1038/431525b</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Complete MHC haplotype sequencing for common disease gene mapping.</h4>
<p class="authors"><span class="author">Stewart CA</span>, <span class="author">Horton R</span>, <span class="author">Allcock RJ</span>, <span class="author">Ashurst JL</span>, <span class="author">Atrazhev AM</span>, <span class="author">Coggill P</span>, <span class="author">Dunham I</span>, <span class="author">Forbes S</span>, <span class="author">Halls K</span>, <span class="author">Howson JM</span>, <span class="author">Humphray SJ</span>, <span class="author">Hunt S</span>, <span class="author">Mungall AJ</span>, <span class="author">Osoegawa K</span>, <span class="author">Palmer S</span>, <span class="author">Roberts AN</span>, <span class="author">Rogers J</span>, <span class="author">Sims S</span>, <span class="author">Wang Y</span>, <span class="author">Wilming LG</span>, <span class="author">Elliott JF</span>, <span class="author">de Jong PJ</span>, <span class="author">Sawcer S</span>, <span class="author">Todd JA</span>, <span class="author">Trowsdale J</span> and <span class="author">Beck S</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom.</p><p class="abstract">The future systematic mapping of variants that confer susceptibility to common diseases requires the construction of a fully informative polymorphism map. Ideally, every base pair of the genome would be sequenced in many individuals. Here, we report 4.75 Mb of contiguous sequence for each of two common haplotypes of the major histocompatibility complex (MHC), to which susceptibility to &gt;100 diseases has been mapped. The autoimmune disease-associated-haplotypes HLA-A3-B7-Cw7-DR15 and HLA-A1-B8-Cw7-DR3 were sequenced in their entirety through a bacterial artificial chromosome (BAC) cloning strategy using the consanguineous cell lines PGF and COX, respectively. The two sequences were annotated to encompass all described splice variants of expressed genes. We defined the complete variation content of the two haplotypes, revealing &gt;18,000 variations between them. Average SNP densities ranged from less than one SNP per kilobase to &gt;60. Acquisition of complete and accurate sequence data over polymorphic regions such as the MHC from large-insert cloned DNA provides a definitive resource for the construction of informative genetic maps, and avoids the limitation of chromosome regions that are refractory to PCR amplification.</p>
<p class="grants">Funded by: Multiple Sclerosis Society: <span class="grant">588</span></p>
<p class="publication"><span class="title">Genome research</span>&nbsp;<span class="year">2004</span>;<span class="volume">14</span>;<span class="number">6</span>;<span class="pagerange">1176-87</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15140828">15140828</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC419796">419796</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgr.2188104">10.1101/gr.2188104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Cancer: understanding the target.</h4>
<p class="authors"><span class="author">Stratton MR</span> and <span class="author">Futreal PA</span></p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">430</span>;<span class="number">6995</span>;<span class="pagerange">30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15229590">15229590</a>; DOI: <a href="http://dx.doi.org/10.1038%2F430030a">10.1038/430030a</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The BRAF gene is frequently mutated in malignant melanoma.</h4>
<p class="authors"><span class="author">Stratton MR</span>, <span class="author">Wooster RW</span> and <span class="author">Futreal PA</span></p>
<p class="affiliation">The Cancer Genome Project Wellcome Trust Sanger Institute, Hinxton, UK.</p>
<p class="publication"><span class="title">Journal of drugs in dermatology : JDD</span>&nbsp;<span class="year">2004</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">573-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15552615" rel="external">15552615</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">In silico analysis of the sigma54-dependent enhancer-binding proteins in Pirellula species strain 1.</h4>
<p class="authors"><span class="author">Studholme DJ</span> and <span class="author">Dixon R</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The planctomycetes are a phylogenetically distinct group of bacteria, widespread in aquatic and terrestrial environments. Their cell walls lack peptidoglycan and their compartmentalised cells undergo a yeast-like budding cell division process. Many bacteria regulate a subset of their genes by an enhancer-dependent mechanism involving the alternative sigma factor sigma54 (RpoN, sigmaN) in association with sigma54-dependent transcriptional activators known as enhancer-binding proteins (EBPs). The sigma54-dependent regulon has previously been studied in several groups of bacteria, but not in the planctomycetes. We wished to exploit the recently published complete genome sequence of Pirellula species strain 1 to predict and analyse the sigma54-dependent regulon in this interesting group of bacteria. The genome of Pirellula species strain 1 encodes one homologue of sigma54, and 16 sigma54-dependent EBPs, including 10 two-component response regulators and a homologue of Escherichia coli RtcR. Two EBPs contain forkhead-associated domains, representing a novel protein domain combination not previously observed in bacterial EBPs and suggesting a novel link between the enhancer-dependent regulon and &#39;eukaryotic-like&#39; protein phosphorylation in bacterial signal transduction. We identified several potential sigma54-dependent promoters upstream of genes and operons including two homologues of csrA, which encodes the global regulator CsrA, and rtcBA, encoding a RNA 3&#39;-terminal phosphate cyclase. Phylogenetic analysis of EBP sequences from a wide range of bacterial taxa suggested that planctomycete EBPs fall into several distinct clades. Also the phylogeny of the sigma54 factors is broadly consistent with that of the host organisms. These results are consistent with a very ancient origin of sigma54 within the bacterial lineage. The repertoire of functions predicted to be under the control of the sigma54-dependent regulon in Pirellula shares some similarities (e.g. rtcBA) as well as exhibiting differences with that in other taxonomic groups of bacteria, reinforcing the evolutionarily dynamic nature of this regulon.</p>
<p class="publication"><span class="title">FEMS microbiology letters</span>&nbsp;<span class="year">2004</span>;<span class="volume">230</span>;<span class="number">2</span>;<span class="pagerange">215-25</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14757243" rel="external">14757243</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Bioinformatic identification of novel regulatory DNA sequence motifs in Streptomyces coelicolor.</h4>
<p class="authors"><span class="author">Studholme DJ</span>, <span class="author">Bentley SD</span> and <span class="author">Kormanec J</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract"><strong>Background</strong>: Streptomyces coelicolor is a bacterium with a vast repertoire of metabolic functions and complex systems of cellular development. Its genome sequence is rich in genes that encode regulatory proteins to control these processes in response to its changing environment. We wished to apply a recently published bioinformatic method for identifying novel regulatory sequence signals to gain new insights into regulation in S. coelicolor.</p><p class="abstract"><strong>Results</strong>: The method involved production of position-specific weight matrices from alignments of over-represented words of DNA sequence. We generated 2497 weight matrices, each representing a candidate regulatory DNA sequence motif. We scanned the genome sequence of S. coelicolor against each of these matrices. A DNA sequence motif represented by one of the matrices was found preferentially in non-coding sequences immediately upstream of genes involved in polysaccharide degradation, including several that encode chitinases. This motif (TGGTCTAGACCA) was also found upstream of genes encoding components of the phosphoenolpyruvate phosphotransfer system (PTS). We hypothesise that this DNA sequence motif represents a regulatory element that is responsive to availability of carbon-sources. Other motifs of potential biological significance were found upstream of genes implicated in secondary metabolism (TTAGGTtAGgCTaACCTAA), sigma factors (TGACN19TGAC), DNA replication and repair (ttgtCAGTGN13TGGA), nucleotide conversions (CTACgcNCGTAG), and ArsR (TCAGN12TCAG). A motif found upstream of genes involved in chromosome replication (TGTCagtgcN7Tagg) was similar to a previously described motif found in UV-responsive promoters.</p><p class="abstract"><strong>Conclusions</strong>: We successfully applied a recently published in silico method to identify conserved sequence motifs in S. coelicolor that may be biologically significant as regulatory elements. Our data are broadly consistent with and further extend data from previously published studies. We invite experimental testing of our hypotheses in vitro and in vivo.</p>
<p class="publication"><span class="title">BMC microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">4</span>;<span class="pagerange">14</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15072583" rel="external">15072583</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC450296" rel="external">450296</a>; DOI: <a href="http://dx.doi.org/10.1186%2F1471-2180-4-14" rel="external">10.1186/1471-2180-4-14</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Novel protein domains and motifs in the marine planctomycete Rhodopirellula baltica.</h4>
<p class="authors"><span class="author">Studholme DJ</span>, <span class="author">Fuerst JA</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK. <a href="mailto:%64%73%32%40%73%61%6E%67%65%72.%61%63.%75%6B">&#100;&#115;&#50;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The planctomycetes are a phylum of bacteria that have a unique cell compartmentalisation and yeast-like budding cell division and peptidoglycan-less proteinaceous cell walls. We wished to further our understanding of these unique organisms at the molecular level by searching for conserved amino acid sequence motifs and domains in the proteins encoded by Rhodopirellula baltica. Using BLAST and single-linkage clustering, we have discovered several new protein domains and sequence motifs in this planctomycete. R. baltica has multiple members of the newly discovered GEFGR protein family and the ASPIC C-terminal domain family, whilst most other organisms for which whole genome sequence is available have no more than one. Many of the domains and motifs appear to be restricted to the planctomycetes. It is possible that these protein domains and motifs may have been lost or replaced in other phyla, or they may have undergone multiple duplication events in the planctomycete lineage. One of the novel motifs probably represents a novel N-terminal export signal peptide. With their unique cell biology, it may be that the planctomycete cell compartmentalisation plan in particular needs special membrane transport mechanisms. The discovery of these new domains and motifs, many of which are associated with secretion and cell-surface functions, will help to stimulate experimental work and thus enhance further understanding of this fascinating group of organisms.</p>
<p class="publication"><span class="title">FEMS microbiology letters</span>&nbsp;<span class="year">2004</span>;<span class="volume">236</span>;<span class="number">2</span>;<span class="pagerange">333-40</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15251216" rel="external">15251216</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.femsle.2004.06.007" rel="external">10.1016/j.femsle.2004.06.007</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations in the DLG3 gene cause nonsyndromic X-linked mental retardation.</h4>
<p class="authors"><span class="author">Tarpey P</span>, <span class="author">Parnau J</span>, <span class="author">Blow M</span>, <span class="author">Woffendin H</span>, <span class="author">Bignell G</span>, <span class="author">Cox C</span>, <span class="author">Cox J</span>, <span class="author">Davies H</span>, <span class="author">Edkins S</span>, <span class="author">Holden S</span>, <span class="author">Korny A</span>, <span class="author">Mallya U</span>, <span class="author">Moon J</span>, <span class="author">O&#39;Meara S</span>, <span class="author">Parker A</span>, <span class="author">Stephens P</span>, <span class="author">Stevens C</span>, <span class="author">Teague J</span>, <span class="author">Donnelly A</span>, <span class="author">Mangelsdorf M</span>, <span class="author">Mulley J</span>, <span class="author">Partington M</span>, <span class="author">Turner G</span>, <span class="author">Stevenson R</span>, <span class="author">Schwartz C</span>, <span class="author">Young I</span>, <span class="author">Easton D</span>, <span class="author">Bobrow M</span>, <span class="author">Futreal PA</span>, <span class="author">Stratton MR</span>, <span class="author">Gecz J</span>, <span class="author">Wooster R</span> and <span class="author">Raymond FL</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, United Kingdom.</p><p class="abstract">We have identified truncating mutations in the human DLG3 (neuroendocrine dlg) gene in 4 of 329 families with moderate to severe X-linked mental retardation. DLG3 encodes synapse-associated protein 102 (SAP102), a member of the membrane-associated guanylate kinase protein family. Neuronal SAP102 is expressed during early brain development and is localized to the postsynaptic density of excitatory synapses. It is composed of three amino-terminal PDZ domains, an src homology domain, and a carboxyl-terminal guanylate kinase domain. The PDZ domains interact directly with the NR2 subunits of the NMDA glutamate receptor and with other proteins responsible for NMDA receptor localization, immobilization, and signaling. The mutations identified in this study all introduce premature stop codons within or before the third PDZ domain, and it is likely that this impairs the ability of SAP102 to interact with the NMDA receptor and/or other proteins involved in downstream NMDA receptor signaling pathways. NMDA receptors have been implicated in the induction of certain forms of synaptic plasticity, such as long-term potentiation and long-term depression, and these changes in synaptic efficacy have been proposed as neural mechanisms underlying memory and learning. The disruption of NMDA receptor targeting or signaling, as a result of the loss of SAP102, may lead to altered synaptic plasticity and may explain the intellectual impairment observed in individuals with DLG3 mutations.</p>
<p class="grants">Funded by: NICHD NIH HHS: <span class="grant">HD 26202</span></p>
<p class="publication"><span class="title">American journal of human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">75</span>;<span class="number">2</span>;<span class="pagerange">318-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15185169" rel="external">15185169</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC1216066" rel="external">1216066</a>; DOI: <a href="http://dx.doi.org/10.1086%2F422703" rel="external">10.1086/422703</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Autocatalytic RNA cleavage in the human beta-globin pre-mRNA promotes transcription termination.</h4>
<p class="authors"><span class="author">Teixeira A</span>, <span class="author">Tahiri-Alaoui A</span>, <span class="author">West S</span>, <span class="author">Thomas B</span>, <span class="author">Ramadass A</span>, <span class="author">Martianov I</span>, <span class="author">Dye M</span>, <span class="author">James W</span>, <span class="author">Proudfoot NJ</span> and <span class="author">Akoulitchev A</span></p>
<p class="affiliation">Sir William Dunn School of Pathology, University of Oxford, South Parks Road, Oxford OX1 3RE, UK.</p><p class="abstract">New evidence indicates that termination of transcription is an important regulatory step, closely related to transcriptional interference and even transcriptional initiation. However, how this occurs is poorly understood. Recently, in vivo analysis of transcriptional termination for the human beta-globin gene revealed a new phenomenon--co-transcriptional cleavage (CoTC). This primary cleavage event within beta-globin pre-messenger RNA, downstream of the poly(A) site, is critical for efficient transcriptional termination by RNA polymerase II. Here we show that the CoTC process in the human beta-globin gene involves an RNA self-cleaving activity. We characterize the autocatalytic core of the CoTC ribozyme and show its functional role in efficient termination in vivo. The identified core CoTC is highly conserved in the 3&#39; flanking regions of other primate beta-globin genes. Functionally, it resembles the 3&#39; processive, self-cleaving ribozymes described for the protein-encoding genes from the myxomycetes Didymium iridis and Physarum polycephalum, indicating evolutionary conservation of this molecular process. We predict that regulated autocatalytic cleavage elements within pre-mRNAs may be a general phenomenon and that functionally it may provide the entry point for exonucleases involved in mRNA maturation, turnover and, in particular, transcriptional termination.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">2004</span>;<span class="volume">432</span>;<span class="number">7016</span>;<span class="pagerange">526-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15565159">15565159</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnature03032">10.1038/nature03032</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The use of genome annotation data and its impact on biological conclusions.</h4>
<p class="authors"><span class="author">Tettelin H</span> and <span class="author">Parkhill J</span></p>
<p class="publication"><span class="title">Nature genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">36</span>;<span class="number">10</span>;<span class="pagerange">1028-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15454934">15454934</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fng1004-1028b">10.1038/ng1004-1028b</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The role of prophage-like elements in the diversity of Salmonella enterica serovars.</h4>
<p class="authors"><span class="author">Thomson N</span>, <span class="author">Baker S</span>, <span class="author">Pickard D</span>, <span class="author">Fookes M</span>, <span class="author">Anjum M</span>, <span class="author">Hamlin N</span>, <span class="author">Wain J</span>, <span class="author">House D</span>, <span class="author">Bhutta Z</span>, <span class="author">Chan K</span>, <span class="author">Falkow S</span>, <span class="author">Parkhill J</span>, <span class="author">Woodward M</span>, <span class="author">Ivens A</span> and <span class="author">Dougan G</span></p>
<p class="affiliation">The Pathogen Sequencing Unit, The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, United Kingdom. <a href="mailto:%6E%72%74%40%73%61%6E%67%65%72.%61%63.%75%6B">&#110;&#114;&#116;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">The Salmonella enterica serovar Typhi CT18 (S.Typhi) chromosome harbours seven distinct prophage-like elements, some of which may encode functional bacteriophages. In silico analyses were used to investigate these regions in S.Typhi CT18, and ultimately compare these integrated bacteriophages against 40 other Salmonella isolates using DNA microarray technology. S.Typhi CT18 contains prophages that show similarity to the lambda, Mu, P2 and P4 bacteriophage families. When compared to other S.Typhi isolates, these elements were generally conserved, supporting a clonal origin of this serovar. However, distinct variation was detected within a broad range of Salmonella serovars; many of the prophage regions are predicted to be specific to S.Typhi. Some of the P2 family prophage analysed have the potential to carry non-essential &quot;cargo&quot; genes within the hyper-variable tail region, an observation that suggests that these bacteriophage may confer a level of specialisation on their host. Lysogenic bacteriophages therefore play a crucial role in the generation of genetic diversity within S.enterica.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">339</span>;<span class="number">2</span>;<span class="pagerange">279-300</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15136033" rel="external">15136033</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.jmb.2004.03.058" rel="external">10.1016/j.jmb.2004.03.058</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shrinking genomics.</h4>
<p class="authors"><span class="author">Thomson NR</span>, <span class="author">Sebaihia M</span>, <span class="author">Cerde&ntilde;o-T&aacute;rraga AM</span>, <span class="author">Holden MT</span> and <span class="author">Parkhill J</span></p>
<p class="affiliation">Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK.</p><p class="abstract">Two bacteria are featured this month, and both are at the lower end of the genome size scale. The first, Mycoplasma gallisepticum, belongs to a group of bacteria that have been studied both as important human and animal pathogens and in the pursuit of understanding the essential functions of a self-replicating minimal cell. The second, Nanoarchaeum equitans, is an obligate symbiont that only grows in co-culture with another archaeon. N. equitans seems to be the coelacanth of the microbial world--it has been assigned to a new phylum and represents a primitive form of prokaroytic life.</p>
<p class="publication"><span class="title">Nature reviews. Microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">2</span>;<span class="number">1</span>;<span class="pagerange">11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15035003">15035003</a>; DOI: <a href="http://dx.doi.org/10.1038%2Fnrmicro800">10.1038/nrmicro800</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A giant novel gene undergoing extensive alternative splicing is severed by a Cornelia de Lange-associated translocation breakpoint at 3q26.3.</h4>
<p class="authors"><span class="author">Tonkin ET</span>, <span class="author">Smith M</span>, <span class="author">Eichhorn P</span>, <span class="author">Jones S</span>, <span class="author">Imamwerdi B</span>, <span class="author">Lindsay S</span>, <span class="author">Jackson M</span>, <span class="author">Wang TJ</span>, <span class="author">Ireland M</span>, <span class="author">Burn J</span>, <span class="author">Krantz ID</span>, <span class="author">Carr P</span> and <span class="author">Strachan T</span></p>
<p class="affiliation">Institute of Human Genetics, International Centre for Life, University of Newcastle, NE1 3BZ, Newcastle upon Tyne, Central Parkway, UK.</p><p class="abstract">Cornelia de Lange syndrome (CdLS) is a rare developmental malformation syndrome characterised by mental handicap, growth retardation, distinctive facial features and limb reduction defects. The vast majority of CdLS cases are sporadic. We carried out a high density bacterial artificial chromosome (BAC) microarray comparative genome hybridisation screen but no evidence was found for a consistent pattern of microdeletion/microduplication. As an alternative, we focused on identifying chromosomal regions spanning associated translocation breakpoints. We prioritised the distal 3q region because of the occurrence, in a classical CdLS patient, of a de novo balanced translocation with a breakpoint at 3q26.3 and of reports of phenotypic overlap between cases of mild CdLS and individuals trisomic for the 3q26-q27 region. We show that the 3q26.3 breakpoint severs a previously uncharacterised giant gene, NAALADL2, containing at least 32 exons spanning 1.37 Mb. Northern blot analysis identified up to six different transcripts in the 1-10 kb range with strongest expression in kidney and placenta; embryonic expression was largely confined to duodenal and stomach endoderm, mesonephros, metanephros and pancreas. Transcript analysis identified extensive alternative splicing leading to multiple 5&#39; and 3&#39; untranslated regions and variable coding sequences. Multiple protein isoforms were defined by different N-terminal regions (with at least four alternative initiating methionine codons), and by differential protein truncation/use of alternative C-terminal sequences attributable to alternative splicing/polyadenylation. Outside the N-terminal regions, the predicted proteins showed significant homology to N-acetylated alpha-linked acidic dipeptidase and transferrin receptors. Mutation screening of NAALADL2 in a panel of CdLS patient DNA samples failed to identify patient-specific mutations. We discuss the possibility that the 3q26.3 translocation could nevertheless contribute to pathogenesis.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">115</span>;<span class="number">2</span>;<span class="pagerange">139-48</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15168106" rel="external">15168106</a>; DOI: <a href="http://dx.doi.org/10.1007%2Fs00439-004-1134-6" rel="external">10.1007/s00439-004-1134-6</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">DNA binding activity of the Escherichia coli nitric oxide sensor NorR suggests a conserved target sequence in diverse proteobacteria.</h4>
<p class="authors"><span class="author">Tucker NP</span>, <span class="author">D&#39;Autr&eacute;aux B</span>, <span class="author">Studholme DJ</span>, <span class="author">Spiro S</span> and <span class="author">Dixon R</span></p>
<p class="affiliation">John Innes Centre, Colney, Norwich, United Kingdom.</p><p class="abstract">The Escherichia coli nitric oxide sensor NorR was shown to bind to the promoter region of the norVW transcription unit, forming at least two distinct complexes detectable by gel retardation. Three binding sites for NorR and two integration host factor binding sites were identified in the norR-norV intergenic region. The derived consensus sequence for NorR binding sites was used to search for novel members of the E. coli NorR regulon and to show that NorR binding sites are partially conserved in other members of the proteobacteria.</p>
<p class="publication"><span class="title">Journal of bacteriology</span>&nbsp;<span class="year">2004</span>;<span class="volume">186</span>;<span class="number">19</span>;<span class="pagerange">6656-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15375149" rel="external">15375149</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC516610" rel="external">516610</a>; DOI: <a href="http://dx.doi.org/10.1128%2FJB.186.19.6656-6660.2004" rel="external">10.1128/JB.186.19.6656-6660.2004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">NaCl restriction upregulates renal Slc26a4 through subcellular redistribution: role in Cl- conservation.</h4>
<p class="authors"><span class="author">Wall SM</span>, <span class="author">Kim YH</span>, <span class="author">Stanley L</span>, <span class="author">Glapion DM</span>, <span class="author">Everett LA</span>, <span class="author">Green ED</span> and <span class="author">Verlander JW</span></p>
<p class="affiliation">Department of Medicine, Emory University, Atlanta, Ga 30322, USA. <a href="mailto:%73%6D%77%61%6C%6C%40%65%6D%6F%72%79.%65%64%75">&#115;&#109;&#119;&#97;&#108;&#108;&#64;&#101;&#109;&#111;&#114;&#121;&#46;&#101;&#100;&#117;</a></p><p class="abstract">Slc26a4 (Pds, pendrin) is an anion transporter expressed in the apical region of type B and non-A, non-B intercalated cells of the distal nephron. It is upregulated by aldosterone analogues and is critical in the development of mineralocorticoid-induced hypertension. Thus, Slc26a4 expression and its role in blood pressure and fluid and electrolyte homeostasis was explored during NaCl restriction, a treatment model in which aldosterone is appropriately increased. Ultrastructural immunolocalization, balance studies, and cortical collecting ducts (CCDs) perfused in vitro were used. With moderate physiological NaCl restriction, Slc26a4 expression in the apical plasma membrane increased 2- to 3-fold in type B intercalated cells. Because Slc26a4 transports Cl-, we tested whether NaCl balance differs in Slc26a4(+/+) and Slc26a4(-/-) mice during NaCl restriction. Cl- absorption was observed in CCDs from Slc26a4(+/+) but not from Slc26a4(-/-) mice. After moderate NaCl restriction, urinary volume and Cl- excretion were increased in Slc26a4(-/-) relative to Slc26a4(+/+) mice. Moreover, Slc26a4(-/-) mice had evidence of relative vascular volume depletion because they had a higher arterial pH, hematocrit, and blood urea nitrogen than wild-type mice. With moderate NaCl restriction, blood pressure was similar in Slc26a4(+/+) and Slc26a4(-/-) mice. However, on a severely restricted intake of NaCl, Slc26a4(-/-) mice were hypotensive relative to wild-type mice. We conclude that Slc26a4 is upregulated with NaCl restriction and is critical in the maintenance of acid-base balance and in the renal conservation of Cl- and water during NaCl restriction.</p>
<p class="grants">Funded by: NIDDK NIH HHS: <span class="grant">DK 52935</span></p>
<p class="publication"><span class="title">Hypertension</span>&nbsp;<span class="year">2004</span>;<span class="volume">44</span>;<span class="number">6</span>;<span class="pagerange">982-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15477386" rel="external">15477386</a>; DOI: <a href="http://dx.doi.org/10.1161%2F01.HYP.0000145863.96091.89" rel="external">10.1161/01.HYP.0000145863.96091.89</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mechanism of activation of the RAF-ERK signaling pathway by oncogenic mutations of B-RAF.</h4>
<p class="authors"><span class="author">Wan PT</span>, <span class="author">Garnett MJ</span>, <span class="author">Roe SM</span>, <span class="author">Lee S</span>, <span class="author">Niculescu-Duvaz D</span>, <span class="author">Good VM</span>, <span class="author">Jones CM</span>, <span class="author">Marshall CJ</span>, <span class="author">Springer CJ</span>, <span class="author">Barford D</span>, <span class="author">Marais R</span> and <span class="author">Cancer Genome Project</span></p>
<p class="affiliation">Section of Structural Biology, The Institute of Cancer Research, Chester Beatty Laboratories, 237 Fulham Road, London SW3 6JB, UK.</p><p class="abstract">Over 30 mutations of the B-RAF gene associated with human cancers have been identified, the majority of which are located within the kinase domain. Here we show that of 22 B-RAF mutants analyzed, 18 have elevated kinase activity and signal to ERK in vivo. Surprisingly, three mutants have reduced kinase activity towards MEK in vitro but, by activating C-RAF in vivo, signal to ERK in cells. The structures of wild type and oncogenic V599EB-RAF kinase domains in complex with the RAF inhibitor BAY43-9006 show that the activation segment is held in an inactive conformation by association with the P loop. The clustering of most mutations to these two regions suggests that disruption of this interaction converts B-RAF into its active conformation. The high activity mutants signal to ERK by directly phosphorylating MEK, whereas the impaired activity mutants stimulate MEK by activating endogenous C-RAF, possibly via an allosteric or transphosphorylation mechanism.</p>
<p class="publication"><span class="title">Cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">116</span>;<span class="number">6</span>;<span class="pagerange">855-67</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15035987">15035987</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Polybromo protein BAF180 functions in mammalian cardiac chamber maturation.</h4>
<p class="authors"><span class="author">Wang Z</span>, <span class="author">Zhai W</span>, <span class="author">Richardson JA</span>, <span class="author">Olson EN</span>, <span class="author">Meneses JJ</span>, <span class="author">Firpo MT</span>, <span class="author">Kang C</span>, <span class="author">Skarnes WC</span> and <span class="author">Tjian R</span></p>
<p class="affiliation">Department of Molecular and Cell Biology, University of California, Berkeley, California 94720-3204, USA.</p><p class="abstract">BAF and PBAF are two related mammalian chromatin remodeling complexes essential for gene expression and development. PBAF, but not BAF, is able to potentiate transcriptional activation in vitro mediated by nuclear receptors, such as RXRalpha, VDR, and PPARgamma. Here we show that the ablation of PBAF-specific subunit BAF180 in mouse embryos results in severe hypoplastic ventricle development and trophoblast placental defects, similar to those found in mice lacking RXRalpha and PPARgamma. Embryonic aggregation analyses reveal that in contrast to PPARgamma-deficient mice, the heart defects are likely a direct result of BAF180 ablation, rather than an indirect consequence of trophoblast placental defects. We identified potential target genes for BAF180 in heart development, such as S100A13 as well as retinoic acid (RA)-induced targets RARbeta2 and CRABPII. Importantly, BAF180 is recruited to the promoter of these target genes and BAF180 deficiency affects the RA response for CRABPII and RARbeta2. These studies reveal unique functions of PBAF in cardiac chamber maturation.</p>
<p class="publication"><span class="title">Genes &amp; development</span>&nbsp;<span class="year">2004</span>;<span class="volume">18</span>;<span class="number">24</span>;<span class="pagerange">3106-16</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15601824">15601824</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC535920">535920</a>; DOI: <a href="http://dx.doi.org/10.1101%2Fgad.1238104">10.1101/gad.1238104</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tissue microarrays: fast-tracking protein expression at the cellular level.</h4>
<p class="authors"><span class="author">Warford A</span></p>
<p class="affiliation">Wellcome Trust Sanger Institute, Hinxton, Cambridge, CB10 1SA, UK. <a href="mailto:%61%77%31%40%73%61%6E%67%65%72.%61%63.%75%6B">&#97;&#119;&#49;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">Tissue microarrays maximize returns in cellular pathology whilst minimizing the use of cells and tissues. They are made by arraying cores of tissue taken from multiple donor blocks into a single recipient block. Accordingly, the histology and pathology of several hundred tissues can be represented in one tissue microarray that, when stained by immunohistochemistry, provides comprehensive topographic information on protein expression. Used with complimentary techniques, such as complementary DNA microarray analysis, tissue microarrays are providing valuable data for the identification of new markers of disease and assisting in the discovery of therapeutic targets. They are also leading a revolution in cellular pathology as high-throughput technology is introduced to maximize the information provided.</p>
<p class="publication"><span class="title">Expert review of proteomics</span>&nbsp;<span class="year">2004</span>;<span class="volume">1</span>;<span class="number">3</span>;<span class="pagerange">283-92</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15966825" rel="external">15966825</a>; DOI: <a href="http://dx.doi.org/10.1586%2F14789450.1.3.283" rel="external">10.1586/14789450.1.3.283</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Global gene expression responses of fission yeast to ionizing radiation.</h4>
<p class="authors"><span class="author">Watson A</span>, <span class="author">Mata J</span>, <span class="author">B&auml;hler J</span>, <span class="author">Carr A</span> and <span class="author">Humphrey T</span></p>
<p class="affiliation">Genome Damage and Stability Centre, University of Sussex, Brighton BN1 9RQ, United Kingdom.</p><p class="abstract">A coordinated transcriptional response to DNA-damaging agents is required to maintain genome stability. We have examined the global gene expression responses of the fission yeast Schizosaccharomyces pombe to ionizing radiation (IR) by using DNA microarrays. We identified approximately 200 genes whose transcript levels were significantly altered at least twofold in response to 500 Gy of gamma IR in a temporally defined manner. The majority of induced genes were core environmental stress response genes, whereas the remaining genes define a transcriptional response to DNA damage in fission yeast. Surprisingly, few DNA repair and checkpoint genes were transcriptionally modulated in response to IR. We define a role for the stress-activated mitogen-activated protein kinase Sty1/Spc1 and the DNA damage checkpoint kinase Rad3 in regulating core environmental stress response genes and IR-specific response genes, both independently and in concert. These findings suggest a complex network of regulatory pathways coordinate gene expression responses to IR in eukaryotes.</p>
<p class="grants">Funded by: Cancer Research UK: <span class="grant">A6517</span>; Wellcome Trust: <span class="grant">077118</span></p>
<p class="publication"><span class="title">Molecular biology of the cell</span>&nbsp;<span class="year">2004</span>;<span class="volume">15</span>;<span class="number">2</span>;<span class="pagerange">851-60</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14668484" rel="external">14668484</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC329398" rel="external">329398</a>; DOI: <a href="http://dx.doi.org/10.1091%2Fmbc.E03-08-0569" rel="external">10.1091/mbc.E03-08-0569</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Tyrosine site-specific recombinases mediate DNA inversions affecting the expression of outer surface proteins of Bacteroides fragilis.</h4>
<p class="authors"><span class="author">Weinacht KG</span>, <span class="author">Roche H</span>, <span class="author">Krinos CM</span>, <span class="author">Coyne MJ</span>, <span class="author">Parkhill J</span> and <span class="author">Comstock LE</span></p>
<p class="affiliation">Channing Laboratory, Brigham and Women&#39;s Hospital, Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA.</p><p class="abstract">The chromosome of Bacteroides fragilis has been shown to undergo 13 distinct DNA inversions affecting the expression of capsular polysaccharides and mediated by a serine site-specific recombinase designated Mpi. In this study, we demonstrate that members of the tyrosine site-specific recombinase family, conserved in B. fragilis, mediate additional DNA inversions of the B. fragilis genome. These DNA invertases flip promoter regions in their immediate downstream region. The genetic organization of the genes regulated by these invertible promoter regions suggests that they are operons and many of the products are predicted to be outer membrane proteins. Phenotypic analysis of a deletion mutant of one of these DNA invertases, tsr15 (aapI), which resulted in the promoter region for the downstream genes being locked ON, confirmed the synthesis of multiple surface proteins by this operon. In addition, this deletion mutant demonstrated an autoaggregative phenotype and showed significantly greater adherence than wild-type organisms in a biofilm assay, suggesting a possible functional role for these phase-variable outer surface proteins. This study demonstrates that DNA inversion is a universal mechanism used by this commensal microorganism to phase vary expression of its surface molecules and involves at least three conserved DNA invertases from two evolutionarily distinct families.</p>
<p class="grants">Funded by: NIAID NIH HHS: <span class="grant">AI44193</span>, <span class="grant">R01 AI044193-05</span>, <span class="grant">R01 AI044193-06</span></p>
<p class="publication"><span class="title">Molecular microbiology</span>&nbsp;<span class="year">2004</span>;<span class="volume">53</span>;<span class="number">5</span>;<span class="pagerange">1319-30</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15387812" rel="external">15387812</a>; DOI: <a href="http://dx.doi.org/10.1111%2Fj.1365-2958.2004.04219.x" rel="external">10.1111/j.1365-2958.2004.04219.x</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A genome sequence survey of the filarial nematode Brugia malayi: repeats, gene discovery, and comparative genomics.</h4>
<p class="authors"><span class="author">Whitton C</span>, <span class="author">Daub J</span>, <span class="author">Quail M</span>, <span class="author">Hall N</span>, <span class="author">Foster J</span>, <span class="author">Ware J</span>, <span class="author">Ganatra M</span>, <span class="author">Slatko B</span>, <span class="author">Barrell B</span> and <span class="author">Blaxter M</span></p>
<p class="affiliation">Ashworth Laboratories, Institution of Cell, Animal and Polulation Biology, School of Biological Sciences, University of Edinburgh, King&#39;s Buildings, Edinburgh EH9 3JT, UK.</p><p class="abstract">Comparative nematode genomics has thus far been largely constrained to the genus Caenorhabditis, but a huge diversity of other nematode species, and genomes, exist. The Brugia malayi genome is approximately 100 Mb in size, and distributed across five chromosome pairs. Previous genomic investigations have included definition of major repeat classes and sequencing of selected genes. We have generated over 18,000 sequences from the ends of large-insert clones from bacterial artificial chromosome libraries. These end sequences, totalling over 10 Mb of sequence, contain just under 8 Mb of unique sequence. We identified the known Mbo I and Hha I repeat families in the sequence data, and also identified several new repeats based on their abundance. Genomic copies of 17% of B. malayi genes defined by expressed sequence tags have been identified. Nearly one quarter of end sequences can encode peptides with significant similarity to protein sequences in the public databases, and we estimate that we have identified more than 2700 new B. malayi genes. Importantly, 459 end sequences had homologues in other organisms, but lacked a match in the completely sequenced genomes of Caenorhabditis briggsae and Caenorhabditis elegans, emphasising the role of gene loss in genome evolution. B. malayi is estimated to have over 18,500 protein-coding genes.</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">137</span>;<span class="number">2</span>;<span class="pagerange">215-27</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15383292" rel="external">15383292</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2004.05.013" rel="external">10.1016/j.molbiopara.2004.05.013</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Fine mapping, gene content, comparative sequencing, and expression analyses support Ctla4 and Nramp1 as candidates for Idd5.1 and Idd5.2 in the nonobese diabetic mouse.</h4>
<p class="authors"><span class="author">Wicker LS</span>, <span class="author">Chamberlain G</span>, <span class="author">Hunter K</span>, <span class="author">Rainbow D</span>, <span class="author">Howlett S</span>, <span class="author">Tiffen P</span>, <span class="author">Clark J</span>, <span class="author">Gonzalez-Munoz A</span>, <span class="author">Cumiskey AM</span>, <span class="author">Rosa RL</span>, <span class="author">Howson JM</span>, <span class="author">Smink LJ</span>, <span class="author">Kingsnorth A</span>, <span class="author">Lyons PA</span>, <span class="author">Gregory S</span>, <span class="author">Rogers J</span>, <span class="author">Todd JA</span> and <span class="author">Peterson LB</span></p>
<p class="affiliation">Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 2XY, UK. <a href="mailto:%6C%69%6E%64%61.%77%69%63%6B%65%72%40%63%69%6D%72.%63%61%6D.%61%63.%75%6B">&#108;&#105;&#110;&#100;&#97;&#46;&#119;&#105;&#99;&#107;&#101;&#114;&#64;&#99;&#105;&#109;&#114;&#46;&#99;&#97;&#109;&#46;&#97;&#99;&#46;&#117;&#107;</a></p><p class="abstract">At least two loci that determine susceptibility to type 1 diabetes in the NOD mouse have been mapped to chromosome 1, Idd5.1 (insulin-dependent diabetes 5.1) and Idd5.2. In this study, using a series of novel NOD.B10 congenic strains, Idd5.1 has been defined to a 2.1-Mb region containing only four genes, Ctla4, Icos, Als2cr19, and Nrp2 (neuropilin-2), thereby excluding a major candidate gene, Cd28. Genomic sequence comparison of the two functional candidate genes, Ctla4 and Icos, from the B6 (resistant at Idd5.1) and the NOD (susceptible at Idd5.1) strains revealed 62 single nucleotide polymorphisms (SNPs), only two of which were in coding regions. One of these coding SNPs, base 77 of Ctla4 exon 2, is a synonymous SNP and has been correlated previously with type 1 diabetes susceptibility and differential expression of a CTLA-4 isoform. Additional expression studies in this work support the hypothesis that this SNP in exon 2 is the genetic variation causing the biological effects of Idd5.1. Analysis of additional congenic strains has also localized Idd5.2 to a small region (1.52 Mb) of chromosome 1, but in contrast to the Idd5.1 interval, Idd5.2 contains at least 45 genes. Notably, the Idd5.2 region still includes the functionally polymorphic Nramp1 gene. Future experiments to test the identity of Idd5.1 and Idd5.2 as Ctla4 and Nramp1, respectively, can now be justified using approaches to specifically alter or mimic the candidate causative SNPs.</p>
<p class="publication"><span class="title">Journal of immunology (Baltimore, Md. : 1950)</span>&nbsp;<span class="year">2004</span>;<span class="volume">173</span>;<span class="number">1</span>;<span class="pagerange">164-73</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15210771">15210771</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Replication timing of the human genome.</h4>
<p class="authors"><span class="author">Woodfine K</span>, <span class="author">Fiegler H</span>, <span class="author">Beare DM</span>, <span class="author">Collins JE</span>, <span class="author">McCann OT</span>, <span class="author">Young BD</span>, <span class="author">Debernardi S</span>, <span class="author">Mott R</span>, <span class="author">Dunham I</span> and <span class="author">Carter NP</span></p>
<p class="affiliation">The Welcome Trust Sanger Institute, Welcome Genome Campus, Cambridge, UK.</p><p class="abstract">We have developed a directly quantitative method utilizing genomic clone DNA microarrays to assess the replication timing of sequences during the S phase of the cell cycle. The genomic resolution of the replication timing measurements is limited only by the genomic clone size and density. We demonstrate the power of this approach by constructing a genome-wide map of replication timing in human lymphoblastoid cells using an array with clones spaced at 1 Mb intervals and a high-resolution replication timing map of 22q with an array utilizing overlapping sequencing tile path clones. We show a positive correlation, both genome-wide and at a high resolution, between replication timing and a range of genome parameters including GC content, gene density and transcriptional activity.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13</span>;<span class="number">2</span>;<span class="pagerange">191-202</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14645202" rel="external">14645202</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh016" rel="external">10.1093/hmg/ddh016</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Delta proteins and MAGI proteins: an interaction of Notch ligands with intracellular scaffolding molecules and its significance for zebrafish development.</h4>
<p class="authors"><span class="author">Wright GJ</span>, <span class="author">Leslie JD</span>, <span class="author">Ariza-McNaughton L</span> and <span class="author">Lewis J</span></p>
<p class="affiliation">Vertebrate Development Laboratory, Cancer Research UK London Research Institute, 44 Lincoln&#39;s Inn Fields, London WC2A 3PX, UK.</p><p class="abstract">Delta proteins activate Notch through a binding reaction that depends on their extracellular domains; but the intracellular (C-terminal) domains of the Deltas also have significant functions. All classes of vertebrates possess a subset of Delta proteins with a conserved ATEV* motif at their C termini. These ATEV Deltas include Delta1 and Delta4 in mammals and DeltaD and DeltaC in the zebrafish. We show that these Deltas associate with the membrane-associated scaffolding proteins MAGI1, MAGI2 and MAGI3, through a direct interaction between the C termini of the Deltas and a specific PDZ domain (PDZ4) of the MAGIs. In cultured cells and in subsets of cells in the intact zebrafish embryo, DeltaD and MAGI1 are co-localized at the plasma membrane. The interaction and the co-localization can be abolished by injection of a morpholino that blocks the mRNA splicing reaction that gives DeltaD its terminal valine, on which the interaction depends. Embryos treated in this way appear normal with respect to some known functions of DeltaD as a Notch ligand, including the control of somite segmentation, neurogenesis, and hypochord formation. They do, however, show an anomalous distribution of Rohon-Beard neurons in the dorsal neural tube, suggesting that the Delta-MAGI interaction may play some part in the control of neuron migration.</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">2004</span>;<span class="volume">131</span>;<span class="number">22</span>;<span class="pagerange">5659-69</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15509766" rel="external">15509766</a>; DOI: <a href="http://dx.doi.org/10.1242%2Fdev.01417" rel="external">10.1242/dev.01417</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Reduced penetrance of craniofacial anomalies as a function of deletion size and genetic background in a chromosome engineered partial mouse model for Smith-Magenis syndrome.</h4>
<p class="authors"><span class="author">Yan J</span>, <span class="author">Keener VW</span>, <span class="author">Bi W</span>, <span class="author">Walz K</span>, <span class="author">Bradley A</span>, <span class="author">Justice MJ</span> and <span class="author">Lupski JR</span></p>
<p class="affiliation">Department of Molecular and Human Genetics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA.</p><p class="abstract">Smith-Magenis syndrome (SMS) is a multiple congenital anomaly/mental retardation syndrome associated with del(17)(p11.2p11.2). The phenotype is variable even in patients with deletions of the same size. RAI1 has been recently suggested as a major gene for majority of the SMS phenotypes, but its role in the full spectrum of the phenotype remains unclear. Df(11)17/+ mice contain a heterozygous deletion in the mouse region syntenic to the SMS common deletion, and exhibit craniofacial abnormalities, seizures and marked obesity, partially reproducing the SMS phenotype. To further study the genetic basis for the phenotype, we constructed three lines of mice with smaller deletions [Df(11)17-1, Df(11)17-2 and Df(11)17-3] using retrovirus-mediated chromosome engineering to create nested deletions. Both craniofacial abnormalities and obesity have been observed, but the penetrance of the craniofacial phenotype was markedly reduced when compared with Df(11)17/+ mice. Overt seizures were not observed. Phenotypic variation has been observed in mice with the same deletion size in the same and in different genetic backgrounds, which may reflect the variation documented in the patients. These results indicate that the smaller deletions contain the gene(s), most likely Rai1, causing craniofacial abnormalities and obesity. However, genes or regulatory elements in the larger deletion, which are not located in the smaller deletions, as well as genes located elsewhere, also influence penetrance and expressivity of the phenotype. Our mouse models refined the genomic region important for a portion of the SMS phenotype and provided a basis for further molecular analysis of genes associated with SMS.</p>
<p class="grants">Funded by: NCI NIH HHS: <span class="grant">P01 CA75719</span>; NIDCR NIH HHS: <span class="grant">R01 DE015210</span></p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">2004</span>;<span class="volume">13</span>;<span class="number">21</span>;<span class="pagerange">2613-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15459175" rel="external">15459175</a>; DOI: <a href="http://dx.doi.org/10.1093%2Fhmg%2Fddh288" rel="external">10.1093/hmg/ddh288</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Identification of PSD-95 as a regulator of dopamine-mediated synaptic and behavioral plasticity.</h4>
<p class="authors"><span class="author">Yao WD</span>, <span class="author">Gainetdinov RR</span>, <span class="author">Arbuckle MI</span>, <span class="author">Sotnikova TD</span>, <span class="author">Cyr M</span>, <span class="author">Beaulieu JM</span>, <span class="author">Torres GE</span>, <span class="author">Grant SG</span> and <span class="author">Caron MG</span></p>
<p class="affiliation">Howard Hughes Medical Institute Laboratories, Department of Cell Biology, Duke University Medical Center, Durham, NC 27710, USA.</p><p class="abstract">To identify the molecular mechanisms underlying psychostimulant-elicited plasticity in the brain reward system, we undertook a phenotype-driven approach using genome-wide microarray profiling of striatal transcripts from three genetic and one pharmacological mouse models of psychostimulant or dopamine supersensitivity. A small set of co-affected genes was identified. One of these genes encoding the synaptic scaffolding protein PSD-95 is downregulated in the striatum of all three mutants and in chronically, but not acutely, cocaine-treated mice. At the synaptic level, enhanced long-term potentiation (LTP) of the frontocortico-accumbal glutamatergic synapses correlates with PSD-95 reduction in every case. Finally, targeted deletion of PSD-95 in an independent line of mice enhances LTP, augments the acute locomotor-stimulating effects of cocaine, but leads to no further behavioral plasticity in response to chronic cocaine. Our findings uncover a previously unappreciated role of PSD-95 in psychostimulant action and identify a molecular and cellular mechanism shared between drug-related plasticity and learning.</p>
<p class="grants">Funded by: NIDA NIH HHS: <span class="grant">DA13511</span></p>
<p class="publication"><span class="title">Neuron</span>&nbsp;<span class="year">2004</span>;<span class="volume">41</span>;<span class="number">4</span>;<span class="pagerange">625-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14980210" rel="external">14980210</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The PepSY domain: a regulator of peptidase activity in the microbial environment?</h4>
<p class="authors"><span class="author">Yeats C</span>, <span class="author">Rawlings ND</span> and <span class="author">Bateman A</span></p>
<p class="affiliation">The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK. <a href="mailto:%63%61%79%40%73%61%6E%67%65%72.%61%63.%75%6B">&#99;&#97;&#121;&#64;&#115;&#97;&#110;&#103;&#101;&#114;&#46;&#97;&#99;&#46;&#117;&#107;</a></p>
<p class="publication"><span class="title">Trends in biochemical sciences</span>&nbsp;<span class="year">2004</span>;<span class="volume">29</span>;<span class="number">4</span>;<span class="pagerange">169-72</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15124630" rel="external">15124630</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.tibs.2004.02.004" rel="external">10.1016/j.tibs.2004.02.004</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Resolution of the novel immune-type receptor gene cluster in zebrafish.</h4>
<p class="authors"><span class="author">Yoder JA</span>, <span class="author">Litman RT</span>, <span class="author">Mueller MG</span>, <span class="author">Desai S</span>, <span class="author">Dobrinski KP</span>, <span class="author">Montgomery JS</span>, <span class="author">Buzzeo MP</span>, <span class="author">Ota T</span>, <span class="author">Amemiya CT</span>, <span class="author">Trede NS</span>, <span class="author">Wei S</span>, <span class="author">Djeu JY</span>, <span class="author">Humphray S</span>, <span class="author">Jekosch K</span>, <span class="author">Hernandez Prada JA</span>, <span class="author">Ostrov DA</span> and <span class="author">Litman GW</span></p>
<p class="affiliation">Department of Molecular Biomedical Sciences, College of Veterinary Medicine, North Carolina State University, 4700 Hillsborough Street, Raleigh, NC 27606, USA.</p><p class="abstract">The novel immune-type receptor (NITR) genes encode a unique multigene family of leukocyte regulatory receptors, which possess an extracellular Ig variable (V) domain and may function in innate immunity. Artificial chromosomes that encode zebrafish NITRs have been assembled into a contig spanning approximately 350 kb. Resolution of the complete NITR gene cluster has led to the identification of eight previously undescribed families of NITRs and has revealed the presence of C-type lectins within the locus. A maximum haplotype of 36 NITR genes (138 gene sequences in total) can be grouped into 12 distinct families, including inhibitory and activating receptors. An extreme level of interindividual heterozygosity is reflected in allelic polymorphisms, haplotype variation, and family-specific isoform complexity. In addition, the exceptional diversity of NITR sequences among species suggests divergent evolution of this multigene family with a birth-and-death process of member genes. High-confidence modeling of Nitr V-domain structures reveals a significant shift in the spatial orientation of the Ig fold, in the region of highest interfamily variation, compared with Ig V domains. These studies resolve a complete immune gene cluster in zebrafish and indicate that the NITRs represent the most complex family of activating/inhibitory surface receptors thus far described.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">44</span>;<span class="pagerange">15706-11</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15496470" rel="external">15496470</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC524843" rel="external">524843</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0405242101" rel="external">10.1073/pnas.0405242101</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Repression of nodal expression by maternal B1-type SOXs regulates germ layer formation in Xenopus and zebrafish.</h4>
<p class="authors"><span class="author">Zhang C</span>, <span class="author">Basta T</span>, <span class="author">Hernandez-Lagunas L</span>, <span class="author">Simpson P</span>, <span class="author">Stemple DL</span>, <span class="author">Artinger KB</span> and <span class="author">Klymkowsky MW</span></p>
<p class="affiliation">Molecular, Cellular and Developmental Biology, University of Colorado, Boulder, 80309-0347, USA.</p><p class="abstract">B1-type SOXs (SOXs 1, 2, and 3) are the most evolutionarily conserved subgroup of the SOX transcription factor family. To study their maternal functions, we used the affinity-purified antibody antiSOX3c, which inhibits the binding of Xenopus SOX3 to target DNA sequences [Development. 130(2003)5609]. The antibody also cross-reacts with zebrafish embryos. When injected into fertilized Xenopus or zebrafish eggs, antiSOX3c caused a profound gastrulation defect; this defect could be rescued by the injection of RNA encoding SOX3DeltaC-EnR, a SOX3-engrailed repression domain chimera. In antiSOX3c-injected Xenopus embryos, normal animal-vegetal patterning of mesodermal and endodermal markers was disrupted, expression domains were shifted toward the animal pole, and the levels of the endodermal markers SOX17 and endodermin increased. In Xenopus, SOX3 acts as a negative regulator of Xnr5, which encodes a nodal-related TGFbeta-family protein. Two nodal-related proteins are expressed in the early zebrafish embryo, squint and cyclops; antiSOX3c-injection leads to an increase in the level of cyclops expression. In both Xenopus and zebrafish, the antiSOX3c phenotype was rescued by the injection of RNA encoding the nodal inhibitor Cerberus-short (CerS). In Xenopus, antiSOX3c&#39;s effects on endodermin expression were suppressed by injection of RNA encoding a dominant negative version of Mixer or a morpholino against SOX17alpha2, both of which act downstream of nodal signaling in the endoderm specification pathway. Based on these data, it appears that maternal B1-type SOX functions together with the VegT/beta-catenin system to regulate nodal expression and to establish the normal pattern of germ layer formation in Xenopus. A mechanistically conserved system appears to act in a similar manner in the zebrafish.</p>
<p class="grants">Funded by: NIDCR NIH HHS: <span class="grant">K22DE14200</span>; NIGMS NIH HHS: <span class="grant">GM54001</span></p>
<p class="publication"><span class="title">Developmental biology</span>&nbsp;<span class="year">2004</span>;<span class="volume">273</span>;<span class="number">1</span>;<span class="pagerange">23-37</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15302595" rel="external">15302595</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.ydbio.2004.05.019" rel="external">10.1016/j.ydbio.2004.05.019</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Impact of population structure, effective bottleneck time, and allele frequency on linkage disequilibrium maps.</h4>
<p class="authors"><span class="author">Zhang W</span>, <span class="author">Collins A</span>, <span class="author">Gibson J</span>, <span class="author">Tapper WJ</span>, <span class="author">Hunt S</span>, <span class="author">Deloukas P</span>, <span class="author">Bentley DR</span> and <span class="author">Morton NE</span></p>
<p class="affiliation">Human Genetics Division, Duthie Building (Mailpoint 808), Southampton General Hospital, Tremona Road, Southampton SO16 6YD, United Kingdom.</p><p class="abstract">Genetic maps in linkage disequilibrium (LD) units play the same role for association mapping as maps in centimorgans provide at much lower resolution for linkage mapping. Association mapping of genes determining disease susceptibility and other phenotypes is based on the theory of LD, here applied to relations with three phenomena. To test the theory, markers at high density along a 10-Mb continuous segment of chromosome 20q were studied in African-American, Asian, and Caucasian samples. Population structure, whether created by pooling samples from divergent populations or by the mating pattern in a mixed population, is accurately bioassayed from genotype frequencies. The effective bottleneck time for Eurasians is substantially less than for migration out of Africa, reflecting later bottlenecks. The classical dependence of allele frequency on mutation age does not hold for the generally shorter time span of inbreeding and LD. Limitation of the classical theory to mutation age justifies the assumption of constant time in a LD map, except for alleles that were rare at the effective bottleneck time or have arisen since. This assumption is derived from the Malecot model and verified in all samples. Tested measures of relative efficiency, support intervals, and localization error determine the operating characteristics of LD maps that are applicable to every sexually reproducing species, with implications for association mapping, high-resolution linkage maps, evolutionary inference, and identification of recombinogenic sequences.</p>
<p class="grants">Funded by: NIGMS NIH HHS: <span class="grant">GM42947</span></p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">2004</span>;<span class="volume">101</span>;<span class="number">52</span>;<span class="pagerange">18075-80</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15604137" rel="external">15604137</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC539799" rel="external">539799</a>; DOI: <a href="http://dx.doi.org/10.1073%2Fpnas.0408251102" rel="external">10.1073/pnas.0408251102</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutations of BRAF and KRAS in gastric cancer and their association with microsatellite instability.</h4>
<p class="authors"><span class="author">Zhao W</span>, <span class="author">Chan TL</span>, <span class="author">Chu KM</span>, <span class="author">Chan AS</span>, <span class="author">Stratton MR</span>, <span class="author">Yuen ST</span> and <span class="author">Leung SY</span></p>
<p class="publication"><span class="title">International journal of cancer. Journal international du cancer</span>&nbsp;<span class="year">2004</span>;<span class="volume">108</span>;<span class="number">1</span>;<span class="pagerange">167-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/14618633" rel="external">14618633</a>; DOI: <a href="http://dx.doi.org/10.1002%2Fijc.11553" rel="external">10.1002/ijc.11553</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Shotgun optical mapping of the entire Leishmania major Friedlin genome.</h4>
<p class="authors"><span class="author">Zhou S</span>, <span class="author">Kile A</span>, <span class="author">Kvikstad E</span>, <span class="author">Bechner M</span>, <span class="author">Severin J</span>, <span class="author">Forrest D</span>, <span class="author">Runnheim R</span>, <span class="author">Churas C</span>, <span class="author">Anantharaman TS</span>, <span class="author">Myler P</span>, <span class="author">Vogt C</span>, <span class="author">Ivens A</span>, <span class="author">Stuart K</span> and <span class="author">Schwartz DC</span></p>
<p class="affiliation">Laboratory for Molecular and Computational Genomics, UW Biotechnology Center, University of Wisconsin-Madison, 425 Henry Mall, Madison, WI 53706, USA.</p><p class="abstract">Leishmania is a group of protozoan parasites which causes a broad spectrum of diseases resulting in widespread human suffering and death, as well as economic loss from the infection of some domestic animals and wildlife. To further understand the fundamental genomic architecture of this parasite, and to accelerate the on-going sequencing project, a whole-genome XbaI restriction map was constructed using the optical mapping system. This map supplemented traditional physical maps that were generated by fingerprinting and hybridization of cosmid and P1 clone libraries. Thirty-six optical map contigs were constructed for the corresponding known 36 chromosomes of the Leishmania major Friedlin genome. The chromosome sizes ranged from 326.9 to 2821.3 kb, with a total genome size of 34.7 Mb; the average XbaI restriction fragment was 25.3 kb, and ranged from 15.7 to 77.8 kb on a per chromosomes basis. Comparison between the optical maps and the in silico maps of sequence drawn from completed, nearly finished, or large sequence contigs showed that optical maps served several useful functions within the path to create finished sequence by: guiding aspects of the sequence assembly, identifying misassemblies, detection of cosmid or PAC clones misplacements to chromosomes, and validation of sequence stemming from varying degrees of finishing. Our results also showed the potential use of optical maps as a means to detect and characterize map segmental duplication within genomes.</p>
<p class="publication"><span class="title">Molecular and biochemical parasitology</span>&nbsp;<span class="year">2004</span>;<span class="volume">138</span>;<span class="number">1</span>;<span class="pagerange">97-106</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/15500921" rel="external">15500921</a>; DOI: <a href="http://dx.doi.org/10.1016%2Fj.molbiopara.2004.08.002" rel="external">10.1016/j.molbiopara.2004.08.002</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/z0ia4snc"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/z0ia4snc" src="/qr/z0ia4snc.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:59 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
