<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd"><html lang="en-gb"><head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="language" content="en">
  <meta name="author" content="www-core (Web Team)">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2, minimum-scale=0.5">
  <meta name="template-svn-id" content="$Id$">
  <title>Sanger Institute - Publications 1994 - Wellcome Trust Sanger Institute</title>
  <link rel="Shortcut Icon" href="/i/sanger-ico.png" type="image/png">
  <link rel="search" href="/i/xml/sanger-search.xml" title="Sanger website search" type="application/opensearchdescription+xml">
  <link rel="stylesheet" type="text/css" href="/t/cssjs/y8r9Dyr_b4d9w07TcP9Esw.c.css">
<link rel="apple-touch-icon" href="/s-inst/gfx/sanger-icon-57.png">
<link rel="apple-touch-icon" sizes="72x72" href="/s-inst/gfx/sanger-icon-72.png">
<link rel="apple-touch-icon" sizes="114x114" href="/s-inst/gfx/sanger-icon-114.png">
<link rel="apple-touch-icon" sizes="144x144" href="/s-inst/gfx/sanger-icon-144.png">
<!--[if lt IE 9]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/h6u4LGzDI2IgPFovKGuY5A.c.css">
<![endif]-->
<!--[if lt IE 8]>
  <link rel="stylesheet" type="text/css" href="/t/cssjs/va3qODlm_3EBCL7UE3s2nw.c.css">
<![endif]-->
  <meta name="svn-id" content="$Id: 1994.html 14733 2013-01-30 10:18:24Z www-core $">
</head>

<body>
  <div id="outer">
    <ul id="skip">
      <li><a href="#navLeft" class="skip">Jump to navigation</a></li>
      <li><a href="#wrap" class="skip">Jump to content</a></li>
    </ul>
    <div id="header">
      <div id="logo"><a class="follow" href="/"><img height="60" width="205" src="/s-inst/gfx/sanger-logo.png" alt="Wellcome Trust Sanger Institute" ></a></div>
      <div id="search"><form action="/s/" method="get"><div>
        <label for="q">Search for</label>
        <input name="q" id="q" type="text" value="Enter search here..." ><input style="height:20px;width:20px" id="q_submit" type="image" value="Search" alt="search websites" src="/core/gfx/blank.gif" >
      </div></form></div>
    </div>
    <div id="navTabs">
      <div id="access"><span id="r-s" title="Small font">A</span><span id="r-n" title="Normal font">A</span><span id="r-l" title="Large font">A</span><span id="r-x" title="Extra large font">A</span></div>
      <div id="navRight"></div>
      <ul id="navLeft"><li><a href="/">Home</a></li><li class="active"><a href="/research/">Research</a></li><li><a href="/resources/">Scientific resources</a></li><li><a href="/workstudy/">Work &amp; study</a></li><li><a href="/about/">About us</a></li></ul>
    </div>
    <div id="NavTabs2"><ul id="Tabs2"><li><a href="/research/areas/">Areas of research</a></li><li><a href="/research/projects/">Projects</a></li><li><a href="/research/faculty/">Academic faculty</a></li><li class="active"><a href="/research/publications/">Scientific publications</a></li></ul></div>
    <div id="wrap">
      
      
      
    <div id="main">
      <div class="panel">
        <h1>Sanger Institute - Publications 1994</h1><p>Number of papers published in 1994: <strong>20</strong></p><ul class="references"><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Structure and expression of the Huntington&#39;s disease gene: evidence against simple inactivation due to an expanded CAG repeat.</h4>
<p class="authors"><span class="author">Ambrose CM</span>, <span class="author">Duyao MP</span>, <span class="author">Barnes G</span>, <span class="author">Bates GP</span>, <span class="author">Lin CS</span>, <span class="author">Srinidhi J</span>, <span class="author">Baxendale S</span>, <span class="author">Hummerich H</span>, <span class="author">Lehrach H</span>, <span class="author">Altherr M</span> <em>et al.</em></p>
<p class="affiliation">Molecular Neurogenetics Unit, Massachusetts General Hospital, Boston 02114.</p><p class="abstract">Huntington&#39;s disease, a neurodegenerative disorder characterized by loss of striatal neurons, is caused by an expanded, unstable trinucleotide repeat in a novel 4p16.3 gene. To lay the foundation for exploring the pathogenic mechanism in HD, we have determined the structure of the disease gene and examined its expression. The HD locus spans 180 kb and consists of 67 exons ranging in size from 48 bp to 341 bp with an average of 138 bp. Scanning of the HD transcript failed to reveal any additional sequence alterations characteristic of HD chromosomes. A codon loss polymorphism in linkage disequilibrium with the disorder revealed that both normal and HD alleles are represented in the mRNA population in HD heterozygotes, indicating that the defect does not eliminate transcription. The gene is ubiquitously expressed as two alternatively polyadenylated forms displaying different relative abundance in various fetal and adult tissues, suggesting the operation of interacting factors in determining specificity of cell loss. The HD gene was disrupted in a female carrying a balanced translocation with a breakpoint between exons 40 and 41. The absence of any abnormal phenotype in this individual argues against simple inactivation of the gene as the mechanism by which the expanded trinucleotide repeat causes HD. Taken together, these observations suggest that the dominant HD mutation either confers a new property on the mRNA or, more likely, alters an interaction at the protein level.</p>
<p class="grants">Funded by: NINDS NIH HHS: <span class="grant">NS16367</span>; Wellcome Trust</p>
<p class="publication"><span class="title">Somatic cell and molecular genetics</span>&nbsp;<span class="year">1994</span>;<span class="volume">20</span>;<span class="number">1</span>;<span class="pagerange">27-38</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8197474" rel="external">8197474</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Dissecting the temporal requirements for homeotic gene function.</h4>
<p class="authors"><span class="author">Castelli-Gair J</span>, <span class="author">Greig S</span>, <span class="author">Micklem G</span> and <span class="author">Akam M</span></p>
<p class="affiliation">Wellcome/CRC Institute, Cambridge, UK.</p><p class="abstract">Homeotic genes confer identity to the different segments of Drosophila. These genes are expressed in many cell types over long periods of time. To determine when the homeotic genes are required for specific developmental events we have expressed the Ultrabithorax, abdominal-A and Abdominal-Bm proteins at different times during development using the GAL4 targeting technique. We find that early transient homeotic gene expression has no lasting effects on the differentiation of the larval epidermis, but it switches the fate of other cell types irreversibly (e.g. the spiracle primordia). We describe one cell type in the peripheral nervous system that makes sequential, independent responses to homeotic gene expression. We also provide evidence that supports the hypothesis of in vivo competition between the bithorax complex proteins for the regulation of their down-stream targets.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Development (Cambridge, England)</span>&nbsp;<span class="year">1994</span>;<span class="volume">120</span>;<span class="number">7</span>;<span class="pagerange">1983-95</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7925003" rel="external">7925003</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">RNA sequence analysis using covariance models.</h4>
<p class="authors"><span class="author">Eddy SR</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Cambridge, UK.</p><p class="abstract">We describe a general approach to several RNA sequence analysis problems using probabilistic models that flexibly describe the secondary structure and primary sequence consensus of an RNA sequence family. We call these models &#39;covariance models&#39;. A covariance model of tRNA sequences is an extremely sensitive and discriminative tool for searching for additional tRNAs and tRNA-related sequences in sequence databases. A model can be built automatically from an existing sequence alignment. We also describe an algorithm for learning a model and hence a consensus secondary structure from initially unaligned example sequences and no prior structural information. Models trained on unaligned tRNA examples correctly predict tRNA secondary structure and produce high-quality multiple alignments. The approach may be applied to any family of small RNA sequences.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1994</span>;<span class="volume">22</span>;<span class="number">11</span>;<span class="pagerange">2079-88</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8029015" rel="external">8029015</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308124" rel="external">308124</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Volume changes in protein evolution.</h4>
<p class="authors"><span class="author">Gerstein M</span>, <span class="author">Sonnhammer EL</span> and <span class="author">Chothia C</span></p>
<p class="affiliation">MRC Laboratory of Molecular Biology, Cambridge, U.K.</p><p class="abstract">We have determined the variations in volume that occur during evolution in the buried core of three different families of proteins. The variation of the whole core is very small (approximately 2.5%) compared to the variation at individual sites (approximately 13%). However, by comparing our results to those expected from random sequences with no correlations between sites, we show that the small variation observed may simply be a manifestation of the statistical &quot;law of large numbers&quot; and not reflect any compensating changes in, or global constraints upon, protein sequences. We have also analysed in detail the volume variations at individual sites, both in the core and on the surface, and compared these variations with those expected from random sequences. Individual sites on the surface have nearly the same variation as random sequences (24% versus 28% variation). However, individual sites in the core have about half the variation of random sequences (13% versus 30%). Roughly, half of these core sites strongly conserve their volume (0 to 10% variation); one quarter have moderate variation (10 to 20%); and the remaining quarter vary randomly (20 to 40%). Our results have clear implications for the relationship between protein sequence and structure. For our analysis, we have developed a new and simple method for weighting protein sequences to correct for unequal representation, which we describe in an Appendix.</p>
<p class="publication"><span class="title">Journal of molecular biology</span>&nbsp;<span class="year">1994</span>;<span class="volume">236</span>;<span class="number">4</span>;<span class="pagerange">1067-78</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8120887" rel="external">8120887</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An extended panel of hamster-human hybrids for chromosome 2q.</h4>
<p class="authors"><span class="author">Hafezparast M</span>, <span class="author">Cole CG</span>, <span class="author">Kaur GP</span>, <span class="author">Athwal RS</span> and <span class="author">Jeggo PA</span></p>
<p class="affiliation">MRC Cell Mutation Unit, University of Sussex, Falmer, Brighton, UK.</p><p class="abstract">A hamster-human hybrid containing only the q arm of chromosome 2 has been used to construct a panel of hybrids bearing reduced regions of chromosome 2 using the technique of irradiation fusion gene transfer. The human chromosome 2 carried the Ecogpt gene and all hybrids were selected using this marker. The integrated Ecogpt gene was localized to the region 2q33-34, resulting in the selective retention of this region in the hybrids. These data were combined with another previously constructed panel of hybrids containing regions of 2q, which were enriched for the region 2q36-37. The combined hybrid panel is useful for the mapping of new markers to defined regions of chromosome 2 and for the cloning of genes located on 2q by a positional strategy.</p>
<p class="publication"><span class="title">Somatic cell and molecular genetics</span>&nbsp;<span class="year">1994</span>;<span class="volume">20</span>;<span class="number">6</span>;<span class="pagerange">541-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7892651" rel="external">7892651</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Mutation detection by fluorescent chemical cleavage: application to hemophilia B.</h4>
<p class="authors"><span class="author">Haris II</span>, <span class="author">Green PM</span>, <span class="author">Bentley DR</span> and <span class="author">Giannelli F</span></p>
<p class="affiliation">Division of Medical and Molecular Genetics, United Medical School of Guy&#39;s Hospital, London, UK.</p><p class="abstract">Chemical mismatch detection when combined with PCR or RT-PCR amplification represents one of the most efficient mutation screening methods. This procedure has been used for the analysis of both large populations of mutants and large and complex genes because it detects and locates virtually all sequence variations in segments of DNA of up to 1.7 kb cleaving at, and/or adjacent to, mismatches in heteroduplexes formed by target and probe DNA. By using fluorescent tags and an automatic gel scanning system, the efficiency of the method has been increased fourfold with gains also in the safety and quality control of probes and in the flexibility of the procedure. Using 12 hemophilia B patients, 8 with known mutations and 4 chosen at random, we show how the chemical mismatch cleavage of up to four DNA segments can be multiplexed and examined in a single gel lane and how the increase in efficiency thus obtained can be used either totally to maximize the DNA screened per lane or partly to locate the mutation unequivocally so that only one sequencing reaction is needed to characterize it fully.</p>
<p class="publication"><span class="title">PCR methods and applications</span>&nbsp;<span class="year">1994</span>;<span class="volume">3</span>;<span class="number">5</span>;<span class="pagerange">268-71</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8038695" rel="external">8038695</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Primary structure and function of a second essential member of the heterooligomeric TCP1 chaperonin complex of yeast, TCP1 beta.</h4>
<p class="authors"><span class="author">Miklos D</span>, <span class="author">Caplan S</span>, <span class="author">Mertens D</span>, <span class="author">Hynes G</span>, <span class="author">Pitluk Z</span>, <span class="author">Kashi Y</span>, <span class="author">Harrison-Lavoie K</span>, <span class="author">Stevenson S</span>, <span class="author">Brown C</span>, <span class="author">Barrell B</span> <em>et al.</em></p>
<p class="affiliation">Howard Hughes Medical Institute, Yale School of Medicine, New Haven, CT 06510.</p><p class="abstract">A role for heterooligomeric TCP1 complex as a chaperonin in the eukaryotic cytosol has recently been suggested both by structural similarities with other chaperonins and by in vitro experiments showing it to mediate ATP-dependent folding of actin, tubulin, and luciferase. Here we present the primary structure of a second subunit of the complex and present genetic and functional analyses. The TCP1 beta amino acid sequence, predicted from the cloned gene, bears 35% identity to TCP1, termed here TCP1 alpha, containing the same highly conserved residues found in the collective sequence of chaperonins. The predicted product was identified as the fastest-migrating species of the TCP1 complex purified from soluble extracts of yeast. The TCP1 beta gene, like TCP1 alpha, is essential. Strains containing lethal disruptions of either gene could not be rescued by additional copies of the other. Spores bearing disruption of either gene germinated as single, large-budded cells. Similarly, large-budded cells were observed following shift to 37 degrees C of strains carrying temperature-sensitive mutations in either TCP1 alpha or TCP1 beta. The arrested cells contained replicated DNA present in single nuclear masses, associated with abnormal tubulin staining patterns, supporting the assertion that mitotic spindle formation and function are impaired. We conclude that TCP1 beta supplies an essential function that partially overlaps with that of TCP1 alpha in acting as a molecular chaperone in tubulin and spindle biogenesis.</p>
<p class="publication"><span class="title">Proceedings of the National Academy of Sciences of the United States of America</span>&nbsp;<span class="year">1994</span>;<span class="volume">91</span>;<span class="number">7</span>;<span class="pagerange">2743-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7908441" rel="external">7908441</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC43446" rel="external">43446</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The sequence complexity of exons trapped from the mouse genome.</h4>
<p class="authors"><span class="author">Nehls M</span>, <span class="author">Pfeifer D</span>, <span class="author">Micklem G</span>, <span class="author">Schmoor C</span> and <span class="author">Boehm T</span></p>
<p class="affiliation">Department of Medicine I, University of Freiburg, Germany.</p><p class="abstract"><strong>Background</strong>: A central issue in genome analysis is the identification and characterization of coding regions. Estimating the coding complexity of vertebrate genomes by measuring the kinetic complexity of mRNA populations and by sequence analysis of cDNAs is limited by the fact that any given source of mRNA represents a very biased sample of all genes. Exon trapping is a method that enables the identification of genes irrespective of their transcriptional status.</p><p class="abstract"><strong>Results</strong>: Exons were trapped from the entire mouse genome, and the resulting fragments cloned. About 7% of a random sample of exons taken from this library have significant structural homology or sequence similarity to previously sequenced genes. Using cDNAs derived from several stages of mouse development, evidence for expression of about 62% of this sample of exons was found. These data suggest that the great majority of &#39;exons&#39; in the library are derived from genes. We estimate that the fraction of the genome contained in trapped exons is 2.4%; this corresponds to a sequence complexity of about 72 megabases.</p><p class="abstract"><strong>Conclusions</strong>: The library of exons trapped from the entire mouse genome probably represents one of the least biased and most comprehensive libraries of mouse coding regions, and should therefore prove very useful for finding genes during genome mapping and sequencing.</p>
<p class="publication"><span class="title">Current biology : CB</span>&nbsp;<span class="year">1994</span>;<span class="volume">4</span>;<span class="number">11</span>;<span class="pagerange">983-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7874497" rel="external">7874497</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">[Analysis of Ki-ras in pancreatic fluid aspirate. A new diagnostic possibility in investigation of pancreatic cancer].</h4>
<p class="authors"><span class="author">Ogreid D</span>, <span class="author">Ulvik A</span>, <span class="author">Horn A</span> and <span class="author">S&oslash;ndenaa K</span></p>
<p class="affiliation">Senter for klinisk molekylaermedisin, Haukeland Sykehus, Bergen.</p><p class="abstract">The prognosis for pancreatic carcinoma is generally poor. The chance of survival could be improved if curative surgery were performed, but early diagnosis is then essential. Fine-needle aspiration cytology and endoscopic retrograde cholepancreaticography (ERCP) are widely used in order to obtain a precise diagnosis before laparotomy. In almost all patients with pancreatic adenocarcinomas, the oncogene Ki-ras is activated by point mutation. We describe a patient where conventional diagnostic procedures were inconclusive. However, DNA-analysis of aspirate from the pancreatic duct and from a pancreatic cyst showed activated Ki-ras oncogene. A malignant diagnosis was later confirmed. We propose that DNA-analyses of pancreatic fluid or tissues should be used whenever facing problems with an early and accurate diagnosis of pancreatic lesions.</p>
<p class="publication"><span class="title">Tidsskrift for den Norske l&aelig;geforening : tidsskrift for praktisk medicin, ny r&aelig;kke</span>&nbsp;<span class="year">1994</span>;<span class="volume">114</span>;<span class="number">26</span>;<span class="pagerange">3079-81</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7974428" rel="external">7974428</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Characterization of a new member of the human beta-adaptin gene family from chromosome 22q12, a candidate meningioma gene.</h4>
<p class="authors"><span class="author">Peyrard M</span>, <span class="author">Fransson I</span>, <span class="author">Xie YG</span>, <span class="author">Han FY</span>, <span class="author">Ruttledge MH</span>, <span class="author">Swahn S</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Collins VP</span> and <span class="author">Dumanski JP</span></p>
<p class="affiliation">Department of Molecular Medicine, Karolinska Hospital, Stockholm, Sweden.</p><p class="abstract">A 140 kb homozygous deletion from 22q12 in one meningioma directed us towards the cloning and characterization of a new member of the human beta-adaptin gene family (named BAM22). Adaptins are essential for the formation of clathrin coated vesicles in the course of intracellular transport of receptor-ligand complexes. The BAM22 gene is totally inactivated in the tumor with homozygous deletion. Northern blot analysis of 70 sporadic meningiomas showed specific loss of expression in 8 tumors, suggesting inactivation of BAM22. Based on this, we propose BAM22 as a second chromosome 22 locus important in meningioma development, after the neurofibromatosis type 2 gene.</p>
<p class="publication"><span class="title">Human molecular genetics</span>&nbsp;<span class="year">1994</span>;<span class="volume">3</span>;<span class="number">8</span>;<span class="pagerange">1393-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7987321" rel="external">7987321</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A YAC contig spanning the ataxia-telangiectasia locus (groups A and C) at 11q22-q23.</h4>
<p class="authors"><span class="author">Rotman G</span>, <span class="author">Savitsky K</span>, <span class="author">Ziv Y</span>, <span class="author">Cole CG</span>, <span class="author">Higgins MJ</span>, <span class="author">Bar-Am I</span>, <span class="author">Dunham I</span>, <span class="author">Bar-Shira A</span>, <span class="author">Vanagaite L</span>, <span class="author">Qin S</span>, <span class="author">Zhang J</span>, <span class="author">Nowak NJ</span>, <span class="author">Chandrasekharappa SC</span>, <span class="author">Lehrach H</span>, <span class="author">Avivi L</span>, <span class="author">Shows TB</span>, <span class="author">Collins FS</span>, <span class="author">Bentley DR</span> and <span class="author">Shiloh Y</span></p>
<p class="affiliation">Department of Human Genetics, Sackler School of Medicine, Tel Aviv University, Ramat Aviv, Israel.</p><p class="abstract">Ataxia-telangiectasia (A-T) is an autosomal recessive disease involving cerebellar degeneration, immunodeficiency, cancer predisposition, chromosomal instability and radiosensitivity. A-T is heterogeneous, and the majority of A-T cases are associated with two complementation groups, A and C. The ATA and ATC loci are closely linked at chromosome 11q22-q23. Recombination mapping and linkage disequilibrium analysis have confined both loci between the markers D11S1817 and D11S927, spaced approximately 3.5 Mb apart. Isolation in yeast artificial chromosomes of the genomic segment defined by these loci is essential to identify the gene or genes containing the ATA and ATC mutations. A YAC contig spanning 4.5 Mb, which includes the D11S1817-D11S927 interval, was constructed using two whole genome libraries (ICRF and St. Louis), and a chromosome 11-specific library. Construction of this contig was expedited by prior generation of a region-specific ICRF sublibrary using Alu-PCR products derived from a radiation hybrid. The contig was expanded further by screening the libraries with Alu-PCR products derived from YAC clones and with STSs from YAC ends. YAC clones were aligned by fingerprinting with moderately repetitive probes.</p>
<p class="grants">Funded by: NHGRI NIH HHS: <span class="grant">HG00359</span></p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1994</span>;<span class="volume">24</span>;<span class="number">2</span>;<span class="pagerange">234-42</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7698744" rel="external">7698744</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Selective regulation of epithelial gene expression in rabbit Peyer&#39;s patch tissue.</h4>
<p class="authors"><span class="author">Savidge TC</span>, <span class="author">Smith MW</span>, <span class="author">Mayel-Afshar S</span>, <span class="author">Collins AJ</span> and <span class="author">Freeman TC</span></p>
<p class="affiliation">Department of Cellular Physiology, Babraham Institute, Cambridge, UK.</p><p class="abstract">The physiological mechanisms that regulate epithelial gene expression during enterocyte migration and differentiation are still poorly understood. The present study has used a combination of quantitative in situ hybridisation, immunohistochemistry and enzyme cytochemistry to examine epithelial cell differentiation in rabbit small intestine. We have measured and compared the levels of mRNA and enzyme activity of the enterocyte brush border markers alkaline phosphatase, amino-peptidase N and lactase in normal villus epithelia and in epithelial cells exposed directly to the Peyer&#39;s patch immune environment. All three genes appeared to be expressed in parallel, but in each epithelial population examined, the pattern of gene expression was different. The level of these mRNAs was markedly reduced in Peyer&#39;s patch-associated epithelia, this being most pronounced in the follicle-associated epithelium, compared with normal villi. The activities of alkaline phosphatase and aminopeptidase N approximated the expression of their genes, whereas additional post-transcriptional events were shown to clearly contribute to the level of lactase activity in these tissues. These findings demonstrate that the reduced brush border hydrolase activity in Peyer&#39;s patch tissue that has been observed previously, is due to a down-regulation of epithelial gene expression in this location. These observations have been used to discuss epithelial differentiation in Peyer&#39;s patch tissue and the possible role of local immune factors in regulating such events.</p>
<p class="publication"><span class="title">Pfl&uuml;gers Archiv : European journal of physiology</span>&nbsp;<span class="year">1994</span>;<span class="volume">428</span>;<span class="number">3-4</span>;<span class="pagerange">391-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7816561" rel="external">7816561</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">GRAM and genfragII: solving and testing the single-digest, partially ordered restriction map problem.</h4>
<p class="authors"><span class="author">Soderlund C</span> and <span class="author">Burks C</span></p>
<p class="affiliation">Theoretical Biology and Biophysics Group, Los Alamos National Laboratory, NM 87545.</p><p class="abstract">GRAM (Genomic Restriction map AsseMbly) takes as input single-digest restriction fragments for a set of overlapping clones and outputs one or more plausible partially ordered restriction maps. For each restriction map, GRAM shows the corresponding alignment of the input clone fragments. Due to the error and uncertainty in experimental data, this problem is computationally difficult to solve; therefore, the principle objective in the design of GRAM is to facilitate man-machine collaborative problem solving. GRAM quickly approximates a solution, as follows. (i) A clustering algorithm determines a probable set of restriction fragments. (ii) An assembly algorithm permutes the set of restriction fragments such that the maximal number of clone fragments are contiguous. The output of the GRAM algorithm is displayed for the user to query and edit. This paper describes the stochastic assembly algorithm and shows how it works with the interactive graphics to support man-machine problem solving. In order to test and verify the performance of GRAM, we have developed a program called genfragII to simulate the digestion of clones and fragments; this program is described and results are presented. GRAM is also being used for a number of genome mapping projects.</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1994</span>;<span class="volume">10</span>;<span class="number">3</span>;<span class="pagerange">349-58</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7922694" rel="external">7922694</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">An expert system for processing sequence homology data.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Hinxton, Cambridge, UK.</p><p class="abstract">When confronted with the task of finding homology to large numbers of sequences, database searching tools such as Blast and Fasta generate prohibitively large amounts of information. An automatic way of making most of the decisions a trained sequence analyst would make was developed by means of a rule-based expert system combined with an algorithm to avoid non-informative biased residue composition matches. The results found relevant by the system are presented in a very concise and clear way, so that the homology can be assessed with minimum effort. The expert system, HSPcrunch, was implemented to process the output to the programs in the BLAST suite. HSPcrunch embodies rules on detecting distant similarities when pairs of weak matches are consistent with a larger gapped alignment, i.e. when Blast has broken a longer gapped alignment up into smaller ungapped ones. This way, more distant similarities can be detected with no or little side-effects of more spurious matches. The rules for how small the gaps must be to be considered significant have been derived empirically. Currently a set of rules are used that operate on two different scoring levels, one for very weak matches that have very small gaps and one for medium weak matches that have slightly larger gaps. This set of rules proved to be robust for most cases and gives high fidelity separation between real homologies and spurious matches. One of the most important rules for reducing the amount of output is to limit the number of overlapping matches to the same region of the query sequence.(ABSTRACT TRUNCATED AT 250 WORDS)</p>
<p class="publication"><span class="title">Proceedings / ... International Conference on Intelligent Systems for Molecular Biology ; ISMB. International Conference on Intelligent Systems for Molecular Biology</span>&nbsp;<span class="year">1994</span>;<span class="volume">2</span>;<span class="pagerange">363-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7584413" rel="external">7584413</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">A workbench for large-scale sequence homology analysis.</h4>
<p class="authors"><span class="author">Sonnhammer EL</span> and <span class="author">Durbin R</span></p>
<p class="affiliation">Sanger Centre, Hinxton Hall, Cambridge, UK.</p><p class="abstract">When routinely analysing very long stretches of DNA sequences produced by genome sequencing projects, detailed analysis of database search results becomes exceedingly time consuming. To reduce the tedious browsing of large quantities of protein similarities, two programs, MSPcrunch and Blixem, were developed, which assist in processing the results from the database search programs in the BLAST suite. MSPcrunch removes biased composition and redundant matches while keeping weak matches that are consistent with a larger gapped alignment. This makes BLAST searching in practice more sensitive and reduces the risk of overlooking distant similarities. Blixem is a multiple sequence alignment viewer for X-windows which makes it significantly easier to scan and evaluate the matches ratified by MSPcrunch. In Blixem, matches to the translated DNA query sequence are simultaneously aligned in three frames. Also, the distribution of matches over the whole DNA query is displayed. Examples of usage are drawn from 36 C. elegans cosmid clones totalling 1.2 megabases, to which these tools were applied.</p>
<p class="grants">Funded by: Wellcome Trust</p>
<p class="publication"><span class="title">Computer applications in the biosciences : CABIOS</span>&nbsp;<span class="year">1994</span>;<span class="volume">10</span>;<span class="number">3</span>;<span class="pagerange">301-7</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7922687">7922687</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Genetic mapping of 14 short tandem repeat polymorphisms on human chromosome 22.</h4>
<p class="authors"><span class="author">Vallada HP</span>, <span class="author">Collins JE</span>, <span class="author">Dunham I</span>, <span class="author">Dawson E</span>, <span class="author">Murray RM</span>, <span class="author">Gill M</span> and <span class="author">Collier DA</span></p>
<p class="affiliation">Genetics Section, Institute of Psychiatry, De Crespigny Park, London, UK.</p><p class="abstract">We have constructed a linkage map of 14 short tandem repeat polymorphisms (11 with heterozygosity &gt; 70%) on the long arm of human chromosome 22 using 23 non-CEPH pedigrees. Twelve of the markers could be positioned uniquely with a likelihood of at least 1,000:1, and distributed at an average distance of 6.62 cM (range 1.5-16.1 cM). The sex-combined map covers a total of 79.6 cM, the female map 93.2 cM and the male map 64.6 cM. Based on comparisons between physical maps and other genetic maps, we estimate that our map covers 70%-80% of the chromosome. The map integrates markers from previous genetic maps and uniquely positions one marker (D22S307). Data from physical mapping on the location of four genetic markers correlates well with our linkage map, and provides information on an additional marker (D22S315). This map will facilitate high resolution mapping of additional polymorphic loci and disease genes on chromosome 22, and act as a reference for building and verifying physical maps.</p>
<p class="publication"><span class="title">Human genetics</span>&nbsp;<span class="year">1994</span>;<span class="volume">93</span>;<span class="number">6</span>;<span class="pagerange">688-90</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8005593" rel="external">8005593</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">Isolation of cosmid and cDNA clones in the region surrounding the BTK gene at Xq21.3-q22.</h4>
<p class="authors"><span class="author">Vorechovsk&yacute; I</span>, <span class="author">Vetrie D</span>, <span class="author">Holland J</span>, <span class="author">Bentley DR</span>, <span class="author">Thomas K</span>, <span class="author">Zhou JN</span>, <span class="author">Notarangelo LD</span>, <span class="author">Plebani A</span>, <span class="author">Font&aacute;n G</span>, <span class="author">Ochs HD</span> <em>et al.</em></p>
<p class="affiliation">Center for BioTechnology, Karolinska Institute at NOVUM, Huddinge, Sweden.</p><p class="abstract">A regional physical and transcription map involving yeast artificial chromosomes (YACs), cosmids, and cDNAs has been constructed for Xq21.3-q22 around the gene BTK (formerly atk or BPK) defective in X-linked agammaglobulinemia (XLA). With a positional cloning strategy employing direct cDNA selection, novel cDNAs were found to cluster in the region of approximately 100 kb flanking the XLA and alpha-galactosidase A loci. While these widely expressed transcripts are in the area known to contain CpG islands, a less evolutionarily conserved gene, located more than 130 kb distal of DXS178, maps to cosmid clones that could not be digested with rare-cutting restriction enzymes. The presence of transcribed sequences flanking the BTK allowed us to investigate their involvement in complex XLA phenotypes. Southern blot analysis using cDNA clones isolated from this region permitted us to exclude a contiguous deletion syndrome as an underlying defect in three patients with XLA and associated growth hormone deficiency. A single XLA patient with torsion dystonia and cosegregating X-linked deafness has been found with a deletion in the 3&#39; part of BTK extending centromerically into the flanking expressed sequence DXS1274E. This suggests a possible involvement of the DXS1274E in this phenotype. The GenBank accession numbers for novel cDNA sequences are as follows: DXS1269E (L20773), DXS1271E (UO1923), DXS1273E (UO1925), and DXS1274E (UO1922).</p>
<p class="publication"><span class="title">Genomics</span>&nbsp;<span class="year">1994</span>;<span class="volume">21</span>;<span class="number">3</span>;<span class="pagerange">517-24</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7959728" rel="external">7959728</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">ECD--a totally integrated database of Escherichia coli K12.</h4>
<p class="authors"><span class="author">Wahl R</span>, <span class="author">Rice P</span>, <span class="author">Rice CM</span> and <span class="author">Kr&ouml;ger M</span></p>
<p class="affiliation">Institut f&uuml;r Mikrobiologie und Molekularbiologie, Fachbereich Biologie, Justus-Liebig-Universt&auml;t Giessen, Germany.</p><p class="abstract">We have compiled the DNA sequence data for E. coli available from the GENBANK and EMBL data libraries and independently from the literature. Starting with this update of our Escherichia coli database (ECD release 20) we provide major changes compared to previous issues. This update not only represents another substantial increase in sequence information, it also allows now to find the exact physical location of each individual gene or regulatory region, even regarding discrepancies in nomenclature. In order to save space this printed version does not contain the database itself anymore, but we provide several examples. The complete database is publically available in electronic form together with a self explaining application program or as a flat file. The complete compilation including a full set of genetic map data and the E. coli protein index can be obtained in machine readable form from the EMBL data library as a part of the CD-ROM issue of the EMBL sequence database, released and updated every three months. After deletion of all detected overlaps a total of 2,878,364 individual bp is found to be determined till the end of June 1994. This corresponds to a total of 60.98% of the entire E. coli chromosome consisting of about 4,720 kbp. This number may actually be higher by 9161 bp derived from other strains of E. coli.</p>
<p class="publication"><span class="title">Nucleic acids research</span>&nbsp;<span class="year">1994</span>;<span class="volume">22</span>;<span class="number">17</span>;<span class="pagerange">3450-5</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7937044" rel="external">7937044</a>; PMC: <a href="http://ukpmc.ac.uk/articles/PMC308300" rel="external">308300</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">2.2 Mb of contiguous nucleotide sequence from chromosome III of C. elegans.</h4>
<p class="authors"><span class="author">Wilson R</span>, <span class="author">Ainscough R</span>, <span class="author">Anderson K</span>, <span class="author">Baynes C</span>, <span class="author">Berks M</span>, <span class="author">Bonfield J</span>, <span class="author">Burton J</span>, <span class="author">Connell M</span>, <span class="author">Copsey T</span>, <span class="author">Cooper J</span> <em>et al.</em></p>
<p class="affiliation">Department of Genetics, Washington University School of Medicine, St Louis, Missouri 63110.</p><p class="abstract">As part of our effort to sequence the 100-megabase (Mb) genome of the nematode Caenorhabditis elegans, we have completed the nucleotide sequence of a contiguous 2,181,032 base pairs in the central gene cluster of chromosome III. Analysis of the finished sequence has indicated an average density of about one gene per five kilobases; comparison with the public sequence databases reveals similarities to previously known genes for about one gene in three. In addition, the genomic sequence contains several intriguing features, including putative gene duplications and a variety of other repeats with potential evolutionary implications.</p>
<p class="publication"><span class="title">Nature</span>&nbsp;<span class="year">1994</span>;<span class="volume">368</span>;<span class="number">6466</span>;<span class="pagerange">32-8</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/7906398">7906398</a>; DOI: <a href="http://dx.doi.org/10.1038%2F368032a0">10.1038/368032a0</a></p></li><li class="periodical ref-coll ref-closed">
<h4 title="click to show/hide abstract..." class="article">The STL1 gene of Saccharomyces cerevisiae is predicted to encode a sugar transporter-like protein.</h4>
<p class="authors"><span class="author">Zhao S</span>, <span class="author">Douglas NW</span>, <span class="author">Heine MJ</span>, <span class="author">Williams GM</span>, <span class="author">Winther-Larsen HC</span> and <span class="author">Meaden PG</span></p>
<p class="affiliation">International Centre for Brewing and Distilling, Heriot-Watt University, Riccarton, Edinburgh, UK.</p><p class="abstract">A gene has been cloned from the yeast Saccharomyces cerevisiae which, on the basis of the deduced translation product, encodes a sugar transporter-like protein. This gene, STL1, was identified as an open reading frame (ORF) closely linked to the cinnamic-acid-resistance gene POF1 on chromosome IV. The putative translation product of STL1 (STL1) contains 536 amino acids, with a M(r) of 60,507. Hydropathy analysis of STL1 suggests that it contains the twelve transmembrane (TM) domains characteristic of a family of sugar transporters from S. cerevisiae and other organisms. STL1 displays greatest homology (28% identity) to the products of the yeast HXT2 (hexose transporter) and GAL2 (galactose transporter) genes. Disruption of STL1 had no detectable effect on yeast growth on glucose, galactose, mannose, maltose or glycerol as sole carbon source. The transport function of the gene product remains unknown at present.</p>
<p class="publication"><span class="title">Gene</span>&nbsp;<span class="year">1994</span>;<span class="volume">146</span>;<span class="number">2</span>;<span class="pagerange">215-9</span></p>
<p class="links">PUBMED: <a href="http://ukpmc.ac.uk/abstract/MED/8076821" rel="external">8076821</a></p></li></ul>
      </div>
    </div>
    <div id="rhs">
      <div class="panel">
<h3>Publications by year</h3>
<ul>
<li><a href="/research/publications/2014.html">2014</a></li>
<li><a href="/research/publications/2013.html">2013</a></li>
<li><a href="/research/publications/2012.html">2012</a></li>
<li><a href="/research/publications/2011.html">2011</a></li>
<li><a href="/research/publications/2010.html">2010</a></li>
<li><a href="/research/publications/2009.html">2009</a></li>
<li><a href="/research/publications/2008.html">2008</a></li>
<li><a href="/research/publications/2007.html">2007</a></li>
<li><a href="/research/publications/2006.html">2006</a></li>
<li><a href="/research/publications/2005.html">2005</a></li>
<li><a href="/research/publications/2004.html">2004</a></li>
<li><a href="/research/publications/2003.html">2003</a></li>
<li><a href="/research/publications/2002.html">2002</a></li>
<li><a href="/research/publications/2001.html">2001</a></li>
<li><a href="/research/publications/2000.html">2000</a></li>
<li><a href="/research/publications/1999.html">1999</a></li>
<li><a href="/research/publications/1998.html">1998</a></li>
<li><a href="/research/publications/1997.html">1997</a></li>
<li><a href="/research/publications/1996.html">1996</a></li>
<li><a href="/research/publications/1995.html">1995</a></li>
<li><a href="/research/publications/1994.html">1994</a></li>
<li><a href="/research/publications/1993.html">1993</a></li>
<li><a href="/research/publications/1992.html">1992</a></li>
</ul>
</div>

    </div>
  
      
    </div>
    <div class="qr"><a href="http://q.sanger.ac.uk/yuiod891"><img height="62" width="62" alt="* quick link - http://q.sanger.ac.uk/yuiod891" src="/qr/yuiod891.png" ></a></div>
    <div id="footer">
      <ul id="footer_links">
        <li><a href="/help/">Help</a></li>
        <li><a href="/about/contact/">Contact us</a></li>
        <li><a href="/legal/">Legal</a></li>
        <li><a href="/legal/cookiespolicy.html">Cookies policy</a></li>
        <li><a href="/datasharing/">Data sharing</a></li>
      </ul>

      <ul class="social">
        <li class="connect">Connect with us: </li>
        <li class="fb"><a href="https://www.facebook.com/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="Connect with us on Facebook" alt="Connect with us on Facebook" ></a></li>
        <li class="tw"><a href="https://twitter.com/sangerinstitute"     ><img height="20" width="21" src="/core/gfx/blank.gif" title="Follow up on Twitter" alt="Follow us on Twitter" ></a></li>
        <li class="yt"><a href="http://www.youtube.com/user/sangerinstitute"><img height="20" width="21" src="/core/gfx/blank.gif" title="View us on Youtube" alt="View us on Youtube" ></a></li>
        <li class="wp"><a href="http://sangerinstitute.wordpress.com/"   ><img height="20" width="21" src="/core/gfx/blank.gif" title="Read our blog" alt="Read our blog" ></a></li>
      </ul>

      <div class="charity">
        <div id="footer_print">
          Unless otherwise stated the content of this website is copyright Wellcome
          Trust Sanger Institute, licenced under the Creative Commons Attribution
          3.0 Unported License.
        </div>
        Wellcome Trust Sanger Institute, Genome Research Limited (reg no. 2742969) is a charity registered in England with number 1021457<br >
        Last modified: Wed, 30 Jan 2013 10:53:47 GMT
      </div>

    </div>
    
  </div>
  <noscript><div><img src="/piwik.php" alt="" style="width:1px;height:1px" ></div></noscript>
  <script type="text/javascript" src="/t/cssjs/uRJ6NC5bLpfNEqAAt5jY4Q.c.js"></script>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WV3FP6"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">//<![CDATA[
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WV3FP6');
//]]></script>
<!-- End Google Tag Manager -->
</body>
</html>
